<link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/bootstrap/4.0.0/css/bootstrap.min.css" integrity="sha384-Gn5384xqQ1aoWXA+058RXPxPg6fy4IWvTNh0E263XmFcJlSAwiGgFAW/dAiS6JXm" crossorigin="anonymous"><div class='container mt-4 bg-light p-4 text-center'><h1 class='text-primary'>Phenotype: blood pressure medication use</h1><p class='text-muted'><strong>URL:</strong> http://ftp.ebi.ac.uk/pub/databases/gwas/summary_statistics/GCST90081001-GCST90082000/GCST90081464</p><table border="1" class="dataframe table table-bordered">
  <thead>
    <tr style="text-align: right;">
      <th>firstAuthor</th>
      <th>accessionId</th>
      <th>publicationDate</th>
      <th>journal</th>
      <th>title</th>
      <th>reportedTrait</th>
      <th>efoTraits</th>
      <th>bgTraits</th>
      <th>discoverySampleAncestry</th>
      <th>replicationSampleAncestry</th>
      <th>associationCount</th>
      <th>summaryStatistics</th>
      <th>ssApiFlag</th>
      <th>agreedToCc0</th>
      <th>genotypingTechnologies</th>
      <th>pubmedId</th>
      <th>initialSampleDescription</th>
      <th>replicateSampleDescription</th>
      <th>UKB_FIELD</th>
      <th>FuzzySimilarity_blood pressure medication</th>
      <th>SearchPhenotype</th>
    </tr>
  </thead>
  <tbody>
    <tr>
      <td>Backman JD</td>
      <td>GCST90081464</td>
      <td>10/18/2021</td>
      <td>Nature</td>
      <td>Exome sequencing and analysis of 454,787 UK Biobank participants.</td>
      <td>blood pressure medication use</td>
      <td>drug use measurement</td>
      <td>-</td>
      <td>320758 european</td>
      <td>NaN</td>
      <td>0</td>
      <td>http://ftp.ebi.ac.uk/pub/databases/gwas/summary_statistics/GCST90081001-GCST90082000/GCST90081464</td>
      <td>no</td>
      <td>yes</td>
      <td>Exome-wide sequencing,Genome-wide genotyping array</td>
      <td>34662886</td>
      <td>99,813 European ancestry cases,220,945 European ancestry controls</td>
      <td>NaN</td>
      <td>-</td>
      <td>93</td>
      <td>blood pressure medication</td>
    </tr>
  </tbody>
</table><h2 class='text-muted'>Information file</h2><table border="1" class="dataframe table table-bordered">
  <thead>
    <tr style="text-align: right;">
      <th>Unnamed: 0</th>
      <th>Name</th>
      <th>Last modified</th>
      <th>Size</th>
      <th>Description</th>
      <th>URL</th>
    </tr>
  </thead>
  <tbody>
    <tr>
      <td>1</td>
      <td>GCST90081464_buildGRCh38.tsv.gz-meta.yaml</td>
      <td>2024-01-04 00:33</td>
      <td>901</td>
      <td>NaN</td>
      <td>http://ftp.ebi.ac.uk/pub/databases/gwas/summary_statistics/GCST90081001-GCST90082000/GCST90081464/GCST90081464_buildGRCh38.tsv.gz-meta.yaml</td>
    </tr>
    <tr>
      <td>2</td>
      <td>README.txt</td>
      <td>2021-11-18 00:35</td>
      <td>698</td>
      <td>NaN</td>
      <td>http://ftp.ebi.ac.uk/pub/databases/gwas/summary_statistics/GCST90081001-GCST90082000/GCST90081464/README.txt</td>
    </tr>
    <tr>
      <td>3</td>
      <td>harmonised/</td>
      <td>2024-02-02 15:18</td>
      <td>-</td>
      <td>NaN</td>
      <td>http://ftp.ebi.ac.uk/pub/databases/gwas/summary_statistics/GCST90081001-GCST90082000/GCST90081464/harmonised/</td>
    </tr>
    <tr>
      <td>4</td>
      <td>md5sum.txt</td>
      <td>2021-11-18 00:35</td>
      <td>109</td>
      <td>NaN</td>
      <td>http://ftp.ebi.ac.uk/pub/databases/gwas/summary_statistics/GCST90081001-GCST90082000/GCST90081464/md5sum.txt</td>
    </tr>
    <tr>
      <td>0</td>
      <td>GCST90081464_buildGRCh38.tsv.gz</td>
      <td>2021-11-18 00:35</td>
      <td>14M</td>
      <td>NaN</td>
      <td>http://ftp.ebi.ac.uk/pub/databases/gwas/summary_statistics/GCST90081001-GCST90082000/GCST90081464/GCST90081464_buildGRCh38.tsv.gz</td>
    </tr>
  </tbody>
</table><h2 class='text-muted'>Processed file</h2><p>Processing file: GCST90081464_buildGRCh38.tsv.gz!</p><h2 class='text-muted'>GWAS file head</h2><table border="1" class="dataframe table table-bordered">
  <thead>
    <tr style="text-align: right;">
      <th>name</th>
      <th>chromosome</th>
      <th>base_pair_location</th>
      <th>other_allele</th>
      <th>effect_allele</th>
      <th>trait</th>
      <th>cohort</th>
      <th>model</th>
      <th>odds_ratio</th>
      <th>ci_lower</th>
      <th>ci_upper</th>
      <th>p_value</th>
      <th>effect_allele_frequency</th>
      <th>standard_error</th>
    </tr>
  </thead>
  <tbody>
    <tr>
      <td>1_930158_C_T</td>
      <td>1</td>
      <td>930158</td>
      <td>C</td>
      <td>T</td>
      <td>RGC_BP_Med__BIN</td>
      <td>UKB_Freeze_450_EUR</td>
      <td>ADD-WGR-FIRTH</td>
      <td>0.561652</td>
      <td>0.080306</td>
      <td>3.92815</td>
      <td>0.561039</td>
      <td>0.000008</td>
      <td>0.992386</td>
    </tr>
    <tr>
      <td>1_930165_G_A</td>
      <td>1</td>
      <td>930165</td>
      <td>G</td>
      <td>A</td>
      <td>RGC_BP_Med__BIN</td>
      <td>UKB_Freeze_450_EUR</td>
      <td>ADD-WGR-FIRTH</td>
      <td>0.835754</td>
      <td>0.333924</td>
      <td>2.09175</td>
      <td>0.701489</td>
      <td>0.000039</td>
      <td>0.468080</td>
    </tr>
  </tbody>
</table></div><div class='container mt-4 bg-light p-4 text-center'><h2 class='text-muted'>Original GWAS file headers</h2><table border="1" class="dataframe table table-bordered">
  <thead>
    <tr style="text-align: right;">
      <th>name</th>
      <th>chromosome</th>
      <th>base_pair_location</th>
      <th>other_allele</th>
      <th>effect_allele</th>
      <th>trait</th>
      <th>cohort</th>
      <th>model</th>
      <th>odds_ratio</th>
      <th>ci_lower</th>
      <th>ci_upper</th>
      <th>p_value</th>
      <th>effect_allele_frequency</th>
      <th>standard_error</th>
    </tr>
  </thead>
  <tbody>
    <tr>
      <td>1_930158_C_T</td>
      <td>1</td>
      <td>930158</td>
      <td>C</td>
      <td>T</td>
      <td>RGC_BP_Med__BIN</td>
      <td>UKB_Freeze_450_EUR</td>
      <td>ADD-WGR-FIRTH</td>
      <td>0.561652</td>
      <td>0.080306</td>
      <td>3.92815</td>
      <td>0.561039</td>
      <td>0.000008</td>
      <td>0.992386</td>
    </tr>
    <tr>
      <td>1_930165_G_A</td>
      <td>1</td>
      <td>930165</td>
      <td>G</td>
      <td>A</td>
      <td>RGC_BP_Med__BIN</td>
      <td>UKB_Freeze_450_EUR</td>
      <td>ADD-WGR-FIRTH</td>
      <td>0.835754</td>
      <td>0.333924</td>
      <td>2.09175</td>
      <td>0.701489</td>
      <td>0.000039</td>
      <td>0.468080</td>
    </tr>
  </tbody>
</table><p style='font-weight: bold; color: blue;'>CHR -> ['chromosome']</p><p style='font-weight: bold; color: blue;'>SNP -> ['name']</p><p style='font-weight: bold; color: blue;'>BP -> ['name', 'base_pair_location']</p><p style='font-weight: bold; color: blue;'>A1 -> ['effect_allele']</p><p style='font-weight: bold; color: blue;'>A2 -> ['other_allele']</p><p style='font-weight: bold; color: blue;'>N -> []</p><p style='font-weight: bold; color: blue;'>P -> ['p_value']</p><p style='font-weight: bold; color: blue;'>INFO -> []</p><p style='font-weight: bold; color: blue;'>MAF -> ['effect_allele_frequency']</p><p style='font-weight: bold; color: blue;'>BETA -> []</p><p style='font-weight: bold; color: blue;'>SE -> ['standard_error']</p><p style='font-weight: bold; color: blue;'>OR -> ['odds_ratio']</p><p style='font-weight: bold; color: blue;'>Z -> []</p><p style='font-weight: bold; color: blue;'>D -> []</p><h2 class='text-muted'>GWAS columns</h2>name, chromosome, base_pair_location, other_allele, effect_allele, trait, cohort, model, odds_ratio, ci_lower, ci_upper, p_value, effect_allele_frequency, standard_error<h2 class='text-muted'>Identified columns</h2>chromosome, name, name, base_pair_location, effect_allele, other_allele, p_value, effect_allele_frequency, standard_error, odds_ratio<h2 class='text-muted'>Unidentified Columns headers</h2>cohort, trait, ci_lower, model, ci_upper<h2 class='text-muted'>PMID</h2>34662886<h2 class='text-muted'>DOI</h2>10.1038/s41586-021-04103-z<div class='bibtex-container'><h2 class='text-muted'>BibTeX</h2><div style='font-family: monospace; white-space: pre-wrap;'> @article{Backman_2021, title={Exome sequencing and analysis of 454,787 UK Biobank participants}, volume={599}, ISSN={1476-4687}, url={http://dx.doi.org/10.1038/s41586-021-04103-z}, DOI={10.1038/s41586-021-04103-z}, number={7886}, journal={Nature}, publisher={Springer Science and Business Media LLC}, author={Backman, Joshua D. and Li, Alexander H. and Marcketta, Anthony and Sun, Dylan and Mbatchou, Joelle and Kessler, Michael D. and Benner, Christian and Liu, Daren and Locke, Adam E. and Balasubramanian, Suganthi and Yadav, Ashish and Banerjee, Nilanjana and Gillies, Christopher E. and Damask, Amy and Liu, Simon and Bai, Xiaodong and Hawes, Alicia and Maxwell, Evan and Gurski, Lauren and Watanabe, Kyoko and Kosmicki, Jack A. and Rajagopal, Veera and Mighty, Jason and Jones, Marcus and Mitnaul, Lyndon and Stahl, Eli and Coppola, Giovanni and Jorgenson, Eric and Habegger, Lukas and Salerno, William J. and Shuldiner, Alan R. and Lotta, Luca A. and Overton, John D. and Cantor, Michael N. and Reid, Jeffrey G. and Yancopoulos, George and Kang, Hyun M. and Marchini, Jonathan and Baras, Aris and Abecasis, Gonçalo R. and Ferreira, Manuel A. R.}, year={2021}, month=oct, pages={628–634} }
</div></div><br><br><br><br><br><br></div><div class='container mt-4 bg-light p-4 text-center'><h1 class='text-primary'>Phenotype: blood pressure medication use (ukb data field 6153)</h1><p class='text-muted'><strong>URL:</strong> http://ftp.ebi.ac.uk/pub/databases/gwas/summary_statistics/GCST90079001-GCST90080000/GCST90079458</p><table border="1" class="dataframe table table-bordered">
  <thead>
    <tr style="text-align: right;">
      <th>firstAuthor</th>
      <th>accessionId</th>
      <th>publicationDate</th>
      <th>journal</th>
      <th>title</th>
      <th>reportedTrait</th>
      <th>efoTraits</th>
      <th>bgTraits</th>
      <th>discoverySampleAncestry</th>
      <th>replicationSampleAncestry</th>
      <th>associationCount</th>
      <th>summaryStatistics</th>
      <th>ssApiFlag</th>
      <th>agreedToCc0</th>
      <th>genotypingTechnologies</th>
      <th>pubmedId</th>
      <th>initialSampleDescription</th>
      <th>replicateSampleDescription</th>
      <th>UKB_FIELD</th>
      <th>FuzzySimilarity_blood pressure medication</th>
      <th>SearchPhenotype</th>
    </tr>
  </thead>
  <tbody>
    <tr>
      <td>Backman JD</td>
      <td>GCST90079458</td>
      <td>10/18/2021</td>
      <td>Nature</td>
      <td>Exome sequencing and analysis of 454,787 UK Biobank participants.</td>
      <td>blood pressure medication use (ukb data field 6153)</td>
      <td>drug use measurement</td>
      <td>-</td>
      <td>75554 european</td>
      <td>NaN</td>
      <td>0</td>
      <td>http://ftp.ebi.ac.uk/pub/databases/gwas/summary_statistics/GCST90079001-GCST90080000/GCST90079458</td>
      <td>no</td>
      <td>yes</td>
      <td>Exome-wide sequencing,Genome-wide genotyping array</td>
      <td>34662886</td>
      <td>42,217 European ancestry cases,33,337 European ancestry controls</td>
      <td>NaN</td>
      <td>blood pressure medication use (ukb data field 6153)</td>
      <td>68</td>
      <td>blood pressure medication</td>
    </tr>
  </tbody>
</table><h2 class='text-muted'>Information file</h2><table border="1" class="dataframe table table-bordered">
  <thead>
    <tr style="text-align: right;">
      <th>Unnamed: 0</th>
      <th>Name</th>
      <th>Last modified</th>
      <th>Size</th>
      <th>Description</th>
      <th>URL</th>
    </tr>
  </thead>
  <tbody>
    <tr>
      <td>1</td>
      <td>GCST90079458_buildGRCh38.tsv.gz-meta.yaml</td>
      <td>2024-01-04 00:09</td>
      <td>922</td>
      <td>NaN</td>
      <td>http://ftp.ebi.ac.uk/pub/databases/gwas/summary_statistics/GCST90079001-GCST90080000/GCST90079458/GCST90079458_buildGRCh38.tsv.gz-meta.yaml</td>
    </tr>
    <tr>
      <td>2</td>
      <td>README.txt</td>
      <td>2021-11-18 01:01</td>
      <td>698</td>
      <td>NaN</td>
      <td>http://ftp.ebi.ac.uk/pub/databases/gwas/summary_statistics/GCST90079001-GCST90080000/GCST90079458/README.txt</td>
    </tr>
    <tr>
      <td>3</td>
      <td>harmonised/</td>
      <td>2024-02-02 15:15</td>
      <td>-</td>
      <td>NaN</td>
      <td>http://ftp.ebi.ac.uk/pub/databases/gwas/summary_statistics/GCST90079001-GCST90080000/GCST90079458/harmonised/</td>
    </tr>
    <tr>
      <td>4</td>
      <td>md5sum.txt</td>
      <td>2021-11-18 01:01</td>
      <td>109</td>
      <td>NaN</td>
      <td>http://ftp.ebi.ac.uk/pub/databases/gwas/summary_statistics/GCST90079001-GCST90080000/GCST90079458/md5sum.txt</td>
    </tr>
    <tr>
      <td>0</td>
      <td>GCST90079458_buildGRCh38.tsv.gz</td>
      <td>2021-11-18 01:01</td>
      <td>872K</td>
      <td>NaN</td>
      <td>http://ftp.ebi.ac.uk/pub/databases/gwas/summary_statistics/GCST90079001-GCST90080000/GCST90079458/GCST90079458_buildGRCh38.tsv.gz</td>
    </tr>
  </tbody>
</table><h2 class='text-muted'>Processed file</h2><p>Processing file: GCST90079458_buildGRCh38.tsv.gz!</p><h2 class='text-muted'>GWAS file head</h2><table border="1" class="dataframe table table-bordered">
  <thead>
    <tr style="text-align: right;">
      <th>name</th>
      <th>chromosome</th>
      <th>base_pair_location</th>
      <th>other_allele</th>
      <th>effect_allele</th>
      <th>trait</th>
      <th>cohort</th>
      <th>model</th>
      <th>odds_ratio</th>
      <th>ci_lower</th>
      <th>ci_upper</th>
      <th>p_value</th>
      <th>effect_allele_frequency</th>
      <th>standard_error</th>
    </tr>
  </thead>
  <tbody>
    <tr>
      <td>1_930165_G_A</td>
      <td>1</td>
      <td>930165</td>
      <td>G</td>
      <td>A</td>
      <td>6153_Medication_for_cholesterol_blood_pressure_diabetes_or_take_exogenous_hormones_2_Blood_pressure_medication_AnyInst__BIN</td>
      <td>UKB_Freeze_450_EUR</td>
      <td>ADD-WGR-FIRTH</td>
      <td>1.4783</td>
      <td>0.339541</td>
      <td>6.43629</td>
      <td>0.602499</td>
      <td>0.000060</td>
      <td>0.750553</td>
    </tr>
    <tr>
      <td>1_930204_G_A</td>
      <td>1</td>
      <td>930204</td>
      <td>G</td>
      <td>A</td>
      <td>6153_Medication_for_cholesterol_blood_pressure_diabetes_or_take_exogenous_hormones_2_Blood_pressure_medication_AnyInst__BIN</td>
      <td>UKB_Freeze_450_EUR</td>
      <td>ADD-WGR-FIRTH</td>
      <td>1.0135</td>
      <td>0.279152</td>
      <td>3.67967</td>
      <td>0.983735</td>
      <td>0.000073</td>
      <td>0.657875</td>
    </tr>
  </tbody>
</table></div><div class='container mt-4 bg-light p-4 text-center'><h2 class='text-muted'>Original GWAS file headers</h2><table border="1" class="dataframe table table-bordered">
  <thead>
    <tr style="text-align: right;">
      <th>name</th>
      <th>chromosome</th>
      <th>base_pair_location</th>
      <th>other_allele</th>
      <th>effect_allele</th>
      <th>trait</th>
      <th>cohort</th>
      <th>model</th>
      <th>odds_ratio</th>
      <th>ci_lower</th>
      <th>ci_upper</th>
      <th>p_value</th>
      <th>effect_allele_frequency</th>
      <th>standard_error</th>
    </tr>
  </thead>
  <tbody>
    <tr>
      <td>1_930165_G_A</td>
      <td>1</td>
      <td>930165</td>
      <td>G</td>
      <td>A</td>
      <td>6153_Medication_for_cholesterol_blood_pressure_diabetes_or_take_exogenous_hormones_2_Blood_pressure_medication_AnyInst__BIN</td>
      <td>UKB_Freeze_450_EUR</td>
      <td>ADD-WGR-FIRTH</td>
      <td>1.4783</td>
      <td>0.339541</td>
      <td>6.43629</td>
      <td>0.602499</td>
      <td>0.000060</td>
      <td>0.750553</td>
    </tr>
    <tr>
      <td>1_930204_G_A</td>
      <td>1</td>
      <td>930204</td>
      <td>G</td>
      <td>A</td>
      <td>6153_Medication_for_cholesterol_blood_pressure_diabetes_or_take_exogenous_hormones_2_Blood_pressure_medication_AnyInst__BIN</td>
      <td>UKB_Freeze_450_EUR</td>
      <td>ADD-WGR-FIRTH</td>
      <td>1.0135</td>
      <td>0.279152</td>
      <td>3.67967</td>
      <td>0.983735</td>
      <td>0.000073</td>
      <td>0.657875</td>
    </tr>
  </tbody>
</table><p style='font-weight: bold; color: blue;'>CHR -> ['chromosome']</p><p style='font-weight: bold; color: blue;'>SNP -> ['name']</p><p style='font-weight: bold; color: blue;'>BP -> ['name', 'base_pair_location']</p><p style='font-weight: bold; color: blue;'>A1 -> ['effect_allele']</p><p style='font-weight: bold; color: blue;'>A2 -> ['other_allele']</p><p style='font-weight: bold; color: blue;'>N -> []</p><p style='font-weight: bold; color: blue;'>P -> ['p_value']</p><p style='font-weight: bold; color: blue;'>INFO -> []</p><p style='font-weight: bold; color: blue;'>MAF -> ['effect_allele_frequency']</p><p style='font-weight: bold; color: blue;'>BETA -> []</p><p style='font-weight: bold; color: blue;'>SE -> ['standard_error']</p><p style='font-weight: bold; color: blue;'>OR -> ['odds_ratio']</p><p style='font-weight: bold; color: blue;'>Z -> []</p><p style='font-weight: bold; color: blue;'>D -> []</p><h2 class='text-muted'>GWAS columns</h2>name, chromosome, base_pair_location, other_allele, effect_allele, trait, cohort, model, odds_ratio, ci_lower, ci_upper, p_value, effect_allele_frequency, standard_error<h2 class='text-muted'>Identified columns</h2>chromosome, name, name, base_pair_location, effect_allele, other_allele, p_value, effect_allele_frequency, standard_error, odds_ratio<h2 class='text-muted'>Unidentified Columns headers</h2>cohort, trait, ci_lower, model, ci_upper<h2 class='text-muted'>PMID</h2>34662886<h2 class='text-muted'>DOI</h2>10.1038/s41586-021-04103-z<div class='bibtex-container'><h2 class='text-muted'>BibTeX</h2><div style='font-family: monospace; white-space: pre-wrap;'> @article{Backman_2021, title={Exome sequencing and analysis of 454,787 UK Biobank participants}, volume={599}, ISSN={1476-4687}, url={http://dx.doi.org/10.1038/s41586-021-04103-z}, DOI={10.1038/s41586-021-04103-z}, number={7886}, journal={Nature}, publisher={Springer Science and Business Media LLC}, author={Backman, Joshua D. and Li, Alexander H. and Marcketta, Anthony and Sun, Dylan and Mbatchou, Joelle and Kessler, Michael D. and Benner, Christian and Liu, Daren and Locke, Adam E. and Balasubramanian, Suganthi and Yadav, Ashish and Banerjee, Nilanjana and Gillies, Christopher E. and Damask, Amy and Liu, Simon and Bai, Xiaodong and Hawes, Alicia and Maxwell, Evan and Gurski, Lauren and Watanabe, Kyoko and Kosmicki, Jack A. and Rajagopal, Veera and Mighty, Jason and Jones, Marcus and Mitnaul, Lyndon and Stahl, Eli and Coppola, Giovanni and Jorgenson, Eric and Habegger, Lukas and Salerno, William J. and Shuldiner, Alan R. and Lotta, Luca A. and Overton, John D. and Cantor, Michael N. and Reid, Jeffrey G. and Yancopoulos, George and Kang, Hyun M. and Marchini, Jonathan and Baras, Aris and Abecasis, Gonçalo R. and Ferreira, Manuel A. R.}, year={2021}, month=oct, pages={628–634} }
</div></div><br><br><br><br><br><br></div>