<link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/bootstrap/4.0.0/css/bootstrap.min.css" integrity="sha384-Gn5384xqQ1aoWXA+058RXPxPg6fy4IWvTNh0E263XmFcJlSAwiGgFAW/dAiS6JXm" crossorigin="anonymous"><div class='container mt-4 bg-light p-4 text-center'><h1 class='text-primary'>Phenotype: body mass index</h1><p class='text-muted'><strong>URL:</strong> http://ftp.ebi.ac.uk/pub/databases/gwas/summary_statistics/GCST006001-GCST007000/GCST006802</p><table border="1" class="dataframe table table-bordered">
  <thead>
    <tr style="text-align: right;">
      <th>firstAuthor</th>
      <th>accessionId</th>
      <th>publicationDate</th>
      <th>journal</th>
      <th>title</th>
      <th>reportedTrait</th>
      <th>efoTraits</th>
      <th>bgTraits</th>
      <th>discoverySampleAncestry</th>
      <th>replicationSampleAncestry</th>
      <th>associationCount</th>
      <th>summaryStatistics</th>
      <th>ssApiFlag</th>
      <th>agreedToCc0</th>
      <th>genotypingTechnologies</th>
      <th>pubmedId</th>
      <th>initialSampleDescription</th>
      <th>replicateSampleDescription</th>
      <th>UKB_FIELD</th>
      <th>FuzzySimilarity_body mass index bmi</th>
      <th>SearchPhenotype</th>
    </tr>
  </thead>
  <tbody>
    <tr>
      <td>Wood AR</td>
      <td>GCST006802</td>
      <td>3/10/2016</td>
      <td>Diabetologia</td>
      <td>Variants in the FTO and CDKAL1 loci have recessive effects on risk of obesity and type 2 diabetes, respectively.</td>
      <td>body mass index</td>
      <td>body mass index</td>
      <td>-</td>
      <td>119688 european</td>
      <td>NaN</td>
      <td>36</td>
      <td>http://ftp.ebi.ac.uk/pub/databases/gwas/summary_statistics/GCST006001-GCST007000/GCST006802</td>
      <td>yes</td>
      <td>-</td>
      <td>Genome-wide genotyping array</td>
      <td>26961502</td>
      <td>29,925 British ancestry cases,89,763 British ancestry controls</td>
      <td>NaN</td>
      <td>-</td>
      <td>88</td>
      <td>body mass index bmi</td>
    </tr>
  </tbody>
</table><h2 class='text-muted'>Information file</h2><table border="1" class="dataframe table table-bordered">
  <thead>
    <tr style="text-align: right;">
      <th>Unnamed: 0</th>
      <th>Name</th>
      <th>Last modified</th>
      <th>Size</th>
      <th>Description</th>
      <th>URL</th>
    </tr>
  </thead>
  <tbody>
    <tr>
      <td>1</td>
      <td>harmonised/</td>
      <td>2019-12-05 11:48</td>
      <td>-</td>
      <td>NaN</td>
      <td>http://ftp.ebi.ac.uk/pub/databases/gwas/summary_statistics/GCST006001-GCST007000/GCST006802/harmonised/</td>
    </tr>
    <tr>
      <td>0</td>
      <td>bmi_dom_dev.txt</td>
      <td>2018-12-27 21:10</td>
      <td>479M</td>
      <td>NaN</td>
      <td>http://ftp.ebi.ac.uk/pub/databases/gwas/summary_statistics/GCST006001-GCST007000/GCST006802/bmi_dom_dev.txt</td>
    </tr>
  </tbody>
</table><h2 class='text-muted'>Processed file</h2><p>Processing file: bmi_dom_dev.txt!</p><h2 class='text-muted'>GWAS file head</h2><table border="1" class="dataframe table table-bordered">
  <thead>
    <tr style="text-align: right;">
      <th>markername</th>
      <th>rsid</th>
      <th>allele1</th>
      <th>allele2</th>
      <th>freq</th>
      <th>beta</th>
      <th>se</th>
      <th>p</th>
      <th>n</th>
    </tr>
  </thead>
  <tbody>
    <tr>
      <td>1_749963</td>
      <td>rs529266287</td>
      <td>T</td>
      <td>TAA</td>
      <td>0.134</td>
      <td>0.001700</td>
      <td>0.014</td>
      <td>0.9</td>
      <td>113309</td>
    </tr>
    <tr>
      <td>1_751343</td>
      <td>rs28544273</td>
      <td>A</td>
      <td>T</td>
      <td>0.126</td>
      <td>-0.000035</td>
      <td>0.014</td>
      <td>1.0</td>
      <td>118061</td>
    </tr>
  </tbody>
</table></div><div class='container mt-4 bg-light p-4 text-center'><h2 class='text-muted'>Original GWAS file headers</h2><table border="1" class="dataframe table table-bordered">
  <thead>
    <tr style="text-align: right;">
      <th>markername</th>
      <th>rsid</th>
      <th>allele1</th>
      <th>allele2</th>
      <th>freq</th>
      <th>beta</th>
      <th>se</th>
      <th>p</th>
      <th>n</th>
    </tr>
  </thead>
  <tbody>
    <tr>
      <td>1_749963</td>
      <td>rs529266287</td>
      <td>T</td>
      <td>TAA</td>
      <td>0.134</td>
      <td>0.001700</td>
      <td>0.014</td>
      <td>0.9</td>
      <td>113309</td>
    </tr>
    <tr>
      <td>1_751343</td>
      <td>rs28544273</td>
      <td>A</td>
      <td>T</td>
      <td>0.126</td>
      <td>-0.000035</td>
      <td>0.014</td>
      <td>1.0</td>
      <td>118061</td>
    </tr>
  </tbody>
</table><p style='font-weight: bold; color: blue;'>CHR -> []</p><p style='font-weight: bold; color: blue;'>SNP -> ['markername', 'rsid']</p><p style='font-weight: bold; color: blue;'>BP -> ['markername']</p><p style='font-weight: bold; color: blue;'>A1 -> ['allele1']</p><p style='font-weight: bold; color: blue;'>A2 -> ['allele1', 'allele2']</p><p style='font-weight: bold; color: blue;'>N -> ['n']</p><p style='font-weight: bold; color: blue;'>P -> ['p']</p><p style='font-weight: bold; color: blue;'>INFO -> []</p><p style='font-weight: bold; color: blue;'>MAF -> ['freq']</p><p style='font-weight: bold; color: blue;'>BETA -> ['beta']</p><p style='font-weight: bold; color: blue;'>SE -> ['se']</p><p style='font-weight: bold; color: blue;'>OR -> []</p><p style='font-weight: bold; color: blue;'>Z -> []</p><p style='font-weight: bold; color: blue;'>D -> []</p><h2 class='text-muted'>GWAS columns</h2>markername, rsid, allele1, allele2, freq, beta, se, p, n<h2 class='text-muted'>Identified columns</h2>markername, rsid, markername, allele1, allele1, allele2, n, p, freq, beta, se<h2 class='text-muted'>Unidentified Columns headers</h2><h2 class='text-muted'>PMID</h2>26961502<h2 class='text-muted'>DOI</h2>10.1007/s00125-016-3908-5<div class='bibtex-container'><h2 class='text-muted'>BibTeX</h2><div style='font-family: monospace; white-space: pre-wrap;'> @article{Wood_2016, title={Variants in the FTO and CDKAL1 loci have recessive effects on risk of obesity and type 2 diabetes, respectively}, volume={59}, ISSN={1432-0428}, url={http://dx.doi.org/10.1007/s00125-016-3908-5}, DOI={10.1007/s00125-016-3908-5}, number={6}, journal={Diabetologia}, publisher={Springer Science and Business Media LLC}, author={Wood, Andrew R. and Tyrrell, Jessica and Beaumont, Robin and Jones, Samuel E. and Tuke, Marcus A. and Ruth, Katherine S. and Yaghootkar, Hanieh and Freathy, Rachel M. and Murray, Anna and Frayling, Timothy M. and Weedon, Michael N.}, year={2016}, month=mar, pages={1214â€“1221} }
</div></div><br><br><br><br><br><br></div><div class='container mt-4 bg-light p-4 text-center'><h1 class='text-primary'>Phenotype: body mass index</h1><p class='text-muted'><strong>URL:</strong> http://ftp.ebi.ac.uk/pub/databases/gwas/summary_statistics/GCST001001-GCST002000/GCST001676</p><table border="1" class="dataframe table table-bordered">
  <thead>
    <tr style="text-align: right;">
      <th>firstAuthor</th>
      <th>accessionId</th>
      <th>publicationDate</th>
      <th>journal</th>
      <th>title</th>
      <th>reportedTrait</th>
      <th>efoTraits</th>
      <th>bgTraits</th>
      <th>discoverySampleAncestry</th>
      <th>replicationSampleAncestry</th>
      <th>associationCount</th>
      <th>summaryStatistics</th>
      <th>ssApiFlag</th>
      <th>agreedToCc0</th>
      <th>genotypingTechnologies</th>
      <th>pubmedId</th>
      <th>initialSampleDescription</th>
      <th>replicateSampleDescription</th>
      <th>UKB_FIELD</th>
      <th>FuzzySimilarity_body mass index bmi</th>
      <th>SearchPhenotype</th>
    </tr>
  </thead>
  <tbody>
    <tr>
      <td>Yang J</td>
      <td>GCST001676</td>
      <td>9/12/2012</td>
      <td>Nature</td>
      <td>FTO genotype is associated with phenotypic variability of body mass index.</td>
      <td>body mass index</td>
      <td>body mass index</td>
      <td>-</td>
      <td>133154 european</td>
      <td>59325 European</td>
      <td>1</td>
      <td>http://ftp.ebi.ac.uk/pub/databases/gwas/summary_statistics/GCST001001-GCST002000/GCST001676</td>
      <td>no</td>
      <td>-</td>
      <td>Genome-wide genotyping array</td>
      <td>22982992</td>
      <td>133,154 European ancestry individuals</td>
      <td>59,325 European ancestry individuals</td>
      <td>-</td>
      <td>88</td>
      <td>body mass index bmi</td>
    </tr>
  </tbody>
</table><h2 class='text-muted'>Information file</h2><table border="1" class="dataframe table table-bordered">
  <thead>
    <tr style="text-align: right;">
      <th>Unnamed: 0</th>
      <th>Name</th>
      <th>Last modified</th>
      <th>Size</th>
      <th>Description</th>
      <th>URL</th>
    </tr>
  </thead>
  <tbody>
    <tr>
      <td>0</td>
      <td>GIANT_Yang2012Nature_publicrelease_HapMapCeuFreq_BMI.txt.gz</td>
      <td>2019-04-13 23:12</td>
      <td>30M</td>
      <td>NaN</td>
      <td>http://ftp.ebi.ac.uk/pub/databases/gwas/summary_statistics/GCST001001-GCST002000/GCST001676/GIANT_Yang2012Nature_publicrelease_HapMapCeuFreq_BMI.txt.gz</td>
    </tr>
  </tbody>
</table><h2 class='text-muted'>Processed file</h2><p>Processing file: GIANT_Yang2012Nature_publicrelease_HapMapCeuFreq_BMI.txt.gz!</p><h2 class='text-muted'>GWAS file head</h2><table border="1" class="dataframe table table-bordered">
  <thead>
    <tr style="text-align: right;">
      <th>markername</th>
      <th>allele1</th>
      <th>allele2</th>
      <th>freq.allele1.hapmapceu</th>
      <th>b</th>
      <th>se</th>
      <th>p</th>
      <th>n</th>
    </tr>
  </thead>
  <tbody>
    <tr>
      <td>rs3094315</td>
      <td>a</td>
      <td>g</td>
      <td>0.864</td>
      <td>0.0068</td>
      <td>0.012</td>
      <td>0.57</td>
      <td>54178.0</td>
    </tr>
    <tr>
      <td>rs2905035</td>
      <td>a</td>
      <td>g</td>
      <td>0.106</td>
      <td>-0.0075</td>
      <td>0.014</td>
      <td>0.59</td>
      <td>51152.0</td>
    </tr>
  </tbody>
</table></div><div class='container mt-4 bg-light p-4 text-center'><h2 class='text-muted'>Original GWAS file headers</h2><table border="1" class="dataframe table table-bordered">
  <thead>
    <tr style="text-align: right;">
      <th>markername</th>
      <th>allele1</th>
      <th>allele2</th>
      <th>freq.allele1.hapmapceu</th>
      <th>b</th>
      <th>se</th>
      <th>p</th>
      <th>n</th>
    </tr>
  </thead>
  <tbody>
    <tr>
      <td>rs3094315</td>
      <td>a</td>
      <td>g</td>
      <td>0.864</td>
      <td>0.0068</td>
      <td>0.012</td>
      <td>0.57</td>
      <td>54178.0</td>
    </tr>
    <tr>
      <td>rs2905035</td>
      <td>a</td>
      <td>g</td>
      <td>0.106</td>
      <td>-0.0075</td>
      <td>0.014</td>
      <td>0.59</td>
      <td>51152.0</td>
    </tr>
  </tbody>
</table><p style='font-weight: bold; color: blue;'>CHR -> []</p><p style='font-weight: bold; color: blue;'>SNP -> ['markername']</p><p style='font-weight: bold; color: blue;'>BP -> ['markername', 'b']</p><p style='font-weight: bold; color: blue;'>A1 -> ['allele1']</p><p style='font-weight: bold; color: blue;'>A2 -> ['allele1', 'allele2']</p><p style='font-weight: bold; color: blue;'>N -> ['n']</p><p style='font-weight: bold; color: blue;'>P -> ['p']</p><p style='font-weight: bold; color: blue;'>INFO -> []</p><p style='font-weight: bold; color: blue;'>MAF -> ['freq.allele1.hapmapceu']</p><p style='font-weight: bold; color: blue;'>BETA -> ['b']</p><p style='font-weight: bold; color: blue;'>SE -> ['se']</p><p style='font-weight: bold; color: blue;'>OR -> []</p><p style='font-weight: bold; color: blue;'>Z -> []</p><p style='font-weight: bold; color: blue;'>D -> []</p><h2 class='text-muted'>GWAS columns</h2>markername, allele1, allele2, freq.allele1.hapmapceu, b, se, p, n<h2 class='text-muted'>Identified columns</h2>markername, markername, b, allele1, allele1, allele2, n, p, freq.allele1.hapmapceu, b, se<h2 class='text-muted'>Unidentified Columns headers</h2><h2 class='text-muted'>PMID</h2>22982992<h2 class='text-muted'>DOI</h2>10.1038/nature11401<div class='bibtex-container'><h2 class='text-muted'>BibTeX</h2><div style='font-family: monospace; white-space: pre-wrap;'> @article{Yang_2012, title={FTO genotype is associated with phenotypic variability of body mass index}, volume={490}, ISSN={1476-4687}, url={http://dx.doi.org/10.1038/nature11401}, DOI={10.1038/nature11401}, number={7419}, journal={Nature}, publisher={Springer Science and Business Media LLC}, author={Yang, Jian and Loos, Ruth J. F. and Powell, Joseph E. and Medland, Sarah E. and Speliotes, Elizabeth K. and Chasman, Daniel I. and Rose, Lynda M. and Thorleifsson, Gudmar and Steinthorsdottir, Valgerdur and Mägi, Reedik and Waite, Lindsay and Vernon Smith, Albert and Yerges-Armstrong, Laura M. and Monda, Keri L. and Hadley, David and Mahajan, Anubha and Li, Guo and Kapur, Karen and Vitart, Veronique and Huffman, Jennifer E. and Wang, Sophie R. and Palmer, Cameron and Esko, Tõnu and Fischer, Krista and Hua Zhao, Jing and Demirkan, Ayşe and Isaacs, Aaron and Feitosa, Mary F. and Luan, Jian’an and Heard-Costa, Nancy L. and White, Charles and Jackson, Anne U. and Preuss, Michael and Ziegler, Andreas and Eriksson, Joel and Kutalik, Zoltán and Frau, Francesca and Nolte, Ilja M. and Van Vliet-Ostaptchouk, Jana V. and Hottenga, Jouke-Jan and Jacobs, Kevin B. and Verweij, Niek and Goel, Anuj and Medina-Gomez, Carolina and Estrada, Karol and Lynn Bragg-Gresham, Jennifer and Sanna, Serena and Sidore, Carlo and Tyrer, Jonathan and Teumer, Alexander and Prokopenko, Inga and Mangino, Massimo and Lindgren, Cecilia M. and Assimes, Themistocles L. and Shuldiner, Alan R. and Hui, Jennie and Beilby, John P. and McArdle, Wendy L. and Hall, Per and Haritunians, Talin and Zgaga, Lina and Kolcic, Ivana and Polasek, Ozren and Zemunik, Tatijana and Oostra, Ben A. and Juhani Junttila, M. and Grönberg, Henrik and Schreiber, Stefan and Peters, Annette and Hicks, Andrew A. and Stephens, Jonathan and Foad, Nicola S. and Laitinen, Jaana and Pouta, Anneli and Kaakinen, Marika and Willemsen, Gonneke and Vink, Jacqueline M. and Wild, Sarah H. and Navis, Gerjan and Asselbergs, Folkert W. and Homuth, Georg and John, Ulrich and Iribarren, Carlos and Harris, Tamara and Launer, Lenore and Gudnason, Vilmundur and O’Connell, Jeffrey R. and Boerwinkle, Eric and Cadby, Gemma and Palmer, Lyle J. and James, Alan L. and Musk, Arthur W. and Ingelsson, Erik and Psaty, Bruce M. and Beckmann, Jacques S. and Waeber, Gerard and Vollenweider, Peter and Hayward, Caroline and Wright, Alan F. and Rudan, Igor and Groop, Leif C. and Metspalu, Andres and Khaw, Kay-Tee and van Duijn, Cornelia M. and Borecki, Ingrid B. and Province, Michael A. and Wareham, Nicholas J. and Tardif, Jean-Claude and Huikuri, Heikki V. and Adrienne Cupples, L. and Atwood, Larry D. and Fox, Caroline S. and Boehnke, Michael and Collins, Francis S. and Mohlke, Karen L. and Erdmann, Jeanette and Schunkert, Heribert and Hengstenberg, Christian and Stark, Klaus and Lorentzon, Mattias and Ohlsson, Claes and Cusi, Daniele and Staessen, Jan A. and Van der Klauw, Melanie M. and Pramstaller, Peter P. and Kathiresan, Sekar and Jolley, Jennifer D. and Ripatti, Samuli and Jarvelin, Marjo-Riitta and de Geus, Eco J. C. and Boomsma, Dorret I. and Penninx, Brenda and Wilson, James F. and Campbell, Harry and Chanock, Stephen J. and van der Harst, Pim and Hamsten, Anders and Watkins, Hugh and Hofman, Albert and Witteman, Jacqueline C. and Zillikens, M. Carola and Uitterlinden, André G. and Rivadeneira, Fernando and Carola Zillikens, M. and Kiemeney, Lambertus A. and Vermeulen, Sita H. and Abecasis, Goncalo R. and Schlessinger, David and Schipf, Sabine and Stumvoll, Michael and Tönjes, Anke and Spector, Tim D. and North, Kari E. and Lettre, Guillaume and McCarthy, Mark I. and Berndt, Sonja I. and Heath, Andrew C. and Madden, Pamela A. F. and Nyholt, Dale R. and Montgomery, Grant W. and Martin, Nicholas G. and McKnight, Barbara and Strachan, David P. and Hill, William G. and Snieder, Harold and Ridker, Paul M. and Thorsteinsdottir, Unnur and Stefansson, Kari and Frayling, Timothy M. and Hirschhorn, Joel N. and Goddard, Michael E. and Visscher, Peter M.}, year={2012}, month=sep, pages={267–272} }
</div></div><br><br><br><br><br><br></div><div class='container mt-4 bg-light p-4 text-center'><h1 class='text-primary'>Phenotype: body mass index</h1><p class='text-muted'><strong>URL:</strong> http://ftp.ebi.ac.uk/pub/databases/gwas/summary_statistics/GCST009001-GCST010000/GCST009057</p><table border="1" class="dataframe table table-bordered">
  <thead>
    <tr style="text-align: right;">
      <th>firstAuthor</th>
      <th>accessionId</th>
      <th>publicationDate</th>
      <th>journal</th>
      <th>title</th>
      <th>reportedTrait</th>
      <th>efoTraits</th>
      <th>bgTraits</th>
      <th>discoverySampleAncestry</th>
      <th>replicationSampleAncestry</th>
      <th>associationCount</th>
      <th>summaryStatistics</th>
      <th>ssApiFlag</th>
      <th>agreedToCc0</th>
      <th>genotypingTechnologies</th>
      <th>pubmedId</th>
      <th>initialSampleDescription</th>
      <th>replicateSampleDescription</th>
      <th>UKB_FIELD</th>
      <th>FuzzySimilarity_body mass index bmi</th>
      <th>SearchPhenotype</th>
    </tr>
  </thead>
  <tbody>
    <tr>
      <td>Gurdasani D</td>
      <td>GCST009057</td>
      <td>10/1/2019</td>
      <td>Cell</td>
      <td>Uganda Genome Resource Enables Insights into Population History and Genomic Discovery in Africa.</td>
      <td>body mass index</td>
      <td>body mass index</td>
      <td>-</td>
      <td>14126 sub-saharan african</td>
      <td>NaN</td>
      <td>2</td>
      <td>http://ftp.ebi.ac.uk/pub/databases/gwas/summary_statistics/GCST009001-GCST010000/GCST009057</td>
      <td>no</td>
      <td>-</td>
      <td>Genome-wide sequencing,Genome-wide genotyping array</td>
      <td>31675503</td>
      <td>14,126 African ancestry individuals</td>
      <td>NaN</td>
      <td>-</td>
      <td>88</td>
      <td>body mass index bmi</td>
    </tr>
  </tbody>
</table><h2 class='text-muted'>Information file</h2><table border="1" class="dataframe table table-bordered">
  <thead>
    <tr style="text-align: right;">
      <th>Unnamed: 0</th>
      <th>Name</th>
      <th>Last modified</th>
      <th>Size</th>
      <th>Description</th>
      <th>URL</th>
    </tr>
  </thead>
  <tbody>
    <tr>
      <td>1</td>
      <td>md5sum.txt</td>
      <td>2019-11-19 00:21</td>
      <td>53</td>
      <td>NaN</td>
      <td>http://ftp.ebi.ac.uk/pub/databases/gwas/summary_statistics/GCST009001-GCST010000/GCST009057/md5sum.txt</td>
    </tr>
    <tr>
      <td>0</td>
      <td>bmiannotated.txt.gz</td>
      <td>2019-11-19 00:21</td>
      <td>2.6G</td>
      <td>NaN</td>
      <td>http://ftp.ebi.ac.uk/pub/databases/gwas/summary_statistics/GCST009001-GCST010000/GCST009057/bmiannotated.txt.gz</td>
    </tr>
  </tbody>
</table><h2 class='text-muted'>Processed file</h2><p>Processing file: bmiannotated.txt.gz!</p><h2 class='text-muted'>GWAS file head</h2><table border="1" class="dataframe table table-bordered">
  <thead>
    <tr style="text-align: right;">
      <th>snpid</th>
      <th>beta_uganda</th>
      <th>se_uganda</th>
      <th>beta_dcc</th>
      <th>se_dcc</th>
      <th>beta_dds</th>
      <th>se_dds</th>
      <th>beta_aadm</th>
      <th>se_aadm</th>
      <th>studyno</th>
      <th>pval_fe</th>
      <th>beta_fe</th>
      <th>se_fe</th>
      <th>pval_re</th>
      <th>beta_re</th>
      <th>se_re</th>
      <th>pval_re2</th>
      <th>isquare</th>
      <th>q</th>
      <th>pval_q</th>
      <th>tau_square</th>
      <th>pval_uganda</th>
      <th>pval_dcc</th>
      <th>pval_dds</th>
      <th>pval_aadm</th>
      <th>af_uganda</th>
      <th>af_dcc</th>
      <th>af_dds</th>
      <th>af_aadm</th>
      <th>no_uganda</th>
      <th>no_dcc</th>
      <th>no_dds</th>
      <th>no_aadm</th>
    </tr>
  </thead>
  <tbody>
    <tr>
      <td>10_100000012_G_A</td>
      <td>0.069551</td>
      <td>0.057426</td>
      <td>-0.176789</td>
      <td>0.095124</td>
      <td>-0.005907</td>
      <td>0.116692</td>
      <td>-0.4927</td>
      <td>0.3718</td>
      <td>4</td>
      <td>0.912445</td>
      <td>-0.004945</td>
      <td>0.044973</td>
      <td>0.555139</td>
      <td>-0.048551</td>
      <td>0.082279</td>
      <td>0.458041</td>
      <td>55.0057</td>
      <td>6.66751</td>
      <td>0.083285</td>
      <td>0.013447</td>
      <td>0.225836</td>
      <td>0.063097</td>
      <td>0.959626</td>
      <td>0.185113</td>
      <td>0.972764</td>
      <td>0.960781</td>
      <td>0.965522</td>
      <td>0.976190</td>
      <td>6197</td>
      <td>1478</td>
      <td>1114</td>
      <td>5187</td>
    </tr>
    <tr>
      <td>10_10000010_C_T</td>
      <td>NaN</td>
      <td>NaN</td>
      <td>NaN</td>
      <td>NaN</td>
      <td>NaN</td>
      <td>NaN</td>
      <td>0.5668</td>
      <td>0.7514</td>
      <td>1</td>
      <td>NaN</td>
      <td>NaN</td>
      <td>NaN</td>
      <td>NaN</td>
      <td>NaN</td>
      <td>NaN</td>
      <td>NaN</td>
      <td>NaN</td>
      <td>NaN</td>
      <td>NaN</td>
      <td>NaN</td>
      <td>NaN</td>
      <td>NaN</td>
      <td>NaN</td>
      <td>0.450654</td>
      <td>NaN</td>
      <td>0.999313</td>
      <td>0.999545</td>
      <td>0.994505</td>
      <td>6197</td>
      <td>1478</td>
      <td>1114</td>
      <td>5187</td>
    </tr>
  </tbody>
</table></div><div class='container mt-4 bg-light p-4 text-center'><h2 class='text-muted'>Original GWAS file headers</h2><table border="1" class="dataframe table table-bordered">
  <thead>
    <tr style="text-align: right;">
      <th>snpid</th>
      <th>beta_uganda</th>
      <th>se_uganda</th>
      <th>beta_dcc</th>
      <th>se_dcc</th>
      <th>beta_dds</th>
      <th>se_dds</th>
      <th>beta_aadm</th>
      <th>se_aadm</th>
      <th>studyno</th>
      <th>pval_fe</th>
      <th>beta_fe</th>
      <th>se_fe</th>
      <th>pval_re</th>
      <th>beta_re</th>
      <th>se_re</th>
      <th>pval_re2</th>
      <th>isquare</th>
      <th>q</th>
      <th>pval_q</th>
      <th>tau_square</th>
      <th>pval_uganda</th>
      <th>pval_dcc</th>
      <th>pval_dds</th>
      <th>pval_aadm</th>
      <th>af_uganda</th>
      <th>af_dcc</th>
      <th>af_dds</th>
      <th>af_aadm</th>
      <th>no_uganda</th>
      <th>no_dcc</th>
      <th>no_dds</th>
      <th>no_aadm</th>
    </tr>
  </thead>
  <tbody>
    <tr>
      <td>10_100000012_G_A</td>
      <td>0.069551</td>
      <td>0.057426</td>
      <td>-0.176789</td>
      <td>0.095124</td>
      <td>-0.005907</td>
      <td>0.116692</td>
      <td>-0.4927</td>
      <td>0.3718</td>
      <td>4</td>
      <td>0.912445</td>
      <td>-0.004945</td>
      <td>0.044973</td>
      <td>0.555139</td>
      <td>-0.048551</td>
      <td>0.082279</td>
      <td>0.458041</td>
      <td>55.0057</td>
      <td>6.66751</td>
      <td>0.083285</td>
      <td>0.013447</td>
      <td>0.225836</td>
      <td>0.063097</td>
      <td>0.959626</td>
      <td>0.185113</td>
      <td>0.972764</td>
      <td>0.960781</td>
      <td>0.965522</td>
      <td>0.976190</td>
      <td>6197</td>
      <td>1478</td>
      <td>1114</td>
      <td>5187</td>
    </tr>
    <tr>
      <td>10_10000010_C_T</td>
      <td>NaN</td>
      <td>NaN</td>
      <td>NaN</td>
      <td>NaN</td>
      <td>NaN</td>
      <td>NaN</td>
      <td>0.5668</td>
      <td>0.7514</td>
      <td>1</td>
      <td>NaN</td>
      <td>NaN</td>
      <td>NaN</td>
      <td>NaN</td>
      <td>NaN</td>
      <td>NaN</td>
      <td>NaN</td>
      <td>NaN</td>
      <td>NaN</td>
      <td>NaN</td>
      <td>NaN</td>
      <td>NaN</td>
      <td>NaN</td>
      <td>NaN</td>
      <td>0.450654</td>
      <td>NaN</td>
      <td>0.999313</td>
      <td>0.999545</td>
      <td>0.994505</td>
      <td>6197</td>
      <td>1478</td>
      <td>1114</td>
      <td>5187</td>
    </tr>
  </tbody>
</table><p style='font-weight: bold; color: blue;'>CHR -> []</p><p style='font-weight: bold; color: blue;'>SNP -> ['snpid']</p><p style='font-weight: bold; color: blue;'>BP -> []</p><p style='font-weight: bold; color: blue;'>A1 -> []</p><p style='font-weight: bold; color: blue;'>A2 -> []</p><p style='font-weight: bold; color: blue;'>N -> []</p><p style='font-weight: bold; color: blue;'>P -> ['pval_fe', 'pval_re', 'pval_re2', 'pval_q', 'pval_uganda', 'pval_dcc', 'pval_dds', 'pval_aadm']</p><p style='font-weight: bold; color: blue;'>INFO -> []</p><p style='font-weight: bold; color: blue;'>MAF -> ['af_uganda', 'af_dcc', 'af_dds', 'af_aadm']</p><p style='font-weight: bold; color: blue;'>BETA -> []</p><p style='font-weight: bold; color: blue;'>SE -> []</p><p style='font-weight: bold; color: blue;'>OR -> []</p><p style='font-weight: bold; color: blue;'>Z -> []</p><p style='font-weight: bold; color: blue;'>D -> []</p><h2 class='text-muted'>GWAS columns</h2>snpid, beta_uganda, se_uganda, beta_dcc, se_dcc, beta_dds, se_dds, beta_aadm, se_aadm, studyno, pval_fe, beta_fe, se_fe, pval_re, beta_re, se_re, pval_re2, isquare, q, pval_q, tau_square, pval_uganda, pval_dcc, pval_dds, pval_aadm, af_uganda, af_dcc, af_dds, af_aadm, no_uganda, no_dcc, no_dds, no_aadm<h2 class='text-muted'>Identified columns</h2>snpid, pval_fe, pval_re, pval_re2, pval_q, pval_uganda, pval_dcc, pval_dds, pval_aadm, af_uganda, af_dcc, af_dds, af_aadm<h2 class='text-muted'>Unidentified Columns headers</h2>se_uganda, no_dcc, tau_square, beta_dds, studyno, se_dcc, beta_dcc, se_fe, q, beta_uganda, isquare, beta_re, se_re, beta_fe, se_aadm, no_uganda, no_dds, beta_aadm, no_aadm, se_dds<h2 class='text-muted'>PMID</h2>31675503<h2 class='text-muted'>DOI</h2>10.1016/j.cell.2019.10.004<div class='bibtex-container'><h2 class='text-muted'>BibTeX</h2><div style='font-family: monospace; white-space: pre-wrap;'> @article{Gurdasani_2019, title={Uganda Genome Resource Enables Insights into Population History and Genomic Discovery in Africa}, volume={179}, ISSN={0092-8674}, url={http://dx.doi.org/10.1016/j.cell.2019.10.004}, DOI={10.1016/j.cell.2019.10.004}, number={4}, journal={Cell}, publisher={Elsevier BV}, author={Gurdasani, Deepti and Carstensen, Tommy and Fatumo, Segun and Chen, Guanjie and Franklin, Chris S. and Prado-Martinez, Javier and Bouman, Heleen and Abascal, Federico and Haber, Marc and Tachmazidou, Ioanna and Mathieson, Iain and Ekoru, Kenneth and DeGorter, Marianne K. and Nsubuga, Rebecca N. and Finan, Chris and Wheeler, Eleanor and Chen, Li and Cooper, David N. and Schiffels, Stephan and Chen, Yuan and Ritchie, Graham R.S. and Pollard, Martin O. and Fortune, Mary D. and Mentzer, Alex J. and Garrison, Erik and Bergström, Anders and Hatzikotoulas, Konstantinos and Adeyemo, Adebowale and Doumatey, Ayo and Elding, Heather and Wain, Louise V. and Ehret, Georg and Auer, Paul L. and Kooperberg, Charles L. and Reiner, Alexander P. and Franceschini, Nora and Maher, Dermot and Montgomery, Stephen B. and Kadie, Carl and Widmer, Chris and Xue, Yali and Seeley, Janet and Asiki, Gershim and Kamali, Anatoli and Young, Elizabeth H. and Pomilla, Cristina and Soranzo, Nicole and Zeggini, Eleftheria and Pirie, Fraser and Morris, Andrew P. and Heckerman, David and Tyler-Smith, Chris and Motala, Ayesha A. and Rotimi, Charles and Kaleebu, Pontiano and Barroso, Inês and Sandhu, Manj S.}, year={2019}, month=oct, pages={984-1002.e36} }
</div></div><br><br><br><br><br><br></div><div class='container mt-4 bg-light p-4 text-center'><h1 class='text-primary'>Phenotype: body mass index</h1><p class='text-muted'><strong>URL:</strong> http://ftp.ebi.ac.uk/pub/databases/gwas/summary_statistics/GCST004001-GCST005000/GCST004904</p><table border="1" class="dataframe table table-bordered">
  <thead>
    <tr style="text-align: right;">
      <th>firstAuthor</th>
      <th>accessionId</th>
      <th>publicationDate</th>
      <th>journal</th>
      <th>title</th>
      <th>reportedTrait</th>
      <th>efoTraits</th>
      <th>bgTraits</th>
      <th>discoverySampleAncestry</th>
      <th>replicationSampleAncestry</th>
      <th>associationCount</th>
      <th>summaryStatistics</th>
      <th>ssApiFlag</th>
      <th>agreedToCc0</th>
      <th>genotypingTechnologies</th>
      <th>pubmedId</th>
      <th>initialSampleDescription</th>
      <th>replicateSampleDescription</th>
      <th>UKB_FIELD</th>
      <th>FuzzySimilarity_body mass index bmi</th>
      <th>SearchPhenotype</th>
    </tr>
  </thead>
  <tbody>
    <tr>
      <td>Akiyama M</td>
      <td>GCST004904</td>
      <td>9/11/2017</td>
      <td>Nat Genet</td>
      <td>Genome-wide association study identifies 112 new loci for body mass index in the Japanese population.</td>
      <td>body mass index</td>
      <td>body mass index</td>
      <td>-</td>
      <td>158284 east asian</td>
      <td>322154 European,15146 East Asian</td>
      <td>265</td>
      <td>http://ftp.ebi.ac.uk/pub/databases/gwas/summary_statistics/GCST004001-GCST005000/GCST004904</td>
      <td>yes</td>
      <td>-</td>
      <td>Genome-wide genotyping array</td>
      <td>28892062</td>
      <td>72,390 Japanese ancestry women, up to ,85,894 Japanese ancestry men</td>
      <td>up to 10,048 Japanese ancestry women,up to 5,098 Japanese ancestry men,up to 181,999 European ancestry women,up to 140,155 European ancestry controls</td>
      <td>-</td>
      <td>88</td>
      <td>body mass index bmi</td>
    </tr>
  </tbody>
</table><h2 class='text-muted'>Information file</h2><table border="1" class="dataframe table table-bordered">
  <thead>
    <tr style="text-align: right;">
      <th>Unnamed: 0</th>
      <th>Name</th>
      <th>Last modified</th>
      <th>Size</th>
      <th>Description</th>
      <th>URL</th>
    </tr>
  </thead>
  <tbody>
    <tr>
      <td>3</td>
      <td>harmonised/</td>
      <td>2019-05-23 17:18</td>
      <td>-</td>
      <td>NaN</td>
      <td>http://ftp.ebi.ac.uk/pub/databases/gwas/summary_statistics/GCST004001-GCST005000/GCST004904/harmonised/</td>
    </tr>
    <tr>
      <td>0</td>
      <td>All_2017_BMI_BBJ_autosome.txt.gz</td>
      <td>2019-04-13 23:07</td>
      <td>130M</td>
      <td>NaN</td>
      <td>http://ftp.ebi.ac.uk/pub/databases/gwas/summary_statistics/GCST004001-GCST005000/GCST004904/All_2017_BMI_BBJ_autosome.txt.gz</td>
    </tr>
    <tr>
      <td>1</td>
      <td>Female_2017_BMI_BBJ_autosome.txt.gz</td>
      <td>2019-04-13 23:07</td>
      <td>130M</td>
      <td>NaN</td>
      <td>http://ftp.ebi.ac.uk/pub/databases/gwas/summary_statistics/GCST004001-GCST005000/GCST004904/Female_2017_BMI_BBJ_autosome.txt.gz</td>
    </tr>
    <tr>
      <td>2</td>
      <td>Male_2017_BMI_BBJ_autosome.txt.gz</td>
      <td>2019-04-13 23:07</td>
      <td>130M</td>
      <td>NaN</td>
      <td>http://ftp.ebi.ac.uk/pub/databases/gwas/summary_statistics/GCST004001-GCST005000/GCST004904/Male_2017_BMI_BBJ_autosome.txt.gz</td>
    </tr>
  </tbody>
</table><h2 class='text-muted'>Processed file</h2><p>Processing file: All_2017_BMI_BBJ_autosome.txt.gz!</p><h2 class='text-muted'>GWAS file head</h2><table border="1" class="dataframe table table-bordered">
  <thead>
    <tr style="text-align: right;">
      <th>snp</th>
      <th>chr</th>
      <th>pos</th>
      <th>ref</th>
      <th>alt</th>
      <th>frq</th>
      <th>rsq</th>
      <th>beta</th>
      <th>se</th>
      <th>p</th>
    </tr>
  </thead>
  <tbody>
    <tr>
      <td>chr10_103577643</td>
      <td>10</td>
      <td>103577643</td>
      <td>C</td>
      <td>T</td>
      <td>0.0302</td>
      <td>0.995</td>
      <td>-0.006703</td>
      <td>0.01048</td>
      <td>0.522500</td>
    </tr>
    <tr>
      <td>chr10_104678841</td>
      <td>10</td>
      <td>104678841</td>
      <td>G</td>
      <td>A</td>
      <td>0.0198</td>
      <td>0.759</td>
      <td>-0.044190</td>
      <td>0.01461</td>
      <td>0.002493</td>
    </tr>
  </tbody>
</table></div><div class='container mt-4 bg-light p-4 text-center'><h2 class='text-muted'>Original GWAS file headers</h2><table border="1" class="dataframe table table-bordered">
  <thead>
    <tr style="text-align: right;">
      <th>snp</th>
      <th>chr</th>
      <th>pos</th>
      <th>ref</th>
      <th>alt</th>
      <th>frq</th>
      <th>rsq</th>
      <th>beta</th>
      <th>se</th>
      <th>p</th>
    </tr>
  </thead>
  <tbody>
    <tr>
      <td>chr10_103577643</td>
      <td>10</td>
      <td>103577643</td>
      <td>C</td>
      <td>T</td>
      <td>0.0302</td>
      <td>0.995</td>
      <td>-0.006703</td>
      <td>0.01048</td>
      <td>0.522500</td>
    </tr>
    <tr>
      <td>chr10_104678841</td>
      <td>10</td>
      <td>104678841</td>
      <td>G</td>
      <td>A</td>
      <td>0.0198</td>
      <td>0.759</td>
      <td>-0.044190</td>
      <td>0.01461</td>
      <td>0.002493</td>
    </tr>
  </tbody>
</table><p style='font-weight: bold; color: blue;'>CHR -> ['chr']</p><p style='font-weight: bold; color: blue;'>SNP -> ['snp', 'rsq']</p><p style='font-weight: bold; color: blue;'>BP -> ['pos']</p><p style='font-weight: bold; color: blue;'>A1 -> ['ref']</p><p style='font-weight: bold; color: blue;'>A2 -> ['alt']</p><p style='font-weight: bold; color: blue;'>N -> []</p><p style='font-weight: bold; color: blue;'>P -> ['p']</p><p style='font-weight: bold; color: blue;'>INFO -> []</p><p style='font-weight: bold; color: blue;'>MAF -> ['frq']</p><p style='font-weight: bold; color: blue;'>BETA -> ['beta']</p><p style='font-weight: bold; color: blue;'>SE -> ['se']</p><p style='font-weight: bold; color: blue;'>OR -> []</p><p style='font-weight: bold; color: blue;'>Z -> []</p><p style='font-weight: bold; color: blue;'>D -> []</p><h2 class='text-muted'>GWAS columns</h2>snp, chr, pos, ref, alt, frq, rsq, beta, se, p<h2 class='text-muted'>Identified columns</h2>chr, snp, rsq, pos, ref, alt, p, frq, beta, se<h2 class='text-muted'>Unidentified Columns headers</h2><h2 class='text-muted'>PMID</h2>28892062<h2 class='text-muted'>DOI</h2>Could not find DOI!<div class='bibtex-container'><h2 class='text-muted'>BibTeX</h2><div style='font-family: monospace; white-space: pre-wrap;'></div></div><br><br><br><br><br><br></div><div class='container mt-4 bg-light p-4 text-center'><h1 class='text-primary'>Phenotype: body mass index</h1><p class='text-muted'><strong>URL:</strong> http://ftp.ebi.ac.uk/pub/databases/gwas/summary_statistics/GCST90018001-GCST90019000/GCST90018727</p><table border="1" class="dataframe table table-bordered">
  <thead>
    <tr style="text-align: right;">
      <th>firstAuthor</th>
      <th>accessionId</th>
      <th>publicationDate</th>
      <th>journal</th>
      <th>title</th>
      <th>reportedTrait</th>
      <th>efoTraits</th>
      <th>bgTraits</th>
      <th>discoverySampleAncestry</th>
      <th>replicationSampleAncestry</th>
      <th>associationCount</th>
      <th>summaryStatistics</th>
      <th>ssApiFlag</th>
      <th>agreedToCc0</th>
      <th>genotypingTechnologies</th>
      <th>pubmedId</th>
      <th>initialSampleDescription</th>
      <th>replicateSampleDescription</th>
      <th>UKB_FIELD</th>
      <th>FuzzySimilarity_body mass index bmi</th>
      <th>SearchPhenotype</th>
    </tr>
  </thead>
  <tbody>
    <tr>
      <td>Sakaue S</td>
      <td>GCST90018727</td>
      <td>9/30/2021</td>
      <td>Nat Genet</td>
      <td>A cross-population atlas of genetic associations for 220 human phenotypes.</td>
      <td>body mass index</td>
      <td>body mass index</td>
      <td>-</td>
      <td>163835 east asian</td>
      <td>NaN</td>
      <td>89</td>
      <td>http://ftp.ebi.ac.uk/pub/databases/gwas/summary_statistics/GCST90018001-GCST90019000/GCST90018727</td>
      <td>no</td>
      <td>yes</td>
      <td>Genome-wide genotyping array</td>
      <td>34594039</td>
      <td>163,835 East Asian ancestry individuals</td>
      <td>NaN</td>
      <td>-</td>
      <td>88</td>
      <td>body mass index bmi</td>
    </tr>
  </tbody>
</table><h2 class='text-muted'>Information file</h2><table border="1" class="dataframe table table-bordered">
  <thead>
    <tr style="text-align: right;">
      <th>Unnamed: 0</th>
      <th>Name</th>
      <th>Last modified</th>
      <th>Size</th>
      <th>Description</th>
      <th>URL</th>
    </tr>
  </thead>
  <tbody>
    <tr>
      <td>1</td>
      <td>GCST90018727_buildGRCh37.tsv.gz-meta.yaml</td>
      <td>2024-01-04 00:32</td>
      <td>946</td>
      <td>NaN</td>
      <td>http://ftp.ebi.ac.uk/pub/databases/gwas/summary_statistics/GCST90018001-GCST90019000/GCST90018727/GCST90018727_buildGRCh37.tsv.gz-meta.yaml</td>
    </tr>
    <tr>
      <td>2</td>
      <td>README.txt</td>
      <td>2021-12-16 00:56</td>
      <td>177</td>
      <td>NaN</td>
      <td>http://ftp.ebi.ac.uk/pub/databases/gwas/summary_statistics/GCST90018001-GCST90019000/GCST90018727/README.txt</td>
    </tr>
    <tr>
      <td>3</td>
      <td>harmonised/</td>
      <td>2024-02-02 15:17</td>
      <td>-</td>
      <td>NaN</td>
      <td>http://ftp.ebi.ac.uk/pub/databases/gwas/summary_statistics/GCST90018001-GCST90019000/GCST90018727/harmonised/</td>
    </tr>
    <tr>
      <td>4</td>
      <td>md5sum.txt</td>
      <td>2021-12-16 00:56</td>
      <td>109</td>
      <td>NaN</td>
      <td>http://ftp.ebi.ac.uk/pub/databases/gwas/summary_statistics/GCST90018001-GCST90019000/GCST90018727/md5sum.txt</td>
    </tr>
    <tr>
      <td>0</td>
      <td>GCST90018727_buildGRCh37.tsv.gz</td>
      <td>2021-12-16 00:56</td>
      <td>232M</td>
      <td>NaN</td>
      <td>http://ftp.ebi.ac.uk/pub/databases/gwas/summary_statistics/GCST90018001-GCST90019000/GCST90018727/GCST90018727_buildGRCh37.tsv.gz</td>
    </tr>
  </tbody>
</table><h2 class='text-muted'>Processed file</h2><p>Processing file: GCST90018727_buildGRCh37.tsv.gz!</p><h2 class='text-muted'>GWAS file head</h2><table border="1" class="dataframe table table-bordered">
  <thead>
    <tr style="text-align: right;">
      <th>chromosome</th>
      <th>base_pair_location</th>
      <th>effect_allele</th>
      <th>other_allele</th>
      <th>effect_allele_frequency</th>
      <th>beta</th>
      <th>standard_error</th>
      <th>p_value</th>
      <th>variant_id</th>
    </tr>
  </thead>
  <tbody>
    <tr>
      <td>1</td>
      <td>751343</td>
      <td>A</td>
      <td>T</td>
      <td>0.148008</td>
      <td>-0.000594</td>
      <td>0.004687</td>
      <td>0.9</td>
      <td>NaN</td>
    </tr>
    <tr>
      <td>1</td>
      <td>751756</td>
      <td>C</td>
      <td>T</td>
      <td>0.147545</td>
      <td>-0.000571</td>
      <td>0.004676</td>
      <td>0.9</td>
      <td>NaN</td>
    </tr>
  </tbody>
</table></div><div class='container mt-4 bg-light p-4 text-center'><h2 class='text-muted'>Original GWAS file headers</h2><table border="1" class="dataframe table table-bordered">
  <thead>
    <tr style="text-align: right;">
      <th>chromosome</th>
      <th>base_pair_location</th>
      <th>effect_allele</th>
      <th>other_allele</th>
      <th>effect_allele_frequency</th>
      <th>beta</th>
      <th>standard_error</th>
      <th>p_value</th>
      <th>variant_id</th>
    </tr>
  </thead>
  <tbody>
    <tr>
      <td>1</td>
      <td>751343</td>
      <td>A</td>
      <td>T</td>
      <td>0.148008</td>
      <td>-0.000594</td>
      <td>0.004687</td>
      <td>0.9</td>
      <td>NaN</td>
    </tr>
    <tr>
      <td>1</td>
      <td>751756</td>
      <td>C</td>
      <td>T</td>
      <td>0.147545</td>
      <td>-0.000571</td>
      <td>0.004676</td>
      <td>0.9</td>
      <td>NaN</td>
    </tr>
  </tbody>
</table><p style='font-weight: bold; color: blue;'>CHR -> ['chromosome']</p><p style='font-weight: bold; color: blue;'>SNP -> ['variant_id']</p><p style='font-weight: bold; color: blue;'>BP -> ['base_pair_location']</p><p style='font-weight: bold; color: blue;'>A1 -> ['effect_allele']</p><p style='font-weight: bold; color: blue;'>A2 -> ['other_allele']</p><p style='font-weight: bold; color: blue;'>N -> []</p><p style='font-weight: bold; color: blue;'>P -> ['p_value']</p><p style='font-weight: bold; color: blue;'>INFO -> []</p><p style='font-weight: bold; color: blue;'>MAF -> ['effect_allele_frequency']</p><p style='font-weight: bold; color: blue;'>BETA -> ['beta']</p><p style='font-weight: bold; color: blue;'>SE -> ['standard_error']</p><p style='font-weight: bold; color: blue;'>OR -> []</p><p style='font-weight: bold; color: blue;'>Z -> []</p><p style='font-weight: bold; color: blue;'>D -> []</p><h2 class='text-muted'>GWAS columns</h2>chromosome, base_pair_location, effect_allele, other_allele, effect_allele_frequency, beta, standard_error, p_value, variant_id<h2 class='text-muted'>Identified columns</h2>chromosome, variant_id, base_pair_location, effect_allele, other_allele, p_value, effect_allele_frequency, beta, standard_error<h2 class='text-muted'>Unidentified Columns headers</h2><h2 class='text-muted'>PMID</h2>34594039<h2 class='text-muted'>DOI</h2>Could not find DOI!<div class='bibtex-container'><h2 class='text-muted'>BibTeX</h2><div style='font-family: monospace; white-space: pre-wrap;'></div></div><br><br><br><br><br><br></div><div class='container mt-4 bg-light p-4 text-center'><h1 class='text-primary'>Phenotype: body mass index</h1><p class='text-muted'><strong>URL:</strong> http://ftp.ebi.ac.uk/pub/databases/gwas/summary_statistics/GCST008001-GCST009000/GCST008025</p><table border="1" class="dataframe table table-bordered">
  <thead>
    <tr style="text-align: right;">
      <th>firstAuthor</th>
      <th>accessionId</th>
      <th>publicationDate</th>
      <th>journal</th>
      <th>title</th>
      <th>reportedTrait</th>
      <th>efoTraits</th>
      <th>bgTraits</th>
      <th>discoverySampleAncestry</th>
      <th>replicationSampleAncestry</th>
      <th>associationCount</th>
      <th>summaryStatistics</th>
      <th>ssApiFlag</th>
      <th>agreedToCc0</th>
      <th>genotypingTechnologies</th>
      <th>pubmedId</th>
      <th>initialSampleDescription</th>
      <th>replicateSampleDescription</th>
      <th>UKB_FIELD</th>
      <th>FuzzySimilarity_body mass index bmi</th>
      <th>SearchPhenotype</th>
    </tr>
  </thead>
  <tbody>
    <tr>
      <td>Wojcik GL</td>
      <td>GCST008025</td>
      <td>6/19/2019</td>
      <td>Nature</td>
      <td>Genetic analyses of diverse populations improves discovery for complex traits.</td>
      <td>body mass index</td>
      <td>body mass index</td>
      <td>-</td>
      <td>17127 african american or afro-caribbean,21955 hispanic or latin american,1025 nr,3936 oceanian,645 native american,4647 east asian, asian unspecified</td>
      <td>NaN</td>
      <td>63</td>
      <td>http://ftp.ebi.ac.uk/pub/databases/gwas/summary_statistics/GCST008001-GCST009000/GCST008025</td>
      <td>yes</td>
      <td>-</td>
      <td>Genome-wide genotyping array</td>
      <td>31217584</td>
      <td>17,127 African American individuals,21,955 Hispanic/Latino individuals,4,647 Asian ancestry individuals,3,936 Native Hawaiian ancestry individuals, 645 Native American ancestry individuals,1,025 individuals</td>
      <td>NaN</td>
      <td>-</td>
      <td>88</td>
      <td>body mass index bmi</td>
    </tr>
  </tbody>
</table><h2 class='text-muted'>Information file</h2><table border="1" class="dataframe table table-bordered">
  <thead>
    <tr style="text-align: right;">
      <th>Unnamed: 0</th>
      <th>Name</th>
      <th>Last modified</th>
      <th>Size</th>
      <th>Description</th>
      <th>URL</th>
    </tr>
  </thead>
  <tbody>
    <tr>
      <td>2</td>
      <td>harmonised/</td>
      <td>2019-12-06 04:22</td>
      <td>-</td>
      <td>NaN</td>
      <td>http://ftp.ebi.ac.uk/pub/databases/gwas/summary_statistics/GCST008001-GCST009000/GCST008025/harmonised/</td>
    </tr>
    <tr>
      <td>0</td>
      <td>README</td>
      <td>2019-07-12 12:09</td>
      <td>3.0K</td>
      <td>NaN</td>
      <td>http://ftp.ebi.ac.uk/pub/databases/gwas/summary_statistics/GCST008001-GCST009000/GCST008025/README</td>
    </tr>
    <tr>
      <td>1</td>
      <td>WojcikG_PMID_invn_rbmi_alls.gz</td>
      <td>2019-07-12 12:09</td>
      <td>1.2G</td>
      <td>NaN</td>
      <td>http://ftp.ebi.ac.uk/pub/databases/gwas/summary_statistics/GCST008001-GCST009000/GCST008025/WojcikG_PMID_invn_rbmi_alls.gz</td>
    </tr>
  </tbody>
</table><h2 class='text-muted'>Processed file</h2><p>Processing file: WojcikG_PMID_invn_rbmi_alls.gz!</p><h2 class='text-muted'>GWAS file head</h2><table border="1" class="dataframe table table-bordered">
  <thead>
    <tr style="text-align: right;">
      <th>chr</th>
      <th>position_hg19</th>
      <th>snp</th>
      <th>other-allele</th>
      <th>effect-allele</th>
      <th>effect-allele-frequency</th>
      <th>sample-size</th>
      <th>beta</th>
      <th>se</th>
      <th>p-val</th>
      <th>info-score</th>
      <th>rsid</th>
    </tr>
  </thead>
  <tbody>
    <tr>
      <td>1</td>
      <td>10539</td>
      <td>1_10539_C_A</td>
      <td>C</td>
      <td>A</td>
      <td>0.004358</td>
      <td>49335</td>
      <td>0.019406</td>
      <td>0.069027</td>
      <td>0.778600</td>
      <td>0.460</td>
      <td>rs537182016</td>
    </tr>
    <tr>
      <td>1</td>
      <td>10616</td>
      <td>rs376342519_10616_CCGCCGTTGCAAAGGCGCGCCG_C</td>
      <td>CCGCCGTTGCAAAGGCGCGCCG</td>
      <td>C</td>
      <td>0.991607</td>
      <td>49335</td>
      <td>-0.042520</td>
      <td>0.041136</td>
      <td>0.301305</td>
      <td>0.604</td>
      <td>rs376342519</td>
    </tr>
  </tbody>
</table></div><div class='container mt-4 bg-light p-4 text-center'><h2 class='text-muted'>Original GWAS file headers</h2><table border="1" class="dataframe table table-bordered">
  <thead>
    <tr style="text-align: right;">
      <th>chr</th>
      <th>position_hg19</th>
      <th>snp</th>
      <th>other-allele</th>
      <th>effect-allele</th>
      <th>effect-allele-frequency</th>
      <th>sample-size</th>
      <th>beta</th>
      <th>se</th>
      <th>p-val</th>
      <th>info-score</th>
      <th>rsid</th>
    </tr>
  </thead>
  <tbody>
    <tr>
      <td>1</td>
      <td>10539</td>
      <td>1_10539_C_A</td>
      <td>C</td>
      <td>A</td>
      <td>0.004358</td>
      <td>49335</td>
      <td>0.019406</td>
      <td>0.069027</td>
      <td>0.778600</td>
      <td>0.460</td>
      <td>rs537182016</td>
    </tr>
    <tr>
      <td>1</td>
      <td>10616</td>
      <td>rs376342519_10616_CCGCCGTTGCAAAGGCGCGCCG_C</td>
      <td>CCGCCGTTGCAAAGGCGCGCCG</td>
      <td>C</td>
      <td>0.991607</td>
      <td>49335</td>
      <td>-0.042520</td>
      <td>0.041136</td>
      <td>0.301305</td>
      <td>0.604</td>
      <td>rs376342519</td>
    </tr>
  </tbody>
</table><p style='font-weight: bold; color: blue;'>CHR -> ['chr']</p><p style='font-weight: bold; color: blue;'>SNP -> ['snp', 'rsid']</p><p style='font-weight: bold; color: blue;'>BP -> ['position_hg19']</p><p style='font-weight: bold; color: blue;'>A1 -> ['effect-allele']</p><p style='font-weight: bold; color: blue;'>A2 -> ['other-allele']</p><p style='font-weight: bold; color: blue;'>N -> ['sample-size']</p><p style='font-weight: bold; color: blue;'>P -> ['p-val']</p><p style='font-weight: bold; color: blue;'>INFO -> ['info-score']</p><p style='font-weight: bold; color: blue;'>MAF -> ['effect-allele-frequency']</p><p style='font-weight: bold; color: blue;'>BETA -> ['beta']</p><p style='font-weight: bold; color: blue;'>SE -> ['se']</p><p style='font-weight: bold; color: blue;'>OR -> []</p><p style='font-weight: bold; color: blue;'>Z -> []</p><p style='font-weight: bold; color: blue;'>D -> []</p><h2 class='text-muted'>GWAS columns</h2>chr, position_hg19, snp, other-allele, effect-allele, effect-allele-frequency, sample-size, beta, se, p-val, info-score, rsid<h2 class='text-muted'>Identified columns</h2>chr, snp, rsid, position_hg19, effect-allele, other-allele, sample-size, p-val, info-score, effect-allele-frequency, beta, se<h2 class='text-muted'>Unidentified Columns headers</h2><h2 class='text-muted'>PMID</h2>31217584<h2 class='text-muted'>DOI</h2>10.1038/s41586-019-1310-4<div class='bibtex-container'><h2 class='text-muted'>BibTeX</h2><div style='font-family: monospace; white-space: pre-wrap;'> @article{Wojcik_2019, title={Genetic analyses of diverse populations improves discovery for complex traits}, volume={570}, ISSN={1476-4687}, url={http://dx.doi.org/10.1038/s41586-019-1310-4}, DOI={10.1038/s41586-019-1310-4}, number={7762}, journal={Nature}, publisher={Springer Science and Business Media LLC}, author={Wojcik, Genevieve L. and Graff, Mariaelisa and Nishimura, Katherine K. and Tao, Ran and Haessler, Jeffrey and Gignoux, Christopher R. and Highland, Heather M. and Patel, Yesha M. and Sorokin, Elena P. and Avery, Christy L. and Belbin, Gillian M. and Bien, Stephanie A. and Cheng, Iona and Cullina, Sinead and Hodonsky, Chani J. and Hu, Yao and Huckins, Laura M. and Jeff, Janina and Justice, Anne E. and Kocarnik, Jonathan M. and Lim, Unhee and Lin, Bridget M. and Lu, Yingchang and Nelson, Sarah C. and Park, Sung-Shim L. and Poisner, Hannah and Preuss, Michael H. and Richard, Melissa A. and Schurmann, Claudia and Setiawan, Veronica W. and Sockell, Alexandra and Vahi, Karan and Verbanck, Marie and Vishnu, Abhishek and Walker, Ryan W. and Young, Kristin L. and Zubair, Niha and Acuña-Alonso, Victor and Ambite, Jose Luis and Barnes, Kathleen C. and Boerwinkle, Eric and Bottinger, Erwin P. and Bustamante, Carlos D. and Caberto, Christian and Canizales-Quinteros, Samuel and Conomos, Matthew P. and Deelman, Ewa and Do, Ron and Doheny, Kimberly and Fernández-Rhodes, Lindsay and Fornage, Myriam and Hailu, Benyam and Heiss, Gerardo and Henn, Brenna M. and Hindorff, Lucia A. and Jackson, Rebecca D. and Laurie, Cecelia A. and Laurie, Cathy C. and Li, Yuqing and Lin, Dan-Yu and Moreno-Estrada, Andres and Nadkarni, Girish and Norman, Paul J. and Pooler, Loreall C. and Reiner, Alexander P. and Romm, Jane and Sabatti, Chiara and Sandoval, Karla and Sheng, Xin and Stahl, Eli A. and Stram, Daniel O. and Thornton, Timothy A. and Wassel, Christina L. and Wilkens, Lynne R. and Winkler, Cheryl A. and Yoneyama, Sachi and Buyske, Steven and Haiman, Christopher A. and Kooperberg, Charles and Le Marchand, Loic and Loos, Ruth J. F. and Matise, Tara C. and North, Kari E. and Peters, Ulrike and Kenny, Eimear E. and Carlson, Christopher S.}, year={2019}, month=jun, pages={514–518} }
</div></div><br><br><br><br><br><br></div><div class='container mt-4 bg-light p-4 text-center'><h1 class='text-primary'>Phenotype: body mass index</h1><p class='text-muted'><strong>URL:</strong> http://ftp.ebi.ac.uk/pub/databases/gwas/summary_statistics/GCST002001-GCST003000/GCST002783</p><table border="1" class="dataframe table table-bordered">
  <thead>
    <tr style="text-align: right;">
      <th>firstAuthor</th>
      <th>accessionId</th>
      <th>publicationDate</th>
      <th>journal</th>
      <th>title</th>
      <th>reportedTrait</th>
      <th>efoTraits</th>
      <th>bgTraits</th>
      <th>discoverySampleAncestry</th>
      <th>replicationSampleAncestry</th>
      <th>associationCount</th>
      <th>summaryStatistics</th>
      <th>ssApiFlag</th>
      <th>agreedToCc0</th>
      <th>genotypingTechnologies</th>
      <th>pubmedId</th>
      <th>initialSampleDescription</th>
      <th>replicateSampleDescription</th>
      <th>UKB_FIELD</th>
      <th>FuzzySimilarity_body mass index bmi</th>
      <th>SearchPhenotype</th>
    </tr>
  </thead>
  <tbody>
    <tr>
      <td>Locke AE</td>
      <td>GCST002783</td>
      <td>2/12/2015</td>
      <td>Nature</td>
      <td>Genetic studies of body mass index yield new insights for obesity biology.</td>
      <td>body mass index</td>
      <td>body mass index</td>
      <td>-</td>
      <td>236781 european,1276 hispanic or latin american,887 african american or afro-caribbean</td>
      <td>88138 European,492 NR,1747 Asian unspecified,3622 South Asian,8755 African American or Afro-Caribbean</td>
      <td>634</td>
      <td>http://ftp.ebi.ac.uk/pub/databases/gwas/summary_statistics/GCST002001-GCST003000/GCST002783</td>
      <td>yes</td>
      <td>-</td>
      <td>Genome-wide genotyping array</td>
      <td>25673413</td>
      <td>104,666 European ancestry male individuals, up to ,132,115 European ancestry female individuals, 370 African American male individuals, 517 African American female individuals, 512 Hispanic male individuals, 764 Hispanic female individuals</td>
      <td>up to 48,274 European ancestry male individuals,up to 39,864 European ancestry female individuals,2,441 African American or Afro-Caribbean male individuals,6,314 African American or Afro-Caribbean female individuals,919 Filipino male individuals,828 Filipino female individuals,205 Seychellois male individuals,287 Seychellois female individuals,2,964 South Asian ancestry male individuals,658 South Asian ancestry female individuals</td>
      <td>-</td>
      <td>88</td>
      <td>body mass index bmi</td>
    </tr>
  </tbody>
</table><h2 class='text-muted'>Information file</h2><table border="1" class="dataframe table table-bordered">
  <thead>
    <tr style="text-align: right;">
      <th>Unnamed: 0</th>
      <th>Name</th>
      <th>Last modified</th>
      <th>Size</th>
      <th>Description</th>
      <th>URL</th>
    </tr>
  </thead>
  <tbody>
    <tr>
      <td>1</td>
      <td>harmonised/</td>
      <td>2019-12-05 11:21</td>
      <td>-</td>
      <td>NaN</td>
      <td>http://ftp.ebi.ac.uk/pub/databases/gwas/summary_statistics/GCST002001-GCST003000/GCST002783/harmonised/</td>
    </tr>
    <tr>
      <td>0</td>
      <td>All_ancestries_SNP_gwas_mc_merge_nogc.tbl.uniq.gz</td>
      <td>2019-05-15 14:00</td>
      <td>34M</td>
      <td>NaN</td>
      <td>http://ftp.ebi.ac.uk/pub/databases/gwas/summary_statistics/GCST002001-GCST003000/GCST002783/All_ancestries_SNP_gwas_mc_merge_nogc.tbl.uniq.gz</td>
    </tr>
  </tbody>
</table><h2 class='text-muted'>Processed file</h2><p>Processing file: All_ancestries_SNP_gwas_mc_merge_nogc.tbl.uniq.gz!</p><h2 class='text-muted'>GWAS file head</h2><table border="1" class="dataframe table table-bordered">
  <thead>
    <tr style="text-align: right;">
      <th>snp</th>
      <th>a1</th>
      <th>a2</th>
      <th>freq1.hapmap</th>
      <th>b</th>
      <th>se</th>
      <th>p</th>
      <th>n</th>
    </tr>
  </thead>
  <tbody>
    <tr>
      <td>rs1000000</td>
      <td>G</td>
      <td>A</td>
      <td>0.6333</td>
      <td>0.0001</td>
      <td>0.0043</td>
      <td>0.9814</td>
      <td>233572.0</td>
    </tr>
    <tr>
      <td>rs10000010</td>
      <td>T</td>
      <td>C</td>
      <td>0.5750</td>
      <td>-0.0022</td>
      <td>0.0029</td>
      <td>0.4384</td>
      <td>339148.0</td>
    </tr>
  </tbody>
</table></div><div class='container mt-4 bg-light p-4 text-center'><h2 class='text-muted'>Original GWAS file headers</h2><table border="1" class="dataframe table table-bordered">
  <thead>
    <tr style="text-align: right;">
      <th>snp</th>
      <th>a1</th>
      <th>a2</th>
      <th>freq1.hapmap</th>
      <th>b</th>
      <th>se</th>
      <th>p</th>
      <th>n</th>
    </tr>
  </thead>
  <tbody>
    <tr>
      <td>rs1000000</td>
      <td>G</td>
      <td>A</td>
      <td>0.6333</td>
      <td>0.0001</td>
      <td>0.0043</td>
      <td>0.9814</td>
      <td>233572.0</td>
    </tr>
    <tr>
      <td>rs10000010</td>
      <td>T</td>
      <td>C</td>
      <td>0.5750</td>
      <td>-0.0022</td>
      <td>0.0029</td>
      <td>0.4384</td>
      <td>339148.0</td>
    </tr>
  </tbody>
</table><p style='font-weight: bold; color: blue;'>CHR -> []</p><p style='font-weight: bold; color: blue;'>SNP -> ['snp']</p><p style='font-weight: bold; color: blue;'>BP -> ['b']</p><p style='font-weight: bold; color: blue;'>A1 -> ['a1']</p><p style='font-weight: bold; color: blue;'>A2 -> ['a1', 'a2']</p><p style='font-weight: bold; color: blue;'>N -> ['n']</p><p style='font-weight: bold; color: blue;'>P -> ['p']</p><p style='font-weight: bold; color: blue;'>INFO -> []</p><p style='font-weight: bold; color: blue;'>MAF -> ['freq1.hapmap']</p><p style='font-weight: bold; color: blue;'>BETA -> ['b']</p><p style='font-weight: bold; color: blue;'>SE -> ['se']</p><p style='font-weight: bold; color: blue;'>OR -> []</p><p style='font-weight: bold; color: blue;'>Z -> []</p><p style='font-weight: bold; color: blue;'>D -> []</p><h2 class='text-muted'>GWAS columns</h2>snp, a1, a2, freq1.hapmap, b, se, p, n<h2 class='text-muted'>Identified columns</h2>snp, b, a1, a1, a2, n, p, freq1.hapmap, b, se<h2 class='text-muted'>Unidentified Columns headers</h2><h2 class='text-muted'>PMID</h2>25673413<h2 class='text-muted'>DOI</h2>10.1038/nature14177<div class='bibtex-container'><h2 class='text-muted'>BibTeX</h2><div style='font-family: monospace; white-space: pre-wrap;'> @article{Locke_2015, title={Genetic studies of body mass index yield new insights for obesity biology}, volume={518}, ISSN={1476-4687}, url={http://dx.doi.org/10.1038/nature14177}, DOI={10.1038/nature14177}, number={7538}, journal={Nature}, publisher={Springer Science and Business Media LLC}, author={Locke, Adam E. and Kahali, Bratati and Berndt, Sonja I. and Justice, Anne E. and Pers, Tune H. and Day, Felix R. and Powell, Corey and Vedantam, Sailaja and Buchkovich, Martin L. and Yang, Jian and Croteau-Chonka, Damien C. and Esko, Tonu and Fall, Tove and Ferreira, Teresa and Gustafsson, Stefan and Kutalik, Zoltán and Luan, Jian’an and Mägi, Reedik and Randall, Joshua C. and Winkler, Thomas W. and Wood, Andrew R. and Workalemahu, Tsegaselassie and Faul, Jessica D. and Smith, Jennifer A. and Hua Zhao, Jing and Zhao, Wei and Chen, Jin and Fehrmann, Rudolf and Hedman, Åsa K. and Karjalainen, Juha and Schmidt, Ellen M. and Absher, Devin and Amin, Najaf and Anderson, Denise and Beekman, Marian and Bolton, Jennifer L. and Bragg-Gresham, Jennifer L. and Buyske, Steven and Demirkan, Ayse and Deng, Guohong and Ehret, Georg B. and Feenstra, Bjarke and Feitosa, Mary F. and Fischer, Krista and Goel, Anuj and Gong, Jian and Jackson, Anne U. and Kanoni, Stavroula and Kleber, Marcus E. and Kristiansson, Kati and Lim, Unhee and Lotay, Vaneet and Mangino, Massimo and Mateo Leach, Irene and Medina-Gomez, Carolina and Medland, Sarah E. and Nalls, Michael A. and Palmer, Cameron D. and Pasko, Dorota and Pechlivanis, Sonali and Peters, Marjolein J. and Prokopenko, Inga and Shungin, Dmitry and Stančáková, Alena and Strawbridge, Rona J. and Ju Sung, Yun and Tanaka, Toshiko and Teumer, Alexander and Trompet, Stella and van der Laan, Sander W. and van Setten, Jessica and Van Vliet-Ostaptchouk, Jana V. and Wang, Zhaoming and Yengo, Loïc and Zhang, Weihua and Isaacs, Aaron and Albrecht, Eva and Ärnlöv, Johan and Arscott, Gillian M. and Attwood, Antony P. and Bandinelli, Stefania and Barrett, Amy and Bas, Isabelita N. and Bellis, Claire and Bennett, Amanda J. and Berne, Christian and Blagieva, Roza and Blüher, Matthias and Böhringer, Stefan and Bonnycastle, Lori L. and Böttcher, Yvonne and Boyd, Heather A. and Bruinenberg, Marcel and Caspersen, Ida H. and Ida Chen, Yii-Der and Clarke, Robert and Warwick Daw, E. and de Craen, Anton J. M. and Delgado, Graciela and Dimitriou, Maria and Doney, Alex S. F. and Eklund, Niina and Estrada, Karol and Eury, Elodie and Folkersen, Lasse and Fraser, Ross M. and Garcia, Melissa E. and Geller, Frank and Giedraitis, Vilmantas and Gigante, Bruna and Go, Alan S. and Golay, Alain and Goodall, Alison H. and Gordon, Scott D. and Gorski, Mathias and Grabe, Hans-Jörgen and Grallert, Harald and Grammer, Tanja B. and Gräßler, Jürgen and Grönberg, Henrik and Groves, Christopher J. and Gusto, Gaëlle and Haessler, Jeffrey and Hall, Per and Haller, Toomas and Hallmans, Goran and Hartman, Catharina A. and Hassinen, Maija and Hayward, Caroline and Heard-Costa, Nancy L. and Helmer, Quinta and Hengstenberg, Christian and Holmen, Oddgeir and Hottenga, Jouke-Jan and James, Alan L. and Jeff, Janina M. and Johansson, Åsa and Jolley, Jennifer and Juliusdottir, Thorhildur and Kinnunen, Leena and Koenig, Wolfgang and Koskenvuo, Markku and Kratzer, Wolfgang and Laitinen, Jaana and Lamina, Claudia and Leander, Karin and Lee, Nanette R. and Lichtner, Peter and Lind, Lars and Lindström, Jaana and Sin Lo, Ken and Lobbens, Stéphane and Lorbeer, Roberto and Lu, Yingchang and Mach, François and Magnusson, Patrik K. E. and Mahajan, Anubha and McArdle, Wendy L. and McLachlan, Stela and Menni, Cristina and Merger, Sigrun and Mihailov, Evelin and Milani, Lili and Moayyeri, Alireza and Monda, Keri L. and Morken, Mario A. and Mulas, Antonella and Müller, Gabriele and Müller-Nurasyid, Martina and Musk, Arthur W. and Nagaraja, Ramaiah and Nöthen, Markus M. and Nolte, Ilja M. and Pilz, Stefan and Rayner, Nigel W. and Renstrom, Frida and Rettig, Rainer and Ried, Janina S. and Ripke, Stephan and Robertson, Neil R. and Rose, Lynda M. and Sanna, Serena and Scharnagl, Hubert and Scholtens, Salome and Schumacher, Fredrick R. and Scott, William R. and Seufferlein, Thomas and Shi, Jianxin and Vernon Smith, Albert and Smolonska, Joanna and Stanton, Alice V. and Steinthorsdottir, Valgerdur and Stirrups, Kathleen and Stringham, Heather M. and Sundström, Johan and Swertz, Morris A. and Swift, Amy J. and Syvänen, Ann-Christine and Tan, Sian-Tsung and Tayo, Bamidele O. and Thorand, Barbara and Thorleifsson, Gudmar and Tyrer, Jonathan P. and Uh, Hae-Won and Vandenput, Liesbeth and Verhulst, Frank C. and Vermeulen, Sita H. and Verweij, Niek and Vonk, Judith M. and Waite, Lindsay L. and Warren, Helen R. and Waterworth, Dawn and Weedon, Michael N. and Wilkens, Lynne R. and Willenborg, Christina and Wilsgaard, Tom and Wojczynski, Mary K. and Wong, Andrew and Wright, Alan F. and Zhang, Qunyuan and Brennan, Eoin P. and Choi, Murim and Dastani, Zari and Drong, Alexander W. and Eriksson, Per and Franco-Cereceda, Anders and Gådin, Jesper R. and Gharavi, Ali G. and Goddard, Michael E. and Handsaker, Robert E. and Huang, Jinyan and Karpe, Fredrik and Kathiresan, Sekar and Keildson, Sarah and Kiryluk, Krzysztof and Kubo, Michiaki and Lee, Jong-Young and Liang, Liming and Lifton, Richard P. and Ma, Baoshan and McCarroll, Steven A. and McKnight, Amy J. and Min, Josine L. and Moffatt, Miriam F. and Montgomery, Grant W. and Murabito, Joanne M. and Nicholson, George and Nyholt, Dale R. and Okada, Yukinori and Perry, John R. B. and Dorajoo, Rajkumar and Reinmaa, Eva and Salem, Rany M. and Sandholm, Niina and Scott, Robert A. and Stolk, Lisette and Takahashi, Atsushi and Tanaka, Toshihiro and van’t Hooft, Ferdinand M. and Vinkhuyzen, Anna A. E. and Westra, Harm-Jan and Zheng, Wei and Zondervan, Krina T. and Heath, Andrew C. and Arveiler, Dominique and Bakker, Stephan J. L. and Beilby, John and Bergman, Richard N. and Blangero, John and Bovet, Pascal and Campbell, Harry and Caulfield, Mark J. and Cesana, Giancarlo and Chakravarti, Aravinda and Chasman, Daniel I. and Chines, Peter S. and Collins, Francis S. and Crawford, Dana C. and Adrienne Cupples, L. and Cusi, Daniele and Danesh, John and de Faire, Ulf and den Ruijter, Hester M. and Dominiczak, Anna F. and Erbel, Raimund and Erdmann, Jeanette and Eriksson, Johan G. and Farrall, Martin and Felix, Stephan B. and Ferrannini, Ele and Ferrières, Jean and Ford, Ian and Forouhi, Nita G. and Forrester, Terrence and Franco, Oscar H. and Gansevoort, Ron T. and Gejman, Pablo V. and Gieger, Christian and Gottesman, Omri and Gudnason, Vilmundur and Gyllensten, Ulf and Hall, Alistair S. and Harris, Tamara B. and Hattersley, Andrew T. and Hicks, Andrew A. and Hindorff, Lucia A. and Hingorani, Aroon D. and Hofman, Albert and Homuth, Georg and Kees Hovingh, G. and Humphries, Steve E. and Hunt, Steven C. and Hyppönen, Elina and Illig, Thomas and Jacobs, Kevin B. and Jarvelin, Marjo-Riitta and Jöckel, Karl-Heinz and Johansen, Berit and Jousilahti, Pekka and Wouter Jukema, J. and Jula, Antti M. and Kaprio, Jaakko and Kastelein, John J. P. and Keinanen-Kiukaanniemi, Sirkka M. and Kiemeney, Lambertus A. and Knekt, Paul and Kooner, Jaspal S. and Kooperberg, Charles and Kovacs, Peter and Kraja, Aldi T. and Kumari, Meena and Kuusisto, Johanna and Lakka, Timo A. and Langenberg, Claudia and Le Marchand, Loic and Lehtimäki, Terho and Lyssenko, Valeriya and Männistö, Satu and Marette, André and Matise, Tara C. and McKenzie, Colin A. and McKnight, Barbara and Moll, Frans L. and Morris, Andrew D. and Morris, Andrew P. and Murray, Jeffrey C. and Nelis, Mari and Ohlsson, Claes and Oldehinkel, Albertine J. and Ong, Ken K. and Madden, Pamela A. F. and Pasterkamp, Gerard and Peden, John F. and Peters, Annette and Postma, Dirkje S. and Pramstaller, Peter P. and Price, Jackie F. and Qi, Lu and Raitakari, Olli T. and Rankinen, Tuomo and Rao, D. C. and Rice, Treva K. and Ridker, Paul M. and Rioux, John D. and Ritchie, Marylyn D. and Rudan, Igor and Salomaa, Veikko and Samani, Nilesh J. and Saramies, Jouko and Sarzynski, Mark A. and Schunkert, Heribert and Schwarz, Peter E. H. and Sever, Peter and Shuldiner, Alan R. and Sinisalo, Juha and Stolk, Ronald P. and Strauch, Konstantin and Tönjes, Anke and Trégouët, David-Alexandre and Tremblay, Angelo and Tremoli, Elena and Virtamo, Jarmo and Vohl, Marie-Claude and Völker, Uwe and Waeber, Gérard and Willemsen, Gonneke and Witteman, Jacqueline C. and Carola Zillikens, M. and Adair, Linda S. and Amouyel, Philippe and Asselbergs, Folkert W. and Assimes, Themistocles L. and Bochud, Murielle and Boehm, Bernhard O. and Boerwinkle, Eric and Bornstein, Stefan R. and Bottinger, Erwin P. and Bouchard, Claude and Cauchi, Stéphane and Chambers, John C. and Chanock, Stephen J. and Cooper, Richard S. and de Bakker, Paul I. W. and Dedoussis, George and Ferrucci, Luigi and Franks, Paul W. and Froguel, Philippe and Groop, Leif C. and Haiman, Christopher A. and Hamsten, Anders and Hui, Jennie and Hunter, David J. and Hveem, Kristian and Kaplan, Robert C. and Kivimaki, Mika and Kuh, Diana and Laakso, Markku and Liu, Yongmei and Martin, Nicholas G. and März, Winfried and Melbye, Mads and Metspalu, Andres and Moebus, Susanne and Munroe, Patricia B. and Njølstad, Inger and Oostra, Ben A. and Palmer, Colin N. A. and Pedersen, Nancy L. and Perola, Markus and Pérusse, Louis and Peters, Ulrike and Power, Chris and Quertermous, Thomas and Rauramaa, Rainer and Rivadeneira, Fernando and Saaristo, Timo E. and Saleheen, Danish and Sattar, Naveed and Schadt, Eric E. and Schlessinger, David and Eline Slagboom, P. and Snieder, Harold and Spector, Tim D. and Thorsteinsdottir, Unnur and Stumvoll, Michael and Tuomilehto, Jaakko and Uitterlinden, André G. and Uusitupa, Matti and van der Harst, Pim and Walker, Mark and Wallaschofski, Henri and Wareham, Nicholas J. and Watkins, Hugh and Weir, David R. and Wichmann, H-Erich and Wilson, James F. and Zanen, Pieter and Borecki, Ingrid B. and Deloukas, Panos and Fox, Caroline S. and Heid, Iris M. and O’Connell, Jeffrey R. and Strachan, David P. and Stefansson, Kari and van Duijn, Cornelia M. and Abecasis, Gonçalo R. and Franke, Lude and Frayling, Timothy M. and McCarthy, Mark I. and Visscher, Peter M. and Scherag, André and Willer, Cristen J. and Boehnke, Michael and Mohlke, Karen L. and Lindgren, Cecilia M. and Beckmann, Jacques S. and Barroso, Inês and North, Kari E. and Ingelsson, Erik and Hirschhorn, Joel N. and Loos, Ruth J. F. and Speliotes, Elizabeth K.}, year={2015}, month=feb, pages={197–206} }
</div></div><br><br><br><br><br><br></div><div class='container mt-4 bg-light p-4 text-center'><h1 class='text-primary'>Phenotype: body mass index</h1><p class='text-muted'><strong>URL:</strong> http://ftp.ebi.ac.uk/pub/databases/gwas/summary_statistics/GCST90018001-GCST90019000/GCST90018947</p><table border="1" class="dataframe table table-bordered">
  <thead>
    <tr style="text-align: right;">
      <th>firstAuthor</th>
      <th>accessionId</th>
      <th>publicationDate</th>
      <th>journal</th>
      <th>title</th>
      <th>reportedTrait</th>
      <th>efoTraits</th>
      <th>bgTraits</th>
      <th>discoverySampleAncestry</th>
      <th>replicationSampleAncestry</th>
      <th>associationCount</th>
      <th>summaryStatistics</th>
      <th>ssApiFlag</th>
      <th>agreedToCc0</th>
      <th>genotypingTechnologies</th>
      <th>pubmedId</th>
      <th>initialSampleDescription</th>
      <th>replicateSampleDescription</th>
      <th>UKB_FIELD</th>
      <th>FuzzySimilarity_body mass index bmi</th>
      <th>SearchPhenotype</th>
    </tr>
  </thead>
  <tbody>
    <tr>
      <td>Sakaue S</td>
      <td>GCST90018947</td>
      <td>9/30/2021</td>
      <td>Nat Genet</td>
      <td>A cross-population atlas of genetic associations for 220 human phenotypes.</td>
      <td>body mass index</td>
      <td>body mass index</td>
      <td>-</td>
      <td>359983 european,163835 east asian</td>
      <td>NaN</td>
      <td>535</td>
      <td>http://ftp.ebi.ac.uk/pub/databases/gwas/summary_statistics/GCST90018001-GCST90019000/GCST90018947</td>
      <td>no</td>
      <td>yes</td>
      <td>Genome-wide genotyping array</td>
      <td>34594039</td>
      <td>359,983 European ancestry individuals,163,835 East Asian ancestry individuals</td>
      <td>NaN</td>
      <td>-</td>
      <td>88</td>
      <td>body mass index bmi</td>
    </tr>
  </tbody>
</table><h2 class='text-muted'>Information file</h2><table border="1" class="dataframe table table-bordered">
  <thead>
    <tr style="text-align: right;">
      <th>Unnamed: 0</th>
      <th>Name</th>
      <th>Last modified</th>
      <th>Size</th>
      <th>Description</th>
      <th>URL</th>
    </tr>
  </thead>
  <tbody>
    <tr>
      <td>2</td>
      <td>README.txt</td>
      <td>2021-12-16 01:16</td>
      <td>177</td>
      <td>NaN</td>
      <td>http://ftp.ebi.ac.uk/pub/databases/gwas/summary_statistics/GCST90018001-GCST90019000/GCST90018947/README.txt</td>
    </tr>
    <tr>
      <td>3</td>
      <td>harmonised/</td>
      <td>2024-02-02 15:18</td>
      <td>-</td>
      <td>NaN</td>
      <td>http://ftp.ebi.ac.uk/pub/databases/gwas/summary_statistics/GCST90018001-GCST90019000/GCST90018947/harmonised/</td>
    </tr>
    <tr>
      <td>4</td>
      <td>md5sum.txt</td>
      <td>2021-12-16 01:16</td>
      <td>109</td>
      <td>NaN</td>
      <td>http://ftp.ebi.ac.uk/pub/databases/gwas/summary_statistics/GCST90018001-GCST90019000/GCST90018947/md5sum.txt</td>
    </tr>
    <tr>
      <td>1</td>
      <td>GCST90018947_buildGRCh37.tsv.gz-meta.yaml</td>
      <td>2024-01-04 00:23</td>
      <td>1.1K</td>
      <td>NaN</td>
      <td>http://ftp.ebi.ac.uk/pub/databases/gwas/summary_statistics/GCST90018001-GCST90019000/GCST90018947/GCST90018947_buildGRCh37.tsv.gz-meta.yaml</td>
    </tr>
    <tr>
      <td>0</td>
      <td>GCST90018947_buildGRCh37.tsv.gz</td>
      <td>2021-12-16 01:16</td>
      <td>330M</td>
      <td>NaN</td>
      <td>http://ftp.ebi.ac.uk/pub/databases/gwas/summary_statistics/GCST90018001-GCST90019000/GCST90018947/GCST90018947_buildGRCh37.tsv.gz</td>
    </tr>
  </tbody>
</table><h2 class='text-muted'>Processed file</h2><p>Processing file: GCST90018947_buildGRCh37.tsv.gz!</p><h2 class='text-muted'>GWAS file head</h2><table border="1" class="dataframe table table-bordered">
  <thead>
    <tr style="text-align: right;">
      <th>chromosome</th>
      <th>base_pair_location</th>
      <th>effect_allele</th>
      <th>other_allele</th>
      <th>effect_allele_frequency</th>
      <th>beta</th>
      <th>standard_error</th>
      <th>p_value</th>
      <th>variant_id</th>
    </tr>
  </thead>
  <tbody>
    <tr>
      <td>1</td>
      <td>100000012</td>
      <td>T</td>
      <td>G</td>
      <td>0.274416</td>
      <td>0.0026</td>
      <td>0.0021</td>
      <td>0.2272</td>
      <td>NaN</td>
    </tr>
    <tr>
      <td>1</td>
      <td>10000006</td>
      <td>A</td>
      <td>G</td>
      <td>0.004699</td>
      <td>-0.0105</td>
      <td>0.0185</td>
      <td>0.5705</td>
      <td>NaN</td>
    </tr>
  </tbody>
</table></div><div class='container mt-4 bg-light p-4 text-center'><h2 class='text-muted'>Original GWAS file headers</h2><table border="1" class="dataframe table table-bordered">
  <thead>
    <tr style="text-align: right;">
      <th>chromosome</th>
      <th>base_pair_location</th>
      <th>effect_allele</th>
      <th>other_allele</th>
      <th>effect_allele_frequency</th>
      <th>beta</th>
      <th>standard_error</th>
      <th>p_value</th>
      <th>variant_id</th>
    </tr>
  </thead>
  <tbody>
    <tr>
      <td>1</td>
      <td>100000012</td>
      <td>T</td>
      <td>G</td>
      <td>0.274416</td>
      <td>0.0026</td>
      <td>0.0021</td>
      <td>0.2272</td>
      <td>NaN</td>
    </tr>
    <tr>
      <td>1</td>
      <td>10000006</td>
      <td>A</td>
      <td>G</td>
      <td>0.004699</td>
      <td>-0.0105</td>
      <td>0.0185</td>
      <td>0.5705</td>
      <td>NaN</td>
    </tr>
  </tbody>
</table><p style='font-weight: bold; color: blue;'>CHR -> ['chromosome']</p><p style='font-weight: bold; color: blue;'>SNP -> ['variant_id']</p><p style='font-weight: bold; color: blue;'>BP -> ['base_pair_location']</p><p style='font-weight: bold; color: blue;'>A1 -> ['effect_allele']</p><p style='font-weight: bold; color: blue;'>A2 -> ['other_allele']</p><p style='font-weight: bold; color: blue;'>N -> []</p><p style='font-weight: bold; color: blue;'>P -> ['p_value']</p><p style='font-weight: bold; color: blue;'>INFO -> []</p><p style='font-weight: bold; color: blue;'>MAF -> ['effect_allele_frequency']</p><p style='font-weight: bold; color: blue;'>BETA -> ['beta']</p><p style='font-weight: bold; color: blue;'>SE -> ['standard_error']</p><p style='font-weight: bold; color: blue;'>OR -> []</p><p style='font-weight: bold; color: blue;'>Z -> []</p><p style='font-weight: bold; color: blue;'>D -> []</p><h2 class='text-muted'>GWAS columns</h2>chromosome, base_pair_location, effect_allele, other_allele, effect_allele_frequency, beta, standard_error, p_value, variant_id<h2 class='text-muted'>Identified columns</h2>chromosome, variant_id, base_pair_location, effect_allele, other_allele, p_value, effect_allele_frequency, beta, standard_error<h2 class='text-muted'>Unidentified Columns headers</h2><h2 class='text-muted'>PMID</h2>34594039<h2 class='text-muted'>DOI</h2>Could not find DOI!<div class='bibtex-container'><h2 class='text-muted'>BibTeX</h2><div style='font-family: monospace; white-space: pre-wrap;'></div></div><br><br><br><br><br><br></div><div class='container mt-4 bg-light p-4 text-center'><h1 class='text-primary'>Phenotype: body mass index</h1><p class='text-muted'><strong>URL:</strong> http://ftp.ebi.ac.uk/pub/databases/gwas/summary_statistics/GCST008001-GCST009000/GCST008129</p><table border="1" class="dataframe table table-bordered">
  <thead>
    <tr style="text-align: right;">
      <th>firstAuthor</th>
      <th>accessionId</th>
      <th>publicationDate</th>
      <th>journal</th>
      <th>title</th>
      <th>reportedTrait</th>
      <th>efoTraits</th>
      <th>bgTraits</th>
      <th>discoverySampleAncestry</th>
      <th>replicationSampleAncestry</th>
      <th>associationCount</th>
      <th>summaryStatistics</th>
      <th>ssApiFlag</th>
      <th>agreedToCc0</th>
      <th>genotypingTechnologies</th>
      <th>pubmedId</th>
      <th>initialSampleDescription</th>
      <th>replicateSampleDescription</th>
      <th>UKB_FIELD</th>
      <th>FuzzySimilarity_body mass index bmi</th>
      <th>SearchPhenotype</th>
    </tr>
  </thead>
  <tbody>
    <tr>
      <td>Turcot V</td>
      <td>GCST008129</td>
      <td>12/22/2017</td>
      <td>Nat Genet</td>
      <td>Protein-altering variants associated with body mass index implicate pathways that control energy intake and expenditure in obesity.</td>
      <td>body mass index</td>
      <td>body mass index</td>
      <td>-</td>
      <td>449889 european,27610 african american or afro-caribbean,8839 east asian,29398 south asian,10772 hispanic or latin american</td>
      <td>196766 European</td>
      <td>92</td>
      <td>http://ftp.ebi.ac.uk/pub/databases/gwas/summary_statistics/GCST008001-GCST009000/GCST008129</td>
      <td>no</td>
      <td>-</td>
      <td>Genome-wide genotyping array,Targeted genotyping array</td>
      <td>29273807</td>
      <td>449,889 European ancestry individuals, up to ,29,398 South Asian ancestry individuals, up to ,27,610 African American individuals, up to ,8,839 East Asian individuals, up to ,10,772 Hispanic ancestry individuals</td>
      <td>up to 196,766 European ancestry individuals</td>
      <td>-</td>
      <td>88</td>
      <td>body mass index bmi</td>
    </tr>
  </tbody>
</table><h2 class='text-muted'>Information file</h2><table border="1" class="dataframe table table-bordered">
  <thead>
    <tr style="text-align: right;">
      <th>Unnamed: 0</th>
      <th>Name</th>
      <th>Last modified</th>
      <th>Size</th>
      <th>Description</th>
      <th>URL</th>
    </tr>
  </thead>
  <tbody>
    <tr>
      <td>1</td>
      <td>TurcotBMIreadme.txt</td>
      <td>2019-07-30 10:24</td>
      <td>4.2K</td>
      <td>NaN</td>
      <td>http://ftp.ebi.ac.uk/pub/databases/gwas/summary_statistics/GCST008001-GCST009000/GCST008129/TurcotBMIreadme.txt</td>
    </tr>
    <tr>
      <td>0</td>
      <td>BMI_All_ancestry.txt.gz</td>
      <td>2019-07-30 10:24</td>
      <td>4.8M</td>
      <td>NaN</td>
      <td>http://ftp.ebi.ac.uk/pub/databases/gwas/summary_statistics/GCST008001-GCST009000/GCST008129/BMI_All_ancestry.txt.gz</td>
    </tr>
  </tbody>
</table><h2 class='text-muted'>GWAS file</h2><p>We were unable to process the GWAS file!</p><div class='container mt-4 bg-light p-4 text-center'><h1 class='text-primary'>Phenotype: body mass index</h1><p class='text-muted'><strong>URL:</strong> http://ftp.ebi.ac.uk/pub/databases/gwas/summary_statistics/GCST90025001-GCST90026000/GCST90025994</p><table border="1" class="dataframe table table-bordered">
  <thead>
    <tr style="text-align: right;">
      <th>firstAuthor</th>
      <th>accessionId</th>
      <th>publicationDate</th>
      <th>journal</th>
      <th>title</th>
      <th>reportedTrait</th>
      <th>efoTraits</th>
      <th>bgTraits</th>
      <th>discoverySampleAncestry</th>
      <th>replicationSampleAncestry</th>
      <th>associationCount</th>
      <th>summaryStatistics</th>
      <th>ssApiFlag</th>
      <th>agreedToCc0</th>
      <th>genotypingTechnologies</th>
      <th>pubmedId</th>
      <th>initialSampleDescription</th>
      <th>replicateSampleDescription</th>
      <th>UKB_FIELD</th>
      <th>FuzzySimilarity_body mass index bmi</th>
      <th>SearchPhenotype</th>
    </tr>
  </thead>
  <tbody>
    <tr>
      <td>Barton AR</td>
      <td>GCST90025994</td>
      <td>7/5/2021</td>
      <td>Nat Genet</td>
      <td>Whole-exome imputation within UK Biobank powers rare coding variant association and fine-mapping analyses.</td>
      <td>body mass index</td>
      <td>body mass index</td>
      <td>-</td>
      <td>457756 european</td>
      <td>NaN</td>
      <td>3</td>
      <td>http://ftp.ebi.ac.uk/pub/databases/gwas/summary_statistics/GCST90025001-GCST90026000/GCST90025994</td>
      <td>no</td>
      <td>no</td>
      <td>Genome-wide genotyping array</td>
      <td>34226706</td>
      <td>457,756 European ancestry individuals</td>
      <td>NaN</td>
      <td>-</td>
      <td>88</td>
      <td>body mass index bmi</td>
    </tr>
  </tbody>
</table><h2 class='text-muted'>Information file</h2><table border="1" class="dataframe table table-bordered">
  <thead>
    <tr style="text-align: right;">
      <th>Unnamed: 0</th>
      <th>Name</th>
      <th>Last modified</th>
      <th>Size</th>
      <th>Description</th>
      <th>URL</th>
    </tr>
  </thead>
  <tbody>
    <tr>
      <td>1</td>
      <td>GCST90025994_buildGRCh37.tsv-meta.yaml</td>
      <td>2024-01-04 00:38</td>
      <td>672</td>
      <td>NaN</td>
      <td>http://ftp.ebi.ac.uk/pub/databases/gwas/summary_statistics/GCST90025001-GCST90026000/GCST90025994/GCST90025994_buildGRCh37.tsv-meta.yaml</td>
    </tr>
    <tr>
      <td>3</td>
      <td>harmonised/</td>
      <td>2024-02-02 15:16</td>
      <td>-</td>
      <td>NaN</td>
      <td>http://ftp.ebi.ac.uk/pub/databases/gwas/summary_statistics/GCST90025001-GCST90026000/GCST90025994/harmonised/</td>
    </tr>
    <tr>
      <td>4</td>
      <td>md5sum.txt</td>
      <td>2021-12-02 00:34</td>
      <td>141</td>
      <td>NaN</td>
      <td>http://ftp.ebi.ac.uk/pub/databases/gwas/summary_statistics/GCST90025001-GCST90026000/GCST90025994/md5sum.txt</td>
    </tr>
    <tr>
      <td>2</td>
      <td>bolt_460K_selfRepWhite.body_BMIz.WES.stats.gz</td>
      <td>2021-12-02 00:34</td>
      <td>235M</td>
      <td>NaN</td>
      <td>http://ftp.ebi.ac.uk/pub/databases/gwas/summary_statistics/GCST90025001-GCST90026000/GCST90025994/bolt_460K_selfRepWhite.body_BMIz.WES.stats.gz</td>
    </tr>
    <tr>
      <td>0</td>
      <td>GCST90025994_buildGRCh37.tsv</td>
      <td>2021-12-02 00:34</td>
      <td>602M</td>
      <td>NaN</td>
      <td>http://ftp.ebi.ac.uk/pub/databases/gwas/summary_statistics/GCST90025001-GCST90026000/GCST90025994/GCST90025994_buildGRCh37.tsv</td>
    </tr>
  </tbody>
</table><h2 class='text-muted'>Processed file</h2><p>Processing file: GCST90025994_buildGRCh37.tsv.1!</p><h2 class='text-muted'>GWAS file head</h2><table border="1" class="dataframe table table-bordered">
  <thead>
    <tr style="text-align: right;">
      <th>chromosome</th>
      <th>base_pair_location</th>
      <th>genpos</th>
      <th>allele1</th>
      <th>allele0</th>
      <th>a1freq</th>
      <th>info</th>
      <th>chisq_linreg</th>
      <th>p_linreg</th>
      <th>beta</th>
      <th>standard_error</th>
      <th>chisq_bolt_lmm_inf</th>
      <th>p_bolt_lmm_inf</th>
      <th>chisq_bolt_lmm</th>
      <th>p_value</th>
    </tr>
  </thead>
  <tbody>
    <tr>
      <td>1</td>
      <td>69081</td>
      <td>0</td>
      <td>C</td>
      <td>G</td>
      <td>0.000120</td>
      <td>0.121072</td>
      <td>4.961220e-01</td>
      <td>0.48</td>
      <td>1.178180e-01</td>
      <td>0.234327</td>
      <td>2.528010e-01</td>
      <td>0.62</td>
      <td>8.684510e-02</td>
      <td>0.77</td>
    </tr>
    <tr>
      <td>1</td>
      <td>69224</td>
      <td>0</td>
      <td>T</td>
      <td>A</td>
      <td>0.000007</td>
      <td>0.009547</td>
      <td>-1.000000e+09</td>
      <td>1.00</td>
      <td>-9.131080e+09</td>
      <td>NaN</td>
      <td>-1.000000e+09</td>
      <td>1.00</td>
      <td>-1.000000e+09</td>
      <td>1.00</td>
    </tr>
  </tbody>
</table></div><div class='container mt-4 bg-light p-4 text-center'><h2 class='text-muted'>Original GWAS file headers</h2><table border="1" class="dataframe table table-bordered">
  <thead>
    <tr style="text-align: right;">
      <th>chromosome</th>
      <th>base_pair_location</th>
      <th>genpos</th>
      <th>allele1</th>
      <th>allele0</th>
      <th>a1freq</th>
      <th>info</th>
      <th>chisq_linreg</th>
      <th>p_linreg</th>
      <th>beta</th>
      <th>standard_error</th>
      <th>chisq_bolt_lmm_inf</th>
      <th>p_bolt_lmm_inf</th>
      <th>chisq_bolt_lmm</th>
      <th>p_value</th>
    </tr>
  </thead>
  <tbody>
    <tr>
      <td>1</td>
      <td>69081</td>
      <td>0</td>
      <td>C</td>
      <td>G</td>
      <td>0.000120</td>
      <td>0.121072</td>
      <td>4.961220e-01</td>
      <td>0.48</td>
      <td>1.178180e-01</td>
      <td>0.234327</td>
      <td>2.528010e-01</td>
      <td>0.62</td>
      <td>8.684510e-02</td>
      <td>0.77</td>
    </tr>
    <tr>
      <td>1</td>
      <td>69224</td>
      <td>0</td>
      <td>T</td>
      <td>A</td>
      <td>0.000007</td>
      <td>0.009547</td>
      <td>-1.000000e+09</td>
      <td>1.00</td>
      <td>-9.131080e+09</td>
      <td>NaN</td>
      <td>-1.000000e+09</td>
      <td>1.00</td>
      <td>-1.000000e+09</td>
      <td>1.00</td>
    </tr>
  </tbody>
</table><p style='font-weight: bold; color: blue;'>CHR -> ['chromosome']</p><p style='font-weight: bold; color: blue;'>SNP -> []</p><p style='font-weight: bold; color: blue;'>BP -> ['base_pair_location', 'genpos']</p><p style='font-weight: bold; color: blue;'>A1 -> ['allele1', 'allele0']</p><p style='font-weight: bold; color: blue;'>A2 -> ['allele1']</p><p style='font-weight: bold; color: blue;'>N -> []</p><p style='font-weight: bold; color: blue;'>P -> ['p_value']</p><p style='font-weight: bold; color: blue;'>INFO -> ['info']</p><p style='font-weight: bold; color: blue;'>MAF -> ['a1freq']</p><p style='font-weight: bold; color: blue;'>BETA -> ['beta', 'p_bolt_lmm_inf']</p><p style='font-weight: bold; color: blue;'>SE -> ['standard_error']</p><p style='font-weight: bold; color: blue;'>OR -> []</p><p style='font-weight: bold; color: blue;'>Z -> []</p><p style='font-weight: bold; color: blue;'>D -> []</p><h2 class='text-muted'>GWAS columns</h2>chromosome, base_pair_location, genpos, allele1, allele0, a1freq, info, chisq_linreg, p_linreg, beta, standard_error, chisq_bolt_lmm_inf, p_bolt_lmm_inf, chisq_bolt_lmm, p_value<h2 class='text-muted'>Identified columns</h2>chromosome, base_pair_location, genpos, allele1, allele0, allele1, p_value, info, a1freq, beta, p_bolt_lmm_inf, standard_error<h2 class='text-muted'>Unidentified Columns headers</h2>chisq_bolt_lmm_inf, chisq_bolt_lmm, chisq_linreg, p_linreg<h2 class='text-muted'>PMID</h2>34226706<h2 class='text-muted'>DOI</h2>10.1038/s41588-021-00892-1<div class='bibtex-container'><h2 class='text-muted'>BibTeX</h2><div style='font-family: monospace; white-space: pre-wrap;'> @article{Barton_2021, title={Whole-exome imputation within UK Biobank powers rare coding variant association and fine-mapping analyses}, volume={53}, ISSN={1546-1718}, url={http://dx.doi.org/10.1038/s41588-021-00892-1}, DOI={10.1038/s41588-021-00892-1}, number={8}, journal={Nature Genetics}, publisher={Springer Science and Business Media LLC}, author={Barton, Alison R. and Sherman, Maxwell A. and Mukamel, Ronen E. and Loh, Po-Ru}, year={2021}, month=jul, pages={1260â€“1269} }
</div></div><br><br><br><br><br><br></div><div class='container mt-4 bg-light p-4 text-center'><h1 class='text-primary'>Phenotype: body mass index</h1><p class='text-muted'><strong>URL:</strong> http://ftp.ebi.ac.uk/pub/databases/gwas/summary_statistics/GCST90095001-GCST90096000/GCST90095034</p><table border="1" class="dataframe table table-bordered">
  <thead>
    <tr style="text-align: right;">
      <th>firstAuthor</th>
      <th>accessionId</th>
      <th>publicationDate</th>
      <th>journal</th>
      <th>title</th>
      <th>reportedTrait</th>
      <th>efoTraits</th>
      <th>bgTraits</th>
      <th>discoverySampleAncestry</th>
      <th>replicationSampleAncestry</th>
      <th>associationCount</th>
      <th>summaryStatistics</th>
      <th>ssApiFlag</th>
      <th>agreedToCc0</th>
      <th>genotypingTechnologies</th>
      <th>pubmedId</th>
      <th>initialSampleDescription</th>
      <th>replicateSampleDescription</th>
      <th>UKB_FIELD</th>
      <th>FuzzySimilarity_body mass index bmi</th>
      <th>SearchPhenotype</th>
    </tr>
  </thead>
  <tbody>
    <tr>
      <td>Fern??ndez-Rhodes L</td>
      <td>GCST90095034</td>
      <td>3/11/2022</td>
      <td>HGG Adv</td>
      <td>Ancestral diversity improves discovery and fine-mapping of genetic loci for anthropometric traits-The Hispanic/Latino Anthropometry Consortium.</td>
      <td>body mass index</td>
      <td>body mass index</td>
      <td>-</td>
      <td>56161 hispanic or latin american</td>
      <td>10538 Hispanic or Latin American</td>
      <td>31</td>
      <td>http://ftp.ebi.ac.uk/pub/databases/gwas/summary_statistics/GCST90095001-GCST90096000/GCST90095034</td>
      <td>no</td>
      <td>yes</td>
      <td>Genome-wide genotyping array</td>
      <td>35399580</td>
      <td>56,161 Hispanic or Latin American individuals</td>
      <td>10,538 Hispanic or Latin American individuals</td>
      <td>-</td>
      <td>88</td>
      <td>body mass index bmi</td>
    </tr>
  </tbody>
</table><h2 class='text-muted'>Information file</h2><table border="1" class="dataframe table table-bordered">
  <thead>
    <tr style="text-align: right;">
      <th>Unnamed: 0</th>
      <th>Name</th>
      <th>Last modified</th>
      <th>Size</th>
      <th>Description</th>
      <th>URL</th>
    </tr>
  </thead>
  <tbody>
    <tr>
      <td>1</td>
      <td>GCST90095034_buildGRCh37.tsv.gz-meta.yaml</td>
      <td>2024-01-04 00:12</td>
      <td>692</td>
      <td>NaN</td>
      <td>http://ftp.ebi.ac.uk/pub/databases/gwas/summary_statistics/GCST90095001-GCST90096000/GCST90095034/GCST90095034_buildGRCh37.tsv.gz-meta.yaml</td>
    </tr>
    <tr>
      <td>2</td>
      <td>harmonised/</td>
      <td>2024-02-02 15:16</td>
      <td>-</td>
      <td>NaN</td>
      <td>http://ftp.ebi.ac.uk/pub/databases/gwas/summary_statistics/GCST90095001-GCST90096000/GCST90095034/harmonised/</td>
    </tr>
    <tr>
      <td>3</td>
      <td>md5sum.txt</td>
      <td>2022-02-26 00:02</td>
      <td>65</td>
      <td>NaN</td>
      <td>http://ftp.ebi.ac.uk/pub/databases/gwas/summary_statistics/GCST90095001-GCST90096000/GCST90095034/md5sum.txt</td>
    </tr>
    <tr>
      <td>0</td>
      <td>GCST90095034_buildGRCh37.tsv.gz</td>
      <td>2022-02-26 00:02</td>
      <td>208M</td>
      <td>NaN</td>
      <td>http://ftp.ebi.ac.uk/pub/databases/gwas/summary_statistics/GCST90095001-GCST90096000/GCST90095034/GCST90095034_buildGRCh37.tsv.gz</td>
    </tr>
  </tbody>
</table><h2 class='text-muted'>Processed file</h2><p>Processing file: GCST90095034_buildGRCh37.tsv.gz!</p><h2 class='text-muted'>GWAS file head</h2><table border="1" class="dataframe table table-bordered">
  <thead>
    <tr style="text-align: right;">
      <th>chromosome</th>
      <th>base_pair_location</th>
      <th>variant_id</th>
      <th>effect_allele</th>
      <th>other_allele</th>
      <th>effect_allele_frequency</th>
      <th>beta</th>
      <th>standard_error</th>
      <th>p_value</th>
      <th>hetisq</th>
      <th>hetpval</th>
      <th>totalsamplesize</th>
    </tr>
  </thead>
  <tbody>
    <tr>
      <td>10</td>
      <td>10000018</td>
      <td>rs6602381</td>
      <td>A</td>
      <td>G</td>
      <td>0.5706</td>
      <td>-0.0051</td>
      <td>0.0064</td>
      <td>0.4283</td>
      <td>38.5</td>
      <td>0.008601</td>
      <td>53972.9</td>
    </tr>
    <tr>
      <td>10</td>
      <td>100000625</td>
      <td>rs7899632</td>
      <td>A</td>
      <td>G</td>
      <td>0.5144</td>
      <td>-0.0001</td>
      <td>0.0063</td>
      <td>0.9856</td>
      <td>0.0</td>
      <td>0.519800</td>
      <td>50707.0</td>
    </tr>
  </tbody>
</table></div><div class='container mt-4 bg-light p-4 text-center'><h2 class='text-muted'>Original GWAS file headers</h2><table border="1" class="dataframe table table-bordered">
  <thead>
    <tr style="text-align: right;">
      <th>chromosome</th>
      <th>base_pair_location</th>
      <th>variant_id</th>
      <th>effect_allele</th>
      <th>other_allele</th>
      <th>effect_allele_frequency</th>
      <th>beta</th>
      <th>standard_error</th>
      <th>p_value</th>
      <th>hetisq</th>
      <th>hetpval</th>
      <th>totalsamplesize</th>
    </tr>
  </thead>
  <tbody>
    <tr>
      <td>10</td>
      <td>10000018</td>
      <td>rs6602381</td>
      <td>A</td>
      <td>G</td>
      <td>0.5706</td>
      <td>-0.0051</td>
      <td>0.0064</td>
      <td>0.4283</td>
      <td>38.5</td>
      <td>0.008601</td>
      <td>53972.9</td>
    </tr>
    <tr>
      <td>10</td>
      <td>100000625</td>
      <td>rs7899632</td>
      <td>A</td>
      <td>G</td>
      <td>0.5144</td>
      <td>-0.0001</td>
      <td>0.0063</td>
      <td>0.9856</td>
      <td>0.0</td>
      <td>0.519800</td>
      <td>50707.0</td>
    </tr>
  </tbody>
</table><p style='font-weight: bold; color: blue;'>CHR -> ['chromosome']</p><p style='font-weight: bold; color: blue;'>SNP -> ['variant_id']</p><p style='font-weight: bold; color: blue;'>BP -> ['base_pair_location']</p><p style='font-weight: bold; color: blue;'>A1 -> ['effect_allele']</p><p style='font-weight: bold; color: blue;'>A2 -> ['other_allele']</p><p style='font-weight: bold; color: blue;'>N -> ['totalsamplesize']</p><p style='font-weight: bold; color: blue;'>P -> ['p_value', 'hetpval']</p><p style='font-weight: bold; color: blue;'>INFO -> []</p><p style='font-weight: bold; color: blue;'>MAF -> ['effect_allele_frequency']</p><p style='font-weight: bold; color: blue;'>BETA -> ['beta']</p><p style='font-weight: bold; color: blue;'>SE -> ['standard_error']</p><p style='font-weight: bold; color: blue;'>OR -> []</p><p style='font-weight: bold; color: blue;'>Z -> []</p><p style='font-weight: bold; color: blue;'>D -> []</p><h2 class='text-muted'>GWAS columns</h2>chromosome, base_pair_location, variant_id, effect_allele, other_allele, effect_allele_frequency, beta, standard_error, p_value, hetisq, hetpval, totalsamplesize<h2 class='text-muted'>Identified columns</h2>chromosome, variant_id, base_pair_location, effect_allele, other_allele, totalsamplesize, p_value, hetpval, effect_allele_frequency, beta, standard_error<h2 class='text-muted'>Unidentified Columns headers</h2>hetisq<h2 class='text-muted'>PMID</h2>35399580<h2 class='text-muted'>DOI</h2>10.1016/j.xhgg.2022.100099<div class='bibtex-container'><h2 class='text-muted'>BibTeX</h2><div style='font-family: monospace; white-space: pre-wrap;'> @article{Fern_ndez_Rhodes_2022, title={Ancestral diversity improves discovery and fine-mapping of genetic loci for anthropometric traits—The Hispanic/Latino Anthropometry Consortium}, volume={3}, ISSN={2666-2477}, url={http://dx.doi.org/10.1016/j.xhgg.2022.100099}, DOI={10.1016/j.xhgg.2022.100099}, number={2}, journal={Human Genetics and Genomics Advances}, publisher={Elsevier BV}, author={Fernández-Rhodes, Lindsay and Graff, Mariaelisa and Buchanan, Victoria L. and Justice, Anne E. and Highland, Heather M. and Guo, Xiuqing and Zhu, Wanying and Chen, Hung-Hsin and Young, Kristin L. and Adhikari, Kaustubh and Palmer, Nicholette D. and Below, Jennifer E. and Bradfield, Jonathan and Pereira, Alexandre C. and Glover, LáShauntá and Kim, Daeeun and Lilly, Adam G. and Shrestha, Poojan and Thomas, Alvin G. and Zhang, Xinruo and Chen, Minhui and Chiang, Charleston W.K. and Pulit, Sara and Horimoto, Andrea and Krieger, Jose E. and Guindo-Martínez, Marta and Preuss, Michael and Schumann, Claudia and Smit, Roelof A.J. and Torres-Mejía, Gabriela and Acuña-Alonzo, Victor and Bedoya, Gabriel and Bortolini, Maria-Cátira and Canizales-Quinteros, Samuel and Gallo, Carla and González-José, Rolando and Poletti, Giovanni and Rothhammer, Francisco and Hakonarson, Hakon and Igo, Robert and Adler, Sharon G. and Iyengar, Sudha K. and Nicholas, Susanne B. and Gogarten, Stephanie M. and Isasi, Carmen R. and Papnicolaou, George and Stilp, Adrienne M. and Qi, Qibin and Kho, Minjung and Smith, Jennifer A. and Langefeld, Carl D. and Wagenknecht, Lynne and Mckean-Cowdin, Roberta and Gao, Xiaoyi Raymond and Nousome, Darryl and Conti, David V. and Feng, Ye and Allison, Matthew A. and Arzumanyan, Zorayr and Buchanan, Thomas A. and Ida Chen, Yii-Der and Genter, Pauline M. and Goodarzi, Mark O. and Hai, Yang and Hsueh, Willa and Ipp, Eli and Kandeel, Fouad R. and Lam, Kelvin and Li, Xiaohui and Nadler, Jerry L. and Raffel, Leslie J. and Roll, Kathryn and Sandow, Kevin and Tan, Jingyi and Taylor, Kent D. and Xiang, Anny H. and Yao, Jie and Audirac-Chalifour, Astride and de Jesus Peralta Romero, Jose and Hartwig, Fernando and Horta, Bernando and Blangero, John and Curran, Joanne E. and Duggirala, Ravindranath and Lehman, Donna E. and Puppala, Sobha and Fejerman, Laura and John, Esther M. and Aguilar-Salinas, Carlos and Burtt, Noël P. and Florez, Jose C. and García-Ortíz, Humberto and González-Villalpando, Clicerio and Mercader, Josep and Orozco, Lorena and Tusié-Luna, Teresa and Blanco, Estela and Gahagan, Sheila and Cox, Nancy J. and Hanis, Craig and Butte, Nancy F. and Cole, Shelley A. and Comuzzie, Anthony G. and Voruganti, V. Saroja and Rohde, Rebecca and Wang, Yujie and Sofer, Tamar and Ziv, Elad and Grant, Struan F.A. and Ruiz-Linares, Andres and Rotter, Jerome I. and Haiman, Christopher A. and Parra, Esteban J. and Cruz, Miguel and Loos, Ruth J.F. and North, Kari E.}, year={2022}, month=apr, pages={100099} }
</div></div><br><br><br><br><br><br></div><div class='container mt-4 bg-light p-4 text-center'><h1 class='text-primary'>Phenotype: body mass index</h1><p class='text-muted'><strong>URL:</strong> http://ftp.ebi.ac.uk/pub/databases/gwas/summary_statistics/GCST009001-GCST010000/GCST009004</p><table border="1" class="dataframe table table-bordered">
  <thead>
    <tr style="text-align: right;">
      <th>firstAuthor</th>
      <th>accessionId</th>
      <th>publicationDate</th>
      <th>journal</th>
      <th>title</th>
      <th>reportedTrait</th>
      <th>efoTraits</th>
      <th>bgTraits</th>
      <th>discoverySampleAncestry</th>
      <th>replicationSampleAncestry</th>
      <th>associationCount</th>
      <th>summaryStatistics</th>
      <th>ssApiFlag</th>
      <th>agreedToCc0</th>
      <th>genotypingTechnologies</th>
      <th>pubmedId</th>
      <th>initialSampleDescription</th>
      <th>replicateSampleDescription</th>
      <th>UKB_FIELD</th>
      <th>FuzzySimilarity_body mass index bmi</th>
      <th>SearchPhenotype</th>
    </tr>
  </thead>
  <tbody>
    <tr>
      <td>Pulit SL</td>
      <td>GCST009004</td>
      <td>9/14/2018</td>
      <td>Hum Mol Genet</td>
      <td>Meta-analysis of genome-wide association studies for body fat distribution in 694,649 individuals of European ancestry.</td>
      <td>body mass index</td>
      <td>body mass index</td>
      <td>-</td>
      <td>806834 european</td>
      <td>NaN</td>
      <td>670</td>
      <td>http://ftp.ebi.ac.uk/pub/databases/gwas/summary_statistics/GCST009001-GCST010000/GCST009004</td>
      <td>no</td>
      <td>-</td>
      <td>Genome-wide genotyping array</td>
      <td>30239722</td>
      <td>806,834 European ancestry individuals</td>
      <td>NaN</td>
      <td>-</td>
      <td>88</td>
      <td>body mass index bmi</td>
    </tr>
  </tbody>
</table><h2 class='text-muted'>Information file</h2><table border="1" class="dataframe table table-bordered">
  <thead>
    <tr style="text-align: right;">
      <th>Unnamed: 0</th>
      <th>Name</th>
      <th>Last modified</th>
      <th>Size</th>
      <th>Description</th>
      <th>URL</th>
    </tr>
  </thead>
  <tbody>
    <tr>
      <td>1</td>
      <td>md5sum.txt</td>
      <td>2019-11-06 00:02</td>
      <td>443</td>
      <td>NaN</td>
      <td>http://ftp.ebi.ac.uk/pub/databases/gwas/summary_statistics/GCST009001-GCST010000/GCST009004/md5sum.txt</td>
    </tr>
    <tr>
      <td>2</td>
      <td>readme.txt</td>
      <td>2019-11-06 00:02</td>
      <td>854</td>
      <td>NaN</td>
      <td>http://ftp.ebi.ac.uk/pub/databases/gwas/summary_statistics/GCST009001-GCST010000/GCST009004/readme.txt</td>
    </tr>
    <tr>
      <td>0</td>
      <td>fat-distn.giant.ukbb.meta-analysis.bmi.combined.tbl.gz</td>
      <td>2019-11-06 00:02</td>
      <td>503M</td>
      <td>NaN</td>
      <td>http://ftp.ebi.ac.uk/pub/databases/gwas/summary_statistics/GCST009001-GCST010000/GCST009004/fat-distn.giant.ukbb.meta-analysis.bmi.combined.tbl.gz</td>
    </tr>
  </tbody>
</table><h2 class='text-muted'>Processed file</h2><p>Processing file: fat-distn.giant.ukbb.meta-analysis.bmi.combined.tbl.gz!</p><h2 class='text-muted'>GWAS file head</h2><table border="1" class="dataframe table table-bordered">
  <thead>
    <tr style="text-align: right;">
      <th>snp</th>
      <th>allele1</th>
      <th>allele2</th>
      <th>freq1</th>
      <th>freqse</th>
      <th>minfreq</th>
      <th>maxfreq</th>
      <th>effect</th>
      <th>stderr</th>
      <th>p-value</th>
      <th>direction</th>
    </tr>
  </thead>
  <tbody>
    <tr>
      <td>rs79565750_T_A</td>
      <td>a</td>
      <td>t</td>
      <td>0.0006</td>
      <td>0.0</td>
      <td>0.0006</td>
      <td>0.0006</td>
      <td>-0.0475</td>
      <td>0.0386</td>
      <td>0.2190</td>
      <td>-?</td>
    </tr>
    <tr>
      <td>rs186736998_G_T</td>
      <td>t</td>
      <td>g</td>
      <td>0.0003</td>
      <td>0.0</td>
      <td>0.0003</td>
      <td>0.0003</td>
      <td>0.0302</td>
      <td>0.0611</td>
      <td>0.6213</td>
      <td>+?</td>
    </tr>
  </tbody>
</table></div><div class='container mt-4 bg-light p-4 text-center'><h2 class='text-muted'>Original GWAS file headers</h2><table border="1" class="dataframe table table-bordered">
  <thead>
    <tr style="text-align: right;">
      <th>snp</th>
      <th>allele1</th>
      <th>allele2</th>
      <th>freq1</th>
      <th>freqse</th>
      <th>minfreq</th>
      <th>maxfreq</th>
      <th>effect</th>
      <th>stderr</th>
      <th>p-value</th>
      <th>direction</th>
    </tr>
  </thead>
  <tbody>
    <tr>
      <td>rs79565750_T_A</td>
      <td>a</td>
      <td>t</td>
      <td>0.0006</td>
      <td>0.0</td>
      <td>0.0006</td>
      <td>0.0006</td>
      <td>-0.0475</td>
      <td>0.0386</td>
      <td>0.2190</td>
      <td>-?</td>
    </tr>
    <tr>
      <td>rs186736998_G_T</td>
      <td>t</td>
      <td>g</td>
      <td>0.0003</td>
      <td>0.0</td>
      <td>0.0003</td>
      <td>0.0003</td>
      <td>0.0302</td>
      <td>0.0611</td>
      <td>0.6213</td>
      <td>+?</td>
    </tr>
  </tbody>
</table><p style='font-weight: bold; color: blue;'>CHR -> []</p><p style='font-weight: bold; color: blue;'>SNP -> ['snp']</p><p style='font-weight: bold; color: blue;'>BP -> []</p><p style='font-weight: bold; color: blue;'>A1 -> ['allele1']</p><p style='font-weight: bold; color: blue;'>A2 -> ['allele1', 'allele2']</p><p style='font-weight: bold; color: blue;'>N -> []</p><p style='font-weight: bold; color: blue;'>P -> ['p-value']</p><p style='font-weight: bold; color: blue;'>INFO -> []</p><p style='font-weight: bold; color: blue;'>MAF -> ['freq1']</p><p style='font-weight: bold; color: blue;'>BETA -> ['effect']</p><p style='font-weight: bold; color: blue;'>SE -> ['freqse', 'stderr']</p><p style='font-weight: bold; color: blue;'>OR -> []</p><p style='font-weight: bold; color: blue;'>Z -> []</p><p style='font-weight: bold; color: blue;'>D -> ['direction']</p><h2 class='text-muted'>GWAS columns</h2>snp, allele1, allele2, freq1, freqse, minfreq, maxfreq, effect, stderr, p-value, direction<h2 class='text-muted'>Identified columns</h2>snp, allele1, allele1, allele2, p-value, freq1, effect, freqse, stderr, direction<h2 class='text-muted'>Unidentified Columns headers</h2>maxfreq, minfreq<h2 class='text-muted'>PMID</h2>30239722<h2 class='text-muted'>DOI</h2>10.1093/hmg/ddy327<div class='bibtex-container'><h2 class='text-muted'>BibTeX</h2><div style='font-family: monospace; white-space: pre-wrap;'> @article{Pulit_2018, title={Meta-analysis of genome-wide association studies
 for body fat distribution in 694 649 individuals of European ancestry}, volume={28}, ISSN={1460-2083}, url={http://dx.doi.org/10.1093/hmg/ddy327}, DOI={10.1093/hmg/ddy327}, number={1}, journal={Human Molecular Genetics}, publisher={Oxford University Press (OUP)}, author={Pulit, Sara L and Stoneman, Charli and Morris, Andrew P and Wood, Andrew R and Glastonbury, Craig A and Tyrrell, Jessica and Yengo, Loïc and Ferreira, Teresa and Marouli, Eirini and Ji, Yingjie and Yang, Jian and Jones, Samuel and Beaumont, Robin and Croteau-Chonka, Damien C and Winkler, Thomas W and Hattersley, Andrew T and Loos, Ruth J F and Hirschhorn, Joel N and Visscher, Peter M and Frayling, Timothy M and Yaghootkar, Hanieh and Lindgren, Cecilia M}, year={2018}, month=sep, pages={166–174} }
</div></div><br><br><br><br><br><br></div><div class='container mt-4 bg-light p-4 text-center'><h1 class='text-primary'>Phenotype: body mass index</h1><p class='text-muted'><strong>URL:</strong> http://ftp.ebi.ac.uk/pub/databases/gwas/summary_statistics/GCST90278001-GCST90279000/GCST90278622</p><table border="1" class="dataframe table table-bordered">
  <thead>
    <tr style="text-align: right;">
      <th>firstAuthor</th>
      <th>accessionId</th>
      <th>publicationDate</th>
      <th>journal</th>
      <th>title</th>
      <th>reportedTrait</th>
      <th>efoTraits</th>
      <th>bgTraits</th>
      <th>discoverySampleAncestry</th>
      <th>replicationSampleAncestry</th>
      <th>associationCount</th>
      <th>summaryStatistics</th>
      <th>ssApiFlag</th>
      <th>agreedToCc0</th>
      <th>genotypingTechnologies</th>
      <th>pubmedId</th>
      <th>initialSampleDescription</th>
      <th>replicateSampleDescription</th>
      <th>UKB_FIELD</th>
      <th>FuzzySimilarity_body mass index bmi</th>
      <th>SearchPhenotype</th>
    </tr>
  </thead>
  <tbody>
    <tr>
      <td>Chen CY</td>
      <td>GCST90278622</td>
      <td>11/16/2023</td>
      <td>Cell Genom</td>
      <td>Analysis across Taiwan Biobank, Biobank Japan, and UK Biobank identifies hundreds of novel loci for 36 quantitative traits.</td>
      <td>body mass index</td>
      <td>body mass index</td>
      <td>-</td>
      <td>92615 east asian</td>
      <td>NaN</td>
      <td>51</td>
      <td>http://ftp.ebi.ac.uk/pub/databases/gwas/summary_statistics/GCST90278001-GCST90279000/GCST90278622</td>
      <td>no</td>
      <td>yes</td>
      <td>Genome-wide genotyping array</td>
      <td>38116116</td>
      <td>92,615 Taiwanese ancestry individuals</td>
      <td>NaN</td>
      <td>-</td>
      <td>88</td>
      <td>body mass index bmi</td>
    </tr>
  </tbody>
</table><h2 class='text-muted'>Information file</h2><table border="1" class="dataframe table table-bordered">
  <thead>
    <tr style="text-align: right;">
      <th>Unnamed: 0</th>
      <th>Name</th>
      <th>Last modified</th>
      <th>Size</th>
      <th>Description</th>
      <th>URL</th>
    </tr>
  </thead>
  <tbody>
    <tr>
      <td>2</td>
      <td>harmonised/</td>
      <td>2024-02-02 15:15</td>
      <td>-</td>
      <td>NaN</td>
      <td>http://ftp.ebi.ac.uk/pub/databases/gwas/summary_statistics/GCST90278001-GCST90279000/GCST90278622/harmonised/</td>
    </tr>
    <tr>
      <td>3</td>
      <td>md5sum.txt</td>
      <td>2023-09-13 00:07</td>
      <td>116</td>
      <td>NaN</td>
      <td>http://ftp.ebi.ac.uk/pub/databases/gwas/summary_statistics/GCST90278001-GCST90279000/GCST90278622/md5sum.txt</td>
    </tr>
    <tr>
      <td>1</td>
      <td>GCST90278622.tsv.gz-meta.yaml</td>
      <td>2024-01-17 00:07</td>
      <td>1.1K</td>
      <td>NaN</td>
      <td>http://ftp.ebi.ac.uk/pub/databases/gwas/summary_statistics/GCST90278001-GCST90279000/GCST90278622/GCST90278622.tsv.gz-meta.yaml</td>
    </tr>
    <tr>
      <td>0</td>
      <td>GCST90278622.tsv.gz</td>
      <td>2023-09-13 00:07</td>
      <td>274M</td>
      <td>NaN</td>
      <td>http://ftp.ebi.ac.uk/pub/databases/gwas/summary_statistics/GCST90278001-GCST90279000/GCST90278622/GCST90278622.tsv.gz</td>
    </tr>
  </tbody>
</table><h2 class='text-muted'>Processed file</h2><p>Processing file: GCST90278622.tsv.gz!</p><h2 class='text-muted'>GWAS file head</h2><table border="1" class="dataframe table table-bordered">
  <thead>
    <tr style="text-align: right;">
      <th>chromosome</th>
      <th>base_pair_location</th>
      <th>effect_allele</th>
      <th>other_allele</th>
      <th>beta</th>
      <th>standard_error</th>
      <th>effect_allele_frequency</th>
      <th>p_value</th>
      <th>minus_log_10_p_value</th>
      <th>rs_id</th>
      <th>variant_id</th>
      <th>n</th>
    </tr>
  </thead>
  <tbody>
    <tr>
      <td>1</td>
      <td>15211</td>
      <td>t</td>
      <td>g</td>
      <td>-0.018648</td>
      <td>0.019323</td>
      <td>0.4606</td>
      <td>0.331131</td>
      <td>0.48</td>
      <td>rs3982632</td>
      <td>1_15211_g_t</td>
      <td>92615</td>
    </tr>
    <tr>
      <td>1</td>
      <td>15903</td>
      <td>g</td>
      <td>gc</td>
      <td>-0.005025</td>
      <td>0.017914</td>
      <td>0.4346</td>
      <td>0.776247</td>
      <td>0.11</td>
      <td>rs557514207</td>
      <td>1_15903_gc_g</td>
      <td>92615</td>
    </tr>
  </tbody>
</table></div><div class='container mt-4 bg-light p-4 text-center'><h2 class='text-muted'>Original GWAS file headers</h2><table border="1" class="dataframe table table-bordered">
  <thead>
    <tr style="text-align: right;">
      <th>chromosome</th>
      <th>base_pair_location</th>
      <th>effect_allele</th>
      <th>other_allele</th>
      <th>beta</th>
      <th>standard_error</th>
      <th>effect_allele_frequency</th>
      <th>p_value</th>
      <th>minus_log_10_p_value</th>
      <th>rs_id</th>
      <th>variant_id</th>
      <th>n</th>
    </tr>
  </thead>
  <tbody>
    <tr>
      <td>1</td>
      <td>15211</td>
      <td>t</td>
      <td>g</td>
      <td>-0.018648</td>
      <td>0.019323</td>
      <td>0.4606</td>
      <td>0.331131</td>
      <td>0.48</td>
      <td>rs3982632</td>
      <td>1_15211_g_t</td>
      <td>92615</td>
    </tr>
    <tr>
      <td>1</td>
      <td>15903</td>
      <td>g</td>
      <td>gc</td>
      <td>-0.005025</td>
      <td>0.017914</td>
      <td>0.4346</td>
      <td>0.776247</td>
      <td>0.11</td>
      <td>rs557514207</td>
      <td>1_15903_gc_g</td>
      <td>92615</td>
    </tr>
  </tbody>
</table><p style='font-weight: bold; color: blue;'>CHR -> ['chromosome']</p><p style='font-weight: bold; color: blue;'>SNP -> ['rs_id', 'variant_id']</p><p style='font-weight: bold; color: blue;'>BP -> ['base_pair_location']</p><p style='font-weight: bold; color: blue;'>A1 -> ['effect_allele']</p><p style='font-weight: bold; color: blue;'>A2 -> ['other_allele']</p><p style='font-weight: bold; color: blue;'>N -> ['n']</p><p style='font-weight: bold; color: blue;'>P -> ['p_value']</p><p style='font-weight: bold; color: blue;'>INFO -> []</p><p style='font-weight: bold; color: blue;'>MAF -> ['effect_allele_frequency']</p><p style='font-weight: bold; color: blue;'>BETA -> ['beta']</p><p style='font-weight: bold; color: blue;'>SE -> ['standard_error']</p><p style='font-weight: bold; color: blue;'>OR -> []</p><p style='font-weight: bold; color: blue;'>Z -> []</p><p style='font-weight: bold; color: blue;'>D -> []</p><h2 class='text-muted'>GWAS columns</h2>chromosome, base_pair_location, effect_allele, other_allele, beta, standard_error, effect_allele_frequency, p_value, minus_log_10_p_value, rs_id, variant_id, n<h2 class='text-muted'>Identified columns</h2>chromosome, rs_id, variant_id, base_pair_location, effect_allele, other_allele, n, p_value, effect_allele_frequency, beta, standard_error<h2 class='text-muted'>Unidentified Columns headers</h2>minus_log_10_p_value<h2 class='text-muted'>PMID</h2>38116116<h2 class='text-muted'>DOI</h2>10.1016/j.xgen.2023.100436<div class='bibtex-container'><h2 class='text-muted'>BibTeX</h2><div style='font-family: monospace; white-space: pre-wrap;'> @article{Chen_2023, title={Analysis across Taiwan Biobank, Biobank Japan, and UK Biobank identifies hundreds of novel loci for 36 quantitative traits}, volume={3}, ISSN={2666-979X}, url={http://dx.doi.org/10.1016/j.xgen.2023.100436}, DOI={10.1016/j.xgen.2023.100436}, number={12}, journal={Cell Genomics}, publisher={Elsevier BV}, author={Chen, Chia-Yen and Chen, Tzu-Ting and Feng, Yen-Chen Anne and Yu, Mingrui and Lin, Shu-Chin and Longchamps, Ryan J. and Wang, Shi-Heng and Hsu, Yi-Hsiang and Yang, Hwai-I. and Kuo, Po-Hsiu and Daly, Mark J. and Chen, Wei J. and Huang, Hailiang and Ge, Tian and Lin, Yen-Feng}, year={2023}, month=dec, pages={100436} }
</div></div><br><br><br><br><br><br></div>