<link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/bootstrap/4.0.0/css/bootstrap.min.css" integrity="sha384-Gn5384xqQ1aoWXA+058RXPxPg6fy4IWvTNh0E263XmFcJlSAwiGgFAW/dAiS6JXm" crossorigin="anonymous"><div class='container mt-4 bg-light p-4 text-center'><h1 class='text-primary'>Phenotype: depression</h1><p class='text-muted'><strong>URL:</strong> http://ftp.ebi.ac.uk/pub/databases/gwas/summary_statistics/GCST003001-GCST004000/GCST003769</p><table border="1" class="dataframe table table-bordered">
  <thead>
    <tr style="text-align: right;">
      <th>firstAuthor</th>
      <th>accessionId</th>
      <th>publicationDate</th>
      <th>journal</th>
      <th>title</th>
      <th>reportedTrait</th>
      <th>efoTraits</th>
      <th>bgTraits</th>
      <th>discoverySampleAncestry</th>
      <th>replicationSampleAncestry</th>
      <th>associationCount</th>
      <th>summaryStatistics</th>
      <th>ssApiFlag</th>
      <th>agreedToCc0</th>
      <th>genotypingTechnologies</th>
      <th>pubmedId</th>
      <th>initialSampleDescription</th>
      <th>replicateSampleDescription</th>
      <th>UKB_FIELD</th>
      <th>FuzzySimilarity_depression</th>
      <th>SearchPhenotype</th>
    </tr>
  </thead>
  <tbody>
    <tr>
      <td>Okbay A</td>
      <td>GCST003769</td>
      <td>4/18/2016</td>
      <td>Nat Genet</td>
      <td>Genetic variants associated with subjective well-being, depressive symptoms, and neuroticism identified through genome-wide analyses.</td>
      <td>depression</td>
      <td>unipolar depression,depressive symptom measurement</td>
      <td>-</td>
      <td>180866 european</td>
      <td>368890 European</td>
      <td>15</td>
      <td>http://ftp.ebi.ac.uk/pub/databases/gwas/summary_statistics/GCST003001-GCST004000/GCST003769</td>
      <td>yes</td>
      <td>-</td>
      <td>Genome-wide genotyping array</td>
      <td>27089181</td>
      <td>105,739 European ancestry individuals,16,471 European ancestry major depressive disorder cases,58,835 European ancestry controls</td>
      <td>368,890 European ancestry individuals</td>
      <td>-</td>
      <td>100</td>
      <td>depression</td>
    </tr>
  </tbody>
</table><h2 class='text-muted'>Information file</h2><table border="1" class="dataframe table table-bordered">
  <thead>
    <tr style="text-align: right;">
      <th>Unnamed: 0</th>
      <th>Name</th>
      <th>Last modified</th>
      <th>Size</th>
      <th>Description</th>
      <th>URL</th>
    </tr>
  </thead>
  <tbody>
    <tr>
      <td>2</td>
      <td>harmonised/</td>
      <td>2019-03-25 11:15</td>
      <td>-</td>
      <td>NaN</td>
      <td>http://ftp.ebi.ac.uk/pub/databases/gwas/summary_statistics/GCST003001-GCST004000/GCST003769/harmonised/</td>
    </tr>
    <tr>
      <td>1</td>
      <td>Okbay A, 2016_readme.docx</td>
      <td>2018-01-05 13:52</td>
      <td>73K</td>
      <td>NaN</td>
      <td>http://ftp.ebi.ac.uk/pub/databases/gwas/summary_statistics/GCST003001-GCST004000/GCST003769/Okbay%20A,%202016_readme.docx</td>
    </tr>
    <tr>
      <td>0</td>
      <td>DS_Full.txt.gz</td>
      <td>2018-01-05 13:52</td>
      <td>98M</td>
      <td>NaN</td>
      <td>http://ftp.ebi.ac.uk/pub/databases/gwas/summary_statistics/GCST003001-GCST004000/GCST003769/DS_Full.txt.gz</td>
    </tr>
  </tbody>
</table><h2 class='text-muted'>Processed file</h2><p>Processing file: DS_Full.txt.gz!</p><h2 class='text-muted'>GWAS file head</h2><table border="1" class="dataframe table table-bordered">
  <thead>
    <tr style="text-align: right;">
      <th>markername</th>
      <th>chr</th>
      <th>pos</th>
      <th>a1</th>
      <th>a2</th>
      <th>eaf</th>
      <th>beta</th>
      <th>se</th>
      <th>pval</th>
    </tr>
  </thead>
  <tbody>
    <tr>
      <td>rs7973260</td>
      <td>12</td>
      <td>118375486</td>
      <td>A</td>
      <td>G</td>
      <td>0.1903</td>
      <td>0.031</td>
      <td>0.005</td>
      <td>1.776000e-09</td>
    </tr>
    <tr>
      <td>rs62100776</td>
      <td>18</td>
      <td>50754633</td>
      <td>A</td>
      <td>T</td>
      <td>0.5280</td>
      <td>-0.025</td>
      <td>0.004</td>
      <td>8.454000e-09</td>
    </tr>
  </tbody>
</table></div><div class='container mt-4 bg-light p-4 text-center'><h2 class='text-muted'>Original GWAS file headers</h2><table border="1" class="dataframe table table-bordered">
  <thead>
    <tr style="text-align: right;">
      <th>markername</th>
      <th>chr</th>
      <th>pos</th>
      <th>a1</th>
      <th>a2</th>
      <th>eaf</th>
      <th>beta</th>
      <th>se</th>
      <th>pval</th>
    </tr>
  </thead>
  <tbody>
    <tr>
      <td>rs7973260</td>
      <td>12</td>
      <td>118375486</td>
      <td>A</td>
      <td>G</td>
      <td>0.1903</td>
      <td>0.031</td>
      <td>0.005</td>
      <td>1.776000e-09</td>
    </tr>
    <tr>
      <td>rs62100776</td>
      <td>18</td>
      <td>50754633</td>
      <td>A</td>
      <td>T</td>
      <td>0.5280</td>
      <td>-0.025</td>
      <td>0.004</td>
      <td>8.454000e-09</td>
    </tr>
  </tbody>
</table><p style='font-weight: bold; color: blue;'>CHR -> ['chr']</p><p style='font-weight: bold; color: blue;'>SNP -> ['markername']</p><p style='font-weight: bold; color: blue;'>BP -> ['markername', 'pos']</p><p style='font-weight: bold; color: blue;'>A1 -> ['a1']</p><p style='font-weight: bold; color: blue;'>A2 -> ['a1', 'a2']</p><p style='font-weight: bold; color: blue;'>N -> []</p><p style='font-weight: bold; color: blue;'>P -> ['pval']</p><p style='font-weight: bold; color: blue;'>INFO -> []</p><p style='font-weight: bold; color: blue;'>MAF -> ['eaf']</p><p style='font-weight: bold; color: blue;'>BETA -> ['beta']</p><p style='font-weight: bold; color: blue;'>SE -> ['se']</p><p style='font-weight: bold; color: blue;'>OR -> []</p><p style='font-weight: bold; color: blue;'>Z -> []</p><p style='font-weight: bold; color: blue;'>D -> []</p><h2 class='text-muted'>GWAS columns</h2>markername, chr, pos, a1, a2, eaf, beta, se, pval<h2 class='text-muted'>Identified columns</h2>chr, markername, markername, pos, a1, a1, a2, pval, eaf, beta, se<h2 class='text-muted'>Unidentified Columns headers</h2><h2 class='text-muted'>PMID</h2>27089181<h2 class='text-muted'>DOI</h2>10.1038/ng.3552<div class='bibtex-container'><h2 class='text-muted'>BibTeX</h2><div style='font-family: monospace; white-space: pre-wrap;'> @article{Okbay_2016, title={Genetic variants associated with subjective well-being, depressive symptoms, and neuroticism identified through genome-wide analyses}, volume={48}, ISSN={1546-1718}, url={http://dx.doi.org/10.1038/ng.3552}, DOI={10.1038/ng.3552}, number={6}, journal={Nature Genetics}, publisher={Springer Science and Business Media LLC}, author={Okbay, Aysu and Baselmans, Bart M L and De Neve, Jan-Emmanuel and Turley, Patrick and Nivard, Michel G and Fontana, Mark Alan and Meddens, S Fleur W and Linnér, Richard Karlsson and Rietveld, Cornelius A and Derringer, Jaime and Gratten, Jacob and Lee, James J and Liu, Jimmy Z and de Vlaming, Ronald and Ahluwalia, Tarunveer S and Buchwald, Jadwiga and Cavadino, Alana and Frazier-Wood, Alexis C and Furlotte, Nicholas A and Garfield, Victoria and Geisel, Marie Henrike and Gonzalez, Juan R and Haitjema, Saskia and Karlsson, Robert and van der Laan, Sander W and Ladwig, Karl-Heinz and Lahti, Jari and van der Lee, Sven J and Lind, Penelope A and Liu, Tian and Matteson, Lindsay and Mihailov, Evelin and Miller, Michael B and Minica, Camelia C and Nolte, Ilja M and Mook-Kanamori, Dennis and van der Most, Peter J and Oldmeadow, Christopher and Qian, Yong and Raitakari, Olli and Rawal, Rajesh and Realo, Anu and Rueedi, Rico and Schmidt, Börge and Smith, Albert V and Stergiakouli, Evie and Tanaka, Toshiko and Taylor, Kent and Thorleifsson, Gudmar and Wedenoja, Juho and Wellmann, Juergen and Westra, Harm-Jan and Willems, Sara M and Zhao, Wei and Amin, Najaf and Bakshi, Andrew and Bergmann, Sven and Bjornsdottir, Gyda and Boyle, Patricia A and Cherney, Samantha and Cox, Simon R and Davies, Gail and Davis, Oliver S P and Ding, Jun and Direk, Nese and Eibich, Peter and Emeny, Rebecca T and Fatemifar, Ghazaleh and Faul, Jessica D and Ferrucci, Luigi and Forstner, Andreas J and Gieger, Christian and Gupta, Richa and Harris, Tamara B and Harris, Juliette M and Holliday, Elizabeth G and Hottenga, Jouke-Jan and De Jager, Philip L and Kaakinen, Marika A and Kajantie, Eero and Karhunen, Ville and Kolcic, Ivana and Kumari, Meena and Launer, Lenore J and Franke, Lude and Li-Gao, Ruifang and Liewald, David C and Koini, Marisa and Loukola, Anu and Marques-Vidal, Pedro and Montgomery, Grant W and Mosing, Miriam A and Paternoster, Lavinia and Pattie, Alison and Petrovic, Katja E and Pulkki-Råback, Laura and Quaye, Lydia and Räikkönen, Katri and Rudan, Igor and Scott, Rodney J and Smith, Jennifer A and Sutin, Angelina R and Trzaskowski, Maciej and Vinkhuyzen, Anna E and Yu, Lei and Zabaneh, Delilah and Attia, John R and Bennett, David A and Berger, Klaus and Bertram, Lars and Boomsma, Dorret I and Snieder, Harold and Chang, Shun-Chiao and Cucca, Francesco and Deary, Ian J and van Duijn, Cornelia M and Eriksson, Johan G and Bültmann, Ute and de Geus, Eco J C and Groenen, Patrick J F and Gudnason, Vilmundur and Hansen, Torben and Hartman, Catharine A and Haworth, Claire M A and Hayward, Caroline and Heath, Andrew C and Hinds, David A and Hyppönen, Elina and Iacono, William G and Järvelin, Marjo-Riitta and Jöckel, Karl-Heinz and Kaprio, Jaakko and Kardia, Sharon L R and Keltikangas-Järvinen, Liisa and Kraft, Peter and Kubzansky, Laura D and Lehtimäki, Terho and Magnusson, Patrik K E and Martin, Nicholas G and McGue, Matt and Metspalu, Andres and Mills, Melinda and de Mutsert, Renée and Oldehinkel, Albertine J and Pasterkamp, Gerard and Pedersen, Nancy L and Plomin, Robert and Polasek, Ozren and Power, Christine and Rich, Stephen S and Rosendaal, Frits R and den Ruijter, Hester M and Schlessinger, David and Schmidt, Helena and Svento, Rauli and Schmidt, Reinhold and Alizadeh, Behrooz Z and Sørensen, Thorkild I A and Spector, Tim D and Starr, John M and Stefansson, Kari and Steptoe, Andrew and Terracciano, Antonio and Thorsteinsdottir, Unnur and Thurik, A Roy and Timpson, Nicholas J and Tiemeier, Henning and Uitterlinden, André G and Vollenweider, Peter and Wagner, Gert G and Weir, David R and Yang, Jian and Conley, Dalton C and Smith, George Davey and Hofman, Albert and Johannesson, Magnus and Laibson, David I and Medland, Sarah E and Meyer, Michelle N and Pickrell, Joseph K and Esko, Tõnu and Krueger, Robert F and Beauchamp, Jonathan P and Koellinger, Philipp D and Benjamin, Daniel J and Bartels, Meike and Cesarini, David}, year={2016}, month=apr, pages={624–633} }
</div></div><br><br><br><br><br><br></div><div class='container mt-4 bg-light p-4 text-center'><h1 class='text-primary'>Phenotype: depression</h1><p class='text-muted'><strong>URL:</strong> http://ftp.ebi.ac.uk/pub/databases/gwas/summary_statistics/GCST006001-GCST007000/GCST006477</p><table border="1" class="dataframe table table-bordered">
  <thead>
    <tr style="text-align: right;">
      <th>firstAuthor</th>
      <th>accessionId</th>
      <th>publicationDate</th>
      <th>journal</th>
      <th>title</th>
      <th>reportedTrait</th>
      <th>efoTraits</th>
      <th>bgTraits</th>
      <th>discoverySampleAncestry</th>
      <th>replicationSampleAncestry</th>
      <th>associationCount</th>
      <th>summaryStatistics</th>
      <th>ssApiFlag</th>
      <th>agreedToCc0</th>
      <th>genotypingTechnologies</th>
      <th>pubmedId</th>
      <th>initialSampleDescription</th>
      <th>replicateSampleDescription</th>
      <th>UKB_FIELD</th>
      <th>FuzzySimilarity_depression</th>
      <th>SearchPhenotype</th>
    </tr>
  </thead>
  <tbody>
    <tr>
      <td>Nagel M</td>
      <td>GCST006477</td>
      <td>6/25/2018</td>
      <td>Nat Genet</td>
      <td>Meta-analysis of genome-wide association studies for neuroticism in 449,484 individuals identifies novel genetic loci and pathways.</td>
      <td>depression</td>
      <td>unipolar depression</td>
      <td>-</td>
      <td>688809 european</td>
      <td>NaN</td>
      <td>82</td>
      <td>http://ftp.ebi.ac.uk/pub/databases/gwas/summary_statistics/GCST006001-GCST007000/GCST006477</td>
      <td>yes</td>
      <td>-</td>
      <td>Genome-wide genotyping array</td>
      <td>29942085</td>
      <td>688,809 European ancestry individuals</td>
      <td>NaN</td>
      <td>-</td>
      <td>100</td>
      <td>depression</td>
    </tr>
  </tbody>
</table><h2 class='text-muted'>Information file</h2><table border="1" class="dataframe table table-bordered">
  <thead>
    <tr style="text-align: right;">
      <th>Unnamed: 0</th>
      <th>Name</th>
      <th>Last modified</th>
      <th>Size</th>
      <th>Description</th>
      <th>URL</th>
    </tr>
  </thead>
  <tbody>
    <tr>
      <td>0</td>
      <td>harmonised/</td>
      <td>2019-03-25 16:09</td>
      <td>-</td>
      <td>NaN</td>
      <td>http://ftp.ebi.ac.uk/pub/databases/gwas/summary_statistics/GCST006001-GCST007000/GCST006477/harmonised/</td>
    </tr>
    <tr>
      <td>1</td>
      <td>readme_ctg.txt</td>
      <td>2018-11-26 14:34</td>
      <td>2.4K</td>
      <td>NaN</td>
      <td>http://ftp.ebi.ac.uk/pub/databases/gwas/summary_statistics/GCST006001-GCST007000/GCST006477/readme_ctg.txt</td>
    </tr>
    <tr>
      <td>2</td>
      <td>sumstats_depression_ctg_format.txt.gz</td>
      <td>2018-11-26 14:34</td>
      <td>298M</td>
      <td>NaN</td>
      <td>http://ftp.ebi.ac.uk/pub/databases/gwas/summary_statistics/GCST006001-GCST007000/GCST006477/sumstats_depression_ctg_format.txt.gz</td>
    </tr>
  </tbody>
</table><h2 class='text-muted'>Processed file</h2><p>Processing file: sumstats_depression_ctg_format.txt.gz!</p><h2 class='text-muted'>GWAS file head</h2><table border="1" class="dataframe table table-bordered">
  <thead>
    <tr style="text-align: right;">
      <th>rsid</th>
      <th>snp</th>
      <th>chr</th>
      <th>pos</th>
      <th>a1</th>
      <th>a2</th>
      <th>eaf_ukb</th>
      <th>maf_ukb</th>
      <th>z</th>
      <th>p</th>
      <th>n</th>
      <th>info_ukb</th>
    </tr>
  </thead>
  <tbody>
    <tr>
      <td>rs12184267</td>
      <td>1_715265_C_T</td>
      <td>1</td>
      <td>715265</td>
      <td>T</td>
      <td>C</td>
      <td>0.03316</td>
      <td>0.03316</td>
      <td>-0.120</td>
      <td>0.9048</td>
      <td>357703</td>
      <td>0.926915</td>
    </tr>
    <tr>
      <td>rs12184277</td>
      <td>1_715367_A_G</td>
      <td>1</td>
      <td>715367</td>
      <td>A</td>
      <td>G</td>
      <td>0.96670</td>
      <td>0.03330</td>
      <td>0.249</td>
      <td>0.8033</td>
      <td>358093</td>
      <td>0.931148</td>
    </tr>
  </tbody>
</table></div><div class='container mt-4 bg-light p-4 text-center'><h2 class='text-muted'>Original GWAS file headers</h2><table border="1" class="dataframe table table-bordered">
  <thead>
    <tr style="text-align: right;">
      <th>rsid</th>
      <th>snp</th>
      <th>chr</th>
      <th>pos</th>
      <th>a1</th>
      <th>a2</th>
      <th>eaf_ukb</th>
      <th>maf_ukb</th>
      <th>z</th>
      <th>p</th>
      <th>n</th>
      <th>info_ukb</th>
    </tr>
  </thead>
  <tbody>
    <tr>
      <td>rs12184267</td>
      <td>1_715265_C_T</td>
      <td>1</td>
      <td>715265</td>
      <td>T</td>
      <td>C</td>
      <td>0.03316</td>
      <td>0.03316</td>
      <td>-0.120</td>
      <td>0.9048</td>
      <td>357703</td>
      <td>0.926915</td>
    </tr>
    <tr>
      <td>rs12184277</td>
      <td>1_715367_A_G</td>
      <td>1</td>
      <td>715367</td>
      <td>A</td>
      <td>G</td>
      <td>0.96670</td>
      <td>0.03330</td>
      <td>0.249</td>
      <td>0.8033</td>
      <td>358093</td>
      <td>0.931148</td>
    </tr>
  </tbody>
</table><p style='font-weight: bold; color: blue;'>CHR -> ['chr']</p><p style='font-weight: bold; color: blue;'>SNP -> ['rsid', 'snp']</p><p style='font-weight: bold; color: blue;'>BP -> ['pos']</p><p style='font-weight: bold; color: blue;'>A1 -> ['a1']</p><p style='font-weight: bold; color: blue;'>A2 -> ['a1', 'a2']</p><p style='font-weight: bold; color: blue;'>N -> ['n']</p><p style='font-weight: bold; color: blue;'>P -> ['p']</p><p style='font-weight: bold; color: blue;'>INFO -> ['info_ukb']</p><p style='font-weight: bold; color: blue;'>MAF -> ['eaf_ukb', 'maf_ukb']</p><p style='font-weight: bold; color: blue;'>BETA -> []</p><p style='font-weight: bold; color: blue;'>SE -> []</p><p style='font-weight: bold; color: blue;'>OR -> []</p><p style='font-weight: bold; color: blue;'>Z -> ['z']</p><p style='font-weight: bold; color: blue;'>D -> []</p><h2 class='text-muted'>GWAS columns</h2>rsid, snp, chr, pos, a1, a2, eaf_ukb, maf_ukb, z, p, n, info_ukb<h2 class='text-muted'>Identified columns</h2>chr, rsid, snp, pos, a1, a1, a2, n, p, info_ukb, eaf_ukb, maf_ukb, z<h2 class='text-muted'>Unidentified Columns headers</h2><h2 class='text-muted'>PMID</h2>29942085<h2 class='text-muted'>DOI</h2>Could not find DOI!<div class='bibtex-container'><h2 class='text-muted'>BibTeX</h2><div style='font-family: monospace; white-space: pre-wrap;'></div></div><br><br><br><br><br><br></div><div class='container mt-4 bg-light p-4 text-center'><h1 class='text-primary'>Phenotype: depression</h1><p class='text-muted'><strong>URL:</strong> http://ftp.ebi.ac.uk/pub/databases/gwas/summary_statistics/GCST010001-GCST011000/GCST010016</p><table border="1" class="dataframe table table-bordered">
  <thead>
    <tr style="text-align: right;">
      <th>firstAuthor</th>
      <th>accessionId</th>
      <th>publicationDate</th>
      <th>journal</th>
      <th>title</th>
      <th>reportedTrait</th>
      <th>efoTraits</th>
      <th>bgTraits</th>
      <th>discoverySampleAncestry</th>
      <th>replicationSampleAncestry</th>
      <th>associationCount</th>
      <th>summaryStatistics</th>
      <th>ssApiFlag</th>
      <th>agreedToCc0</th>
      <th>genotypingTechnologies</th>
      <th>pubmedId</th>
      <th>initialSampleDescription</th>
      <th>replicateSampleDescription</th>
      <th>UKB_FIELD</th>
      <th>FuzzySimilarity_depression</th>
      <th>SearchPhenotype</th>
    </tr>
  </thead>
  <tbody>
    <tr>
      <td>Cai N</td>
      <td>GCST010016</td>
      <td>3/30/2020</td>
      <td>Nat Genet</td>
      <td>Minimal phenotyping yields genome-wide association signals of low specificity for major depression.</td>
      <td>depression</td>
      <td>unipolar depression</td>
      <td>-</td>
      <td>212411 european</td>
      <td>NaN</td>
      <td>0</td>
      <td>http://ftp.ebi.ac.uk/pub/databases/gwas/summary_statistics/GCST010001-GCST011000/GCST010016</td>
      <td>no</td>
      <td>-</td>
      <td>Genome-wide genotyping array</td>
      <td>32231276</td>
      <td>9,176 British ancestry cases,203,235 British ancestry controls</td>
      <td>NaN</td>
      <td>-</td>
      <td>100</td>
      <td>depression</td>
    </tr>
  </tbody>
</table><h2 class='text-muted'>Information file</h2><table border="1" class="dataframe table table-bordered">
  <thead>
    <tr style="text-align: right;">
      <th>Unnamed: 0</th>
      <th>Name</th>
      <th>Last modified</th>
      <th>Size</th>
      <th>Description</th>
      <th>URL</th>
    </tr>
  </thead>
  <tbody>
    <tr>
      <td>1</td>
      <td>md5sum.txt</td>
      <td>2020-05-30 00:00</td>
      <td>162</td>
      <td>NaN</td>
      <td>http://ftp.ebi.ac.uk/pub/databases/gwas/summary_statistics/GCST010001-GCST011000/GCST010016/md5sum.txt</td>
    </tr>
    <tr>
      <td>2</td>
      <td>readme.txt</td>
      <td>2020-05-30 00:00</td>
      <td>730</td>
      <td>NaN</td>
      <td>http://ftp.ebi.ac.uk/pub/databases/gwas/summary_statistics/GCST010001-GCST011000/GCST010016/readme.txt</td>
    </tr>
    <tr>
      <td>0</td>
      <td>ICD10Dep.covararraypcs.assoc.logistic.zip</td>
      <td>2020-05-30 00:00</td>
      <td>243M</td>
      <td>NaN</td>
      <td>http://ftp.ebi.ac.uk/pub/databases/gwas/summary_statistics/GCST010001-GCST011000/GCST010016/ICD10Dep.covararraypcs.assoc.logistic.zip</td>
    </tr>
  </tbody>
</table><h2 class='text-muted'>GWAS file</h2><p>We were unable to process the GWAS file!</p><div class='container mt-4 bg-light p-4 text-center'><h1 class='text-primary'>Phenotype: depression</h1><p class='text-muted'><strong>URL:</strong> http://ftp.ebi.ac.uk/pub/databases/gwas/summary_statistics/GCST007001-GCST008000/GCST007342</p><table border="1" class="dataframe table table-bordered">
  <thead>
    <tr style="text-align: right;">
      <th>firstAuthor</th>
      <th>accessionId</th>
      <th>publicationDate</th>
      <th>journal</th>
      <th>title</th>
      <th>reportedTrait</th>
      <th>efoTraits</th>
      <th>bgTraits</th>
      <th>discoverySampleAncestry</th>
      <th>replicationSampleAncestry</th>
      <th>associationCount</th>
      <th>summaryStatistics</th>
      <th>ssApiFlag</th>
      <th>agreedToCc0</th>
      <th>genotypingTechnologies</th>
      <th>pubmedId</th>
      <th>initialSampleDescription</th>
      <th>replicateSampleDescription</th>
      <th>UKB_FIELD</th>
      <th>FuzzySimilarity_depression</th>
      <th>SearchPhenotype</th>
    </tr>
  </thead>
  <tbody>
    <tr>
      <td>Howard DM</td>
      <td>GCST007342</td>
      <td>2/4/2019</td>
      <td>Nat Neurosci</td>
      <td>Genome-wide meta-analysis of depression identifies 102 independent variants and highlights the importance of the prefrontal brain regions.</td>
      <td>depression</td>
      <td>unipolar depression</td>
      <td>-</td>
      <td>361315 nr,446238 european</td>
      <td>1306354 European</td>
      <td>98</td>
      <td>http://ftp.ebi.ac.uk/pub/databases/gwas/summary_statistics/GCST007001-GCST008000/GCST007342</td>
      <td>no</td>
      <td>-</td>
      <td>Genome-wide genotyping array</td>
      <td>30718901</td>
      <td>118,811 European ancestry cases,327,427 European ancestry controls,127,552 cases,233,763 controls</td>
      <td>414,055 European ancestry cases,892,299 European ancestry controls</td>
      <td>-</td>
      <td>100</td>
      <td>depression</td>
    </tr>
  </tbody>
</table><h2 class='text-muted'>Information file</h2><table border="1" class="dataframe table table-bordered">
  <thead>
    <tr style="text-align: right;">
      <th>Unnamed: 0</th>
      <th>Name</th>
      <th>Last modified</th>
      <th>Size</th>
      <th>Description</th>
      <th>URL</th>
    </tr>
  </thead>
  <tbody>
    <tr>
      <td>1</td>
      <td>ReadMe.txt</td>
      <td>2019-04-13 23:21</td>
      <td>2.4K</td>
      <td>NaN</td>
      <td>http://ftp.ebi.ac.uk/pub/databases/gwas/summary_statistics/GCST007001-GCST008000/GCST007342/ReadMe.txt</td>
    </tr>
    <tr>
      <td>0</td>
      <td>PGC_UKB_23andMe_depression_10000.txt</td>
      <td>2019-04-13 23:21</td>
      <td>444K</td>
      <td>NaN</td>
      <td>http://ftp.ebi.ac.uk/pub/databases/gwas/summary_statistics/GCST007001-GCST008000/GCST007342/PGC_UKB_23andMe_depression_10000.txt</td>
    </tr>
  </tbody>
</table><h2 class='text-muted'>Processed file</h2><p>Processing file: PGC_UKB_23andMe_depression_10000.txt!</p><h2 class='text-muted'>GWAS file head</h2><table border="1" class="dataframe table table-bordered">
  <thead>
    <tr style="text-align: right;">
      <th>markername</th>
      <th>a1</th>
      <th>a2</th>
      <th>freq</th>
      <th>logor</th>
      <th>stderrlogor</th>
      <th>p</th>
    </tr>
  </thead>
  <tbody>
    <tr>
      <td>rs10004838</td>
      <td>a</td>
      <td>t</td>
      <td>0.6307</td>
      <td>0.0120</td>
      <td>0.0037</td>
      <td>0.001288</td>
    </tr>
    <tr>
      <td>rs10005823</td>
      <td>t</td>
      <td>c</td>
      <td>0.7232</td>
      <td>-0.0141</td>
      <td>0.0040</td>
      <td>0.000469</td>
    </tr>
  </tbody>
</table></div><div class='container mt-4 bg-light p-4 text-center'><h2 class='text-muted'>Original GWAS file headers</h2><table border="1" class="dataframe table table-bordered">
  <thead>
    <tr style="text-align: right;">
      <th>markername</th>
      <th>a1</th>
      <th>a2</th>
      <th>freq</th>
      <th>logor</th>
      <th>stderrlogor</th>
      <th>p</th>
    </tr>
  </thead>
  <tbody>
    <tr>
      <td>rs10004838</td>
      <td>a</td>
      <td>t</td>
      <td>0.6307</td>
      <td>0.0120</td>
      <td>0.0037</td>
      <td>0.001288</td>
    </tr>
    <tr>
      <td>rs10005823</td>
      <td>t</td>
      <td>c</td>
      <td>0.7232</td>
      <td>-0.0141</td>
      <td>0.0040</td>
      <td>0.000469</td>
    </tr>
  </tbody>
</table><p style='font-weight: bold; color: blue;'>CHR -> []</p><p style='font-weight: bold; color: blue;'>SNP -> ['markername']</p><p style='font-weight: bold; color: blue;'>BP -> ['markername']</p><p style='font-weight: bold; color: blue;'>A1 -> ['a1']</p><p style='font-weight: bold; color: blue;'>A2 -> ['a1', 'a2']</p><p style='font-weight: bold; color: blue;'>N -> []</p><p style='font-weight: bold; color: blue;'>P -> ['p']</p><p style='font-weight: bold; color: blue;'>INFO -> []</p><p style='font-weight: bold; color: blue;'>MAF -> ['freq']</p><p style='font-weight: bold; color: blue;'>BETA -> []</p><p style='font-weight: bold; color: blue;'>SE -> ['stderrlogor']</p><p style='font-weight: bold; color: blue;'>OR -> ['logor']</p><p style='font-weight: bold; color: blue;'>Z -> []</p><p style='font-weight: bold; color: blue;'>D -> []</p><h2 class='text-muted'>GWAS columns</h2>markername, a1, a2, freq, logor, stderrlogor, p<h2 class='text-muted'>Identified columns</h2>markername, markername, a1, a1, a2, p, freq, stderrlogor, logor<h2 class='text-muted'>Unidentified Columns headers</h2><h2 class='text-muted'>PMID</h2>30718901<h2 class='text-muted'>DOI</h2>10.1038/s41593-018-0326-7<div class='bibtex-container'><h2 class='text-muted'>BibTeX</h2><div style='font-family: monospace; white-space: pre-wrap;'> @article{Howard_2019, title={Genome-wide meta-analysis of depression identifies 102 independent variants and highlights the importance of the prefrontal brain regions}, volume={22}, ISSN={1546-1726}, url={http://dx.doi.org/10.1038/s41593-018-0326-7}, DOI={10.1038/s41593-018-0326-7}, number={3}, journal={Nature Neuroscience}, publisher={Springer Science and Business Media LLC}, author={Howard, David M. and Adams, Mark J. and Clarke, Toni-Kim and Hafferty, Jonathan D. and Gibson, Jude and Shirali, Masoud and Coleman, Jonathan R. I. and Hagenaars, Saskia P. and Ward, Joey and Wigmore, Eleanor M. and Alloza, Clara and Shen, Xueyi and Barbu, Miruna C. and Xu, Eileen Y. and Whalley, Heather C. and Marioni, Riccardo E. and Porteous, David J. and Davies, Gail and Deary, Ian J. and Hemani, Gibran and Berger, Klaus and Teismann, Henning and Rawal, Rajesh and Arolt, Volker and Baune, Bernhard T. and Dannlowski, Udo and Domschke, Katharina and Tian, Chao and Hinds, David A. and Trzaskowski, Maciej and Byrne, Enda M. and Ripke, Stephan and Smith, Daniel J. and Sullivan, Patrick F. and Wray, Naomi R. and Breen, Gerome and Lewis, Cathryn M. and McIntosh, Andrew M.}, year={2019}, month=feb, pages={343â€“352} }
</div></div><br><br><br><br><br><br></div><div class='container mt-4 bg-light p-4 text-center'><h1 class='text-primary'>Phenotype: depression</h1><p class='text-muted'><strong>URL:</strong> http://ftp.ebi.ac.uk/pub/databases/gwas/summary_statistics/GCST90038001-GCST90039000/GCST90038650</p><table border="1" class="dataframe table table-bordered">
  <thead>
    <tr style="text-align: right;">
      <th>firstAuthor</th>
      <th>accessionId</th>
      <th>publicationDate</th>
      <th>journal</th>
      <th>title</th>
      <th>reportedTrait</th>
      <th>efoTraits</th>
      <th>bgTraits</th>
      <th>discoverySampleAncestry</th>
      <th>replicationSampleAncestry</th>
      <th>associationCount</th>
      <th>summaryStatistics</th>
      <th>ssApiFlag</th>
      <th>agreedToCc0</th>
      <th>genotypingTechnologies</th>
      <th>pubmedId</th>
      <th>initialSampleDescription</th>
      <th>replicateSampleDescription</th>
      <th>UKB_FIELD</th>
      <th>FuzzySimilarity_depression</th>
      <th>SearchPhenotype</th>
    </tr>
  </thead>
  <tbody>
    <tr>
      <td>D??nerta?� HM</td>
      <td>GCST90038650</td>
      <td>4/8/2021</td>
      <td>Nat Aging</td>
      <td>Common genetic associations between age-related diseases.</td>
      <td>depression</td>
      <td>unipolar depression</td>
      <td>-</td>
      <td>484598 nr</td>
      <td>NaN</td>
      <td>0</td>
      <td>http://ftp.ebi.ac.uk/pub/databases/gwas/summary_statistics/GCST90038001-GCST90039000/GCST90038650</td>
      <td>no</td>
      <td>yes</td>
      <td>Genome-wide genotyping array</td>
      <td>33959723</td>
      <td>27,568 cases,457,030 controls</td>
      <td>NaN</td>
      <td>-</td>
      <td>100</td>
      <td>depression</td>
    </tr>
  </tbody>
</table><h2 class='text-muted'>Information file</h2><table border="1" class="dataframe table table-bordered">
  <thead>
    <tr style="text-align: right;">
      <th>Unnamed: 0</th>
      <th>Name</th>
      <th>Last modified</th>
      <th>Size</th>
      <th>Description</th>
      <th>URL</th>
    </tr>
  </thead>
  <tbody>
    <tr>
      <td>1</td>
      <td>GCST90038650_buildGRCh37.tsv-meta.yaml</td>
      <td>2024-01-04 00:31</td>
      <td>661</td>
      <td>NaN</td>
      <td>http://ftp.ebi.ac.uk/pub/databases/gwas/summary_statistics/GCST90038001-GCST90039000/GCST90038650/GCST90038650_buildGRCh37.tsv-meta.yaml</td>
    </tr>
    <tr>
      <td>2</td>
      <td>harmonised/</td>
      <td>2024-02-02 15:16</td>
      <td>-</td>
      <td>NaN</td>
      <td>http://ftp.ebi.ac.uk/pub/databases/gwas/summary_statistics/GCST90038001-GCST90039000/GCST90038650/harmonised/</td>
    </tr>
    <tr>
      <td>3</td>
      <td>md5sum.txt</td>
      <td>2021-11-14 00:33</td>
      <td>62</td>
      <td>NaN</td>
      <td>http://ftp.ebi.ac.uk/pub/databases/gwas/summary_statistics/GCST90038001-GCST90039000/GCST90038650/md5sum.txt</td>
    </tr>
    <tr>
      <td>0</td>
      <td>GCST90038650_buildGRCh37.tsv</td>
      <td>2021-11-14 00:33</td>
      <td>1.2G</td>
      <td>NaN</td>
      <td>http://ftp.ebi.ac.uk/pub/databases/gwas/summary_statistics/GCST90038001-GCST90039000/GCST90038650/GCST90038650_buildGRCh37.tsv</td>
    </tr>
  </tbody>
</table><h2 class='text-muted'>Processed file</h2><p>Processing file: GCST90038650_buildGRCh37.tsv.1!</p><h2 class='text-muted'>GWAS file head</h2><table border="1" class="dataframe table table-bordered">
  <thead>
    <tr style="text-align: right;">
      <th>variant_id</th>
      <th>chromosome</th>
      <th>base_pair_location</th>
      <th>genpos</th>
      <th>effect_allele</th>
      <th>other_allele</th>
      <th>effect_allele_frequency</th>
      <th>info</th>
      <th>chisq_linreg</th>
      <th>p_linreg</th>
      <th>beta</th>
      <th>standard_error</th>
      <th>chisq_bolt_lmm_inf</th>
      <th>p_bolt_lmm_inf</th>
      <th>chisq_bolt_lmm</th>
      <th>p_value</th>
    </tr>
  </thead>
  <tbody>
    <tr>
      <td>rs533090414</td>
      <td>1</td>
      <td>18849</td>
      <td>0.0</td>
      <td>C</td>
      <td>G</td>
      <td>0.026465</td>
      <td>0.536412</td>
      <td>3.858780</td>
      <td>0.049</td>
      <td>-0.003968</td>
      <td>0.002018</td>
      <td>3.865700</td>
      <td>0.049</td>
      <td>3.932160</td>
      <td>0.047</td>
    </tr>
    <tr>
      <td>rs568927205</td>
      <td>1</td>
      <td>54712</td>
      <td>0.0</td>
      <td>T</td>
      <td>TTTTC</td>
      <td>0.415825</td>
      <td>0.538572</td>
      <td>0.359482</td>
      <td>0.550</td>
      <td>0.000345</td>
      <td>0.000650</td>
      <td>0.281784</td>
      <td>0.600</td>
      <td>0.280051</td>
      <td>0.600</td>
    </tr>
  </tbody>
</table></div><div class='container mt-4 bg-light p-4 text-center'><h2 class='text-muted'>Original GWAS file headers</h2><table border="1" class="dataframe table table-bordered">
  <thead>
    <tr style="text-align: right;">
      <th>variant_id</th>
      <th>chromosome</th>
      <th>base_pair_location</th>
      <th>genpos</th>
      <th>effect_allele</th>
      <th>other_allele</th>
      <th>effect_allele_frequency</th>
      <th>info</th>
      <th>chisq_linreg</th>
      <th>p_linreg</th>
      <th>beta</th>
      <th>standard_error</th>
      <th>chisq_bolt_lmm_inf</th>
      <th>p_bolt_lmm_inf</th>
      <th>chisq_bolt_lmm</th>
      <th>p_value</th>
    </tr>
  </thead>
  <tbody>
    <tr>
      <td>rs533090414</td>
      <td>1</td>
      <td>18849</td>
      <td>0.0</td>
      <td>C</td>
      <td>G</td>
      <td>0.026465</td>
      <td>0.536412</td>
      <td>3.858780</td>
      <td>0.049</td>
      <td>-0.003968</td>
      <td>0.002018</td>
      <td>3.865700</td>
      <td>0.049</td>
      <td>3.932160</td>
      <td>0.047</td>
    </tr>
    <tr>
      <td>rs568927205</td>
      <td>1</td>
      <td>54712</td>
      <td>0.0</td>
      <td>T</td>
      <td>TTTTC</td>
      <td>0.415825</td>
      <td>0.538572</td>
      <td>0.359482</td>
      <td>0.550</td>
      <td>0.000345</td>
      <td>0.000650</td>
      <td>0.281784</td>
      <td>0.600</td>
      <td>0.280051</td>
      <td>0.600</td>
    </tr>
  </tbody>
</table><p style='font-weight: bold; color: blue;'>CHR -> ['chromosome']</p><p style='font-weight: bold; color: blue;'>SNP -> ['variant_id']</p><p style='font-weight: bold; color: blue;'>BP -> ['base_pair_location', 'genpos']</p><p style='font-weight: bold; color: blue;'>A1 -> ['effect_allele']</p><p style='font-weight: bold; color: blue;'>A2 -> ['other_allele']</p><p style='font-weight: bold; color: blue;'>N -> []</p><p style='font-weight: bold; color: blue;'>P -> ['p_value']</p><p style='font-weight: bold; color: blue;'>INFO -> ['info']</p><p style='font-weight: bold; color: blue;'>MAF -> ['effect_allele_frequency']</p><p style='font-weight: bold; color: blue;'>BETA -> ['beta', 'p_bolt_lmm_inf']</p><p style='font-weight: bold; color: blue;'>SE -> ['standard_error']</p><p style='font-weight: bold; color: blue;'>OR -> []</p><p style='font-weight: bold; color: blue;'>Z -> []</p><p style='font-weight: bold; color: blue;'>D -> []</p><h2 class='text-muted'>GWAS columns</h2>variant_id, chromosome, base_pair_location, genpos, effect_allele, other_allele, effect_allele_frequency, info, chisq_linreg, p_linreg, beta, standard_error, chisq_bolt_lmm_inf, p_bolt_lmm_inf, chisq_bolt_lmm, p_value<h2 class='text-muted'>Identified columns</h2>chromosome, variant_id, base_pair_location, genpos, effect_allele, other_allele, p_value, info, effect_allele_frequency, beta, p_bolt_lmm_inf, standard_error<h2 class='text-muted'>Unidentified Columns headers</h2>chisq_bolt_lmm_inf, chisq_bolt_lmm, chisq_linreg, p_linreg<h2 class='text-muted'>PMID</h2>33959723<h2 class='text-muted'>DOI</h2>10.1038/s43587-021-00051-5<div class='bibtex-container'><h2 class='text-muted'>BibTeX</h2><div style='font-family: monospace; white-space: pre-wrap;'> @article{D_nerta__2021, title={Common genetic associations between age-related diseases}, volume={1}, ISSN={2662-8465}, url={http://dx.doi.org/10.1038/s43587-021-00051-5}, DOI={10.1038/s43587-021-00051-5}, number={4}, journal={Nature Aging}, publisher={Springer Science and Business Media LLC}, author={Dönertaş, Handan Melike and Fabian, Daniel K. and Fuentealba, Matías and Partridge, Linda and Thornton, Janet M.}, year={2021}, month=apr, pages={400–412} }
</div></div><br><br><br><br><br><br></div><div class='container mt-4 bg-light p-4 text-center'><h1 class='text-primary'>Phenotype: depression</h1><p class='text-muted'><strong>URL:</strong> http://ftp.ebi.ac.uk/pub/databases/gwas/summary_statistics/GCST90275001-GCST90276000/GCST90275155</p><table border="1" class="dataframe table table-bordered">
  <thead>
    <tr style="text-align: right;">
      <th>firstAuthor</th>
      <th>accessionId</th>
      <th>publicationDate</th>
      <th>journal</th>
      <th>title</th>
      <th>reportedTrait</th>
      <th>efoTraits</th>
      <th>bgTraits</th>
      <th>discoverySampleAncestry</th>
      <th>replicationSampleAncestry</th>
      <th>associationCount</th>
      <th>summaryStatistics</th>
      <th>ssApiFlag</th>
      <th>agreedToCc0</th>
      <th>genotypingTechnologies</th>
      <th>pubmedId</th>
      <th>initialSampleDescription</th>
      <th>replicateSampleDescription</th>
      <th>UKB_FIELD</th>
      <th>FuzzySimilarity_depression</th>
      <th>SearchPhenotype</th>
    </tr>
  </thead>
  <tbody>
    <tr>
      <td>Pedersen EM</td>
      <td>GCST90275155</td>
      <td>9/9/2023</td>
      <td>Nat Commun</td>
      <td>ADuLT: An efficient and robust time-to-event GWAS.</td>
      <td>depression</td>
      <td>unipolar depression</td>
      <td>-</td>
      <td>63875 european</td>
      <td>NaN</td>
      <td>0</td>
      <td>http://ftp.ebi.ac.uk/pub/databases/gwas/summary_statistics/GCST90275001-GCST90276000/GCST90275155</td>
      <td>no</td>
      <td>yes</td>
      <td>Genome-wide genotyping array</td>
      <td>37689771</td>
      <td>27,507 European ancestry cases,36,368 European ancestry controls</td>
      <td>NaN</td>
      <td>-</td>
      <td>100</td>
      <td>depression</td>
    </tr>
  </tbody>
</table><h2 class='text-muted'>Information file</h2><table border="1" class="dataframe table table-bordered">
  <thead>
    <tr style="text-align: right;">
      <th>Unnamed: 0</th>
      <th>Name</th>
      <th>Last modified</th>
      <th>Size</th>
      <th>Description</th>
      <th>URL</th>
    </tr>
  </thead>
  <tbody>
    <tr>
      <td>2</td>
      <td>md5sum.txt</td>
      <td>2023-09-28 00:07</td>
      <td>110</td>
      <td>NaN</td>
      <td>http://ftp.ebi.ac.uk/pub/databases/gwas/summary_statistics/GCST90275001-GCST90276000/GCST90275155/md5sum.txt</td>
    </tr>
    <tr>
      <td>1</td>
      <td>GCST90275155.tsv-meta.yaml</td>
      <td>2024-01-04 00:21</td>
      <td>1.1K</td>
      <td>NaN</td>
      <td>http://ftp.ebi.ac.uk/pub/databases/gwas/summary_statistics/GCST90275001-GCST90276000/GCST90275155/GCST90275155.tsv-meta.yaml</td>
    </tr>
    <tr>
      <td>0</td>
      <td>GCST90275155.tsv</td>
      <td>2023-09-28 00:07</td>
      <td>1.2G</td>
      <td>NaN</td>
      <td>http://ftp.ebi.ac.uk/pub/databases/gwas/summary_statistics/GCST90275001-GCST90276000/GCST90275155/GCST90275155.tsv</td>
    </tr>
  </tbody>
</table><h2 class='text-muted'>Processed file</h2><p>Processing file: GCST90275155.tsv.1!</p><h2 class='text-muted'>GWAS file head</h2><table border="1" class="dataframe table table-bordered">
  <thead>
    <tr style="text-align: right;">
      <th>chromosome</th>
      <th>base_pair_location</th>
      <th>effect_allele</th>
      <th>other_allele</th>
      <th>beta</th>
      <th>standard_error</th>
      <th>effect_allele_frequency</th>
      <th>p_value</th>
      <th>variant_id</th>
      <th>rs_id</th>
      <th>z</th>
      <th>ref_allele</th>
    </tr>
  </thead>
  <tbody>
    <tr>
      <td>1</td>
      <td>693731</td>
      <td>A</td>
      <td>G</td>
      <td>-0.032009</td>
      <td>0.010518</td>
      <td>0.127278</td>
      <td>0.002342</td>
      <td>1_693731_A_G</td>
      <td>rs12238997</td>
      <td>-3.043125</td>
      <td>EA</td>
    </tr>
    <tr>
      <td>1</td>
      <td>729679</td>
      <td>C</td>
      <td>G</td>
      <td>0.025551</td>
      <td>0.008993</td>
      <td>0.186210</td>
      <td>0.004495</td>
      <td>1_729679_C_G</td>
      <td>rs4951859</td>
      <td>2.841287</td>
      <td>EA</td>
    </tr>
  </tbody>
</table></div><div class='container mt-4 bg-light p-4 text-center'><h2 class='text-muted'>Original GWAS file headers</h2><table border="1" class="dataframe table table-bordered">
  <thead>
    <tr style="text-align: right;">
      <th>chromosome</th>
      <th>base_pair_location</th>
      <th>effect_allele</th>
      <th>other_allele</th>
      <th>beta</th>
      <th>standard_error</th>
      <th>effect_allele_frequency</th>
      <th>p_value</th>
      <th>variant_id</th>
      <th>rs_id</th>
      <th>z</th>
      <th>ref_allele</th>
    </tr>
  </thead>
  <tbody>
    <tr>
      <td>1</td>
      <td>693731</td>
      <td>A</td>
      <td>G</td>
      <td>-0.032009</td>
      <td>0.010518</td>
      <td>0.127278</td>
      <td>0.002342</td>
      <td>1_693731_A_G</td>
      <td>rs12238997</td>
      <td>-3.043125</td>
      <td>EA</td>
    </tr>
    <tr>
      <td>1</td>
      <td>729679</td>
      <td>C</td>
      <td>G</td>
      <td>0.025551</td>
      <td>0.008993</td>
      <td>0.186210</td>
      <td>0.004495</td>
      <td>1_729679_C_G</td>
      <td>rs4951859</td>
      <td>2.841287</td>
      <td>EA</td>
    </tr>
  </tbody>
</table><p style='font-weight: bold; color: blue;'>CHR -> ['chromosome']</p><p style='font-weight: bold; color: blue;'>SNP -> ['variant_id', 'rs_id']</p><p style='font-weight: bold; color: blue;'>BP -> ['base_pair_location']</p><p style='font-weight: bold; color: blue;'>A1 -> ['effect_allele', 'ref_allele']</p><p style='font-weight: bold; color: blue;'>A2 -> ['other_allele']</p><p style='font-weight: bold; color: blue;'>N -> []</p><p style='font-weight: bold; color: blue;'>P -> ['p_value']</p><p style='font-weight: bold; color: blue;'>INFO -> []</p><p style='font-weight: bold; color: blue;'>MAF -> ['effect_allele_frequency']</p><p style='font-weight: bold; color: blue;'>BETA -> ['beta']</p><p style='font-weight: bold; color: blue;'>SE -> ['standard_error']</p><p style='font-weight: bold; color: blue;'>OR -> []</p><p style='font-weight: bold; color: blue;'>Z -> ['z']</p><p style='font-weight: bold; color: blue;'>D -> []</p><h2 class='text-muted'>GWAS columns</h2>chromosome, base_pair_location, effect_allele, other_allele, beta, standard_error, effect_allele_frequency, p_value, variant_id, rs_id, z, ref_allele<h2 class='text-muted'>Identified columns</h2>chromosome, variant_id, rs_id, base_pair_location, effect_allele, ref_allele, other_allele, p_value, effect_allele_frequency, beta, standard_error, z<h2 class='text-muted'>Unidentified Columns headers</h2><h2 class='text-muted'>PMID</h2>37689771<h2 class='text-muted'>DOI</h2>10.1038/s41467-023-41210-z<div class='bibtex-container'><h2 class='text-muted'>BibTeX</h2><div style='font-family: monospace; white-space: pre-wrap;'> @article{Pedersen_2023, title={ADuLT: An efficient and robust time-to-event GWAS}, volume={14}, ISSN={2041-1723}, url={http://dx.doi.org/10.1038/s41467-023-41210-z}, DOI={10.1038/s41467-023-41210-z}, number={1}, journal={Nature Communications}, publisher={Springer Science and Business Media LLC}, author={Pedersen, Emil M. and Agerbo, Esben and Plana-Ripoll, Oleguer and Steinbach, Jette and Krebs, Morten D. and Hougaard, David M. and Werge, Thomas and Nordentoft, Merete and Børglum, Anders D. and Musliner, Katherine L. and Ganna, Andrea and Schork, Andrew J. and Mortensen, Preben B. and McGrath, John J. and Privé, Florian and Vilhjálmsson, Bjarni J.}, year={2023}, month=sep }
</div></div><br><br><br><br><br><br></div><div class='container mt-4 bg-light p-4 text-center'><h1 class='text-primary'>Phenotype: depression</h1><p class='text-muted'><strong>URL:</strong> http://ftp.ebi.ac.uk/pub/databases/gwas/summary_statistics/GCST009001-GCST010000/GCST009978</p><table border="1" class="dataframe table table-bordered">
  <thead>
    <tr style="text-align: right;">
      <th>firstAuthor</th>
      <th>accessionId</th>
      <th>publicationDate</th>
      <th>journal</th>
      <th>title</th>
      <th>reportedTrait</th>
      <th>efoTraits</th>
      <th>bgTraits</th>
      <th>discoverySampleAncestry</th>
      <th>replicationSampleAncestry</th>
      <th>associationCount</th>
      <th>summaryStatistics</th>
      <th>ssApiFlag</th>
      <th>agreedToCc0</th>
      <th>genotypingTechnologies</th>
      <th>pubmedId</th>
      <th>initialSampleDescription</th>
      <th>replicateSampleDescription</th>
      <th>UKB_FIELD</th>
      <th>FuzzySimilarity_depression</th>
      <th>SearchPhenotype</th>
    </tr>
  </thead>
  <tbody>
    <tr>
      <td>Shen H</td>
      <td>GCST009978</td>
      <td>1/11/2020</td>
      <td>Neuropsychopharmacology</td>
      <td>Polygenic prediction and GWAS of depression, PTSD, and suicidal ideation/self-harm in a Peruvian cohort.</td>
      <td>depression</td>
      <td>unipolar depression</td>
      <td>-</td>
      <td>3404 hispanic or latin american</td>
      <td>NaN</td>
      <td>1</td>
      <td>http://ftp.ebi.ac.uk/pub/databases/gwas/summary_statistics/GCST009001-GCST010000/GCST009978</td>
      <td>no</td>
      <td>-</td>
      <td>Genome-wide genotyping array</td>
      <td>31926482</td>
      <td>1,076 Peruvian cases,2,328 Peruvian controls</td>
      <td>NaN</td>
      <td>-</td>
      <td>100</td>
      <td>depression</td>
    </tr>
  </tbody>
</table><h2 class='text-muted'>Information file</h2><table border="1" class="dataframe table table-bordered">
  <thead>
    <tr style="text-align: right;">
      <th>Unnamed: 0</th>
      <th>Name</th>
      <th>Last modified</th>
      <th>Size</th>
      <th>Description</th>
      <th>URL</th>
    </tr>
  </thead>
  <tbody>
    <tr>
      <td>1</td>
      <td>md5sum.txt</td>
      <td>2020-05-20 00:00</td>
      <td>61</td>
      <td>NaN</td>
      <td>http://ftp.ebi.ac.uk/pub/databases/gwas/summary_statistics/GCST009001-GCST010000/GCST009978/md5sum.txt</td>
    </tr>
    <tr>
      <td>0</td>
      <td>GWAS_results_Depression.txt</td>
      <td>2020-05-20 00:00</td>
      <td>323M</td>
      <td>NaN</td>
      <td>http://ftp.ebi.ac.uk/pub/databases/gwas/summary_statistics/GCST009001-GCST010000/GCST009978/GWAS_results_Depression.txt</td>
    </tr>
  </tbody>
</table><h2 class='text-muted'>Processed file</h2><p>Processing file: GWAS_results_Depression.txt!</p><h2 class='text-muted'>GWAS file head</h2><table border="1" class="dataframe table table-bordered">
  <thead>
    <tr style="text-align: right;">
      <th>unnamed: 0</th>
      <th>chr</th>
      <th>unnamed: 2</th>
      <th>unnamed: 3</th>
      <th>unnamed: 4</th>
      <th>unnamed: 5</th>
      <th>unnamed: 6</th>
      <th>unnamed: 7</th>
      <th>unnamed: 8</th>
      <th>unnamed: 9</th>
      <th>unnamed: 10</th>
      <th>unnamed: 11</th>
      <th>unnamed: 12</th>
      <th>unnamed: 13</th>
      <th>unnamed: 14</th>
      <th>unnamed: 15</th>
      <th>unnamed: 16</th>
      <th>unnamed: 17</th>
      <th>unnamed: 18</th>
      <th>unnamed: 19</th>
      <th>snp</th>
      <th>unnamed: 21</th>
      <th>unnamed: 22</th>
      <th>unnamed: 23</th>
      <th>unnamed: 24</th>
      <th>unnamed: 25</th>
      <th>unnamed: 26</th>
      <th>unnamed: 27</th>
      <th>unnamed: 28</th>
      <th>bp</th>
      <th>unnamed: 30</th>
      <th>unnamed: 31</th>
      <th>a1</th>
      <th>unnamed: 33</th>
      <th>unnamed: 34</th>
      <th>unnamed: 35</th>
      <th>unnamed: 36</th>
      <th>unnamed: 37</th>
      <th>unnamed: 38</th>
      <th>test</th>
      <th>unnamed: 40</th>
      <th>unnamed: 41</th>
      <th>unnamed: 42</th>
      <th>nmiss</th>
      <th>unnamed: 44</th>
      <th>unnamed: 45</th>
      <th>unnamed: 46</th>
      <th>unnamed: 47</th>
      <th>unnamed: 48</th>
      <th>unnamed: 49</th>
      <th>unnamed: 50</th>
      <th>unnamed: 51</th>
      <th>or</th>
      <th>unnamed: 53</th>
      <th>unnamed: 54</th>
      <th>unnamed: 55</th>
      <th>unnamed: 56</th>
      <th>unnamed: 57</th>
      <th>unnamed: 58</th>
      <th>unnamed: 59</th>
      <th>unnamed: 60</th>
      <th>stat</th>
      <th>unnamed: 62</th>
      <th>unnamed: 63</th>
      <th>unnamed: 64</th>
      <th>unnamed: 65</th>
      <th>unnamed: 66</th>
      <th>unnamed: 67</th>
      <th>unnamed: 68</th>
      <th>unnamed: 69</th>
      <th>unnamed: 70</th>
      <th>unnamed: 71</th>
      <th>unnamed: 72</th>
      <th>p</th>
      <th>unnamed: 74</th>
    </tr>
  </thead>
  <tbody>
    <tr>
      <td>NaN</td>
      <td>NaN</td>
      <td>13</td>
      <td>NaN</td>
      <td>NaN</td>
      <td>NaN</td>
      <td>NaN</td>
      <td>NaN</td>
      <td>NaN</td>
      <td>NaN</td>
      <td>NaN</td>
      <td>NaN</td>
      <td>NaN</td>
      <td>NaN</td>
      <td>NaN</td>
      <td>rs2162231</td>
      <td>NaN</td>
      <td>103619302.0</td>
      <td>NaN</td>
      <td>NaN</td>
      <td>NaN</td>
      <td>C</td>
      <td>NaN</td>
      <td>NaN</td>
      <td>NaN</td>
      <td>NaN</td>
      <td>NaN</td>
      <td>NaN</td>
      <td>NaN</td>
      <td>ADD</td>
      <td>NaN</td>
      <td>NaN</td>
      <td>NaN</td>
      <td>NaN</td>
      <td>3397.0</td>
      <td>NaN</td>
      <td>NaN</td>
      <td>NaN</td>
      <td>NaN</td>
      <td>NaN</td>
      <td>1.306</td>
      <td>NaN</td>
      <td>NaN</td>
      <td>NaN</td>
      <td>NaN</td>
      <td>NaN</td>
      <td>NaN</td>
      <td>NaN</td>
      <td>4.707</td>
      <td>NaN</td>
      <td>NaN</td>
      <td>NaN</td>
      <td>NaN</td>
      <td>0.000003</td>
      <td>NaN</td>
      <td>NaN</td>
      <td>NaN</td>
      <td>NaN</td>
      <td>NaN</td>
      <td>NaN</td>
      <td>NaN</td>
      <td>NaN</td>
      <td>NaN</td>
      <td>NaN</td>
      <td>NaN</td>
      <td>NaN</td>
      <td>NaN</td>
      <td>NaN</td>
      <td>NaN</td>
      <td>NaN</td>
      <td>NaN</td>
      <td>NaN</td>
      <td>NaN</td>
      <td>NaN</td>
      <td>NaN</td>
    </tr>
    <tr>
      <td>NaN</td>
      <td>NaN</td>
      <td>13</td>
      <td>NaN</td>
      <td>NaN</td>
      <td>NaN</td>
      <td>NaN</td>
      <td>NaN</td>
      <td>NaN</td>
      <td>NaN</td>
      <td>NaN</td>
      <td>NaN</td>
      <td>NaN</td>
      <td>NaN</td>
      <td>rs11374334</td>
      <td>NaN</td>
      <td>103613211</td>
      <td>NaN</td>
      <td>NaN</td>
      <td>NaN</td>
      <td>C</td>
      <td>NaN</td>
      <td>NaN</td>
      <td>NaN</td>
      <td>NaN</td>
      <td>NaN</td>
      <td>NaN</td>
      <td>NaN</td>
      <td>ADD</td>
      <td>NaN</td>
      <td>NaN</td>
      <td>NaN</td>
      <td>NaN</td>
      <td>3404.0</td>
      <td>NaN</td>
      <td>NaN</td>
      <td>NaN</td>
      <td>NaN</td>
      <td>NaN</td>
      <td>1.301</td>
      <td>NaN</td>
      <td>NaN</td>
      <td>NaN</td>
      <td>NaN</td>
      <td>NaN</td>
      <td>NaN</td>
      <td>NaN</td>
      <td>4.545</td>
      <td>NaN</td>
      <td>NaN</td>
      <td>NaN</td>
      <td>0.000006</td>
      <td>NaN</td>
      <td>NaN</td>
      <td>NaN</td>
      <td>NaN</td>
      <td>NaN</td>
      <td>NaN</td>
      <td>NaN</td>
      <td>NaN</td>
      <td>NaN</td>
      <td>NaN</td>
      <td>NaN</td>
      <td>NaN</td>
      <td>NaN</td>
      <td>NaN</td>
      <td>NaN</td>
      <td>NaN</td>
      <td>NaN</td>
      <td>NaN</td>
      <td>NaN</td>
      <td>NaN</td>
      <td>NaN</td>
      <td>NaN</td>
      <td>NaN</td>
    </tr>
  </tbody>
</table></div><div class='container mt-4 bg-light p-4 text-center'><h2 class='text-muted'>Original GWAS file headers</h2><table border="1" class="dataframe table table-bordered">
  <thead>
    <tr style="text-align: right;">
      <th>unnamed: 0</th>
      <th>chr</th>
      <th>unnamed: 2</th>
      <th>unnamed: 3</th>
      <th>unnamed: 4</th>
      <th>unnamed: 5</th>
      <th>unnamed: 6</th>
      <th>unnamed: 7</th>
      <th>unnamed: 8</th>
      <th>unnamed: 9</th>
      <th>unnamed: 10</th>
      <th>unnamed: 11</th>
      <th>unnamed: 12</th>
      <th>unnamed: 13</th>
      <th>unnamed: 14</th>
      <th>unnamed: 15</th>
      <th>unnamed: 16</th>
      <th>unnamed: 17</th>
      <th>unnamed: 18</th>
      <th>unnamed: 19</th>
      <th>snp</th>
      <th>unnamed: 21</th>
      <th>unnamed: 22</th>
      <th>unnamed: 23</th>
      <th>unnamed: 24</th>
      <th>unnamed: 25</th>
      <th>unnamed: 26</th>
      <th>unnamed: 27</th>
      <th>unnamed: 28</th>
      <th>bp</th>
      <th>unnamed: 30</th>
      <th>unnamed: 31</th>
      <th>a1</th>
      <th>unnamed: 33</th>
      <th>unnamed: 34</th>
      <th>unnamed: 35</th>
      <th>unnamed: 36</th>
      <th>unnamed: 37</th>
      <th>unnamed: 38</th>
      <th>test</th>
      <th>unnamed: 40</th>
      <th>unnamed: 41</th>
      <th>unnamed: 42</th>
      <th>nmiss</th>
      <th>unnamed: 44</th>
      <th>unnamed: 45</th>
      <th>unnamed: 46</th>
      <th>unnamed: 47</th>
      <th>unnamed: 48</th>
      <th>unnamed: 49</th>
      <th>unnamed: 50</th>
      <th>unnamed: 51</th>
      <th>or</th>
      <th>unnamed: 53</th>
      <th>unnamed: 54</th>
      <th>unnamed: 55</th>
      <th>unnamed: 56</th>
      <th>unnamed: 57</th>
      <th>unnamed: 58</th>
      <th>unnamed: 59</th>
      <th>unnamed: 60</th>
      <th>stat</th>
      <th>unnamed: 62</th>
      <th>unnamed: 63</th>
      <th>unnamed: 64</th>
      <th>unnamed: 65</th>
      <th>unnamed: 66</th>
      <th>unnamed: 67</th>
      <th>unnamed: 68</th>
      <th>unnamed: 69</th>
      <th>unnamed: 70</th>
      <th>unnamed: 71</th>
      <th>unnamed: 72</th>
      <th>p</th>
      <th>unnamed: 74</th>
    </tr>
  </thead>
  <tbody>
    <tr>
      <td>NaN</td>
      <td>NaN</td>
      <td>13</td>
      <td>NaN</td>
      <td>NaN</td>
      <td>NaN</td>
      <td>NaN</td>
      <td>NaN</td>
      <td>NaN</td>
      <td>NaN</td>
      <td>NaN</td>
      <td>NaN</td>
      <td>NaN</td>
      <td>NaN</td>
      <td>NaN</td>
      <td>rs2162231</td>
      <td>NaN</td>
      <td>103619302.0</td>
      <td>NaN</td>
      <td>NaN</td>
      <td>NaN</td>
      <td>C</td>
      <td>NaN</td>
      <td>NaN</td>
      <td>NaN</td>
      <td>NaN</td>
      <td>NaN</td>
      <td>NaN</td>
      <td>NaN</td>
      <td>ADD</td>
      <td>NaN</td>
      <td>NaN</td>
      <td>NaN</td>
      <td>NaN</td>
      <td>3397.0</td>
      <td>NaN</td>
      <td>NaN</td>
      <td>NaN</td>
      <td>NaN</td>
      <td>NaN</td>
      <td>1.306</td>
      <td>NaN</td>
      <td>NaN</td>
      <td>NaN</td>
      <td>NaN</td>
      <td>NaN</td>
      <td>NaN</td>
      <td>NaN</td>
      <td>4.707</td>
      <td>NaN</td>
      <td>NaN</td>
      <td>NaN</td>
      <td>NaN</td>
      <td>0.000003</td>
      <td>NaN</td>
      <td>NaN</td>
      <td>NaN</td>
      <td>NaN</td>
      <td>NaN</td>
      <td>NaN</td>
      <td>NaN</td>
      <td>NaN</td>
      <td>NaN</td>
      <td>NaN</td>
      <td>NaN</td>
      <td>NaN</td>
      <td>NaN</td>
      <td>NaN</td>
      <td>NaN</td>
      <td>NaN</td>
      <td>NaN</td>
      <td>NaN</td>
      <td>NaN</td>
      <td>NaN</td>
      <td>NaN</td>
    </tr>
    <tr>
      <td>NaN</td>
      <td>NaN</td>
      <td>13</td>
      <td>NaN</td>
      <td>NaN</td>
      <td>NaN</td>
      <td>NaN</td>
      <td>NaN</td>
      <td>NaN</td>
      <td>NaN</td>
      <td>NaN</td>
      <td>NaN</td>
      <td>NaN</td>
      <td>NaN</td>
      <td>rs11374334</td>
      <td>NaN</td>
      <td>103613211</td>
      <td>NaN</td>
      <td>NaN</td>
      <td>NaN</td>
      <td>C</td>
      <td>NaN</td>
      <td>NaN</td>
      <td>NaN</td>
      <td>NaN</td>
      <td>NaN</td>
      <td>NaN</td>
      <td>NaN</td>
      <td>ADD</td>
      <td>NaN</td>
      <td>NaN</td>
      <td>NaN</td>
      <td>NaN</td>
      <td>3404.0</td>
      <td>NaN</td>
      <td>NaN</td>
      <td>NaN</td>
      <td>NaN</td>
      <td>NaN</td>
      <td>1.301</td>
      <td>NaN</td>
      <td>NaN</td>
      <td>NaN</td>
      <td>NaN</td>
      <td>NaN</td>
      <td>NaN</td>
      <td>NaN</td>
      <td>4.545</td>
      <td>NaN</td>
      <td>NaN</td>
      <td>NaN</td>
      <td>0.000006</td>
      <td>NaN</td>
      <td>NaN</td>
      <td>NaN</td>
      <td>NaN</td>
      <td>NaN</td>
      <td>NaN</td>
      <td>NaN</td>
      <td>NaN</td>
      <td>NaN</td>
      <td>NaN</td>
      <td>NaN</td>
      <td>NaN</td>
      <td>NaN</td>
      <td>NaN</td>
      <td>NaN</td>
      <td>NaN</td>
      <td>NaN</td>
      <td>NaN</td>
      <td>NaN</td>
      <td>NaN</td>
      <td>NaN</td>
      <td>NaN</td>
      <td>NaN</td>
    </tr>
  </tbody>
</table><p style='font-weight: bold; color: blue;'>CHR -> ['chr']</p><p style='font-weight: bold; color: blue;'>SNP -> ['snp']</p><p style='font-weight: bold; color: blue;'>BP -> ['bp']</p><p style='font-weight: bold; color: blue;'>A1 -> ['a1']</p><p style='font-weight: bold; color: blue;'>A2 -> ['a1']</p><p style='font-weight: bold; color: blue;'>N -> []</p><p style='font-weight: bold; color: blue;'>P -> ['p']</p><p style='font-weight: bold; color: blue;'>INFO -> []</p><p style='font-weight: bold; color: blue;'>MAF -> []</p><p style='font-weight: bold; color: blue;'>BETA -> []</p><p style='font-weight: bold; color: blue;'>SE -> []</p><p style='font-weight: bold; color: blue;'>OR -> ['or']</p><p style='font-weight: bold; color: blue;'>Z -> []</p><p style='font-weight: bold; color: blue;'>D -> []</p><h2 class='text-muted'>GWAS columns</h2>unnamed: 0, chr, unnamed: 2, unnamed: 3, unnamed: 4, unnamed: 5, unnamed: 6, unnamed: 7, unnamed: 8, unnamed: 9, unnamed: 10, unnamed: 11, unnamed: 12, unnamed: 13, unnamed: 14, unnamed: 15, unnamed: 16, unnamed: 17, unnamed: 18, unnamed: 19, snp, unnamed: 21, unnamed: 22, unnamed: 23, unnamed: 24, unnamed: 25, unnamed: 26, unnamed: 27, unnamed: 28, bp, unnamed: 30, unnamed: 31, a1, unnamed: 33, unnamed: 34, unnamed: 35, unnamed: 36, unnamed: 37, unnamed: 38, test, unnamed: 40, unnamed: 41, unnamed: 42, nmiss, unnamed: 44, unnamed: 45, unnamed: 46, unnamed: 47, unnamed: 48, unnamed: 49, unnamed: 50, unnamed: 51, or, unnamed: 53, unnamed: 54, unnamed: 55, unnamed: 56, unnamed: 57, unnamed: 58, unnamed: 59, unnamed: 60, stat, unnamed: 62, unnamed: 63, unnamed: 64, unnamed: 65, unnamed: 66, unnamed: 67, unnamed: 68, unnamed: 69, unnamed: 70, unnamed: 71, unnamed: 72, p, unnamed: 74<h2 class='text-muted'>Identified columns</h2>chr, snp, bp, a1, a1, p, or<h2 class='text-muted'>Unidentified Columns headers</h2>unnamed: 12, unnamed: 65, unnamed: 27, unnamed: 21, unnamed: 49, unnamed: 35, unnamed: 6, unnamed: 16, unnamed: 60, unnamed: 67, unnamed: 7, unnamed: 44, unnamed: 3, unnamed: 40, unnamed: 5, unnamed: 53, unnamed: 9, unnamed: 23, unnamed: 48, unnamed: 24, unnamed: 33, unnamed: 56, unnamed: 4, unnamed: 18, unnamed: 36, unnamed: 71, nmiss, unnamed: 45, unnamed: 66, unnamed: 10, unnamed: 58, unnamed: 26, stat, unnamed: 41, unnamed: 55, unnamed: 70, unnamed: 25, unnamed: 64, unnamed: 72, unnamed: 28, unnamed: 14, unnamed: 22, unnamed: 17, unnamed: 30, unnamed: 34, unnamed: 42, unnamed: 46, unnamed: 63, unnamed: 51, unnamed: 59, unnamed: 54, test, unnamed: 8, unnamed: 62, unnamed: 50, unnamed: 74, unnamed: 47, unnamed: 13, unnamed: 2, unnamed: 31, unnamed: 57, unnamed: 19, unnamed: 37, unnamed: 68, unnamed: 0, unnamed: 11, unnamed: 69, unnamed: 38, unnamed: 15<h2 class='text-muted'>PMID</h2>31926482<h2 class='text-muted'>DOI</h2>10.1038/s41386-020-0603-5<div class='bibtex-container'><h2 class='text-muted'>BibTeX</h2><div style='font-family: monospace; white-space: pre-wrap;'> @article{Shen_2020, title={Polygenic prediction and GWAS of depression, PTSD, and suicidal ideation/self-harm in a Peruvian cohort}, volume={45}, ISSN={1740-634X}, url={http://dx.doi.org/10.1038/s41386-020-0603-5}, DOI={10.1038/s41386-020-0603-5}, number={10}, journal={Neuropsychopharmacology}, publisher={Springer Science and Business Media LLC}, author={Shen, Hanyang and Gelaye, Bizu and Huang, Hailiang and Rondon, Marta B. and Sanchez, Sixto and Duncan, Laramie E.}, year={2020}, month=jan, pages={1595â€“1602} }
</div></div><br><br><br><br><br><br></div><div class='container mt-4 bg-light p-4 text-center'><h1 class='text-primary'>Phenotype: depression</h1><p class='text-muted'><strong>URL:</strong> http://ftp.ebi.ac.uk/pub/databases/gwas/summary_statistics/GCST90018001-GCST90019000/GCST90018833</p><table border="1" class="dataframe table table-bordered">
  <thead>
    <tr style="text-align: right;">
      <th>firstAuthor</th>
      <th>accessionId</th>
      <th>publicationDate</th>
      <th>journal</th>
      <th>title</th>
      <th>reportedTrait</th>
      <th>efoTraits</th>
      <th>bgTraits</th>
      <th>discoverySampleAncestry</th>
      <th>replicationSampleAncestry</th>
      <th>associationCount</th>
      <th>summaryStatistics</th>
      <th>ssApiFlag</th>
      <th>agreedToCc0</th>
      <th>genotypingTechnologies</th>
      <th>pubmedId</th>
      <th>initialSampleDescription</th>
      <th>replicateSampleDescription</th>
      <th>UKB_FIELD</th>
      <th>FuzzySimilarity_depression</th>
      <th>SearchPhenotype</th>
    </tr>
  </thead>
  <tbody>
    <tr>
      <td>Sakaue S</td>
      <td>GCST90018833</td>
      <td>9/30/2021</td>
      <td>Nat Genet</td>
      <td>A cross-population atlas of genetic associations for 220 human phenotypes.</td>
      <td>depression</td>
      <td>unipolar depression</td>
      <td>-</td>
      <td>449414 european,178630 east asian</td>
      <td>NaN</td>
      <td>1</td>
      <td>http://ftp.ebi.ac.uk/pub/databases/gwas/summary_statistics/GCST90018001-GCST90019000/GCST90018833</td>
      <td>no</td>
      <td>yes</td>
      <td>Genome-wide genotyping array</td>
      <td>34594039</td>
      <td>13,559 European ancestry cases,435,855 European ancestry controls, 836 East Asian ancestry cases,177,794 East Asian ancestry controls</td>
      <td>NaN</td>
      <td>-</td>
      <td>100</td>
      <td>depression</td>
    </tr>
  </tbody>
</table><h2 class='text-muted'>Information file</h2><table border="1" class="dataframe table table-bordered">
  <thead>
    <tr style="text-align: right;">
      <th>Unnamed: 0</th>
      <th>Name</th>
      <th>Last modified</th>
      <th>Size</th>
      <th>Description</th>
      <th>URL</th>
    </tr>
  </thead>
  <tbody>
    <tr>
      <td>2</td>
      <td>README.txt</td>
      <td>2021-12-16 01:22</td>
      <td>177</td>
      <td>NaN</td>
      <td>http://ftp.ebi.ac.uk/pub/databases/gwas/summary_statistics/GCST90018001-GCST90019000/GCST90018833/README.txt</td>
    </tr>
    <tr>
      <td>3</td>
      <td>harmonised/</td>
      <td>2024-02-02 15:17</td>
      <td>-</td>
      <td>NaN</td>
      <td>http://ftp.ebi.ac.uk/pub/databases/gwas/summary_statistics/GCST90018001-GCST90019000/GCST90018833/harmonised/</td>
    </tr>
    <tr>
      <td>4</td>
      <td>md5sum.txt</td>
      <td>2021-12-16 01:22</td>
      <td>109</td>
      <td>NaN</td>
      <td>http://ftp.ebi.ac.uk/pub/databases/gwas/summary_statistics/GCST90018001-GCST90019000/GCST90018833/md5sum.txt</td>
    </tr>
    <tr>
      <td>1</td>
      <td>GCST90018833_buildGRCh37.tsv.gz-meta.yaml</td>
      <td>2024-01-04 00:06</td>
      <td>1.1K</td>
      <td>NaN</td>
      <td>http://ftp.ebi.ac.uk/pub/databases/gwas/summary_statistics/GCST90018001-GCST90019000/GCST90018833/GCST90018833_buildGRCh37.tsv.gz-meta.yaml</td>
    </tr>
    <tr>
      <td>0</td>
      <td>GCST90018833_buildGRCh37.tsv.gz</td>
      <td>2021-12-16 01:22</td>
      <td>505M</td>
      <td>NaN</td>
      <td>http://ftp.ebi.ac.uk/pub/databases/gwas/summary_statistics/GCST90018001-GCST90019000/GCST90018833/GCST90018833_buildGRCh37.tsv.gz</td>
    </tr>
  </tbody>
</table><h2 class='text-muted'>Processed file</h2><p>Processing file: GCST90018833_buildGRCh37.tsv.gz!</p><h2 class='text-muted'>GWAS file head</h2><table border="1" class="dataframe table table-bordered">
  <thead>
    <tr style="text-align: right;">
      <th>chromosome</th>
      <th>base_pair_location</th>
      <th>effect_allele</th>
      <th>other_allele</th>
      <th>effect_allele_frequency</th>
      <th>beta</th>
      <th>standard_error</th>
      <th>p_value</th>
      <th>variant_id</th>
    </tr>
  </thead>
  <tbody>
    <tr>
      <td>1</td>
      <td>100000012</td>
      <td>T</td>
      <td>G</td>
      <td>0.284057</td>
      <td>0.0351</td>
      <td>0.0146</td>
      <td>0.01635</td>
      <td>NaN</td>
    </tr>
    <tr>
      <td>1</td>
      <td>10000006</td>
      <td>A</td>
      <td>G</td>
      <td>0.005740</td>
      <td>0.0419</td>
      <td>0.0787</td>
      <td>0.59430</td>
      <td>NaN</td>
    </tr>
  </tbody>
</table></div><div class='container mt-4 bg-light p-4 text-center'><h2 class='text-muted'>Original GWAS file headers</h2><table border="1" class="dataframe table table-bordered">
  <thead>
    <tr style="text-align: right;">
      <th>chromosome</th>
      <th>base_pair_location</th>
      <th>effect_allele</th>
      <th>other_allele</th>
      <th>effect_allele_frequency</th>
      <th>beta</th>
      <th>standard_error</th>
      <th>p_value</th>
      <th>variant_id</th>
    </tr>
  </thead>
  <tbody>
    <tr>
      <td>1</td>
      <td>100000012</td>
      <td>T</td>
      <td>G</td>
      <td>0.284057</td>
      <td>0.0351</td>
      <td>0.0146</td>
      <td>0.01635</td>
      <td>NaN</td>
    </tr>
    <tr>
      <td>1</td>
      <td>10000006</td>
      <td>A</td>
      <td>G</td>
      <td>0.005740</td>
      <td>0.0419</td>
      <td>0.0787</td>
      <td>0.59430</td>
      <td>NaN</td>
    </tr>
  </tbody>
</table><p style='font-weight: bold; color: blue;'>CHR -> ['chromosome']</p><p style='font-weight: bold; color: blue;'>SNP -> ['variant_id']</p><p style='font-weight: bold; color: blue;'>BP -> ['base_pair_location']</p><p style='font-weight: bold; color: blue;'>A1 -> ['effect_allele']</p><p style='font-weight: bold; color: blue;'>A2 -> ['other_allele']</p><p style='font-weight: bold; color: blue;'>N -> []</p><p style='font-weight: bold; color: blue;'>P -> ['p_value']</p><p style='font-weight: bold; color: blue;'>INFO -> []</p><p style='font-weight: bold; color: blue;'>MAF -> ['effect_allele_frequency']</p><p style='font-weight: bold; color: blue;'>BETA -> ['beta']</p><p style='font-weight: bold; color: blue;'>SE -> ['standard_error']</p><p style='font-weight: bold; color: blue;'>OR -> []</p><p style='font-weight: bold; color: blue;'>Z -> []</p><p style='font-weight: bold; color: blue;'>D -> []</p><h2 class='text-muted'>GWAS columns</h2>chromosome, base_pair_location, effect_allele, other_allele, effect_allele_frequency, beta, standard_error, p_value, variant_id<h2 class='text-muted'>Identified columns</h2>chromosome, variant_id, base_pair_location, effect_allele, other_allele, p_value, effect_allele_frequency, beta, standard_error<h2 class='text-muted'>Unidentified Columns headers</h2><h2 class='text-muted'>PMID</h2>34594039<h2 class='text-muted'>DOI</h2>Could not find DOI!<div class='bibtex-container'><h2 class='text-muted'>BibTeX</h2><div style='font-family: monospace; white-space: pre-wrap;'></div></div><br><br><br><br><br><br></div><div class='container mt-4 bg-light p-4 text-center'><h1 class='text-primary'>Phenotype: depression</h1><p class='text-muted'><strong>URL:</strong> http://ftp.ebi.ac.uk/pub/databases/gwas/summary_statistics/GCST90014001-GCST90015000/GCST90014437</p><table border="1" class="dataframe table table-bordered">
  <thead>
    <tr style="text-align: right;">
      <th>firstAuthor</th>
      <th>accessionId</th>
      <th>publicationDate</th>
      <th>journal</th>
      <th>title</th>
      <th>reportedTrait</th>
      <th>efoTraits</th>
      <th>bgTraits</th>
      <th>discoverySampleAncestry</th>
      <th>replicationSampleAncestry</th>
      <th>associationCount</th>
      <th>summaryStatistics</th>
      <th>ssApiFlag</th>
      <th>agreedToCc0</th>
      <th>genotypingTechnologies</th>
      <th>pubmedId</th>
      <th>initialSampleDescription</th>
      <th>replicateSampleDescription</th>
      <th>UKB_FIELD</th>
      <th>FuzzySimilarity_depression</th>
      <th>SearchPhenotype</th>
    </tr>
  </thead>
  <tbody>
    <tr>
      <td>Glanville KP</td>
      <td>GCST90014437</td>
      <td>6/1/2021</td>
      <td>Biol Psychiatry Glob Open Sci</td>
      <td>Investigating Pleiotropy Between Depression and Autoimmune Diseases Using the UK Biobank.</td>
      <td>depression</td>
      <td>unipolar depression</td>
      <td>-</td>
      <td>247177 european</td>
      <td>NaN</td>
      <td>0</td>
      <td>http://ftp.ebi.ac.uk/pub/databases/gwas/summary_statistics/GCST90014001-GCST90015000/GCST90014437</td>
      <td>no</td>
      <td>yes</td>
      <td>Genome-wide genotyping array</td>
      <td>34278373</td>
      <td>14,625 European ancestry cases,232,552 European ancestry controls</td>
      <td>NaN</td>
      <td>-</td>
      <td>100</td>
      <td>depression</td>
    </tr>
  </tbody>
</table><h2 class='text-muted'>Information file</h2><table border="1" class="dataframe table table-bordered">
  <thead>
    <tr style="text-align: right;">
      <th>Unnamed: 0</th>
      <th>Name</th>
      <th>Last modified</th>
      <th>Size</th>
      <th>Description</th>
      <th>URL</th>
    </tr>
  </thead>
  <tbody>
    <tr>
      <td>1</td>
      <td>GCST90014437_buildGRCh37.tsv-meta.yaml</td>
      <td>2024-01-04 00:30</td>
      <td>667</td>
      <td>NaN</td>
      <td>http://ftp.ebi.ac.uk/pub/databases/gwas/summary_statistics/GCST90014001-GCST90015000/GCST90014437/GCST90014437_buildGRCh37.tsv-meta.yaml</td>
    </tr>
    <tr>
      <td>2</td>
      <td>md5sum.txt</td>
      <td>2021-04-15 00:04</td>
      <td>62</td>
      <td>NaN</td>
      <td>http://ftp.ebi.ac.uk/pub/databases/gwas/summary_statistics/GCST90014001-GCST90015000/GCST90014437/md5sum.txt</td>
    </tr>
    <tr>
      <td>0</td>
      <td>GCST90014437_buildGRCh37.tsv</td>
      <td>2021-04-15 00:04</td>
      <td>106M</td>
      <td>NaN</td>
      <td>http://ftp.ebi.ac.uk/pub/databases/gwas/summary_statistics/GCST90014001-GCST90015000/GCST90014437/GCST90014437_buildGRCh37.tsv</td>
    </tr>
  </tbody>
</table><h2 class='text-muted'>Processed file</h2><p>Processing file: GCST90014437_buildGRCh37.tsv.1!</p><h2 class='text-muted'>GWAS file head</h2><table border="1" class="dataframe table table-bordered">
  <thead>
    <tr style="text-align: right;">
      <th>chromosome</th>
      <th>variant_id</th>
      <th>base_pair_location</th>
      <th>other_allele</th>
      <th>effect_allele</th>
      <th>effect_allele_frequency</th>
      <th>info</th>
      <th>beta</th>
      <th>standard_error</th>
      <th>t</th>
      <th>-log10p</th>
      <th>p_value</th>
    </tr>
  </thead>
  <tbody>
    <tr>
      <td>10</td>
      <td>rs12255619</td>
      <td>98481</td>
      <td>A</td>
      <td>C</td>
      <td>0.084928</td>
      <td>0.999328</td>
      <td>-0.002711</td>
      <td>0.001994</td>
      <td>-1.3599</td>
      <td>0.75976</td>
      <td>0.173876</td>
    </tr>
    <tr>
      <td>10</td>
      <td>rs11252546</td>
      <td>104427</td>
      <td>T</td>
      <td>C</td>
      <td>0.352820</td>
      <td>0.998610</td>
      <td>-0.006143</td>
      <td>0.002007</td>
      <td>-3.0604</td>
      <td>2.65540</td>
      <td>0.002211</td>
    </tr>
  </tbody>
</table></div><div class='container mt-4 bg-light p-4 text-center'><h2 class='text-muted'>Original GWAS file headers</h2><table border="1" class="dataframe table table-bordered">
  <thead>
    <tr style="text-align: right;">
      <th>chromosome</th>
      <th>variant_id</th>
      <th>base_pair_location</th>
      <th>other_allele</th>
      <th>effect_allele</th>
      <th>effect_allele_frequency</th>
      <th>info</th>
      <th>beta</th>
      <th>standard_error</th>
      <th>t</th>
      <th>-log10p</th>
      <th>p_value</th>
    </tr>
  </thead>
  <tbody>
    <tr>
      <td>10</td>
      <td>rs12255619</td>
      <td>98481</td>
      <td>A</td>
      <td>C</td>
      <td>0.084928</td>
      <td>0.999328</td>
      <td>-0.002711</td>
      <td>0.001994</td>
      <td>-1.3599</td>
      <td>0.75976</td>
      <td>0.173876</td>
    </tr>
    <tr>
      <td>10</td>
      <td>rs11252546</td>
      <td>104427</td>
      <td>T</td>
      <td>C</td>
      <td>0.352820</td>
      <td>0.998610</td>
      <td>-0.006143</td>
      <td>0.002007</td>
      <td>-3.0604</td>
      <td>2.65540</td>
      <td>0.002211</td>
    </tr>
  </tbody>
</table><p style='font-weight: bold; color: blue;'>CHR -> ['chromosome']</p><p style='font-weight: bold; color: blue;'>SNP -> ['variant_id']</p><p style='font-weight: bold; color: blue;'>BP -> ['base_pair_location']</p><p style='font-weight: bold; color: blue;'>A1 -> ['effect_allele']</p><p style='font-weight: bold; color: blue;'>A2 -> ['other_allele']</p><p style='font-weight: bold; color: blue;'>N -> []</p><p style='font-weight: bold; color: blue;'>P -> ['p_value']</p><p style='font-weight: bold; color: blue;'>INFO -> ['info']</p><p style='font-weight: bold; color: blue;'>MAF -> ['effect_allele_frequency']</p><p style='font-weight: bold; color: blue;'>BETA -> ['beta']</p><p style='font-weight: bold; color: blue;'>SE -> ['standard_error']</p><p style='font-weight: bold; color: blue;'>OR -> []</p><p style='font-weight: bold; color: blue;'>Z -> []</p><p style='font-weight: bold; color: blue;'>D -> []</p><h2 class='text-muted'>GWAS columns</h2>chromosome, variant_id, base_pair_location, other_allele, effect_allele, effect_allele_frequency, info, beta, standard_error, t, -log10p, p_value<h2 class='text-muted'>Identified columns</h2>chromosome, variant_id, base_pair_location, effect_allele, other_allele, p_value, info, effect_allele_frequency, beta, standard_error<h2 class='text-muted'>Unidentified Columns headers</h2>-log10p, t<h2 class='text-muted'>PMID</h2>34278373<h2 class='text-muted'>DOI</h2>10.1016/j.bpsgos.2021.03.002<div class='bibtex-container'><h2 class='text-muted'>BibTeX</h2><div style='font-family: monospace; white-space: pre-wrap;'> @article{Glanville_2021, title={Investigating Pleiotropy Between Depression and Autoimmune Diseases Using the UK Biobank}, volume={1}, ISSN={2667-1743}, url={http://dx.doi.org/10.1016/j.bpsgos.2021.03.002}, DOI={10.1016/j.bpsgos.2021.03.002}, number={1}, journal={Biological Psychiatry Global Open Science}, publisher={Elsevier BV}, author={Glanville, Kylie P. and Coleman, Jonathan R.I. and O’Reilly, Paul F. and Galloway, James and Lewis, Cathryn M.}, year={2021}, month=jun, pages={48–58} }
</div></div><br><br><br><br><br><br></div><div class='container mt-4 bg-light p-4 text-center'><h1 class='text-primary'>Phenotype: depression</h1><p class='text-muted'><strong>URL:</strong> http://ftp.ebi.ac.uk/pub/databases/gwas/summary_statistics/GCST90014001-GCST90015000/GCST90014436</p><table border="1" class="dataframe table table-bordered">
  <thead>
    <tr style="text-align: right;">
      <th>firstAuthor</th>
      <th>accessionId</th>
      <th>publicationDate</th>
      <th>journal</th>
      <th>title</th>
      <th>reportedTrait</th>
      <th>efoTraits</th>
      <th>bgTraits</th>
      <th>discoverySampleAncestry</th>
      <th>replicationSampleAncestry</th>
      <th>associationCount</th>
      <th>summaryStatistics</th>
      <th>ssApiFlag</th>
      <th>agreedToCc0</th>
      <th>genotypingTechnologies</th>
      <th>pubmedId</th>
      <th>initialSampleDescription</th>
      <th>replicateSampleDescription</th>
      <th>UKB_FIELD</th>
      <th>FuzzySimilarity_depression</th>
      <th>SearchPhenotype</th>
    </tr>
  </thead>
  <tbody>
    <tr>
      <td>Glanville KP</td>
      <td>GCST90014436</td>
      <td>6/1/2021</td>
      <td>Biol Psychiatry Glob Open Sci</td>
      <td>Investigating Pleiotropy Between Depression and Autoimmune Diseases Using the UK Biobank.</td>
      <td>depression</td>
      <td>unipolar depression</td>
      <td>-</td>
      <td>297627 european</td>
      <td>NaN</td>
      <td>0</td>
      <td>http://ftp.ebi.ac.uk/pub/databases/gwas/summary_statistics/GCST90014001-GCST90015000/GCST90014436</td>
      <td>no</td>
      <td>yes</td>
      <td>Genome-wide genotyping array</td>
      <td>34278373</td>
      <td>65,075 European ancestry cases,232,552 European ancestry controls</td>
      <td>NaN</td>
      <td>-</td>
      <td>100</td>
      <td>depression</td>
    </tr>
  </tbody>
</table><h2 class='text-muted'>Information file</h2><table border="1" class="dataframe table table-bordered">
  <thead>
    <tr style="text-align: right;">
      <th>Unnamed: 0</th>
      <th>Name</th>
      <th>Last modified</th>
      <th>Size</th>
      <th>Description</th>
      <th>URL</th>
    </tr>
  </thead>
  <tbody>
    <tr>
      <td>1</td>
      <td>GCST90014436_buildGRCh37.tsv-meta.yaml</td>
      <td>2024-01-04 00:15</td>
      <td>667</td>
      <td>NaN</td>
      <td>http://ftp.ebi.ac.uk/pub/databases/gwas/summary_statistics/GCST90014001-GCST90015000/GCST90014436/GCST90014436_buildGRCh37.tsv-meta.yaml</td>
    </tr>
    <tr>
      <td>2</td>
      <td>md5sum.txt</td>
      <td>2021-04-15 00:04</td>
      <td>62</td>
      <td>NaN</td>
      <td>http://ftp.ebi.ac.uk/pub/databases/gwas/summary_statistics/GCST90014001-GCST90015000/GCST90014436/md5sum.txt</td>
    </tr>
    <tr>
      <td>0</td>
      <td>GCST90014436_buildGRCh37.tsv</td>
      <td>2021-04-15 00:04</td>
      <td>106M</td>
      <td>NaN</td>
      <td>http://ftp.ebi.ac.uk/pub/databases/gwas/summary_statistics/GCST90014001-GCST90015000/GCST90014436/GCST90014436_buildGRCh37.tsv</td>
    </tr>
  </tbody>
</table><h2 class='text-muted'>Processed file</h2><p>Processing file: GCST90014436_buildGRCh37.tsv.1!</p><h2 class='text-muted'>GWAS file head</h2><table border="1" class="dataframe table table-bordered">
  <thead>
    <tr style="text-align: right;">
      <th>chromosome</th>
      <th>variant_id</th>
      <th>base_pair_location</th>
      <th>other_allele</th>
      <th>effect_allele</th>
      <th>effect_allele_frequency</th>
      <th>info</th>
      <th>beta</th>
      <th>standard_error</th>
      <th>t</th>
      <th>-log10p</th>
      <th>p_value</th>
    </tr>
  </thead>
  <tbody>
    <tr>
      <td>10</td>
      <td>rs12255619</td>
      <td>98481</td>
      <td>A</td>
      <td>C</td>
      <td>0.084928</td>
      <td>0.999328</td>
      <td>-0.002076</td>
      <td>0.001819</td>
      <td>-1.1413</td>
      <td>0.5956</td>
      <td>0.253746</td>
    </tr>
    <tr>
      <td>10</td>
      <td>rs11252546</td>
      <td>104427</td>
      <td>T</td>
      <td>C</td>
      <td>0.352820</td>
      <td>0.998610</td>
      <td>-0.004704</td>
      <td>0.001830</td>
      <td>-2.5703</td>
      <td>1.9930</td>
      <td>0.010162</td>
    </tr>
  </tbody>
</table></div><div class='container mt-4 bg-light p-4 text-center'><h2 class='text-muted'>Original GWAS file headers</h2><table border="1" class="dataframe table table-bordered">
  <thead>
    <tr style="text-align: right;">
      <th>chromosome</th>
      <th>variant_id</th>
      <th>base_pair_location</th>
      <th>other_allele</th>
      <th>effect_allele</th>
      <th>effect_allele_frequency</th>
      <th>info</th>
      <th>beta</th>
      <th>standard_error</th>
      <th>t</th>
      <th>-log10p</th>
      <th>p_value</th>
    </tr>
  </thead>
  <tbody>
    <tr>
      <td>10</td>
      <td>rs12255619</td>
      <td>98481</td>
      <td>A</td>
      <td>C</td>
      <td>0.084928</td>
      <td>0.999328</td>
      <td>-0.002076</td>
      <td>0.001819</td>
      <td>-1.1413</td>
      <td>0.5956</td>
      <td>0.253746</td>
    </tr>
    <tr>
      <td>10</td>
      <td>rs11252546</td>
      <td>104427</td>
      <td>T</td>
      <td>C</td>
      <td>0.352820</td>
      <td>0.998610</td>
      <td>-0.004704</td>
      <td>0.001830</td>
      <td>-2.5703</td>
      <td>1.9930</td>
      <td>0.010162</td>
    </tr>
  </tbody>
</table><p style='font-weight: bold; color: blue;'>CHR -> ['chromosome']</p><p style='font-weight: bold; color: blue;'>SNP -> ['variant_id']</p><p style='font-weight: bold; color: blue;'>BP -> ['base_pair_location']</p><p style='font-weight: bold; color: blue;'>A1 -> ['effect_allele']</p><p style='font-weight: bold; color: blue;'>A2 -> ['other_allele']</p><p style='font-weight: bold; color: blue;'>N -> []</p><p style='font-weight: bold; color: blue;'>P -> ['p_value']</p><p style='font-weight: bold; color: blue;'>INFO -> ['info']</p><p style='font-weight: bold; color: blue;'>MAF -> ['effect_allele_frequency']</p><p style='font-weight: bold; color: blue;'>BETA -> ['beta']</p><p style='font-weight: bold; color: blue;'>SE -> ['standard_error']</p><p style='font-weight: bold; color: blue;'>OR -> []</p><p style='font-weight: bold; color: blue;'>Z -> []</p><p style='font-weight: bold; color: blue;'>D -> []</p><h2 class='text-muted'>GWAS columns</h2>chromosome, variant_id, base_pair_location, other_allele, effect_allele, effect_allele_frequency, info, beta, standard_error, t, -log10p, p_value<h2 class='text-muted'>Identified columns</h2>chromosome, variant_id, base_pair_location, effect_allele, other_allele, p_value, info, effect_allele_frequency, beta, standard_error<h2 class='text-muted'>Unidentified Columns headers</h2>-log10p, t<h2 class='text-muted'>PMID</h2>34278373<h2 class='text-muted'>DOI</h2>10.1016/j.bpsgos.2021.03.002<div class='bibtex-container'><h2 class='text-muted'>BibTeX</h2><div style='font-family: monospace; white-space: pre-wrap;'> @article{Glanville_2021, title={Investigating Pleiotropy Between Depression and Autoimmune Diseases Using the UK Biobank}, volume={1}, ISSN={2667-1743}, url={http://dx.doi.org/10.1016/j.bpsgos.2021.03.002}, DOI={10.1016/j.bpsgos.2021.03.002}, number={1}, journal={Biological Psychiatry Global Open Science}, publisher={Elsevier BV}, author={Glanville, Kylie P. and Coleman, Jonathan R.I. and O’Reilly, Paul F. and Galloway, James and Lewis, Cathryn M.}, year={2021}, month=jun, pages={48–58} }
</div></div><br><br><br><br><br><br></div><div class='container mt-4 bg-light p-4 text-center'><h1 class='text-primary'>Phenotype: depression</h1><p class='text-muted'><strong>URL:</strong> http://ftp.ebi.ac.uk/pub/databases/gwas/summary_statistics/GCST90018001-GCST90019000/GCST90018613</p><table border="1" class="dataframe table table-bordered">
  <thead>
    <tr style="text-align: right;">
      <th>firstAuthor</th>
      <th>accessionId</th>
      <th>publicationDate</th>
      <th>journal</th>
      <th>title</th>
      <th>reportedTrait</th>
      <th>efoTraits</th>
      <th>bgTraits</th>
      <th>discoverySampleAncestry</th>
      <th>replicationSampleAncestry</th>
      <th>associationCount</th>
      <th>summaryStatistics</th>
      <th>ssApiFlag</th>
      <th>agreedToCc0</th>
      <th>genotypingTechnologies</th>
      <th>pubmedId</th>
      <th>initialSampleDescription</th>
      <th>replicateSampleDescription</th>
      <th>UKB_FIELD</th>
      <th>FuzzySimilarity_depression</th>
      <th>SearchPhenotype</th>
    </tr>
  </thead>
  <tbody>
    <tr>
      <td>Sakaue S</td>
      <td>GCST90018613</td>
      <td>9/30/2021</td>
      <td>Nat Genet</td>
      <td>A cross-population atlas of genetic associations for 220 human phenotypes.</td>
      <td>depression</td>
      <td>unipolar depression</td>
      <td>-</td>
      <td>178630 east asian</td>
      <td>NaN</td>
      <td>0</td>
      <td>http://ftp.ebi.ac.uk/pub/databases/gwas/summary_statistics/GCST90018001-GCST90019000/GCST90018613</td>
      <td>no</td>
      <td>yes</td>
      <td>Genome-wide genotyping array</td>
      <td>34594039</td>
      <td>177,794 East Asian ancestry controls</td>
      <td>NaN</td>
      <td>-</td>
      <td>100</td>
      <td>depression</td>
    </tr>
  </tbody>
</table><h2 class='text-muted'>Information file</h2><table border="1" class="dataframe table table-bordered">
  <thead>
    <tr style="text-align: right;">
      <th>Unnamed: 0</th>
      <th>Name</th>
      <th>Last modified</th>
      <th>Size</th>
      <th>Description</th>
      <th>URL</th>
    </tr>
  </thead>
  <tbody>
    <tr>
      <td>1</td>
      <td>GCST90018613_buildGRCh37.tsv.gz-meta.yaml</td>
      <td>2024-01-04 00:35</td>
      <td>941</td>
      <td>NaN</td>
      <td>http://ftp.ebi.ac.uk/pub/databases/gwas/summary_statistics/GCST90018001-GCST90019000/GCST90018613/GCST90018613_buildGRCh37.tsv.gz-meta.yaml</td>
    </tr>
    <tr>
      <td>2</td>
      <td>README.txt</td>
      <td>2021-12-16 00:23</td>
      <td>177</td>
      <td>NaN</td>
      <td>http://ftp.ebi.ac.uk/pub/databases/gwas/summary_statistics/GCST90018001-GCST90019000/GCST90018613/README.txt</td>
    </tr>
    <tr>
      <td>3</td>
      <td>harmonised/</td>
      <td>2024-02-02 15:17</td>
      <td>-</td>
      <td>NaN</td>
      <td>http://ftp.ebi.ac.uk/pub/databases/gwas/summary_statistics/GCST90018001-GCST90019000/GCST90018613/harmonised/</td>
    </tr>
    <tr>
      <td>4</td>
      <td>md5sum.txt</td>
      <td>2021-12-16 00:23</td>
      <td>109</td>
      <td>NaN</td>
      <td>http://ftp.ebi.ac.uk/pub/databases/gwas/summary_statistics/GCST90018001-GCST90019000/GCST90018613/md5sum.txt</td>
    </tr>
    <tr>
      <td>0</td>
      <td>GCST90018613_buildGRCh37.tsv.gz</td>
      <td>2021-12-16 00:23</td>
      <td>502M</td>
      <td>NaN</td>
      <td>http://ftp.ebi.ac.uk/pub/databases/gwas/summary_statistics/GCST90018001-GCST90019000/GCST90018613/GCST90018613_buildGRCh37.tsv.gz</td>
    </tr>
  </tbody>
</table><h2 class='text-muted'>Processed file</h2><p>Processing file: GCST90018613_buildGRCh37.tsv.gz!</p><h2 class='text-muted'>GWAS file head</h2><table border="1" class="dataframe table table-bordered">
  <thead>
    <tr style="text-align: right;">
      <th>chromosome</th>
      <th>base_pair_location</th>
      <th>effect_allele</th>
      <th>other_allele</th>
      <th>effect_allele_frequency</th>
      <th>beta</th>
      <th>standard_error</th>
      <th>p_value</th>
      <th>variant_id</th>
    </tr>
  </thead>
  <tbody>
    <tr>
      <td>1</td>
      <td>751343</td>
      <td>A</td>
      <td>T</td>
      <td>0.147709</td>
      <td>0.036409</td>
      <td>0.072554</td>
      <td>0.615792</td>
      <td>NaN</td>
    </tr>
    <tr>
      <td>1</td>
      <td>751756</td>
      <td>C</td>
      <td>T</td>
      <td>0.147245</td>
      <td>0.036393</td>
      <td>0.072384</td>
      <td>0.615117</td>
      <td>NaN</td>
    </tr>
  </tbody>
</table></div><div class='container mt-4 bg-light p-4 text-center'><h2 class='text-muted'>Original GWAS file headers</h2><table border="1" class="dataframe table table-bordered">
  <thead>
    <tr style="text-align: right;">
      <th>chromosome</th>
      <th>base_pair_location</th>
      <th>effect_allele</th>
      <th>other_allele</th>
      <th>effect_allele_frequency</th>
      <th>beta</th>
      <th>standard_error</th>
      <th>p_value</th>
      <th>variant_id</th>
    </tr>
  </thead>
  <tbody>
    <tr>
      <td>1</td>
      <td>751343</td>
      <td>A</td>
      <td>T</td>
      <td>0.147709</td>
      <td>0.036409</td>
      <td>0.072554</td>
      <td>0.615792</td>
      <td>NaN</td>
    </tr>
    <tr>
      <td>1</td>
      <td>751756</td>
      <td>C</td>
      <td>T</td>
      <td>0.147245</td>
      <td>0.036393</td>
      <td>0.072384</td>
      <td>0.615117</td>
      <td>NaN</td>
    </tr>
  </tbody>
</table><p style='font-weight: bold; color: blue;'>CHR -> ['chromosome']</p><p style='font-weight: bold; color: blue;'>SNP -> ['variant_id']</p><p style='font-weight: bold; color: blue;'>BP -> ['base_pair_location']</p><p style='font-weight: bold; color: blue;'>A1 -> ['effect_allele']</p><p style='font-weight: bold; color: blue;'>A2 -> ['other_allele']</p><p style='font-weight: bold; color: blue;'>N -> []</p><p style='font-weight: bold; color: blue;'>P -> ['p_value']</p><p style='font-weight: bold; color: blue;'>INFO -> []</p><p style='font-weight: bold; color: blue;'>MAF -> ['effect_allele_frequency']</p><p style='font-weight: bold; color: blue;'>BETA -> ['beta']</p><p style='font-weight: bold; color: blue;'>SE -> ['standard_error']</p><p style='font-weight: bold; color: blue;'>OR -> []</p><p style='font-weight: bold; color: blue;'>Z -> []</p><p style='font-weight: bold; color: blue;'>D -> []</p><h2 class='text-muted'>GWAS columns</h2>chromosome, base_pair_location, effect_allele, other_allele, effect_allele_frequency, beta, standard_error, p_value, variant_id<h2 class='text-muted'>Identified columns</h2>chromosome, variant_id, base_pair_location, effect_allele, other_allele, p_value, effect_allele_frequency, beta, standard_error<h2 class='text-muted'>Unidentified Columns headers</h2><h2 class='text-muted'>PMID</h2>34594039<h2 class='text-muted'>DOI</h2>Could not find DOI!<div class='bibtex-container'><h2 class='text-muted'>BibTeX</h2><div style='font-family: monospace; white-space: pre-wrap;'></div></div><br><br><br><br><br><br></div><div class='container mt-4 bg-light p-4 text-center'><h1 class='text-primary'>Phenotype: depression</h1><p class='text-muted'><strong>URL:</strong> http://ftp.ebi.ac.uk/pub/databases/gwas/summary_statistics/GCST005001-GCST006000/GCST005839</p><table border="1" class="dataframe table table-bordered">
  <thead>
    <tr style="text-align: right;">
      <th>firstAuthor</th>
      <th>accessionId</th>
      <th>publicationDate</th>
      <th>journal</th>
      <th>title</th>
      <th>reportedTrait</th>
      <th>efoTraits</th>
      <th>bgTraits</th>
      <th>discoverySampleAncestry</th>
      <th>replicationSampleAncestry</th>
      <th>associationCount</th>
      <th>summaryStatistics</th>
      <th>ssApiFlag</th>
      <th>agreedToCc0</th>
      <th>genotypingTechnologies</th>
      <th>pubmedId</th>
      <th>initialSampleDescription</th>
      <th>replicateSampleDescription</th>
      <th>UKB_FIELD</th>
      <th>FuzzySimilarity_depression</th>
      <th>SearchPhenotype</th>
    </tr>
  </thead>
  <tbody>
    <tr>
      <td>Wray NR</td>
      <td>GCST005839</td>
      <td>4/26/2018</td>
      <td>Nat Genet</td>
      <td>Genome-wide association analyses identify 44 risk variants and refine the genetic architecture of major depression.</td>
      <td>depression</td>
      <td>unipolar depression</td>
      <td>-</td>
      <td>480359 european</td>
      <td>NaN</td>
      <td>44</td>
      <td>http://ftp.ebi.ac.uk/pub/databases/gwas/summary_statistics/GCST005001-GCST006000/GCST005839</td>
      <td>no</td>
      <td>-</td>
      <td>Genome-wide genotyping array</td>
      <td>29700475</td>
      <td>135,458 European ancestry cases,344,901 European ancestry controls</td>
      <td>NaN</td>
      <td>-</td>
      <td>100</td>
      <td>depression</td>
    </tr>
  </tbody>
</table><h2 class='text-muted'>Information file</h2><table border="1" class="dataframe table table-bordered">
  <thead>
    <tr style="text-align: right;">
      <th>Unnamed: 0</th>
      <th>Name</th>
      <th>Last modified</th>
      <th>Size</th>
      <th>Description</th>
      <th>URL</th>
    </tr>
  </thead>
  <tbody>
    <tr>
      <td>2</td>
      <td>PGC_MDD_2018_README_third_180316.pdf</td>
      <td>2018-07-17 10:45</td>
      <td>70K</td>
      <td>NaN</td>
      <td>http://ftp.ebi.ac.uk/pub/databases/gwas/summary_statistics/GCST005001-GCST006000/GCST005839/PGC_MDD_2018_README_third_180316.pdf</td>
    </tr>
    <tr>
      <td>1</td>
      <td>PGC_MDD2018_10kSNPs_2</td>
      <td>2018-08-04 12:18</td>
      <td>677K</td>
      <td>NaN</td>
      <td>http://ftp.ebi.ac.uk/pub/databases/gwas/summary_statistics/GCST005001-GCST006000/GCST005839/PGC_MDD2018_10kSNPs_2</td>
    </tr>
    <tr>
      <td>0</td>
      <td>MDD2018_ex23andMe.gz</td>
      <td>2018-07-17 10:45</td>
      <td>447M</td>
      <td>NaN</td>
      <td>http://ftp.ebi.ac.uk/pub/databases/gwas/summary_statistics/GCST005001-GCST006000/GCST005839/MDD2018_ex23andMe.gz</td>
    </tr>
  </tbody>
</table><h2 class='text-muted'>Processed file</h2><p>Processing file: MDD2018_ex23andMe.gz!</p><h2 class='text-muted'>GWAS file head</h2><table border="1" class="dataframe table table-bordered">
  <thead>
    <tr style="text-align: right;">
      <th>chr</th>
      <th>snp</th>
      <th>bp</th>
      <th>a1</th>
      <th>a2</th>
      <th>frq_a_59851</th>
      <th>frq_u_113154</th>
      <th>info</th>
      <th>or</th>
      <th>se</th>
      <th>p</th>
      <th>ngt</th>
      <th>direction</th>
      <th>hetisqt</th>
      <th>hetdf</th>
      <th>hetpva</th>
      <th>nca</th>
      <th>nco</th>
      <th>neff</th>
    </tr>
  </thead>
  <tbody>
    <tr>
      <td>8</td>
      <td>rs62513865</td>
      <td>101592213</td>
      <td>T</td>
      <td>C</td>
      <td>0.0747</td>
      <td>0.0733</td>
      <td>0.957</td>
      <td>1.01461</td>
      <td>0.0153</td>
      <td>0.3438</td>
      <td>0</td>
      <td>---+++</td>
      <td>0.0</td>
      <td>5</td>
      <td>0.7906</td>
      <td>59851</td>
      <td>113154</td>
      <td>69115.85</td>
    </tr>
    <tr>
      <td>8</td>
      <td>rs79643588</td>
      <td>106973048</td>
      <td>A</td>
      <td>G</td>
      <td>0.0920</td>
      <td>0.0920</td>
      <td>0.999</td>
      <td>1.02122</td>
      <td>0.0136</td>
      <td>0.1231</td>
      <td>0</td>
      <td>++-+++</td>
      <td>0.0</td>
      <td>5</td>
      <td>0.6847</td>
      <td>59851</td>
      <td>113154</td>
      <td>69115.85</td>
    </tr>
  </tbody>
</table></div><div class='container mt-4 bg-light p-4 text-center'><h2 class='text-muted'>Original GWAS file headers</h2><table border="1" class="dataframe table table-bordered">
  <thead>
    <tr style="text-align: right;">
      <th>chr</th>
      <th>snp</th>
      <th>bp</th>
      <th>a1</th>
      <th>a2</th>
      <th>frq_a_59851</th>
      <th>frq_u_113154</th>
      <th>info</th>
      <th>or</th>
      <th>se</th>
      <th>p</th>
      <th>ngt</th>
      <th>direction</th>
      <th>hetisqt</th>
      <th>hetdf</th>
      <th>hetpva</th>
      <th>nca</th>
      <th>nco</th>
      <th>neff</th>
    </tr>
  </thead>
  <tbody>
    <tr>
      <td>8</td>
      <td>rs62513865</td>
      <td>101592213</td>
      <td>T</td>
      <td>C</td>
      <td>0.0747</td>
      <td>0.0733</td>
      <td>0.957</td>
      <td>1.01461</td>
      <td>0.0153</td>
      <td>0.3438</td>
      <td>0</td>
      <td>---+++</td>
      <td>0.0</td>
      <td>5</td>
      <td>0.7906</td>
      <td>59851</td>
      <td>113154</td>
      <td>69115.85</td>
    </tr>
    <tr>
      <td>8</td>
      <td>rs79643588</td>
      <td>106973048</td>
      <td>A</td>
      <td>G</td>
      <td>0.0920</td>
      <td>0.0920</td>
      <td>0.999</td>
      <td>1.02122</td>
      <td>0.0136</td>
      <td>0.1231</td>
      <td>0</td>
      <td>++-+++</td>
      <td>0.0</td>
      <td>5</td>
      <td>0.6847</td>
      <td>59851</td>
      <td>113154</td>
      <td>69115.85</td>
    </tr>
  </tbody>
</table><p style='font-weight: bold; color: blue;'>CHR -> ['chr']</p><p style='font-weight: bold; color: blue;'>SNP -> ['snp']</p><p style='font-weight: bold; color: blue;'>BP -> ['bp']</p><p style='font-weight: bold; color: blue;'>A1 -> ['a1']</p><p style='font-weight: bold; color: blue;'>A2 -> ['a1', 'a2']</p><p style='font-weight: bold; color: blue;'>N -> []</p><p style='font-weight: bold; color: blue;'>P -> ['p']</p><p style='font-weight: bold; color: blue;'>INFO -> ['info']</p><p style='font-weight: bold; color: blue;'>MAF -> []</p><p style='font-weight: bold; color: blue;'>BETA -> []</p><p style='font-weight: bold; color: blue;'>SE -> ['se']</p><p style='font-weight: bold; color: blue;'>OR -> ['or']</p><p style='font-weight: bold; color: blue;'>Z -> []</p><p style='font-weight: bold; color: blue;'>D -> ['direction']</p><h2 class='text-muted'>GWAS columns</h2>chr, snp, bp, a1, a2, frq_a_59851, frq_u_113154, info, or, se, p, ngt, direction, hetisqt, hetdf, hetpva, nca, nco, neff<h2 class='text-muted'>Identified columns</h2>chr, snp, bp, a1, a1, a2, p, info, se, or, direction<h2 class='text-muted'>Unidentified Columns headers</h2>nca, neff, nco, ngt, frq_u_113154, hetisqt, hetdf, hetpva, frq_a_59851<h2 class='text-muted'>PMID</h2>29700475<h2 class='text-muted'>DOI</h2>10.1038/s41588-018-0090-3<div class='bibtex-container'><h2 class='text-muted'>BibTeX</h2><div style='font-family: monospace; white-space: pre-wrap;'> @article{Wray_2018, title={Genome-wide association analyses identify 44 risk variants and refine the genetic architecture of major depression}, volume={50}, ISSN={1546-1718}, url={http://dx.doi.org/10.1038/s41588-018-0090-3}, DOI={10.1038/s41588-018-0090-3}, number={5}, journal={Nature Genetics}, publisher={Springer Science and Business Media LLC}, author={Wray, Naomi R. and Ripke, Stephan and Mattheisen, Manuel and Trzaskowski, Maciej and Byrne, Enda M. and Abdellaoui, Abdel and Adams, Mark J. and Agerbo, Esben and Air, Tracy M. and Andlauer, Till M. F. and Bacanu, Silviu-Alin and Bækvad-Hansen, Marie and Beekman, Aartjan F. T. and Bigdeli, Tim B. and Binder, Elisabeth B. and Blackwood, Douglas R. H. and Bryois, Julien and Buttenschøn, Henriette N. and Bybjerg-Grauholm, Jonas and Cai, Na and Castelao, Enrique and Christensen, Jane Hvarregaard and Clarke, Toni-Kim and Coleman, Jonathan I. R. and Colodro-Conde, Lucía and Couvy-Duchesne, Baptiste and Craddock, Nick and Crawford, Gregory E. and Crowley, Cheynna A. and Dashti, Hassan S. and Davies, Gail and Deary, Ian J. and Degenhardt, Franziska and Derks, Eske M. and Direk, Nese and Dolan, Conor V. and Dunn, Erin C. and Eley, Thalia C. and Eriksson, Nicholas and Escott-Price, Valentina and Kiadeh, Farnush Hassan Farhadi and Finucane, Hilary K. and Forstner, Andreas J. and Frank, Josef and Gaspar, Héléna A. and Gill, Michael and Giusti-Rodríguez, Paola and Goes, Fernando S. and Gordon, Scott D. and Grove, Jakob and Hall, Lynsey S. and Hannon, Eilis and Hansen, Christine Søholm and Hansen, Thomas F. and Herms, Stefan and Hickie, Ian B. and Hoffmann, Per and Homuth, Georg and Horn, Carsten and Hottenga, Jouke-Jan and Hougaard, David M. and Hu, Ming and Hyde, Craig L. and Ising, Marcus and Jansen, Rick and Jin, Fulai and Jorgenson, Eric and Knowles, James A. and Kohane, Isaac S. and Kraft, Julia and Kretzschmar, Warren W. and Krogh, Jesper and Kutalik, Zoltán and Lane, Jacqueline M. and Li, Yihan and Li, Yun and Lind, Penelope A. and Liu, Xiaoxiao and Lu, Leina and MacIntyre, Donald J. and MacKinnon, Dean F. and Maier, Robert M. and Maier, Wolfgang and Marchini, Jonathan and Mbarek, Hamdi and McGrath, Patrick and McGuffin, Peter and Medland, Sarah E. and Mehta, Divya and Middeldorp, Christel M. and Mihailov, Evelin and Milaneschi, Yuri and Milani, Lili and Mill, Jonathan and Mondimore, Francis M. and Montgomery, Grant W. and Mostafavi, Sara and Mullins, Niamh and Nauck, Matthias and Ng, Bernard and Nivard, Michel G. and Nyholt, Dale R. and O’Reilly, Paul F. and Oskarsson, Hogni and Owen, Michael J. and Painter, Jodie N. and Pedersen, Carsten Bøcker and Pedersen, Marianne Giørtz and Peterson, Roseann E. and Pettersson, Erik and Peyrot, Wouter J. and Pistis, Giorgio and Posthuma, Danielle and Purcell, Shaun M. and Quiroz, Jorge A. and Qvist, Per and Rice, John P. and Riley, Brien P. and Rivera, Margarita and Saeed Mirza, Saira and Saxena, Richa and Schoevers, Robert and Schulte, Eva C. and Shen, Ling and Shi, Jianxin and Shyn, Stanley I. and Sigurdsson, Engilbert and Sinnamon, Grant B. C. and Smit, Johannes H. and Smith, Daniel J. and Stefansson, Hreinn and Steinberg, Stacy and Stockmeier, Craig A. and Streit, Fabian and Strohmaier, Jana and Tansey, Katherine E. and Teismann, Henning and Teumer, Alexander and Thompson, Wesley and Thomson, Pippa A. and Thorgeirsson, Thorgeir E. and Tian, Chao and Traylor, Matthew and Treutlein, Jens and Trubetskoy, Vassily and Uitterlinden, André G. and Umbricht, Daniel and Van der Auwera, Sandra and van Hemert, Albert M. and Viktorin, Alexander and Visscher, Peter M. and Wang, Yunpeng and Webb, Bradley T. and Weinsheimer, Shantel Marie and Wellmann, Jürgen and Willemsen, Gonneke and Witt, Stephanie H. and Wu, Yang and Xi, Hualin S. and Yang, Jian and Zhang, Futao and Arolt, Volker and Baune, Bernhard T. and Berger, Klaus and Boomsma, Dorret I. and Cichon, Sven and Dannlowski, Udo and de Geus, E. C. J. and DePaulo, J. Raymond and Domenici, Enrico and Domschke, Katharina and Esko, Tõnu and Grabe, Hans J. and Hamilton, Steven P. and Hayward, Caroline and Heath, Andrew C. and Hinds, David A. and Kendler, Kenneth S. and Kloiber, Stefan and Lewis, Glyn and Li, Qingqin S. and Lucae, Susanne and Madden, Pamela F. A. and Magnusson, Patrik K. and Martin, Nicholas G. and McIntosh, Andrew M. and Metspalu, Andres and Mors, Ole and Mortensen, Preben Bo and Müller-Myhsok, Bertram and Nordentoft, Merete and Nöthen, Markus M. and O’Donovan, Michael C. and Paciga, Sara A. and Pedersen, Nancy L. and Penninx, Brenda W. J. H. and Perlis, Roy H. and Porteous, David J. and Potash, James B. and Preisig, Martin and Rietschel, Marcella and Schaefer, Catherine and Schulze, Thomas G. and Smoller, Jordan W. and Stefansson, Kari and Tiemeier, Henning and Uher, Rudolf and Völzke, Henry and Weissman, Myrna M. and Werge, Thomas and Winslow, Ashley R. and Lewis, Cathryn M. and Levinson, Douglas F. and Breen, Gerome and Børglum, Anders D. and Sullivan, Patrick F.}, year={2018}, month=apr, pages={668–681} }
</div></div><br><br><br><br><br><br></div>