<link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/bootstrap/4.0.0/css/bootstrap.min.css" integrity="sha384-Gn5384xqQ1aoWXA+058RXPxPg6fy4IWvTNh0E263XmFcJlSAwiGgFAW/dAiS6JXm" crossorigin="anonymous"><div class='container mt-4 bg-light p-4 text-center'><h1 class='text-primary'>Phenotype: gastro oesophageal reflux or gastric reflux</h1><p class='text-muted'><strong>URL:</strong> http://ftp.ebi.ac.uk/pub/databases/gwas/summary_statistics/GCST90077001-GCST90078000/GCST90077692</p><table border="1" class="dataframe table table-bordered">
  <thead>
    <tr style="text-align: right;">
      <th>firstAuthor</th>
      <th>accessionId</th>
      <th>publicationDate</th>
      <th>journal</th>
      <th>title</th>
      <th>reportedTrait</th>
      <th>efoTraits</th>
      <th>bgTraits</th>
      <th>discoverySampleAncestry</th>
      <th>replicationSampleAncestry</th>
      <th>associationCount</th>
      <th>summaryStatistics</th>
      <th>ssApiFlag</th>
      <th>agreedToCc0</th>
      <th>genotypingTechnologies</th>
      <th>pubmedId</th>
      <th>initialSampleDescription</th>
      <th>replicateSampleDescription</th>
      <th>UKB_FIELD</th>
      <th>FuzzySimilarity_gastro oesophageal reflux gord gastric reflux</th>
      <th>SearchPhenotype</th>
    </tr>
  </thead>
  <tbody>
    <tr>
      <td>Backman JD</td>
      <td>GCST90077692</td>
      <td>10/18/2021</td>
      <td>Nature</td>
      <td>Exome sequencing and analysis of 454,787 UK Biobank participants.</td>
      <td>gastro oesophageal reflux or gastric reflux</td>
      <td>gastroesophageal reflux disease</td>
      <td>-</td>
      <td>331754 european</td>
      <td>NaN</td>
      <td>0</td>
      <td>http://ftp.ebi.ac.uk/pub/databases/gwas/summary_statistics/GCST90077001-GCST90078000/GCST90077692</td>
      <td>no</td>
      <td>yes</td>
      <td>Exome-wide sequencing,Genome-wide genotyping array</td>
      <td>34662886</td>
      <td>21,143 European ancestry cases,310,611 European ancestry controls</td>
      <td>NaN</td>
      <td>-</td>
      <td>91</td>
      <td>gastro oesophageal reflux gord gastric reflux</td>
    </tr>
  </tbody>
</table><h2 class='text-muted'>Information file</h2><table border="1" class="dataframe table table-bordered">
  <thead>
    <tr style="text-align: right;">
      <th>Unnamed: 0</th>
      <th>Name</th>
      <th>Last modified</th>
      <th>Size</th>
      <th>Description</th>
      <th>URL</th>
    </tr>
  </thead>
  <tbody>
    <tr>
      <td>1</td>
      <td>GCST90077692_buildGRCh38.tsv.gz-meta.yaml</td>
      <td>2024-01-04 00:10</td>
      <td>915</td>
      <td>NaN</td>
      <td>http://ftp.ebi.ac.uk/pub/databases/gwas/summary_statistics/GCST90077001-GCST90078000/GCST90077692/GCST90077692_buildGRCh38.tsv.gz-meta.yaml</td>
    </tr>
    <tr>
      <td>2</td>
      <td>README.txt</td>
      <td>2021-11-18 01:06</td>
      <td>698</td>
      <td>NaN</td>
      <td>http://ftp.ebi.ac.uk/pub/databases/gwas/summary_statistics/GCST90077001-GCST90078000/GCST90077692/README.txt</td>
    </tr>
    <tr>
      <td>3</td>
      <td>harmonised/</td>
      <td>2024-02-02 15:16</td>
      <td>-</td>
      <td>NaN</td>
      <td>http://ftp.ebi.ac.uk/pub/databases/gwas/summary_statistics/GCST90077001-GCST90078000/GCST90077692/harmonised/</td>
    </tr>
    <tr>
      <td>4</td>
      <td>md5sum.txt</td>
      <td>2021-11-18 01:06</td>
      <td>109</td>
      <td>NaN</td>
      <td>http://ftp.ebi.ac.uk/pub/databases/gwas/summary_statistics/GCST90077001-GCST90078000/GCST90077692/md5sum.txt</td>
    </tr>
    <tr>
      <td>0</td>
      <td>GCST90077692_buildGRCh38.tsv.gz</td>
      <td>2021-11-18 01:06</td>
      <td>15M</td>
      <td>NaN</td>
      <td>http://ftp.ebi.ac.uk/pub/databases/gwas/summary_statistics/GCST90077001-GCST90078000/GCST90077692/GCST90077692_buildGRCh38.tsv.gz</td>
    </tr>
  </tbody>
</table><h2 class='text-muted'>Processed file</h2><p>Processing file: GCST90077692_buildGRCh38.tsv.gz!</p><h2 class='text-muted'>GWAS file head</h2><table border="1" class="dataframe table table-bordered">
  <thead>
    <tr style="text-align: right;">
      <th>name</th>
      <th>chromosome</th>
      <th>base_pair_location</th>
      <th>other_allele</th>
      <th>effect_allele</th>
      <th>trait</th>
      <th>cohort</th>
      <th>model</th>
      <th>odds_ratio</th>
      <th>ci_lower</th>
      <th>ci_upper</th>
      <th>p_value</th>
      <th>effect_allele_frequency</th>
      <th>standard_error</th>
    </tr>
  </thead>
  <tbody>
    <tr>
      <td>1_930165_G_A</td>
      <td>1</td>
      <td>930165</td>
      <td>G</td>
      <td>A</td>
      <td>20002_Illness_code_SR_1138_gastro_oesophageal_reflux_gord_gastric_reflux_AnyInst__BIN</td>
      <td>UKB_Freeze_450_EUR</td>
      <td>ADD-WGR-FIRTH</td>
      <td>0.635651</td>
      <td>0.136270</td>
      <td>2.96509</td>
      <td>0.564166</td>
      <td>0.000042</td>
      <td>0.785735</td>
    </tr>
    <tr>
      <td>1_930204_G_A</td>
      <td>1</td>
      <td>930204</td>
      <td>G</td>
      <td>A</td>
      <td>20002_Illness_code_SR_1138_gastro_oesophageal_reflux_gord_gastric_reflux_AnyInst__BIN</td>
      <td>UKB_Freeze_450_EUR</td>
      <td>ADD-WGR-FIRTH</td>
      <td>0.666279</td>
      <td>0.134238</td>
      <td>3.30701</td>
      <td>0.619366</td>
      <td>0.000041</td>
      <td>0.817408</td>
    </tr>
  </tbody>
</table></div><div class='container mt-4 bg-light p-4 text-center'><h2 class='text-muted'>Original GWAS file headers</h2><table border="1" class="dataframe table table-bordered">
  <thead>
    <tr style="text-align: right;">
      <th>name</th>
      <th>chromosome</th>
      <th>base_pair_location</th>
      <th>other_allele</th>
      <th>effect_allele</th>
      <th>trait</th>
      <th>cohort</th>
      <th>model</th>
      <th>odds_ratio</th>
      <th>ci_lower</th>
      <th>ci_upper</th>
      <th>p_value</th>
      <th>effect_allele_frequency</th>
      <th>standard_error</th>
    </tr>
  </thead>
  <tbody>
    <tr>
      <td>1_930165_G_A</td>
      <td>1</td>
      <td>930165</td>
      <td>G</td>
      <td>A</td>
      <td>20002_Illness_code_SR_1138_gastro_oesophageal_reflux_gord_gastric_reflux_AnyInst__BIN</td>
      <td>UKB_Freeze_450_EUR</td>
      <td>ADD-WGR-FIRTH</td>
      <td>0.635651</td>
      <td>0.136270</td>
      <td>2.96509</td>
      <td>0.564166</td>
      <td>0.000042</td>
      <td>0.785735</td>
    </tr>
    <tr>
      <td>1_930204_G_A</td>
      <td>1</td>
      <td>930204</td>
      <td>G</td>
      <td>A</td>
      <td>20002_Illness_code_SR_1138_gastro_oesophageal_reflux_gord_gastric_reflux_AnyInst__BIN</td>
      <td>UKB_Freeze_450_EUR</td>
      <td>ADD-WGR-FIRTH</td>
      <td>0.666279</td>
      <td>0.134238</td>
      <td>3.30701</td>
      <td>0.619366</td>
      <td>0.000041</td>
      <td>0.817408</td>
    </tr>
  </tbody>
</table><p style='font-weight: bold; color: blue;'>CHR -> ['chromosome']</p><p style='font-weight: bold; color: blue;'>SNP -> ['name']</p><p style='font-weight: bold; color: blue;'>BP -> ['name', 'base_pair_location']</p><p style='font-weight: bold; color: blue;'>A1 -> ['effect_allele']</p><p style='font-weight: bold; color: blue;'>A2 -> ['other_allele']</p><p style='font-weight: bold; color: blue;'>N -> []</p><p style='font-weight: bold; color: blue;'>P -> ['p_value']</p><p style='font-weight: bold; color: blue;'>INFO -> []</p><p style='font-weight: bold; color: blue;'>MAF -> ['effect_allele_frequency']</p><p style='font-weight: bold; color: blue;'>BETA -> []</p><p style='font-weight: bold; color: blue;'>SE -> ['standard_error']</p><p style='font-weight: bold; color: blue;'>OR -> ['odds_ratio']</p><p style='font-weight: bold; color: blue;'>Z -> []</p><p style='font-weight: bold; color: blue;'>D -> []</p><h2 class='text-muted'>GWAS columns</h2>name, chromosome, base_pair_location, other_allele, effect_allele, trait, cohort, model, odds_ratio, ci_lower, ci_upper, p_value, effect_allele_frequency, standard_error<h2 class='text-muted'>Identified columns</h2>chromosome, name, name, base_pair_location, effect_allele, other_allele, p_value, effect_allele_frequency, standard_error, odds_ratio<h2 class='text-muted'>Unidentified Columns headers</h2>model, ci_upper, cohort, ci_lower, trait<h2 class='text-muted'>PMID</h2>34662886<h2 class='text-muted'>DOI</h2>10.1038/s41586-021-04103-z<div class='bibtex-container'><h2 class='text-muted'>BibTeX</h2><div style='font-family: monospace; white-space: pre-wrap;'> @article{Backman_2021, title={Exome sequencing and analysis of 454,787 UK Biobank participants}, volume={599}, ISSN={1476-4687}, url={http://dx.doi.org/10.1038/s41586-021-04103-z}, DOI={10.1038/s41586-021-04103-z}, number={7886}, journal={Nature}, publisher={Springer Science and Business Media LLC}, author={Backman, Joshua D. and Li, Alexander H. and Marcketta, Anthony and Sun, Dylan and Mbatchou, Joelle and Kessler, Michael D. and Benner, Christian and Liu, Daren and Locke, Adam E. and Balasubramanian, Suganthi and Yadav, Ashish and Banerjee, Nilanjana and Gillies, Christopher E. and Damask, Amy and Liu, Simon and Bai, Xiaodong and Hawes, Alicia and Maxwell, Evan and Gurski, Lauren and Watanabe, Kyoko and Kosmicki, Jack A. and Rajagopal, Veera and Mighty, Jason and Jones, Marcus and Mitnaul, Lyndon and Stahl, Eli and Coppola, Giovanni and Jorgenson, Eric and Habegger, Lukas and Salerno, William J. and Shuldiner, Alan R. and Lotta, Luca A. and Overton, John D. and Cantor, Michael N. and Reid, Jeffrey G. and Yancopoulos, George and Kang, Hyun M. and Marchini, Jonathan and Baras, Aris and Abecasis, Gonçalo R. and Ferreira, Manuel A. R.}, year={2021}, month=oct, pages={628–634} }
</div></div><br><br><br><br><br><br></div><div class='container mt-4 bg-light p-4 text-center'><h1 class='text-primary'>Phenotype: icd10 k21: gastro oesophageal reflux or gastric reflux</h1><p class='text-muted'><strong>URL:</strong> http://ftp.ebi.ac.uk/pub/databases/gwas/summary_statistics/GCST90077001-GCST90078000/GCST90077693</p><table border="1" class="dataframe table table-bordered">
  <thead>
    <tr style="text-align: right;">
      <th>firstAuthor</th>
      <th>accessionId</th>
      <th>publicationDate</th>
      <th>journal</th>
      <th>title</th>
      <th>reportedTrait</th>
      <th>efoTraits</th>
      <th>bgTraits</th>
      <th>discoverySampleAncestry</th>
      <th>replicationSampleAncestry</th>
      <th>associationCount</th>
      <th>summaryStatistics</th>
      <th>ssApiFlag</th>
      <th>agreedToCc0</th>
      <th>genotypingTechnologies</th>
      <th>pubmedId</th>
      <th>initialSampleDescription</th>
      <th>replicateSampleDescription</th>
      <th>UKB_FIELD</th>
      <th>FuzzySimilarity_gastro oesophageal reflux gord gastric reflux</th>
      <th>SearchPhenotype</th>
    </tr>
  </thead>
  <tbody>
    <tr>
      <td>Backman JD</td>
      <td>GCST90077693</td>
      <td>10/18/2021</td>
      <td>Nature</td>
      <td>Exome sequencing and analysis of 454,787 UK Biobank participants.</td>
      <td>icd10 k21: gastro oesophageal reflux or gastric reflux</td>
      <td>gastroesophageal reflux disease</td>
      <td>-</td>
      <td>329051 european</td>
      <td>NaN</td>
      <td>0</td>
      <td>http://ftp.ebi.ac.uk/pub/databases/gwas/summary_statistics/GCST90077001-GCST90078000/GCST90077693</td>
      <td>no</td>
      <td>yes</td>
      <td>Exome-wide sequencing,Genome-wide genotyping array</td>
      <td>34662886</td>
      <td>18,674 European ancestry cases,310,377 European ancestry controls</td>
      <td>NaN</td>
      <td>-</td>
      <td>86</td>
      <td>gastro oesophageal reflux gord gastric reflux</td>
    </tr>
  </tbody>
</table><h2 class='text-muted'>Information file</h2><table border="1" class="dataframe table table-bordered">
  <thead>
    <tr style="text-align: right;">
      <th>Unnamed: 0</th>
      <th>Name</th>
      <th>Last modified</th>
      <th>Size</th>
      <th>Description</th>
      <th>URL</th>
    </tr>
  </thead>
  <tbody>
    <tr>
      <td>1</td>
      <td>GCST90077693_buildGRCh38.tsv.gz-meta.yaml</td>
      <td>2024-01-04 00:20</td>
      <td>928</td>
      <td>NaN</td>
      <td>http://ftp.ebi.ac.uk/pub/databases/gwas/summary_statistics/GCST90077001-GCST90078000/GCST90077693/GCST90077693_buildGRCh38.tsv.gz-meta.yaml</td>
    </tr>
    <tr>
      <td>2</td>
      <td>README.txt</td>
      <td>2021-11-18 00:33</td>
      <td>698</td>
      <td>NaN</td>
      <td>http://ftp.ebi.ac.uk/pub/databases/gwas/summary_statistics/GCST90077001-GCST90078000/GCST90077693/README.txt</td>
    </tr>
    <tr>
      <td>3</td>
      <td>harmonised/</td>
      <td>2024-02-02 15:16</td>
      <td>-</td>
      <td>NaN</td>
      <td>http://ftp.ebi.ac.uk/pub/databases/gwas/summary_statistics/GCST90077001-GCST90078000/GCST90077693/harmonised/</td>
    </tr>
    <tr>
      <td>4</td>
      <td>md5sum.txt</td>
      <td>2021-11-18 00:33</td>
      <td>109</td>
      <td>NaN</td>
      <td>http://ftp.ebi.ac.uk/pub/databases/gwas/summary_statistics/GCST90077001-GCST90078000/GCST90077693/md5sum.txt</td>
    </tr>
    <tr>
      <td>0</td>
      <td>GCST90077693_buildGRCh38.tsv.gz</td>
      <td>2021-11-18 00:33</td>
      <td>15M</td>
      <td>NaN</td>
      <td>http://ftp.ebi.ac.uk/pub/databases/gwas/summary_statistics/GCST90077001-GCST90078000/GCST90077693/GCST90077693_buildGRCh38.tsv.gz</td>
    </tr>
  </tbody>
</table><h2 class='text-muted'>Processed file</h2><p>Processing file: GCST90077693_buildGRCh38.tsv.gz!</p><h2 class='text-muted'>GWAS file head</h2><table border="1" class="dataframe table table-bordered">
  <thead>
    <tr style="text-align: right;">
      <th>name</th>
      <th>chromosome</th>
      <th>base_pair_location</th>
      <th>other_allele</th>
      <th>effect_allele</th>
      <th>trait</th>
      <th>cohort</th>
      <th>model</th>
      <th>odds_ratio</th>
      <th>ci_lower</th>
      <th>ci_upper</th>
      <th>p_value</th>
      <th>effect_allele_frequency</th>
      <th>standard_error</th>
    </tr>
  </thead>
  <tbody>
    <tr>
      <td>1_930165_G_A</td>
      <td>1</td>
      <td>930165</td>
      <td>G</td>
      <td>A</td>
      <td>20002_Illness_code_SR_1138_gastro_oesophageal_reflux_gord_gastric_reflux_Phe10_K21_combined_AnyInst__BIN</td>
      <td>UKB_Freeze_450_EUR</td>
      <td>ADD-WGR-FIRTH</td>
      <td>0.686159</td>
      <td>0.135398</td>
      <td>3.47727</td>
      <td>0.649200</td>
      <td>0.000043</td>
      <td>0.828021</td>
    </tr>
    <tr>
      <td>1_930204_G_A</td>
      <td>1</td>
      <td>930204</td>
      <td>G</td>
      <td>A</td>
      <td>20002_Illness_code_SR_1138_gastro_oesophageal_reflux_gord_gastric_reflux_Phe10_K21_combined_AnyInst__BIN</td>
      <td>UKB_Freeze_450_EUR</td>
      <td>ADD-WGR-FIRTH</td>
      <td>0.343703</td>
      <td>0.064547</td>
      <td>1.83017</td>
      <td>0.210707</td>
      <td>0.000041</td>
      <td>0.853274</td>
    </tr>
  </tbody>
</table></div><div class='container mt-4 bg-light p-4 text-center'><h2 class='text-muted'>Original GWAS file headers</h2><table border="1" class="dataframe table table-bordered">
  <thead>
    <tr style="text-align: right;">
      <th>name</th>
      <th>chromosome</th>
      <th>base_pair_location</th>
      <th>other_allele</th>
      <th>effect_allele</th>
      <th>trait</th>
      <th>cohort</th>
      <th>model</th>
      <th>odds_ratio</th>
      <th>ci_lower</th>
      <th>ci_upper</th>
      <th>p_value</th>
      <th>effect_allele_frequency</th>
      <th>standard_error</th>
    </tr>
  </thead>
  <tbody>
    <tr>
      <td>1_930165_G_A</td>
      <td>1</td>
      <td>930165</td>
      <td>G</td>
      <td>A</td>
      <td>20002_Illness_code_SR_1138_gastro_oesophageal_reflux_gord_gastric_reflux_Phe10_K21_combined_AnyInst__BIN</td>
      <td>UKB_Freeze_450_EUR</td>
      <td>ADD-WGR-FIRTH</td>
      <td>0.686159</td>
      <td>0.135398</td>
      <td>3.47727</td>
      <td>0.649200</td>
      <td>0.000043</td>
      <td>0.828021</td>
    </tr>
    <tr>
      <td>1_930204_G_A</td>
      <td>1</td>
      <td>930204</td>
      <td>G</td>
      <td>A</td>
      <td>20002_Illness_code_SR_1138_gastro_oesophageal_reflux_gord_gastric_reflux_Phe10_K21_combined_AnyInst__BIN</td>
      <td>UKB_Freeze_450_EUR</td>
      <td>ADD-WGR-FIRTH</td>
      <td>0.343703</td>
      <td>0.064547</td>
      <td>1.83017</td>
      <td>0.210707</td>
      <td>0.000041</td>
      <td>0.853274</td>
    </tr>
  </tbody>
</table><p style='font-weight: bold; color: blue;'>CHR -> ['chromosome']</p><p style='font-weight: bold; color: blue;'>SNP -> ['name']</p><p style='font-weight: bold; color: blue;'>BP -> ['name', 'base_pair_location']</p><p style='font-weight: bold; color: blue;'>A1 -> ['effect_allele']</p><p style='font-weight: bold; color: blue;'>A2 -> ['other_allele']</p><p style='font-weight: bold; color: blue;'>N -> []</p><p style='font-weight: bold; color: blue;'>P -> ['p_value']</p><p style='font-weight: bold; color: blue;'>INFO -> []</p><p style='font-weight: bold; color: blue;'>MAF -> ['effect_allele_frequency']</p><p style='font-weight: bold; color: blue;'>BETA -> []</p><p style='font-weight: bold; color: blue;'>SE -> ['standard_error']</p><p style='font-weight: bold; color: blue;'>OR -> ['odds_ratio']</p><p style='font-weight: bold; color: blue;'>Z -> []</p><p style='font-weight: bold; color: blue;'>D -> []</p><h2 class='text-muted'>GWAS columns</h2>name, chromosome, base_pair_location, other_allele, effect_allele, trait, cohort, model, odds_ratio, ci_lower, ci_upper, p_value, effect_allele_frequency, standard_error<h2 class='text-muted'>Identified columns</h2>chromosome, name, name, base_pair_location, effect_allele, other_allele, p_value, effect_allele_frequency, standard_error, odds_ratio<h2 class='text-muted'>Unidentified Columns headers</h2>model, ci_upper, cohort, ci_lower, trait<h2 class='text-muted'>PMID</h2>34662886<h2 class='text-muted'>DOI</h2>10.1038/s41586-021-04103-z<div class='bibtex-container'><h2 class='text-muted'>BibTeX</h2><div style='font-family: monospace; white-space: pre-wrap;'> @article{Backman_2021, title={Exome sequencing and analysis of 454,787 UK Biobank participants}, volume={599}, ISSN={1476-4687}, url={http://dx.doi.org/10.1038/s41586-021-04103-z}, DOI={10.1038/s41586-021-04103-z}, number={7886}, journal={Nature}, publisher={Springer Science and Business Media LLC}, author={Backman, Joshua D. and Li, Alexander H. and Marcketta, Anthony and Sun, Dylan and Mbatchou, Joelle and Kessler, Michael D. and Benner, Christian and Liu, Daren and Locke, Adam E. and Balasubramanian, Suganthi and Yadav, Ashish and Banerjee, Nilanjana and Gillies, Christopher E. and Damask, Amy and Liu, Simon and Bai, Xiaodong and Hawes, Alicia and Maxwell, Evan and Gurski, Lauren and Watanabe, Kyoko and Kosmicki, Jack A. and Rajagopal, Veera and Mighty, Jason and Jones, Marcus and Mitnaul, Lyndon and Stahl, Eli and Coppola, Giovanni and Jorgenson, Eric and Habegger, Lukas and Salerno, William J. and Shuldiner, Alan R. and Lotta, Luca A. and Overton, John D. and Cantor, Michael N. and Reid, Jeffrey G. and Yancopoulos, George and Kang, Hyun M. and Marchini, Jonathan and Baras, Aris and Abecasis, Gonçalo R. and Ferreira, Manuel A. R.}, year={2021}, month=oct, pages={628–634} }
</div></div><br><br><br><br><br><br></div><div class='container mt-4 bg-light p-4 text-center'><h1 class='text-primary'>Phenotype: gastroesophageal reflux disease or gastric reflux</h1><p class='text-muted'><strong>URL:</strong> http://ftp.ebi.ac.uk/pub/databases/gwas/summary_statistics/GCST90038001-GCST90039000/GCST90038624</p><table border="1" class="dataframe table table-bordered">
  <thead>
    <tr style="text-align: right;">
      <th>firstAuthor</th>
      <th>accessionId</th>
      <th>publicationDate</th>
      <th>journal</th>
      <th>title</th>
      <th>reportedTrait</th>
      <th>efoTraits</th>
      <th>bgTraits</th>
      <th>discoverySampleAncestry</th>
      <th>replicationSampleAncestry</th>
      <th>associationCount</th>
      <th>summaryStatistics</th>
      <th>ssApiFlag</th>
      <th>agreedToCc0</th>
      <th>genotypingTechnologies</th>
      <th>pubmedId</th>
      <th>initialSampleDescription</th>
      <th>replicateSampleDescription</th>
      <th>UKB_FIELD</th>
      <th>FuzzySimilarity_gastro oesophageal reflux gord gastric reflux</th>
      <th>SearchPhenotype</th>
    </tr>
  </thead>
  <tbody>
    <tr>
      <td>D??nerta?� HM</td>
      <td>GCST90038624</td>
      <td>4/8/2021</td>
      <td>Nat Aging</td>
      <td>Common genetic associations between age-related diseases.</td>
      <td>gastroesophageal reflux disease or gastric reflux</td>
      <td>gastroesophageal reflux disease</td>
      <td>-</td>
      <td>484598 nr</td>
      <td>NaN</td>
      <td>0</td>
      <td>http://ftp.ebi.ac.uk/pub/databases/gwas/summary_statistics/GCST90038001-GCST90039000/GCST90038624</td>
      <td>no</td>
      <td>yes</td>
      <td>Genome-wide genotyping array</td>
      <td>33959723</td>
      <td>20,381 cases,464,217 controls</td>
      <td>NaN</td>
      <td>-</td>
      <td>81</td>
      <td>gastro oesophageal reflux gord gastric reflux</td>
    </tr>
  </tbody>
</table><h2 class='text-muted'>Information file</h2><table border="1" class="dataframe table table-bordered">
  <thead>
    <tr style="text-align: right;">
      <th>Unnamed: 0</th>
      <th>Name</th>
      <th>Last modified</th>
      <th>Size</th>
      <th>Description</th>
      <th>URL</th>
    </tr>
  </thead>
  <tbody>
    <tr>
      <td>1</td>
      <td>GCST90038624_buildGRCh37.tsv-meta.yaml</td>
      <td>2024-01-04 00:24</td>
      <td>700</td>
      <td>NaN</td>
      <td>http://ftp.ebi.ac.uk/pub/databases/gwas/summary_statistics/GCST90038001-GCST90039000/GCST90038624/GCST90038624_buildGRCh37.tsv-meta.yaml</td>
    </tr>
    <tr>
      <td>2</td>
      <td>harmonised/</td>
      <td>2024-02-02 15:16</td>
      <td>-</td>
      <td>NaN</td>
      <td>http://ftp.ebi.ac.uk/pub/databases/gwas/summary_statistics/GCST90038001-GCST90039000/GCST90038624/harmonised/</td>
    </tr>
    <tr>
      <td>3</td>
      <td>md5sum.txt</td>
      <td>2021-11-14 00:27</td>
      <td>62</td>
      <td>NaN</td>
      <td>http://ftp.ebi.ac.uk/pub/databases/gwas/summary_statistics/GCST90038001-GCST90039000/GCST90038624/md5sum.txt</td>
    </tr>
    <tr>
      <td>0</td>
      <td>GCST90038624_buildGRCh37.tsv</td>
      <td>2021-11-14 00:27</td>
      <td>1.2G</td>
      <td>NaN</td>
      <td>http://ftp.ebi.ac.uk/pub/databases/gwas/summary_statistics/GCST90038001-GCST90039000/GCST90038624/GCST90038624_buildGRCh37.tsv</td>
    </tr>
  </tbody>
</table><h2 class='text-muted'>Processed file</h2><p>Processing file: GCST90038624_buildGRCh37.tsv.1!</p><h2 class='text-muted'>GWAS file head</h2><table border="1" class="dataframe table table-bordered">
  <thead>
    <tr style="text-align: right;">
      <th>variant_id</th>
      <th>chromosome</th>
      <th>base_pair_location</th>
      <th>genpos</th>
      <th>effect_allele</th>
      <th>other_allele</th>
      <th>effect_allele_frequency</th>
      <th>info</th>
      <th>chisq_linreg</th>
      <th>p_linreg</th>
      <th>beta</th>
      <th>standard_error</th>
      <th>chisq_bolt_lmm_inf</th>
      <th>p_bolt_lmm_inf</th>
      <th>chisq_bolt_lmm</th>
      <th>p_value</th>
    </tr>
  </thead>
  <tbody>
    <tr>
      <td>rs533090414</td>
      <td>1</td>
      <td>18849</td>
      <td>0.0</td>
      <td>C</td>
      <td>G</td>
      <td>0.026465</td>
      <td>0.536412</td>
      <td>0.599595</td>
      <td>0.440</td>
      <td>-0.001238</td>
      <td>0.001753</td>
      <td>0.499134</td>
      <td>0.480</td>
      <td>0.476313</td>
      <td>0.490</td>
    </tr>
    <tr>
      <td>rs568927205</td>
      <td>1</td>
      <td>54712</td>
      <td>0.0</td>
      <td>T</td>
      <td>TTTTC</td>
      <td>0.415825</td>
      <td>0.538572</td>
      <td>3.911910</td>
      <td>0.048</td>
      <td>-0.001119</td>
      <td>0.000564</td>
      <td>3.931090</td>
      <td>0.047</td>
      <td>3.899680</td>
      <td>0.048</td>
    </tr>
  </tbody>
</table></div><div class='container mt-4 bg-light p-4 text-center'><h2 class='text-muted'>Original GWAS file headers</h2><table border="1" class="dataframe table table-bordered">
  <thead>
    <tr style="text-align: right;">
      <th>variant_id</th>
      <th>chromosome</th>
      <th>base_pair_location</th>
      <th>genpos</th>
      <th>effect_allele</th>
      <th>other_allele</th>
      <th>effect_allele_frequency</th>
      <th>info</th>
      <th>chisq_linreg</th>
      <th>p_linreg</th>
      <th>beta</th>
      <th>standard_error</th>
      <th>chisq_bolt_lmm_inf</th>
      <th>p_bolt_lmm_inf</th>
      <th>chisq_bolt_lmm</th>
      <th>p_value</th>
    </tr>
  </thead>
  <tbody>
    <tr>
      <td>rs533090414</td>
      <td>1</td>
      <td>18849</td>
      <td>0.0</td>
      <td>C</td>
      <td>G</td>
      <td>0.026465</td>
      <td>0.536412</td>
      <td>0.599595</td>
      <td>0.440</td>
      <td>-0.001238</td>
      <td>0.001753</td>
      <td>0.499134</td>
      <td>0.480</td>
      <td>0.476313</td>
      <td>0.490</td>
    </tr>
    <tr>
      <td>rs568927205</td>
      <td>1</td>
      <td>54712</td>
      <td>0.0</td>
      <td>T</td>
      <td>TTTTC</td>
      <td>0.415825</td>
      <td>0.538572</td>
      <td>3.911910</td>
      <td>0.048</td>
      <td>-0.001119</td>
      <td>0.000564</td>
      <td>3.931090</td>
      <td>0.047</td>
      <td>3.899680</td>
      <td>0.048</td>
    </tr>
  </tbody>
</table><p style='font-weight: bold; color: blue;'>CHR -> ['chromosome']</p><p style='font-weight: bold; color: blue;'>SNP -> ['variant_id']</p><p style='font-weight: bold; color: blue;'>BP -> ['base_pair_location', 'genpos']</p><p style='font-weight: bold; color: blue;'>A1 -> ['effect_allele']</p><p style='font-weight: bold; color: blue;'>A2 -> ['other_allele']</p><p style='font-weight: bold; color: blue;'>N -> []</p><p style='font-weight: bold; color: blue;'>P -> ['p_value']</p><p style='font-weight: bold; color: blue;'>INFO -> ['info']</p><p style='font-weight: bold; color: blue;'>MAF -> ['effect_allele_frequency']</p><p style='font-weight: bold; color: blue;'>BETA -> ['beta', 'p_bolt_lmm_inf']</p><p style='font-weight: bold; color: blue;'>SE -> ['standard_error']</p><p style='font-weight: bold; color: blue;'>OR -> []</p><p style='font-weight: bold; color: blue;'>Z -> []</p><p style='font-weight: bold; color: blue;'>D -> []</p><h2 class='text-muted'>GWAS columns</h2>variant_id, chromosome, base_pair_location, genpos, effect_allele, other_allele, effect_allele_frequency, info, chisq_linreg, p_linreg, beta, standard_error, chisq_bolt_lmm_inf, p_bolt_lmm_inf, chisq_bolt_lmm, p_value<h2 class='text-muted'>Identified columns</h2>chromosome, variant_id, base_pair_location, genpos, effect_allele, other_allele, p_value, info, effect_allele_frequency, beta, p_bolt_lmm_inf, standard_error<h2 class='text-muted'>Unidentified Columns headers</h2>p_linreg, chisq_linreg, chisq_bolt_lmm_inf, chisq_bolt_lmm<h2 class='text-muted'>PMID</h2>33959723<h2 class='text-muted'>DOI</h2>10.1038/s43587-021-00051-5<div class='bibtex-container'><h2 class='text-muted'>BibTeX</h2><div style='font-family: monospace; white-space: pre-wrap;'> @article{D_nerta__2021, title={Common genetic associations between age-related diseases}, volume={1}, ISSN={2662-8465}, url={http://dx.doi.org/10.1038/s43587-021-00051-5}, DOI={10.1038/s43587-021-00051-5}, number={4}, journal={Nature Aging}, publisher={Springer Science and Business Media LLC}, author={Dönertaş, Handan Melike and Fabian, Daniel K. and Fuentealba, Matías and Partridge, Linda and Thornton, Janet M.}, year={2021}, month=apr, pages={400–412} }
</div></div><br><br><br><br><br><br></div><div class='container mt-4 bg-light p-4 text-center'><h1 class='text-primary'>Phenotype: gastroesophageal reflux disease</h1><p class='text-muted'><strong>URL:</strong> http://ftp.ebi.ac.uk/pub/databases/gwas/summary_statistics/GCST90018001-GCST90019000/GCST90018628</p><table border="1" class="dataframe table table-bordered">
  <thead>
    <tr style="text-align: right;">
      <th>firstAuthor</th>
      <th>accessionId</th>
      <th>publicationDate</th>
      <th>journal</th>
      <th>title</th>
      <th>reportedTrait</th>
      <th>efoTraits</th>
      <th>bgTraits</th>
      <th>discoverySampleAncestry</th>
      <th>replicationSampleAncestry</th>
      <th>associationCount</th>
      <th>summaryStatistics</th>
      <th>ssApiFlag</th>
      <th>agreedToCc0</th>
      <th>genotypingTechnologies</th>
      <th>pubmedId</th>
      <th>initialSampleDescription</th>
      <th>replicateSampleDescription</th>
      <th>UKB_FIELD</th>
      <th>FuzzySimilarity_gastro oesophageal reflux gord gastric reflux</th>
      <th>SearchPhenotype</th>
    </tr>
  </thead>
  <tbody>
    <tr>
      <td>Sakaue S</td>
      <td>GCST90018628</td>
      <td>9/30/2021</td>
      <td>Nat Genet</td>
      <td>A cross-population atlas of genetic associations for 220 human phenotypes.</td>
      <td>gastroesophageal reflux disease</td>
      <td>gastroesophageal reflux disease</td>
      <td>-</td>
      <td>178464 east asian</td>
      <td>NaN</td>
      <td>0</td>
      <td>http://ftp.ebi.ac.uk/pub/databases/gwas/summary_statistics/GCST90018001-GCST90019000/GCST90018628</td>
      <td>no</td>
      <td>yes</td>
      <td>Genome-wide genotyping array</td>
      <td>34594039</td>
      <td>177,516 East Asian ancestry controls</td>
      <td>NaN</td>
      <td>-</td>
      <td>68</td>
      <td>gastro oesophageal reflux gord gastric reflux</td>
    </tr>
  </tbody>
</table><h2 class='text-muted'>Information file</h2><table border="1" class="dataframe table table-bordered">
  <thead>
    <tr style="text-align: right;">
      <th>Unnamed: 0</th>
      <th>Name</th>
      <th>Last modified</th>
      <th>Size</th>
      <th>Description</th>
      <th>URL</th>
    </tr>
  </thead>
  <tbody>
    <tr>
      <td>1</td>
      <td>GCST90018628_buildGRCh37.tsv.gz-meta.yaml</td>
      <td>2024-01-04 00:22</td>
      <td>962</td>
      <td>NaN</td>
      <td>http://ftp.ebi.ac.uk/pub/databases/gwas/summary_statistics/GCST90018001-GCST90019000/GCST90018628/GCST90018628_buildGRCh37.tsv.gz-meta.yaml</td>
    </tr>
    <tr>
      <td>2</td>
      <td>README.txt</td>
      <td>2021-12-16 00:33</td>
      <td>177</td>
      <td>NaN</td>
      <td>http://ftp.ebi.ac.uk/pub/databases/gwas/summary_statistics/GCST90018001-GCST90019000/GCST90018628/README.txt</td>
    </tr>
    <tr>
      <td>3</td>
      <td>harmonised/</td>
      <td>2024-02-02 15:17</td>
      <td>-</td>
      <td>NaN</td>
      <td>http://ftp.ebi.ac.uk/pub/databases/gwas/summary_statistics/GCST90018001-GCST90019000/GCST90018628/harmonised/</td>
    </tr>
    <tr>
      <td>4</td>
      <td>md5sum.txt</td>
      <td>2021-12-16 00:33</td>
      <td>109</td>
      <td>NaN</td>
      <td>http://ftp.ebi.ac.uk/pub/databases/gwas/summary_statistics/GCST90018001-GCST90019000/GCST90018628/md5sum.txt</td>
    </tr>
    <tr>
      <td>0</td>
      <td>GCST90018628_buildGRCh37.tsv.gz</td>
      <td>2021-12-16 00:33</td>
      <td>503M</td>
      <td>NaN</td>
      <td>http://ftp.ebi.ac.uk/pub/databases/gwas/summary_statistics/GCST90018001-GCST90019000/GCST90018628/GCST90018628_buildGRCh37.tsv.gz</td>
    </tr>
  </tbody>
</table><h2 class='text-muted'>Processed file</h2><p>Processing file: GCST90018628_buildGRCh37.tsv.gz!</p><h2 class='text-muted'>GWAS file head</h2><table border="1" class="dataframe table table-bordered">
  <thead>
    <tr style="text-align: right;">
      <th>chromosome</th>
      <th>base_pair_location</th>
      <th>effect_allele</th>
      <th>other_allele</th>
      <th>effect_allele_frequency</th>
      <th>beta</th>
      <th>standard_error</th>
      <th>p_value</th>
      <th>variant_id</th>
    </tr>
  </thead>
  <tbody>
    <tr>
      <td>1</td>
      <td>751343</td>
      <td>A</td>
      <td>T</td>
      <td>0.147686</td>
      <td>-0.023344</td>
      <td>0.068038</td>
      <td>0.731523</td>
      <td>NaN</td>
    </tr>
    <tr>
      <td>1</td>
      <td>751756</td>
      <td>C</td>
      <td>T</td>
      <td>0.147222</td>
      <td>-0.023485</td>
      <td>0.067879</td>
      <td>0.729360</td>
      <td>NaN</td>
    </tr>
  </tbody>
</table></div><div class='container mt-4 bg-light p-4 text-center'><h2 class='text-muted'>Original GWAS file headers</h2><table border="1" class="dataframe table table-bordered">
  <thead>
    <tr style="text-align: right;">
      <th>chromosome</th>
      <th>base_pair_location</th>
      <th>effect_allele</th>
      <th>other_allele</th>
      <th>effect_allele_frequency</th>
      <th>beta</th>
      <th>standard_error</th>
      <th>p_value</th>
      <th>variant_id</th>
    </tr>
  </thead>
  <tbody>
    <tr>
      <td>1</td>
      <td>751343</td>
      <td>A</td>
      <td>T</td>
      <td>0.147686</td>
      <td>-0.023344</td>
      <td>0.068038</td>
      <td>0.731523</td>
      <td>NaN</td>
    </tr>
    <tr>
      <td>1</td>
      <td>751756</td>
      <td>C</td>
      <td>T</td>
      <td>0.147222</td>
      <td>-0.023485</td>
      <td>0.067879</td>
      <td>0.729360</td>
      <td>NaN</td>
    </tr>
  </tbody>
</table><p style='font-weight: bold; color: blue;'>CHR -> ['chromosome']</p><p style='font-weight: bold; color: blue;'>SNP -> ['variant_id']</p><p style='font-weight: bold; color: blue;'>BP -> ['base_pair_location']</p><p style='font-weight: bold; color: blue;'>A1 -> ['effect_allele']</p><p style='font-weight: bold; color: blue;'>A2 -> ['other_allele']</p><p style='font-weight: bold; color: blue;'>N -> []</p><p style='font-weight: bold; color: blue;'>P -> ['p_value']</p><p style='font-weight: bold; color: blue;'>INFO -> []</p><p style='font-weight: bold; color: blue;'>MAF -> ['effect_allele_frequency']</p><p style='font-weight: bold; color: blue;'>BETA -> ['beta']</p><p style='font-weight: bold; color: blue;'>SE -> ['standard_error']</p><p style='font-weight: bold; color: blue;'>OR -> []</p><p style='font-weight: bold; color: blue;'>Z -> []</p><p style='font-weight: bold; color: blue;'>D -> []</p><h2 class='text-muted'>GWAS columns</h2>chromosome, base_pair_location, effect_allele, other_allele, effect_allele_frequency, beta, standard_error, p_value, variant_id<h2 class='text-muted'>Identified columns</h2>chromosome, variant_id, base_pair_location, effect_allele, other_allele, p_value, effect_allele_frequency, beta, standard_error<h2 class='text-muted'>Unidentified Columns headers</h2><h2 class='text-muted'>PMID</h2>34594039<h2 class='text-muted'>DOI</h2>Could not find DOI!<div class='bibtex-container'><h2 class='text-muted'>BibTeX</h2><div style='font-family: monospace; white-space: pre-wrap;'></div></div><br><br><br><br><br><br></div><div class='container mt-4 bg-light p-4 text-center'><h1 class='text-primary'>Phenotype: gastroesophageal reflux disease</h1><p class='text-muted'><strong>URL:</strong> http://ftp.ebi.ac.uk/pub/databases/gwas/summary_statistics/GCST90000001-GCST90001000/GCST90000514</p><table border="1" class="dataframe table table-bordered">
  <thead>
    <tr style="text-align: right;">
      <th>firstAuthor</th>
      <th>accessionId</th>
      <th>publicationDate</th>
      <th>journal</th>
      <th>title</th>
      <th>reportedTrait</th>
      <th>efoTraits</th>
      <th>bgTraits</th>
      <th>discoverySampleAncestry</th>
      <th>replicationSampleAncestry</th>
      <th>associationCount</th>
      <th>summaryStatistics</th>
      <th>ssApiFlag</th>
      <th>agreedToCc0</th>
      <th>genotypingTechnologies</th>
      <th>pubmedId</th>
      <th>initialSampleDescription</th>
      <th>replicateSampleDescription</th>
      <th>UKB_FIELD</th>
      <th>FuzzySimilarity_gastro oesophageal reflux gord gastric reflux</th>
      <th>SearchPhenotype</th>
    </tr>
  </thead>
  <tbody>
    <tr>
      <td>Ong JS</td>
      <td>GCST90000514</td>
      <td>6/29/2021</td>
      <td>Gut</td>
      <td>Multitrait genetic association analysis identifies 50 new risk loci for gastro-oesophageal reflux, seven new loci for Barrett's oesophagus and provides insights into clinical heterogeneity in reflux diagnosis.</td>
      <td>gastroesophageal reflux disease</td>
      <td>gastroesophageal reflux disease</td>
      <td>-</td>
      <td>602604 european</td>
      <td>1590227 NR</td>
      <td>37</td>
      <td>http://ftp.ebi.ac.uk/pub/databases/gwas/summary_statistics/GCST90000001-GCST90001000/GCST90000514</td>
      <td>yes</td>
      <td>yes</td>
      <td>Genome-wide genotyping array</td>
      <td>34187846</td>
      <td>129,080 European ancestry cases,473,524 European ancestry controls</td>
      <td>462,753 cases,1,127,474 controls</td>
      <td>-</td>
      <td>68</td>
      <td>gastro oesophageal reflux gord gastric reflux</td>
    </tr>
  </tbody>
</table><h2 class='text-muted'>Information file</h2><table border="1" class="dataframe table table-bordered">
  <thead>
    <tr style="text-align: right;">
      <th>Unnamed: 0</th>
      <th>Name</th>
      <th>Last modified</th>
      <th>Size</th>
      <th>Description</th>
      <th>URL</th>
    </tr>
  </thead>
  <tbody>
    <tr>
      <td>2</td>
      <td>README.txt</td>
      <td>2021-07-28 00:02</td>
      <td>758</td>
      <td>NaN</td>
      <td>http://ftp.ebi.ac.uk/pub/databases/gwas/summary_statistics/GCST90000001-GCST90001000/GCST90000514/README.txt</td>
    </tr>
    <tr>
      <td>3</td>
      <td>harmonised/</td>
      <td>2024-02-02 15:18</td>
      <td>-</td>
      <td>NaN</td>
      <td>http://ftp.ebi.ac.uk/pub/databases/gwas/summary_statistics/GCST90000001-GCST90001000/GCST90000514/harmonised/</td>
    </tr>
    <tr>
      <td>4</td>
      <td>md5sum.txt</td>
      <td>2021-07-28 00:02</td>
      <td>109</td>
      <td>NaN</td>
      <td>http://ftp.ebi.ac.uk/pub/databases/gwas/summary_statistics/GCST90000001-GCST90001000/GCST90000514/md5sum.txt</td>
    </tr>
    <tr>
      <td>1</td>
      <td>GCST90000514_buildGRCh37.tsv.gz-meta.yaml</td>
      <td>2024-01-04 00:18</td>
      <td>1.5K</td>
      <td>NaN</td>
      <td>http://ftp.ebi.ac.uk/pub/databases/gwas/summary_statistics/GCST90000001-GCST90001000/GCST90000514/GCST90000514_buildGRCh37.tsv.gz-meta.yaml</td>
    </tr>
    <tr>
      <td>0</td>
      <td>GCST90000514_buildGRCh37.tsv.gz</td>
      <td>2021-07-28 00:02</td>
      <td>97M</td>
      <td>NaN</td>
      <td>http://ftp.ebi.ac.uk/pub/databases/gwas/summary_statistics/GCST90000001-GCST90001000/GCST90000514/GCST90000514_buildGRCh37.tsv.gz</td>
    </tr>
  </tbody>
</table><h2 class='text-muted'>Processed file</h2><p>Processing file: GCST90000514_buildGRCh37.tsv.gz!</p><h2 class='text-muted'>GWAS file head</h2><table border="1" class="dataframe table table-bordered">
  <thead>
    <tr style="text-align: right;">
      <th>variant_id</th>
      <th>chromosome</th>
      <th>base_pair_location</th>
      <th>effect_allele</th>
      <th>other_allele</th>
      <th>premtag_effectiven</th>
      <th>effect_allele_frequency</th>
      <th>beta</th>
      <th>standard_error</th>
      <th>p_value</th>
    </tr>
  </thead>
  <tbody>
    <tr>
      <td>rs10904045</td>
      <td>10</td>
      <td>94426</td>
      <td>C</td>
      <td>T</td>
      <td>54984.0</td>
      <td>0.603101</td>
      <td>0.008232</td>
      <td>0.004944</td>
      <td>0.095935</td>
    </tr>
    <tr>
      <td>rs11252127</td>
      <td>10</td>
      <td>98087</td>
      <td>C</td>
      <td>T</td>
      <td>54778.0</td>
      <td>0.723393</td>
      <td>-0.004030</td>
      <td>0.005411</td>
      <td>0.456405</td>
    </tr>
  </tbody>
</table></div><div class='container mt-4 bg-light p-4 text-center'><h2 class='text-muted'>Original GWAS file headers</h2><table border="1" class="dataframe table table-bordered">
  <thead>
    <tr style="text-align: right;">
      <th>variant_id</th>
      <th>chromosome</th>
      <th>base_pair_location</th>
      <th>effect_allele</th>
      <th>other_allele</th>
      <th>premtag_effectiven</th>
      <th>effect_allele_frequency</th>
      <th>beta</th>
      <th>standard_error</th>
      <th>p_value</th>
    </tr>
  </thead>
  <tbody>
    <tr>
      <td>rs10904045</td>
      <td>10</td>
      <td>94426</td>
      <td>C</td>
      <td>T</td>
      <td>54984.0</td>
      <td>0.603101</td>
      <td>0.008232</td>
      <td>0.004944</td>
      <td>0.095935</td>
    </tr>
    <tr>
      <td>rs11252127</td>
      <td>10</td>
      <td>98087</td>
      <td>C</td>
      <td>T</td>
      <td>54778.0</td>
      <td>0.723393</td>
      <td>-0.004030</td>
      <td>0.005411</td>
      <td>0.456405</td>
    </tr>
  </tbody>
</table><p style='font-weight: bold; color: blue;'>CHR -> ['chromosome']</p><p style='font-weight: bold; color: blue;'>SNP -> ['variant_id']</p><p style='font-weight: bold; color: blue;'>BP -> ['base_pair_location']</p><p style='font-weight: bold; color: blue;'>A1 -> ['effect_allele']</p><p style='font-weight: bold; color: blue;'>A2 -> ['other_allele']</p><p style='font-weight: bold; color: blue;'>N -> []</p><p style='font-weight: bold; color: blue;'>P -> ['p_value']</p><p style='font-weight: bold; color: blue;'>INFO -> []</p><p style='font-weight: bold; color: blue;'>MAF -> ['effect_allele_frequency']</p><p style='font-weight: bold; color: blue;'>BETA -> ['beta']</p><p style='font-weight: bold; color: blue;'>SE -> ['standard_error']</p><p style='font-weight: bold; color: blue;'>OR -> []</p><p style='font-weight: bold; color: blue;'>Z -> []</p><p style='font-weight: bold; color: blue;'>D -> []</p><h2 class='text-muted'>GWAS columns</h2>variant_id, chromosome, base_pair_location, effect_allele, other_allele, premtag_effectiven, effect_allele_frequency, beta, standard_error, p_value<h2 class='text-muted'>Identified columns</h2>chromosome, variant_id, base_pair_location, effect_allele, other_allele, p_value, effect_allele_frequency, beta, standard_error<h2 class='text-muted'>Unidentified Columns headers</h2>premtag_effectiven<h2 class='text-muted'>PMID</h2>34187846<h2 class='text-muted'>DOI</h2>10.1136/gutjnl-2020-323906<div class='bibtex-container'><h2 class='text-muted'>BibTeX</h2><div style='font-family: monospace; white-space: pre-wrap;'> @article{Ong_2021, title={Multitrait genetic association analysis identifies 50 new risk loci for gastro-oesophageal reflux, seven new loci for Barrett’s oesophagus and provides insights into clinical heterogeneity in reflux diagnosis}, volume={71}, ISSN={1468-3288}, url={http://dx.doi.org/10.1136/gutjnl-2020-323906}, DOI={10.1136/gutjnl-2020-323906}, number={6}, journal={Gut}, publisher={BMJ}, author={Ong, Jue-Sheng and An, Jiyuan and Han, Xikun and Law, Matthew H and Nandakumar, Priyanka and Schumacher, Johannes and Gockel, Ines and Bohmer, Anne and Jankowski, Janusz and Palles, Claire and Olsen, Catherine M and Neale, Rachel E and Fitzgerald, Rebecca and Thrift, Aaron P and Vaughan, Thomas L and Buas, Matthew F and Hinds, David A and Gharahkhani, Puya and Kendall, Bradley J and MacGregor, Stuart}, year={2021}, month=jun, pages={1053–1061} }
</div></div><br><br><br><br><br><br></div><div class='container mt-4 bg-light p-4 text-center'><h1 class='text-primary'>Phenotype: gastroesophageal reflux disease</h1><p class='text-muted'><strong>URL:</strong> http://ftp.ebi.ac.uk/pub/databases/gwas/summary_statistics/GCST90018001-GCST90019000/GCST90018848</p><table border="1" class="dataframe table table-bordered">
  <thead>
    <tr style="text-align: right;">
      <th>firstAuthor</th>
      <th>accessionId</th>
      <th>publicationDate</th>
      <th>journal</th>
      <th>title</th>
      <th>reportedTrait</th>
      <th>efoTraits</th>
      <th>bgTraits</th>
      <th>discoverySampleAncestry</th>
      <th>replicationSampleAncestry</th>
      <th>associationCount</th>
      <th>summaryStatistics</th>
      <th>ssApiFlag</th>
      <th>agreedToCc0</th>
      <th>genotypingTechnologies</th>
      <th>pubmedId</th>
      <th>initialSampleDescription</th>
      <th>replicateSampleDescription</th>
      <th>UKB_FIELD</th>
      <th>FuzzySimilarity_gastro oesophageal reflux gord gastric reflux</th>
      <th>SearchPhenotype</th>
    </tr>
  </thead>
  <tbody>
    <tr>
      <td>Sakaue S</td>
      <td>GCST90018848</td>
      <td>9/30/2021</td>
      <td>Nat Genet</td>
      <td>A cross-population atlas of genetic associations for 220 human phenotypes.</td>
      <td>gastroesophageal reflux disease</td>
      <td>gastroesophageal reflux disease</td>
      <td>-</td>
      <td>467253 european,178464 east asian</td>
      <td>NaN</td>
      <td>2</td>
      <td>http://ftp.ebi.ac.uk/pub/databases/gwas/summary_statistics/GCST90018001-GCST90019000/GCST90018848</td>
      <td>no</td>
      <td>yes</td>
      <td>Genome-wide genotyping array</td>
      <td>34594039</td>
      <td>32,957 European ancestry cases,434,296 European ancestry controls, 948 East Asian ancestry cases,177,516 East Asian ancestry controls</td>
      <td>NaN</td>
      <td>-</td>
      <td>68</td>
      <td>gastro oesophageal reflux gord gastric reflux</td>
    </tr>
  </tbody>
</table><h2 class='text-muted'>Information file</h2><table border="1" class="dataframe table table-bordered">
  <thead>
    <tr style="text-align: right;">
      <th>Unnamed: 0</th>
      <th>Name</th>
      <th>Last modified</th>
      <th>Size</th>
      <th>Description</th>
      <th>URL</th>
    </tr>
  </thead>
  <tbody>
    <tr>
      <td>2</td>
      <td>README.txt</td>
      <td>2021-12-16 01:02</td>
      <td>177</td>
      <td>NaN</td>
      <td>http://ftp.ebi.ac.uk/pub/databases/gwas/summary_statistics/GCST90018001-GCST90019000/GCST90018848/README.txt</td>
    </tr>
    <tr>
      <td>3</td>
      <td>harmonised/</td>
      <td>2024-02-02 15:17</td>
      <td>-</td>
      <td>NaN</td>
      <td>http://ftp.ebi.ac.uk/pub/databases/gwas/summary_statistics/GCST90018001-GCST90019000/GCST90018848/harmonised/</td>
    </tr>
    <tr>
      <td>4</td>
      <td>md5sum.txt</td>
      <td>2021-12-16 01:02</td>
      <td>109</td>
      <td>NaN</td>
      <td>http://ftp.ebi.ac.uk/pub/databases/gwas/summary_statistics/GCST90018001-GCST90019000/GCST90018848/md5sum.txt</td>
    </tr>
    <tr>
      <td>1</td>
      <td>GCST90018848_buildGRCh37.tsv.gz-meta.yaml</td>
      <td>2024-01-04 00:20</td>
      <td>1.1K</td>
      <td>NaN</td>
      <td>http://ftp.ebi.ac.uk/pub/databases/gwas/summary_statistics/GCST90018001-GCST90019000/GCST90018848/GCST90018848_buildGRCh37.tsv.gz-meta.yaml</td>
    </tr>
    <tr>
      <td>0</td>
      <td>GCST90018848_buildGRCh37.tsv.gz</td>
      <td>2021-12-16 01:02</td>
      <td>504M</td>
      <td>NaN</td>
      <td>http://ftp.ebi.ac.uk/pub/databases/gwas/summary_statistics/GCST90018001-GCST90019000/GCST90018848/GCST90018848_buildGRCh37.tsv.gz</td>
    </tr>
  </tbody>
</table><h2 class='text-muted'>Processed file</h2><p>Processing file: GCST90018848_buildGRCh37.tsv.gz!</p><h2 class='text-muted'>GWAS file head</h2><table border="1" class="dataframe table table-bordered">
  <thead>
    <tr style="text-align: right;">
      <th>chromosome</th>
      <th>base_pair_location</th>
      <th>effect_allele</th>
      <th>other_allele</th>
      <th>effect_allele_frequency</th>
      <th>beta</th>
      <th>standard_error</th>
      <th>p_value</th>
      <th>variant_id</th>
    </tr>
  </thead>
  <tbody>
    <tr>
      <td>1</td>
      <td>100000012</td>
      <td>T</td>
      <td>G</td>
      <td>0.281747</td>
      <td>0.0065</td>
      <td>0.0095</td>
      <td>0.4928</td>
      <td>NaN</td>
    </tr>
    <tr>
      <td>1</td>
      <td>10000006</td>
      <td>A</td>
      <td>G</td>
      <td>0.005637</td>
      <td>0.0100</td>
      <td>0.0600</td>
      <td>0.8674</td>
      <td>NaN</td>
    </tr>
  </tbody>
</table></div><div class='container mt-4 bg-light p-4 text-center'><h2 class='text-muted'>Original GWAS file headers</h2><table border="1" class="dataframe table table-bordered">
  <thead>
    <tr style="text-align: right;">
      <th>chromosome</th>
      <th>base_pair_location</th>
      <th>effect_allele</th>
      <th>other_allele</th>
      <th>effect_allele_frequency</th>
      <th>beta</th>
      <th>standard_error</th>
      <th>p_value</th>
      <th>variant_id</th>
    </tr>
  </thead>
  <tbody>
    <tr>
      <td>1</td>
      <td>100000012</td>
      <td>T</td>
      <td>G</td>
      <td>0.281747</td>
      <td>0.0065</td>
      <td>0.0095</td>
      <td>0.4928</td>
      <td>NaN</td>
    </tr>
    <tr>
      <td>1</td>
      <td>10000006</td>
      <td>A</td>
      <td>G</td>
      <td>0.005637</td>
      <td>0.0100</td>
      <td>0.0600</td>
      <td>0.8674</td>
      <td>NaN</td>
    </tr>
  </tbody>
</table><p style='font-weight: bold; color: blue;'>CHR -> ['chromosome']</p><p style='font-weight: bold; color: blue;'>SNP -> ['variant_id']</p><p style='font-weight: bold; color: blue;'>BP -> ['base_pair_location']</p><p style='font-weight: bold; color: blue;'>A1 -> ['effect_allele']</p><p style='font-weight: bold; color: blue;'>A2 -> ['other_allele']</p><p style='font-weight: bold; color: blue;'>N -> []</p><p style='font-weight: bold; color: blue;'>P -> ['p_value']</p><p style='font-weight: bold; color: blue;'>INFO -> []</p><p style='font-weight: bold; color: blue;'>MAF -> ['effect_allele_frequency']</p><p style='font-weight: bold; color: blue;'>BETA -> ['beta']</p><p style='font-weight: bold; color: blue;'>SE -> ['standard_error']</p><p style='font-weight: bold; color: blue;'>OR -> []</p><p style='font-weight: bold; color: blue;'>Z -> []</p><p style='font-weight: bold; color: blue;'>D -> []</p><h2 class='text-muted'>GWAS columns</h2>chromosome, base_pair_location, effect_allele, other_allele, effect_allele_frequency, beta, standard_error, p_value, variant_id<h2 class='text-muted'>Identified columns</h2>chromosome, variant_id, base_pair_location, effect_allele, other_allele, p_value, effect_allele_frequency, beta, standard_error<h2 class='text-muted'>Unidentified Columns headers</h2><h2 class='text-muted'>PMID</h2>34594039<h2 class='text-muted'>DOI</h2>Could not find DOI!<div class='bibtex-container'><h2 class='text-muted'>BibTeX</h2><div style='font-family: monospace; white-space: pre-wrap;'></div></div><br><br><br><br><br><br></div><div class='container mt-4 bg-light p-4 text-center'><h1 class='text-primary'>Phenotype: icd10 k21: gastro-esophageal reflux disease</h1><p class='text-muted'><strong>URL:</strong> http://ftp.ebi.ac.uk/pub/databases/gwas/summary_statistics/GCST90080001-GCST90081000/GCST90080188</p><table border="1" class="dataframe table table-bordered">
  <thead>
    <tr style="text-align: right;">
      <th>firstAuthor</th>
      <th>accessionId</th>
      <th>publicationDate</th>
      <th>journal</th>
      <th>title</th>
      <th>reportedTrait</th>
      <th>efoTraits</th>
      <th>bgTraits</th>
      <th>discoverySampleAncestry</th>
      <th>replicationSampleAncestry</th>
      <th>associationCount</th>
      <th>summaryStatistics</th>
      <th>ssApiFlag</th>
      <th>agreedToCc0</th>
      <th>genotypingTechnologies</th>
      <th>pubmedId</th>
      <th>initialSampleDescription</th>
      <th>replicateSampleDescription</th>
      <th>UKB_FIELD</th>
      <th>FuzzySimilarity_gastro oesophageal reflux gord gastric reflux</th>
      <th>SearchPhenotype</th>
    </tr>
  </thead>
  <tbody>
    <tr>
      <td>Backman JD</td>
      <td>GCST90080188</td>
      <td>10/18/2021</td>
      <td>Nature</td>
      <td>Exome sequencing and analysis of 454,787 UK Biobank participants.</td>
      <td>icd10 k21: gastro-esophageal reflux disease</td>
      <td>gastroesophageal reflux disease</td>
      <td>-</td>
      <td>376648 european</td>
      <td>NaN</td>
      <td>0</td>
      <td>http://ftp.ebi.ac.uk/pub/databases/gwas/summary_statistics/GCST90080001-GCST90081000/GCST90080188</td>
      <td>no</td>
      <td>yes</td>
      <td>Exome-wide sequencing,Genome-wide genotyping array</td>
      <td>34662886</td>
      <td>34,552 European ancestry cases,342,096 European ancestry controls</td>
      <td>NaN</td>
      <td>-</td>
      <td>53</td>
      <td>gastro oesophageal reflux gord gastric reflux</td>
    </tr>
  </tbody>
</table><h2 class='text-muted'>Information file</h2><table border="1" class="dataframe table table-bordered">
  <thead>
    <tr style="text-align: right;">
      <th>Unnamed: 0</th>
      <th>Name</th>
      <th>Last modified</th>
      <th>Size</th>
      <th>Description</th>
      <th>URL</th>
    </tr>
  </thead>
  <tbody>
    <tr>
      <td>1</td>
      <td>GCST90080188_buildGRCh38.tsv.gz-meta.yaml</td>
      <td>2024-01-04 00:30</td>
      <td>917</td>
      <td>NaN</td>
      <td>http://ftp.ebi.ac.uk/pub/databases/gwas/summary_statistics/GCST90080001-GCST90081000/GCST90080188/GCST90080188_buildGRCh38.tsv.gz-meta.yaml</td>
    </tr>
    <tr>
      <td>2</td>
      <td>README.txt</td>
      <td>2021-11-18 01:03</td>
      <td>698</td>
      <td>NaN</td>
      <td>http://ftp.ebi.ac.uk/pub/databases/gwas/summary_statistics/GCST90080001-GCST90081000/GCST90080188/README.txt</td>
    </tr>
    <tr>
      <td>3</td>
      <td>harmonised/</td>
      <td>2024-02-02 15:15</td>
      <td>-</td>
      <td>NaN</td>
      <td>http://ftp.ebi.ac.uk/pub/databases/gwas/summary_statistics/GCST90080001-GCST90081000/GCST90080188/harmonised/</td>
    </tr>
    <tr>
      <td>4</td>
      <td>md5sum.txt</td>
      <td>2021-11-18 01:03</td>
      <td>109</td>
      <td>NaN</td>
      <td>http://ftp.ebi.ac.uk/pub/databases/gwas/summary_statistics/GCST90080001-GCST90081000/GCST90080188/md5sum.txt</td>
    </tr>
    <tr>
      <td>0</td>
      <td>GCST90080188_buildGRCh38.tsv.gz</td>
      <td>2021-11-18 01:03</td>
      <td>16M</td>
      <td>NaN</td>
      <td>http://ftp.ebi.ac.uk/pub/databases/gwas/summary_statistics/GCST90080001-GCST90081000/GCST90080188/GCST90080188_buildGRCh38.tsv.gz</td>
    </tr>
  </tbody>
</table><h2 class='text-muted'>Processed file</h2><p>Processing file: GCST90080188_buildGRCh38.tsv.gz!</p><h2 class='text-muted'>GWAS file head</h2><table border="1" class="dataframe table table-bordered">
  <thead>
    <tr style="text-align: right;">
      <th>name</th>
      <th>chromosome</th>
      <th>base_pair_location</th>
      <th>other_allele</th>
      <th>effect_allele</th>
      <th>trait</th>
      <th>cohort</th>
      <th>model</th>
      <th>odds_ratio</th>
      <th>ci_lower</th>
      <th>ci_upper</th>
      <th>p_value</th>
      <th>effect_allele_frequency</th>
      <th>standard_error</th>
    </tr>
  </thead>
  <tbody>
    <tr>
      <td>1_930158_C_T</td>
      <td>1</td>
      <td>930158</td>
      <td>C</td>
      <td>T</td>
      <td>Phe10_K21__BIN</td>
      <td>UKB_Freeze_450_EUR</td>
      <td>ADD-WGR-FIRTH</td>
      <td>6.979020</td>
      <td>1.157500</td>
      <td>42.07930</td>
      <td>0.034047</td>
      <td>0.000008</td>
      <td>0.916673</td>
    </tr>
    <tr>
      <td>1_930165_G_A</td>
      <td>1</td>
      <td>930165</td>
      <td>G</td>
      <td>A</td>
      <td>Phe10_K21__BIN</td>
      <td>UKB_Freeze_450_EUR</td>
      <td>ADD-WGR-FIRTH</td>
      <td>0.676424</td>
      <td>0.206032</td>
      <td>2.22076</td>
      <td>0.519226</td>
      <td>0.000041</td>
      <td>0.606535</td>
    </tr>
  </tbody>
</table></div><div class='container mt-4 bg-light p-4 text-center'><h2 class='text-muted'>Original GWAS file headers</h2><table border="1" class="dataframe table table-bordered">
  <thead>
    <tr style="text-align: right;">
      <th>name</th>
      <th>chromosome</th>
      <th>base_pair_location</th>
      <th>other_allele</th>
      <th>effect_allele</th>
      <th>trait</th>
      <th>cohort</th>
      <th>model</th>
      <th>odds_ratio</th>
      <th>ci_lower</th>
      <th>ci_upper</th>
      <th>p_value</th>
      <th>effect_allele_frequency</th>
      <th>standard_error</th>
    </tr>
  </thead>
  <tbody>
    <tr>
      <td>1_930158_C_T</td>
      <td>1</td>
      <td>930158</td>
      <td>C</td>
      <td>T</td>
      <td>Phe10_K21__BIN</td>
      <td>UKB_Freeze_450_EUR</td>
      <td>ADD-WGR-FIRTH</td>
      <td>6.979020</td>
      <td>1.157500</td>
      <td>42.07930</td>
      <td>0.034047</td>
      <td>0.000008</td>
      <td>0.916673</td>
    </tr>
    <tr>
      <td>1_930165_G_A</td>
      <td>1</td>
      <td>930165</td>
      <td>G</td>
      <td>A</td>
      <td>Phe10_K21__BIN</td>
      <td>UKB_Freeze_450_EUR</td>
      <td>ADD-WGR-FIRTH</td>
      <td>0.676424</td>
      <td>0.206032</td>
      <td>2.22076</td>
      <td>0.519226</td>
      <td>0.000041</td>
      <td>0.606535</td>
    </tr>
  </tbody>
</table><p style='font-weight: bold; color: blue;'>CHR -> ['chromosome']</p><p style='font-weight: bold; color: blue;'>SNP -> ['name']</p><p style='font-weight: bold; color: blue;'>BP -> ['name', 'base_pair_location']</p><p style='font-weight: bold; color: blue;'>A1 -> ['effect_allele']</p><p style='font-weight: bold; color: blue;'>A2 -> ['other_allele']</p><p style='font-weight: bold; color: blue;'>N -> []</p><p style='font-weight: bold; color: blue;'>P -> ['p_value']</p><p style='font-weight: bold; color: blue;'>INFO -> []</p><p style='font-weight: bold; color: blue;'>MAF -> ['effect_allele_frequency']</p><p style='font-weight: bold; color: blue;'>BETA -> []</p><p style='font-weight: bold; color: blue;'>SE -> ['standard_error']</p><p style='font-weight: bold; color: blue;'>OR -> ['odds_ratio']</p><p style='font-weight: bold; color: blue;'>Z -> []</p><p style='font-weight: bold; color: blue;'>D -> []</p><h2 class='text-muted'>GWAS columns</h2>name, chromosome, base_pair_location, other_allele, effect_allele, trait, cohort, model, odds_ratio, ci_lower, ci_upper, p_value, effect_allele_frequency, standard_error<h2 class='text-muted'>Identified columns</h2>chromosome, name, name, base_pair_location, effect_allele, other_allele, p_value, effect_allele_frequency, standard_error, odds_ratio<h2 class='text-muted'>Unidentified Columns headers</h2>model, ci_upper, cohort, ci_lower, trait<h2 class='text-muted'>PMID</h2>34662886<h2 class='text-muted'>DOI</h2>10.1038/s41586-021-04103-z<div class='bibtex-container'><h2 class='text-muted'>BibTeX</h2><div style='font-family: monospace; white-space: pre-wrap;'> @article{Backman_2021, title={Exome sequencing and analysis of 454,787 UK Biobank participants}, volume={599}, ISSN={1476-4687}, url={http://dx.doi.org/10.1038/s41586-021-04103-z}, DOI={10.1038/s41586-021-04103-z}, number={7886}, journal={Nature}, publisher={Springer Science and Business Media LLC}, author={Backman, Joshua D. and Li, Alexander H. and Marcketta, Anthony and Sun, Dylan and Mbatchou, Joelle and Kessler, Michael D. and Benner, Christian and Liu, Daren and Locke, Adam E. and Balasubramanian, Suganthi and Yadav, Ashish and Banerjee, Nilanjana and Gillies, Christopher E. and Damask, Amy and Liu, Simon and Bai, Xiaodong and Hawes, Alicia and Maxwell, Evan and Gurski, Lauren and Watanabe, Kyoko and Kosmicki, Jack A. and Rajagopal, Veera and Mighty, Jason and Jones, Marcus and Mitnaul, Lyndon and Stahl, Eli and Coppola, Giovanni and Jorgenson, Eric and Habegger, Lukas and Salerno, William J. and Shuldiner, Alan R. and Lotta, Luca A. and Overton, John D. and Cantor, Michael N. and Reid, Jeffrey G. and Yancopoulos, George and Kang, Hyun M. and Marchini, Jonathan and Baras, Aris and Abecasis, Gonçalo R. and Ferreira, Manuel A. R.}, year={2021}, month=oct, pages={628–634} }
</div></div><br><br><br><br><br><br></div><div class='container mt-4 bg-light p-4 text-center'><h1 class='text-primary'>Phenotype: icd10 k21: gastro-esophageal reflux disease</h1><p class='text-muted'><strong>URL:</strong> http://ftp.ebi.ac.uk/pub/databases/gwas/summary_statistics/GCST90246001-GCST90247000/GCST90246130</p><table border="1" class="dataframe table table-bordered">
  <thead>
    <tr style="text-align: right;">
      <th>firstAuthor</th>
      <th>accessionId</th>
      <th>publicationDate</th>
      <th>journal</th>
      <th>title</th>
      <th>reportedTrait</th>
      <th>efoTraits</th>
      <th>bgTraits</th>
      <th>discoverySampleAncestry</th>
      <th>replicationSampleAncestry</th>
      <th>associationCount</th>
      <th>summaryStatistics</th>
      <th>ssApiFlag</th>
      <th>agreedToCc0</th>
      <th>genotypingTechnologies</th>
      <th>pubmedId</th>
      <th>initialSampleDescription</th>
      <th>replicateSampleDescription</th>
      <th>UKB_FIELD</th>
      <th>FuzzySimilarity_gastro oesophageal reflux gord gastric reflux</th>
      <th>SearchPhenotype</th>
    </tr>
  </thead>
  <tbody>
    <tr>
      <td>Walters RG</td>
      <td>GCST90246130</td>
      <td>7/20/2023</td>
      <td>Cell Genom</td>
      <td>Genotyping and population characteristics of the China Kadoorie Biobank.</td>
      <td>icd10 k21: gastro-esophageal reflux disease</td>
      <td>gastroesophageal reflux disease</td>
      <td>-</td>
      <td>75899 east asian</td>
      <td>NaN</td>
      <td>0</td>
      <td>http://ftp.ebi.ac.uk/pub/databases/gwas/summary_statistics/GCST90246001-GCST90247000/GCST90246130</td>
      <td>no</td>
      <td>yes</td>
      <td>Genome-wide genotyping array</td>
      <td>37601966</td>
      <td>75,782 Chinese ancestry controls</td>
      <td>NaN</td>
      <td>-</td>
      <td>53</td>
      <td>gastro oesophageal reflux gord gastric reflux</td>
    </tr>
  </tbody>
</table><h2 class='text-muted'>Information file</h2><h2 class='text-muted'>Information file</h2><p>Information file is missing!</p><h2 class='text-muted'>GWAS file</h2><p>We were unable to process the GWAS file!</p>