<link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/bootstrap/4.0.0/css/bootstrap.min.css" integrity="sha384-Gn5384xqQ1aoWXA+058RXPxPg6fy4IWvTNh0E263XmFcJlSAwiGgFAW/dAiS6JXm" crossorigin="anonymous"><div class='container mt-4 bg-light p-4 text-center'><h1 class='text-primary'>Phenotype: high cholesterol</h1><p class='text-muted'><strong>URL:</strong> http://ftp.ebi.ac.uk/pub/databases/gwas/summary_statistics/GCST90038001-GCST90039000/GCST90038690</p><table border="1" class="dataframe table table-bordered">
  <thead>
    <tr style="text-align: right;">
      <th>firstAuthor</th>
      <th>accessionId</th>
      <th>publicationDate</th>
      <th>journal</th>
      <th>title</th>
      <th>reportedTrait</th>
      <th>efoTraits</th>
      <th>bgTraits</th>
      <th>discoverySampleAncestry</th>
      <th>replicationSampleAncestry</th>
      <th>associationCount</th>
      <th>summaryStatistics</th>
      <th>ssApiFlag</th>
      <th>agreedToCc0</th>
      <th>genotypingTechnologies</th>
      <th>pubmedId</th>
      <th>initialSampleDescription</th>
      <th>replicateSampleDescription</th>
      <th>UKB_FIELD</th>
      <th>FuzzySimilarity_high cholesterol</th>
      <th>SearchPhenotype</th>
    </tr>
  </thead>
  <tbody>
    <tr>
      <td>D??nerta?� HM</td>
      <td>GCST90038690</td>
      <td>4/8/2021</td>
      <td>Nat Aging</td>
      <td>Common genetic associations between age-related diseases.</td>
      <td>high cholesterol</td>
      <td>hypercholesterolemia</td>
      <td>-</td>
      <td>484598 nr</td>
      <td>NaN</td>
      <td>0</td>
      <td>http://ftp.ebi.ac.uk/pub/databases/gwas/summary_statistics/GCST90038001-GCST90039000/GCST90038690</td>
      <td>no</td>
      <td>yes</td>
      <td>Genome-wide genotyping array</td>
      <td>33959723</td>
      <td>59,853 cases,424,745 controls</td>
      <td>NaN</td>
      <td>-</td>
      <td>100</td>
      <td>high cholesterol</td>
    </tr>
  </tbody>
</table><h2 class='text-muted'>Information file</h2><table border="1" class="dataframe table table-bordered">
  <thead>
    <tr style="text-align: right;">
      <th>Unnamed: 0</th>
      <th>Name</th>
      <th>Last modified</th>
      <th>Size</th>
      <th>Description</th>
      <th>URL</th>
    </tr>
  </thead>
  <tbody>
    <tr>
      <td>1</td>
      <td>GCST90038690_buildGRCh37.tsv-meta.yaml</td>
      <td>2024-01-04 00:37</td>
      <td>666</td>
      <td>NaN</td>
      <td>http://ftp.ebi.ac.uk/pub/databases/gwas/summary_statistics/GCST90038001-GCST90039000/GCST90038690/GCST90038690_buildGRCh37.tsv-meta.yaml</td>
    </tr>
    <tr>
      <td>2</td>
      <td>harmonised/</td>
      <td>2024-02-02 15:16</td>
      <td>-</td>
      <td>NaN</td>
      <td>http://ftp.ebi.ac.uk/pub/databases/gwas/summary_statistics/GCST90038001-GCST90039000/GCST90038690/harmonised/</td>
    </tr>
    <tr>
      <td>3</td>
      <td>md5sum.txt</td>
      <td>2021-11-14 00:27</td>
      <td>62</td>
      <td>NaN</td>
      <td>http://ftp.ebi.ac.uk/pub/databases/gwas/summary_statistics/GCST90038001-GCST90039000/GCST90038690/md5sum.txt</td>
    </tr>
    <tr>
      <td>0</td>
      <td>GCST90038690_buildGRCh37.tsv</td>
      <td>2021-11-14 00:27</td>
      <td>1.2G</td>
      <td>NaN</td>
      <td>http://ftp.ebi.ac.uk/pub/databases/gwas/summary_statistics/GCST90038001-GCST90039000/GCST90038690/GCST90038690_buildGRCh37.tsv</td>
    </tr>
  </tbody>
</table><h2 class='text-muted'>Processed file</h2><p>Processing file: GCST90038690_buildGRCh37.tsv.1!</p><h2 class='text-muted'>GWAS file head</h2><table border="1" class="dataframe table table-bordered">
  <thead>
    <tr style="text-align: right;">
      <th>variant_id</th>
      <th>chromosome</th>
      <th>base_pair_location</th>
      <th>genpos</th>
      <th>effect_allele</th>
      <th>other_allele</th>
      <th>effect_allele_frequency</th>
      <th>info</th>
      <th>chisq_linreg</th>
      <th>p_linreg</th>
      <th>beta</th>
      <th>standard_error</th>
      <th>chisq_bolt_lmm_inf</th>
      <th>p_bolt_lmm_inf</th>
      <th>chisq_bolt_lmm</th>
      <th>p_value</th>
    </tr>
  </thead>
  <tbody>
    <tr>
      <td>rs533090414</td>
      <td>1</td>
      <td>18849</td>
      <td>0.0</td>
      <td>C</td>
      <td>G</td>
      <td>0.026465</td>
      <td>0.536412</td>
      <td>0.309205</td>
      <td>0.58</td>
      <td>-0.001335</td>
      <td>0.002770</td>
      <td>0.232182</td>
      <td>0.63</td>
      <td>0.171286</td>
      <td>0.68</td>
    </tr>
    <tr>
      <td>rs568927205</td>
      <td>1</td>
      <td>54712</td>
      <td>0.0</td>
      <td>T</td>
      <td>TTTTC</td>
      <td>0.415825</td>
      <td>0.538572</td>
      <td>0.849309</td>
      <td>0.36</td>
      <td>0.000519</td>
      <td>0.000892</td>
      <td>0.338778</td>
      <td>0.56</td>
      <td>0.428563</td>
      <td>0.51</td>
    </tr>
  </tbody>
</table></div><div class='container mt-4 bg-light p-4 text-center'><h2 class='text-muted'>Original GWAS file headers</h2><table border="1" class="dataframe table table-bordered">
  <thead>
    <tr style="text-align: right;">
      <th>variant_id</th>
      <th>chromosome</th>
      <th>base_pair_location</th>
      <th>genpos</th>
      <th>effect_allele</th>
      <th>other_allele</th>
      <th>effect_allele_frequency</th>
      <th>info</th>
      <th>chisq_linreg</th>
      <th>p_linreg</th>
      <th>beta</th>
      <th>standard_error</th>
      <th>chisq_bolt_lmm_inf</th>
      <th>p_bolt_lmm_inf</th>
      <th>chisq_bolt_lmm</th>
      <th>p_value</th>
    </tr>
  </thead>
  <tbody>
    <tr>
      <td>rs533090414</td>
      <td>1</td>
      <td>18849</td>
      <td>0.0</td>
      <td>C</td>
      <td>G</td>
      <td>0.026465</td>
      <td>0.536412</td>
      <td>0.309205</td>
      <td>0.58</td>
      <td>-0.001335</td>
      <td>0.002770</td>
      <td>0.232182</td>
      <td>0.63</td>
      <td>0.171286</td>
      <td>0.68</td>
    </tr>
    <tr>
      <td>rs568927205</td>
      <td>1</td>
      <td>54712</td>
      <td>0.0</td>
      <td>T</td>
      <td>TTTTC</td>
      <td>0.415825</td>
      <td>0.538572</td>
      <td>0.849309</td>
      <td>0.36</td>
      <td>0.000519</td>
      <td>0.000892</td>
      <td>0.338778</td>
      <td>0.56</td>
      <td>0.428563</td>
      <td>0.51</td>
    </tr>
  </tbody>
</table><p style='font-weight: bold; color: blue;'>CHR -> ['chromosome']</p><p style='font-weight: bold; color: blue;'>SNP -> ['variant_id']</p><p style='font-weight: bold; color: blue;'>BP -> ['base_pair_location', 'genpos']</p><p style='font-weight: bold; color: blue;'>A1 -> ['effect_allele']</p><p style='font-weight: bold; color: blue;'>A2 -> ['other_allele']</p><p style='font-weight: bold; color: blue;'>N -> []</p><p style='font-weight: bold; color: blue;'>P -> ['p_value']</p><p style='font-weight: bold; color: blue;'>INFO -> ['info']</p><p style='font-weight: bold; color: blue;'>MAF -> ['effect_allele_frequency']</p><p style='font-weight: bold; color: blue;'>BETA -> ['beta', 'p_bolt_lmm_inf']</p><p style='font-weight: bold; color: blue;'>SE -> ['standard_error']</p><p style='font-weight: bold; color: blue;'>OR -> []</p><p style='font-weight: bold; color: blue;'>Z -> []</p><p style='font-weight: bold; color: blue;'>D -> []</p><h2 class='text-muted'>GWAS columns</h2>variant_id, chromosome, base_pair_location, genpos, effect_allele, other_allele, effect_allele_frequency, info, chisq_linreg, p_linreg, beta, standard_error, chisq_bolt_lmm_inf, p_bolt_lmm_inf, chisq_bolt_lmm, p_value<h2 class='text-muted'>Identified columns</h2>chromosome, variant_id, base_pair_location, genpos, effect_allele, other_allele, p_value, info, effect_allele_frequency, beta, p_bolt_lmm_inf, standard_error<h2 class='text-muted'>Unidentified Columns headers</h2>chisq_bolt_lmm_inf, p_linreg, chisq_bolt_lmm, chisq_linreg<h2 class='text-muted'>PMID</h2>33959723<h2 class='text-muted'>DOI</h2>10.1038/s43587-021-00051-5<div class='bibtex-container'><h2 class='text-muted'>BibTeX</h2><div style='font-family: monospace; white-space: pre-wrap;'> @article{D_nerta__2021, title={Common genetic associations between age-related diseases}, volume={1}, ISSN={2662-8465}, url={http://dx.doi.org/10.1038/s43587-021-00051-5}, DOI={10.1038/s43587-021-00051-5}, number={4}, journal={Nature Aging}, publisher={Springer Science and Business Media LLC}, author={Dönertaş, Handan Melike and Fabian, Daniel K. and Fuentealba, Matías and Partridge, Linda and Thornton, Janet M.}, year={2021}, month=apr, pages={400–412} }
</div></div><br><br><br><br><br><br></div><div class='container mt-4 bg-light p-4 text-center'><h1 class='text-primary'>Phenotype: high cholesterol</h1><p class='text-muted'><strong>URL:</strong> http://ftp.ebi.ac.uk/pub/databases/gwas/summary_statistics/GCST90029001-GCST90030000/GCST90029021</p><table border="1" class="dataframe table table-bordered">
  <thead>
    <tr style="text-align: right;">
      <th>firstAuthor</th>
      <th>accessionId</th>
      <th>publicationDate</th>
      <th>journal</th>
      <th>title</th>
      <th>reportedTrait</th>
      <th>efoTraits</th>
      <th>bgTraits</th>
      <th>discoverySampleAncestry</th>
      <th>replicationSampleAncestry</th>
      <th>associationCount</th>
      <th>summaryStatistics</th>
      <th>ssApiFlag</th>
      <th>agreedToCc0</th>
      <th>genotypingTechnologies</th>
      <th>pubmedId</th>
      <th>initialSampleDescription</th>
      <th>replicateSampleDescription</th>
      <th>UKB_FIELD</th>
      <th>FuzzySimilarity_high cholesterol</th>
      <th>SearchPhenotype</th>
    </tr>
  </thead>
  <tbody>
    <tr>
      <td>Loh PR</td>
      <td>GCST90029021</td>
      <td>7/1/2018</td>
      <td>Nat Genet</td>
      <td>Mixed-model association for biobank-scale datasets.</td>
      <td>high cholesterol</td>
      <td>hypercholesterolemia</td>
      <td>-</td>
      <td>464736 european</td>
      <td>NaN</td>
      <td>0</td>
      <td>http://ftp.ebi.ac.uk/pub/databases/gwas/summary_statistics/GCST90029001-GCST90030000/GCST90029021</td>
      <td>no</td>
      <td>no</td>
      <td>Genome-wide genotyping array</td>
      <td>29892013</td>
      <td>464,736 European ancestry individuals</td>
      <td>NaN</td>
      <td>-</td>
      <td>100</td>
      <td>high cholesterol</td>
    </tr>
  </tbody>
</table><h2 class='text-muted'>Information file</h2><table border="1" class="dataframe table table-bordered">
  <thead>
    <tr style="text-align: right;">
      <th>Unnamed: 0</th>
      <th>Name</th>
      <th>Last modified</th>
      <th>Size</th>
      <th>Description</th>
      <th>URL</th>
    </tr>
  </thead>
  <tbody>
    <tr>
      <td>1</td>
      <td>GCST90029021_buildGRCh37.tsv-meta.yaml</td>
      <td>2024-01-04 00:36</td>
      <td>672</td>
      <td>NaN</td>
      <td>http://ftp.ebi.ac.uk/pub/databases/gwas/summary_statistics/GCST90029001-GCST90030000/GCST90029021/GCST90029021_buildGRCh37.tsv-meta.yaml</td>
    </tr>
    <tr>
      <td>2</td>
      <td>harmonised/</td>
      <td>2024-02-02 15:15</td>
      <td>-</td>
      <td>NaN</td>
      <td>http://ftp.ebi.ac.uk/pub/databases/gwas/summary_statistics/GCST90029001-GCST90030000/GCST90029021/harmonised/</td>
    </tr>
    <tr>
      <td>3</td>
      <td>md5sum.txt</td>
      <td>2021-09-25 00:23</td>
      <td>62</td>
      <td>NaN</td>
      <td>http://ftp.ebi.ac.uk/pub/databases/gwas/summary_statistics/GCST90029001-GCST90030000/GCST90029021/md5sum.txt</td>
    </tr>
    <tr>
      <td>0</td>
      <td>GCST90029021_buildGRCh37.tsv</td>
      <td>2021-09-25 00:23</td>
      <td>962M</td>
      <td>NaN</td>
      <td>http://ftp.ebi.ac.uk/pub/databases/gwas/summary_statistics/GCST90029001-GCST90030000/GCST90029021/GCST90029021_buildGRCh37.tsv</td>
    </tr>
  </tbody>
</table><h2 class='text-muted'>Processed file</h2><p>Processing file: GCST90029021_buildGRCh37.tsv.1!</p><h2 class='text-muted'>GWAS file head</h2><table border="1" class="dataframe table table-bordered">
  <thead>
    <tr style="text-align: right;">
      <th>variant_id</th>
      <th>chromosome</th>
      <th>base_pair_location</th>
      <th>effect_allele</th>
      <th>other_allele</th>
      <th>ref</th>
      <th>effect_allele_frequency</th>
      <th>beta</th>
      <th>standard_error</th>
      <th>p_value</th>
      <th>n</th>
      <th>info</th>
    </tr>
  </thead>
  <tbody>
    <tr>
      <td>rs10399793</td>
      <td>1</td>
      <td>49298</td>
      <td>T</td>
      <td>C</td>
      <td>T</td>
      <td>0.376220</td>
      <td>-0.000217</td>
      <td>0.001172</td>
      <td>0.85</td>
      <td>459324</td>
      <td>0.342797</td>
    </tr>
    <tr>
      <td>rs2462492</td>
      <td>1</td>
      <td>54676</td>
      <td>C</td>
      <td>T</td>
      <td>C</td>
      <td>0.599409</td>
      <td>0.001789</td>
      <td>0.001161</td>
      <td>0.11</td>
      <td>459324</td>
      <td>0.340158</td>
    </tr>
  </tbody>
</table></div><div class='container mt-4 bg-light p-4 text-center'><h2 class='text-muted'>Original GWAS file headers</h2><table border="1" class="dataframe table table-bordered">
  <thead>
    <tr style="text-align: right;">
      <th>variant_id</th>
      <th>chromosome</th>
      <th>base_pair_location</th>
      <th>effect_allele</th>
      <th>other_allele</th>
      <th>ref</th>
      <th>effect_allele_frequency</th>
      <th>beta</th>
      <th>standard_error</th>
      <th>p_value</th>
      <th>n</th>
      <th>info</th>
    </tr>
  </thead>
  <tbody>
    <tr>
      <td>rs10399793</td>
      <td>1</td>
      <td>49298</td>
      <td>T</td>
      <td>C</td>
      <td>T</td>
      <td>0.376220</td>
      <td>-0.000217</td>
      <td>0.001172</td>
      <td>0.85</td>
      <td>459324</td>
      <td>0.342797</td>
    </tr>
    <tr>
      <td>rs2462492</td>
      <td>1</td>
      <td>54676</td>
      <td>C</td>
      <td>T</td>
      <td>C</td>
      <td>0.599409</td>
      <td>0.001789</td>
      <td>0.001161</td>
      <td>0.11</td>
      <td>459324</td>
      <td>0.340158</td>
    </tr>
  </tbody>
</table><p style='font-weight: bold; color: blue;'>CHR -> ['chromosome']</p><p style='font-weight: bold; color: blue;'>SNP -> ['variant_id']</p><p style='font-weight: bold; color: blue;'>BP -> ['base_pair_location']</p><p style='font-weight: bold; color: blue;'>A1 -> ['effect_allele', 'ref']</p><p style='font-weight: bold; color: blue;'>A2 -> ['other_allele']</p><p style='font-weight: bold; color: blue;'>N -> ['n']</p><p style='font-weight: bold; color: blue;'>P -> ['p_value']</p><p style='font-weight: bold; color: blue;'>INFO -> ['info']</p><p style='font-weight: bold; color: blue;'>MAF -> ['effect_allele_frequency']</p><p style='font-weight: bold; color: blue;'>BETA -> ['beta']</p><p style='font-weight: bold; color: blue;'>SE -> ['standard_error']</p><p style='font-weight: bold; color: blue;'>OR -> []</p><p style='font-weight: bold; color: blue;'>Z -> []</p><p style='font-weight: bold; color: blue;'>D -> []</p><h2 class='text-muted'>GWAS columns</h2>variant_id, chromosome, base_pair_location, effect_allele, other_allele, ref, effect_allele_frequency, beta, standard_error, p_value, n, info<h2 class='text-muted'>Identified columns</h2>chromosome, variant_id, base_pair_location, effect_allele, ref, other_allele, n, p_value, info, effect_allele_frequency, beta, standard_error<h2 class='text-muted'>Unidentified Columns headers</h2><h2 class='text-muted'>PMID</h2>29892013<h2 class='text-muted'>DOI</h2>10.1038/s41588-018-0144-6<div class='bibtex-container'><h2 class='text-muted'>BibTeX</h2><div style='font-family: monospace; white-space: pre-wrap;'> @article{Loh_2018, title={Mixed-model association for biobank-scale datasets}, volume={50}, ISSN={1546-1718}, url={http://dx.doi.org/10.1038/s41588-018-0144-6}, DOI={10.1038/s41588-018-0144-6}, number={7}, journal={Nature Genetics}, publisher={Springer Science and Business Media LLC}, author={Loh, Po-Ru and Kichaev, Gleb and Gazal, Steven and Schoech, Armin P. and Price, Alkes L.}, year={2018}, month=jun, pages={906â€“908} }
</div></div><br><br><br><br><br><br></div><div class='container mt-4 bg-light p-4 text-center'><h1 class='text-primary'>Phenotype: high cholesterol</h1><p class='text-muted'><strong>URL:</strong> http://ftp.ebi.ac.uk/pub/databases/gwas/summary_statistics/GCST90296001-GCST90297000/GCST90296626</p><table border="1" class="dataframe table table-bordered">
  <thead>
    <tr style="text-align: right;">
      <th>firstAuthor</th>
      <th>accessionId</th>
      <th>publicationDate</th>
      <th>journal</th>
      <th>title</th>
      <th>reportedTrait</th>
      <th>efoTraits</th>
      <th>bgTraits</th>
      <th>discoverySampleAncestry</th>
      <th>replicationSampleAncestry</th>
      <th>associationCount</th>
      <th>summaryStatistics</th>
      <th>ssApiFlag</th>
      <th>agreedToCc0</th>
      <th>genotypingTechnologies</th>
      <th>pubmedId</th>
      <th>initialSampleDescription</th>
      <th>replicateSampleDescription</th>
      <th>UKB_FIELD</th>
      <th>FuzzySimilarity_high cholesterol</th>
      <th>SearchPhenotype</th>
    </tr>
  </thead>
  <tbody>
    <tr>
      <td>Sohail M</td>
      <td>GCST90296626</td>
      <td>10/11/2023</td>
      <td>Nature</td>
      <td>Mexican Biobank advances population and medical genomics of diverse ancestries.</td>
      <td>high cholesterol</td>
      <td>hypercholesterolemia</td>
      <td>-</td>
      <td>1061 hispanic or latin american</td>
      <td>NaN</td>
      <td>0</td>
      <td>http://ftp.ebi.ac.uk/pub/databases/gwas/summary_statistics/GCST90296001-GCST90297000/GCST90296626</td>
      <td>no</td>
      <td>no</td>
      <td>Genome-wide genotyping array</td>
      <td>37821706</td>
      <td>1,061 Hispanic or Latin American individuals (with &gt;90% Native American ancestry)</td>
      <td>NaN</td>
      <td>-</td>
      <td>100</td>
      <td>high cholesterol</td>
    </tr>
  </tbody>
</table><h2 class='text-muted'>Information file</h2><table border="1" class="dataframe table table-bordered">
  <thead>
    <tr style="text-align: right;">
      <th>Unnamed: 0</th>
      <th>Name</th>
      <th>Last modified</th>
      <th>Size</th>
      <th>Description</th>
      <th>URL</th>
    </tr>
  </thead>
  <tbody>
    <tr>
      <td>2</td>
      <td>harmonised/</td>
      <td>2024-02-07 22:30</td>
      <td>-</td>
      <td>NaN</td>
      <td>http://ftp.ebi.ac.uk/pub/databases/gwas/summary_statistics/GCST90296001-GCST90297000/GCST90296626/harmonised/</td>
    </tr>
    <tr>
      <td>3</td>
      <td>md5sum.txt</td>
      <td>2023-11-02 00:32</td>
      <td>110</td>
      <td>NaN</td>
      <td>http://ftp.ebi.ac.uk/pub/databases/gwas/summary_statistics/GCST90296001-GCST90297000/GCST90296626/md5sum.txt</td>
    </tr>
    <tr>
      <td>1</td>
      <td>GCST90296626.tsv-meta.yaml</td>
      <td>2024-01-04 00:19</td>
      <td>2.4K</td>
      <td>NaN</td>
      <td>http://ftp.ebi.ac.uk/pub/databases/gwas/summary_statistics/GCST90296001-GCST90297000/GCST90296626/GCST90296626.tsv-meta.yaml</td>
    </tr>
    <tr>
      <td>0</td>
      <td>GCST90296626.tsv</td>
      <td>2023-11-02 00:32</td>
      <td>1.0G</td>
      <td>NaN</td>
      <td>http://ftp.ebi.ac.uk/pub/databases/gwas/summary_statistics/GCST90296001-GCST90297000/GCST90296626/GCST90296626.tsv</td>
    </tr>
  </tbody>
</table><h2 class='text-muted'>Processed file</h2><p>Processing file: GCST90296626.tsv.1!</p><h2 class='text-muted'>GWAS file head</h2><table border="1" class="dataframe table table-bordered">
  <thead>
    <tr style="text-align: right;">
      <th>chromosome</th>
      <th>base_pair_location</th>
      <th>effect_allele</th>
      <th>other_allele</th>
      <th>beta</th>
      <th>standard_error</th>
      <th>effect_allele_frequency</th>
      <th>p_value</th>
      <th>variant_id</th>
      <th>info</th>
      <th>chisq</th>
      <th>test</th>
      <th>log10p</th>
      <th>n</th>
    </tr>
  </thead>
  <tbody>
    <tr>
      <td>1</td>
      <td>10930</td>
      <td>G</td>
      <td>A</td>
      <td>0.479937</td>
      <td>5.65167</td>
      <td>0.996414</td>
      <td>0.932325</td>
      <td>NaN</td>
      <td>0.185413</td>
      <td>0.007211</td>
      <td>ADD</td>
      <td>0.030433</td>
      <td>1058</td>
    </tr>
    <tr>
      <td>1</td>
      <td>10989</td>
      <td>G</td>
      <td>A</td>
      <td>-5.265930</td>
      <td>5.41161</td>
      <td>0.996764</td>
      <td>0.330514</td>
      <td>NaN</td>
      <td>0.160295</td>
      <td>0.946883</td>
      <td>ADD</td>
      <td>0.480810</td>
      <td>1058</td>
    </tr>
  </tbody>
</table></div><div class='container mt-4 bg-light p-4 text-center'><h2 class='text-muted'>Original GWAS file headers</h2><table border="1" class="dataframe table table-bordered">
  <thead>
    <tr style="text-align: right;">
      <th>chromosome</th>
      <th>base_pair_location</th>
      <th>effect_allele</th>
      <th>other_allele</th>
      <th>beta</th>
      <th>standard_error</th>
      <th>effect_allele_frequency</th>
      <th>p_value</th>
      <th>variant_id</th>
      <th>info</th>
      <th>chisq</th>
      <th>test</th>
      <th>log10p</th>
      <th>n</th>
    </tr>
  </thead>
  <tbody>
    <tr>
      <td>1</td>
      <td>10930</td>
      <td>G</td>
      <td>A</td>
      <td>0.479937</td>
      <td>5.65167</td>
      <td>0.996414</td>
      <td>0.932325</td>
      <td>NaN</td>
      <td>0.185413</td>
      <td>0.007211</td>
      <td>ADD</td>
      <td>0.030433</td>
      <td>1058</td>
    </tr>
    <tr>
      <td>1</td>
      <td>10989</td>
      <td>G</td>
      <td>A</td>
      <td>-5.265930</td>
      <td>5.41161</td>
      <td>0.996764</td>
      <td>0.330514</td>
      <td>NaN</td>
      <td>0.160295</td>
      <td>0.946883</td>
      <td>ADD</td>
      <td>0.480810</td>
      <td>1058</td>
    </tr>
  </tbody>
</table><p style='font-weight: bold; color: blue;'>CHR -> ['chromosome']</p><p style='font-weight: bold; color: blue;'>SNP -> ['variant_id']</p><p style='font-weight: bold; color: blue;'>BP -> ['base_pair_location']</p><p style='font-weight: bold; color: blue;'>A1 -> ['effect_allele']</p><p style='font-weight: bold; color: blue;'>A2 -> ['other_allele']</p><p style='font-weight: bold; color: blue;'>N -> ['n']</p><p style='font-weight: bold; color: blue;'>P -> ['p_value']</p><p style='font-weight: bold; color: blue;'>INFO -> ['info']</p><p style='font-weight: bold; color: blue;'>MAF -> ['effect_allele_frequency']</p><p style='font-weight: bold; color: blue;'>BETA -> ['beta']</p><p style='font-weight: bold; color: blue;'>SE -> ['standard_error']</p><p style='font-weight: bold; color: blue;'>OR -> []</p><p style='font-weight: bold; color: blue;'>Z -> []</p><p style='font-weight: bold; color: blue;'>D -> []</p><h2 class='text-muted'>GWAS columns</h2>chromosome, base_pair_location, effect_allele, other_allele, beta, standard_error, effect_allele_frequency, p_value, variant_id, info, chisq, test, log10p, n<h2 class='text-muted'>Identified columns</h2>chromosome, variant_id, base_pair_location, effect_allele, other_allele, n, p_value, info, effect_allele_frequency, beta, standard_error<h2 class='text-muted'>Unidentified Columns headers</h2>log10p, test, chisq<h2 class='text-muted'>PMID</h2>37821706<h2 class='text-muted'>DOI</h2>10.1038/s41586-023-06560-0<div class='bibtex-container'><h2 class='text-muted'>BibTeX</h2><div style='font-family: monospace; white-space: pre-wrap;'> @article{Sohail_2023, title={Mexican Biobank advances population and medical genomics of diverse ancestries}, volume={622}, ISSN={1476-4687}, url={http://dx.doi.org/10.1038/s41586-023-06560-0}, DOI={10.1038/s41586-023-06560-0}, number={7984}, journal={Nature}, publisher={Springer Science and Business Media LLC}, author={Sohail, Mashaal and Palma-Martínez, María J. and Chong, Amanda Y. and Quinto-Cortés, Consuelo D. and Barberena-Jonas, Carmina and Medina-Muñoz, Santiago G. and Ragsdale, Aaron and Delgado-Sánchez, Guadalupe and Cruz-Hervert, Luis Pablo and Ferreyra-Reyes, Leticia and Ferreira-Guerrero, Elizabeth and Mongua-Rodríguez, Norma and Canizales-Quintero, Sergio and Jimenez-Kaufmann, Andrés and Moreno-Macías, Hortensia and Aguilar-Salinas, Carlos A. and Auckland, Kathryn and Cortés, Adrián and Acuña-Alonzo, Víctor and Gignoux, Christopher R. and Wojcik, Genevieve L. and Ioannidis, Alexander G. and Fernández-Valverde, Selene L. and Hill, Adrian V. S. and Tusié-Luna, María Teresa and Mentzer, Alexander J. and Novembre, John and García-García, Lourdes and Moreno-Estrada, Andrés}, year={2023}, month=oct, pages={775–783} }
</div></div><br><br><br><br><br><br></div><div class='container mt-4 bg-light p-4 text-center'><h1 class='text-primary'>Phenotype: high cholesterol</h1><p class='text-muted'><strong>URL:</strong> http://ftp.ebi.ac.uk/pub/databases/gwas/summary_statistics/GCST90296001-GCST90297000/GCST90296604</p><table border="1" class="dataframe table table-bordered">
  <thead>
    <tr style="text-align: right;">
      <th>firstAuthor</th>
      <th>accessionId</th>
      <th>publicationDate</th>
      <th>journal</th>
      <th>title</th>
      <th>reportedTrait</th>
      <th>efoTraits</th>
      <th>bgTraits</th>
      <th>discoverySampleAncestry</th>
      <th>replicationSampleAncestry</th>
      <th>associationCount</th>
      <th>summaryStatistics</th>
      <th>ssApiFlag</th>
      <th>agreedToCc0</th>
      <th>genotypingTechnologies</th>
      <th>pubmedId</th>
      <th>initialSampleDescription</th>
      <th>replicateSampleDescription</th>
      <th>UKB_FIELD</th>
      <th>FuzzySimilarity_high cholesterol</th>
      <th>SearchPhenotype</th>
    </tr>
  </thead>
  <tbody>
    <tr>
      <td>Sohail M</td>
      <td>GCST90296604</td>
      <td>10/11/2023</td>
      <td>Nature</td>
      <td>Mexican Biobank advances population and medical genomics of diverse ancestries.</td>
      <td>high cholesterol</td>
      <td>hypercholesterolemia</td>
      <td>-</td>
      <td>5708 hispanic or latin american</td>
      <td>NaN</td>
      <td>0</td>
      <td>http://ftp.ebi.ac.uk/pub/databases/gwas/summary_statistics/GCST90296001-GCST90297000/GCST90296604</td>
      <td>no</td>
      <td>no</td>
      <td>Genome-wide genotyping array</td>
      <td>37821706</td>
      <td>5,363 Hispanic or Latin American controls</td>
      <td>NaN</td>
      <td>-</td>
      <td>100</td>
      <td>high cholesterol</td>
    </tr>
  </tbody>
</table><h2 class='text-muted'>Information file</h2><table border="1" class="dataframe table table-bordered">
  <thead>
    <tr style="text-align: right;">
      <th>Unnamed: 0</th>
      <th>Name</th>
      <th>Last modified</th>
      <th>Size</th>
      <th>Description</th>
      <th>URL</th>
    </tr>
  </thead>
  <tbody>
    <tr>
      <td>2</td>
      <td>harmonised/</td>
      <td>2024-02-07 22:30</td>
      <td>-</td>
      <td>NaN</td>
      <td>http://ftp.ebi.ac.uk/pub/databases/gwas/summary_statistics/GCST90296001-GCST90297000/GCST90296604/harmonised/</td>
    </tr>
    <tr>
      <td>3</td>
      <td>md5sum.txt</td>
      <td>2023-11-02 00:37</td>
      <td>110</td>
      <td>NaN</td>
      <td>http://ftp.ebi.ac.uk/pub/databases/gwas/summary_statistics/GCST90296001-GCST90297000/GCST90296604/md5sum.txt</td>
    </tr>
    <tr>
      <td>1</td>
      <td>GCST90296604.tsv-meta.yaml</td>
      <td>2024-01-04 00:18</td>
      <td>2.4K</td>
      <td>NaN</td>
      <td>http://ftp.ebi.ac.uk/pub/databases/gwas/summary_statistics/GCST90296001-GCST90297000/GCST90296604/GCST90296604.tsv-meta.yaml</td>
    </tr>
    <tr>
      <td>0</td>
      <td>GCST90296604.tsv</td>
      <td>2023-11-02 00:37</td>
      <td>2.9G</td>
      <td>NaN</td>
      <td>http://ftp.ebi.ac.uk/pub/databases/gwas/summary_statistics/GCST90296001-GCST90297000/GCST90296604/GCST90296604.tsv</td>
    </tr>
  </tbody>
</table><h2 class='text-muted'>Processed file</h2><p>Processing file: GCST90296604.tsv.1!</p><h2 class='text-muted'>GWAS file head</h2><table border="1" class="dataframe table table-bordered">
  <thead>
    <tr style="text-align: right;">
      <th>chromosome</th>
      <th>base_pair_location</th>
      <th>effect_allele</th>
      <th>other_allele</th>
      <th>beta</th>
      <th>standard_error</th>
      <th>effect_allele_frequency</th>
      <th>p_value</th>
      <th>variant_id</th>
      <th>info</th>
      <th>log10p</th>
      <th>n</th>
      <th>chisq</th>
      <th>test</th>
    </tr>
  </thead>
  <tbody>
    <tr>
      <td>1</td>
      <td>10894</td>
      <td>G</td>
      <td>A</td>
      <td>5.61969</td>
      <td>4.28185</td>
      <td>0.998906</td>
      <td>0.189370</td>
      <td>NaN</td>
      <td>0.141311</td>
      <td>0.722688</td>
      <td>5708</td>
      <td>1.72251</td>
      <td>ADD</td>
    </tr>
    <tr>
      <td>1</td>
      <td>10915</td>
      <td>G</td>
      <td>A</td>
      <td>5.27127</td>
      <td>4.85177</td>
      <td>0.999358</td>
      <td>0.277274</td>
      <td>NaN</td>
      <td>0.134596</td>
      <td>0.557091</td>
      <td>5708</td>
      <td>1.18040</td>
      <td>ADD</td>
    </tr>
  </tbody>
</table></div><div class='container mt-4 bg-light p-4 text-center'><h2 class='text-muted'>Original GWAS file headers</h2><table border="1" class="dataframe table table-bordered">
  <thead>
    <tr style="text-align: right;">
      <th>chromosome</th>
      <th>base_pair_location</th>
      <th>effect_allele</th>
      <th>other_allele</th>
      <th>beta</th>
      <th>standard_error</th>
      <th>effect_allele_frequency</th>
      <th>p_value</th>
      <th>variant_id</th>
      <th>info</th>
      <th>log10p</th>
      <th>n</th>
      <th>chisq</th>
      <th>test</th>
    </tr>
  </thead>
  <tbody>
    <tr>
      <td>1</td>
      <td>10894</td>
      <td>G</td>
      <td>A</td>
      <td>5.61969</td>
      <td>4.28185</td>
      <td>0.998906</td>
      <td>0.189370</td>
      <td>NaN</td>
      <td>0.141311</td>
      <td>0.722688</td>
      <td>5708</td>
      <td>1.72251</td>
      <td>ADD</td>
    </tr>
    <tr>
      <td>1</td>
      <td>10915</td>
      <td>G</td>
      <td>A</td>
      <td>5.27127</td>
      <td>4.85177</td>
      <td>0.999358</td>
      <td>0.277274</td>
      <td>NaN</td>
      <td>0.134596</td>
      <td>0.557091</td>
      <td>5708</td>
      <td>1.18040</td>
      <td>ADD</td>
    </tr>
  </tbody>
</table><p style='font-weight: bold; color: blue;'>CHR -> ['chromosome']</p><p style='font-weight: bold; color: blue;'>SNP -> ['variant_id']</p><p style='font-weight: bold; color: blue;'>BP -> ['base_pair_location']</p><p style='font-weight: bold; color: blue;'>A1 -> ['effect_allele']</p><p style='font-weight: bold; color: blue;'>A2 -> ['other_allele']</p><p style='font-weight: bold; color: blue;'>N -> ['n']</p><p style='font-weight: bold; color: blue;'>P -> ['p_value']</p><p style='font-weight: bold; color: blue;'>INFO -> ['info']</p><p style='font-weight: bold; color: blue;'>MAF -> ['effect_allele_frequency']</p><p style='font-weight: bold; color: blue;'>BETA -> ['beta']</p><p style='font-weight: bold; color: blue;'>SE -> ['standard_error']</p><p style='font-weight: bold; color: blue;'>OR -> []</p><p style='font-weight: bold; color: blue;'>Z -> []</p><p style='font-weight: bold; color: blue;'>D -> []</p><h2 class='text-muted'>GWAS columns</h2>chromosome, base_pair_location, effect_allele, other_allele, beta, standard_error, effect_allele_frequency, p_value, variant_id, info, log10p, n, chisq, test<h2 class='text-muted'>Identified columns</h2>chromosome, variant_id, base_pair_location, effect_allele, other_allele, n, p_value, info, effect_allele_frequency, beta, standard_error<h2 class='text-muted'>Unidentified Columns headers</h2>log10p, test, chisq<h2 class='text-muted'>PMID</h2>37821706<h2 class='text-muted'>DOI</h2>10.1038/s41586-023-06560-0<div class='bibtex-container'><h2 class='text-muted'>BibTeX</h2><div style='font-family: monospace; white-space: pre-wrap;'> @article{Sohail_2023, title={Mexican Biobank advances population and medical genomics of diverse ancestries}, volume={622}, ISSN={1476-4687}, url={http://dx.doi.org/10.1038/s41586-023-06560-0}, DOI={10.1038/s41586-023-06560-0}, number={7984}, journal={Nature}, publisher={Springer Science and Business Media LLC}, author={Sohail, Mashaal and Palma-Martínez, María J. and Chong, Amanda Y. and Quinto-Cortés, Consuelo D. and Barberena-Jonas, Carmina and Medina-Muñoz, Santiago G. and Ragsdale, Aaron and Delgado-Sánchez, Guadalupe and Cruz-Hervert, Luis Pablo and Ferreyra-Reyes, Leticia and Ferreira-Guerrero, Elizabeth and Mongua-Rodríguez, Norma and Canizales-Quintero, Sergio and Jimenez-Kaufmann, Andrés and Moreno-Macías, Hortensia and Aguilar-Salinas, Carlos A. and Auckland, Kathryn and Cortés, Adrián and Acuña-Alonzo, Víctor and Gignoux, Christopher R. and Wojcik, Genevieve L. and Ioannidis, Alexander G. and Fernández-Valverde, Selene L. and Hill, Adrian V. S. and Tusié-Luna, María Teresa and Mentzer, Alexander J. and Novembre, John and García-García, Lourdes and Moreno-Estrada, Andrés}, year={2023}, month=oct, pages={775–783} }
</div></div><br><br><br><br><br><br></div><div class='container mt-4 bg-light p-4 text-center'><h1 class='text-primary'>Phenotype: high cholesterol</h1><p class='text-muted'><strong>URL:</strong> http://ftp.ebi.ac.uk/pub/databases/gwas/summary_statistics/GCST90077001-GCST90078000/GCST90077883</p><table border="1" class="dataframe table table-bordered">
  <thead>
    <tr style="text-align: right;">
      <th>firstAuthor</th>
      <th>accessionId</th>
      <th>publicationDate</th>
      <th>journal</th>
      <th>title</th>
      <th>reportedTrait</th>
      <th>efoTraits</th>
      <th>bgTraits</th>
      <th>discoverySampleAncestry</th>
      <th>replicationSampleAncestry</th>
      <th>associationCount</th>
      <th>summaryStatistics</th>
      <th>ssApiFlag</th>
      <th>agreedToCc0</th>
      <th>genotypingTechnologies</th>
      <th>pubmedId</th>
      <th>initialSampleDescription</th>
      <th>replicateSampleDescription</th>
      <th>UKB_FIELD</th>
      <th>FuzzySimilarity_high cholesterol</th>
      <th>SearchPhenotype</th>
    </tr>
  </thead>
  <tbody>
    <tr>
      <td>Backman JD</td>
      <td>GCST90077883</td>
      <td>10/18/2021</td>
      <td>Nature</td>
      <td>Exome sequencing and analysis of 454,787 UK Biobank participants.</td>
      <td>high cholesterol</td>
      <td>hypercholesterolemia</td>
      <td>-</td>
      <td>331754 european</td>
      <td>NaN</td>
      <td>2</td>
      <td>http://ftp.ebi.ac.uk/pub/databases/gwas/summary_statistics/GCST90077001-GCST90078000/GCST90077883</td>
      <td>no</td>
      <td>yes</td>
      <td>Exome-wide sequencing,Genome-wide genotyping array</td>
      <td>34662886</td>
      <td>57,365 European ancestry cases,274,389 European ancestry controls</td>
      <td>NaN</td>
      <td>-</td>
      <td>100</td>
      <td>high cholesterol</td>
    </tr>
  </tbody>
</table><h2 class='text-muted'>Information file</h2><table border="1" class="dataframe table table-bordered">
  <thead>
    <tr style="text-align: right;">
      <th>Unnamed: 0</th>
      <th>Name</th>
      <th>Last modified</th>
      <th>Size</th>
      <th>Description</th>
      <th>URL</th>
    </tr>
  </thead>
  <tbody>
    <tr>
      <td>1</td>
      <td>GCST90077883_buildGRCh38.tsv.gz-meta.yaml</td>
      <td>2024-01-04 00:32</td>
      <td>887</td>
      <td>NaN</td>
      <td>http://ftp.ebi.ac.uk/pub/databases/gwas/summary_statistics/GCST90077001-GCST90078000/GCST90077883/GCST90077883_buildGRCh38.tsv.gz-meta.yaml</td>
    </tr>
    <tr>
      <td>2</td>
      <td>README.txt</td>
      <td>2021-11-18 00:59</td>
      <td>698</td>
      <td>NaN</td>
      <td>http://ftp.ebi.ac.uk/pub/databases/gwas/summary_statistics/GCST90077001-GCST90078000/GCST90077883/README.txt</td>
    </tr>
    <tr>
      <td>3</td>
      <td>harmonised/</td>
      <td>2024-02-02 15:16</td>
      <td>-</td>
      <td>NaN</td>
      <td>http://ftp.ebi.ac.uk/pub/databases/gwas/summary_statistics/GCST90077001-GCST90078000/GCST90077883/harmonised/</td>
    </tr>
    <tr>
      <td>4</td>
      <td>md5sum.txt</td>
      <td>2021-11-18 00:59</td>
      <td>109</td>
      <td>NaN</td>
      <td>http://ftp.ebi.ac.uk/pub/databases/gwas/summary_statistics/GCST90077001-GCST90078000/GCST90077883/md5sum.txt</td>
    </tr>
    <tr>
      <td>0</td>
      <td>GCST90077883_buildGRCh38.tsv.gz</td>
      <td>2021-11-18 00:59</td>
      <td>15M</td>
      <td>NaN</td>
      <td>http://ftp.ebi.ac.uk/pub/databases/gwas/summary_statistics/GCST90077001-GCST90078000/GCST90077883/GCST90077883_buildGRCh38.tsv.gz</td>
    </tr>
  </tbody>
</table><h2 class='text-muted'>Processed file</h2><p>Processing file: GCST90077883_buildGRCh38.tsv.gz!</p><h2 class='text-muted'>GWAS file head</h2><table border="1" class="dataframe table table-bordered">
  <thead>
    <tr style="text-align: right;">
      <th>name</th>
      <th>chromosome</th>
      <th>base_pair_location</th>
      <th>other_allele</th>
      <th>effect_allele</th>
      <th>trait</th>
      <th>cohort</th>
      <th>model</th>
      <th>odds_ratio</th>
      <th>ci_lower</th>
      <th>ci_upper</th>
      <th>p_value</th>
      <th>effect_allele_frequency</th>
      <th>standard_error</th>
    </tr>
  </thead>
  <tbody>
    <tr>
      <td>1_930165_G_A</td>
      <td>1</td>
      <td>930165</td>
      <td>G</td>
      <td>A</td>
      <td>20002_Illness_code_SR_1473_high_cholesterol_AnyInst__BIN</td>
      <td>UKB_Freeze_450_EUR</td>
      <td>ADD-WGR-FIRTH</td>
      <td>0.854479</td>
      <td>0.303101</td>
      <td>2.40888</td>
      <td>0.766163</td>
      <td>0.000042</td>
      <td>0.528798</td>
    </tr>
    <tr>
      <td>1_930204_G_A</td>
      <td>1</td>
      <td>930204</td>
      <td>G</td>
      <td>A</td>
      <td>20002_Illness_code_SR_1473_high_cholesterol_AnyInst__BIN</td>
      <td>UKB_Freeze_450_EUR</td>
      <td>ADD-WGR-FIRTH</td>
      <td>1.771850</td>
      <td>0.597674</td>
      <td>5.25281</td>
      <td>0.302228</td>
      <td>0.000041</td>
      <td>0.554468</td>
    </tr>
  </tbody>
</table></div><div class='container mt-4 bg-light p-4 text-center'><h2 class='text-muted'>Original GWAS file headers</h2><table border="1" class="dataframe table table-bordered">
  <thead>
    <tr style="text-align: right;">
      <th>name</th>
      <th>chromosome</th>
      <th>base_pair_location</th>
      <th>other_allele</th>
      <th>effect_allele</th>
      <th>trait</th>
      <th>cohort</th>
      <th>model</th>
      <th>odds_ratio</th>
      <th>ci_lower</th>
      <th>ci_upper</th>
      <th>p_value</th>
      <th>effect_allele_frequency</th>
      <th>standard_error</th>
    </tr>
  </thead>
  <tbody>
    <tr>
      <td>1_930165_G_A</td>
      <td>1</td>
      <td>930165</td>
      <td>G</td>
      <td>A</td>
      <td>20002_Illness_code_SR_1473_high_cholesterol_AnyInst__BIN</td>
      <td>UKB_Freeze_450_EUR</td>
      <td>ADD-WGR-FIRTH</td>
      <td>0.854479</td>
      <td>0.303101</td>
      <td>2.40888</td>
      <td>0.766163</td>
      <td>0.000042</td>
      <td>0.528798</td>
    </tr>
    <tr>
      <td>1_930204_G_A</td>
      <td>1</td>
      <td>930204</td>
      <td>G</td>
      <td>A</td>
      <td>20002_Illness_code_SR_1473_high_cholesterol_AnyInst__BIN</td>
      <td>UKB_Freeze_450_EUR</td>
      <td>ADD-WGR-FIRTH</td>
      <td>1.771850</td>
      <td>0.597674</td>
      <td>5.25281</td>
      <td>0.302228</td>
      <td>0.000041</td>
      <td>0.554468</td>
    </tr>
  </tbody>
</table><p style='font-weight: bold; color: blue;'>CHR -> ['chromosome']</p><p style='font-weight: bold; color: blue;'>SNP -> ['name']</p><p style='font-weight: bold; color: blue;'>BP -> ['name', 'base_pair_location']</p><p style='font-weight: bold; color: blue;'>A1 -> ['effect_allele']</p><p style='font-weight: bold; color: blue;'>A2 -> ['other_allele']</p><p style='font-weight: bold; color: blue;'>N -> []</p><p style='font-weight: bold; color: blue;'>P -> ['p_value']</p><p style='font-weight: bold; color: blue;'>INFO -> []</p><p style='font-weight: bold; color: blue;'>MAF -> ['effect_allele_frequency']</p><p style='font-weight: bold; color: blue;'>BETA -> []</p><p style='font-weight: bold; color: blue;'>SE -> ['standard_error']</p><p style='font-weight: bold; color: blue;'>OR -> ['odds_ratio']</p><p style='font-weight: bold; color: blue;'>Z -> []</p><p style='font-weight: bold; color: blue;'>D -> []</p><h2 class='text-muted'>GWAS columns</h2>name, chromosome, base_pair_location, other_allele, effect_allele, trait, cohort, model, odds_ratio, ci_lower, ci_upper, p_value, effect_allele_frequency, standard_error<h2 class='text-muted'>Identified columns</h2>chromosome, name, name, base_pair_location, effect_allele, other_allele, p_value, effect_allele_frequency, standard_error, odds_ratio<h2 class='text-muted'>Unidentified Columns headers</h2>ci_lower, cohort, model, trait, ci_upper<h2 class='text-muted'>PMID</h2>34662886<h2 class='text-muted'>DOI</h2>10.1038/s41586-021-04103-z<div class='bibtex-container'><h2 class='text-muted'>BibTeX</h2><div style='font-family: monospace; white-space: pre-wrap;'> @article{Backman_2021, title={Exome sequencing and analysis of 454,787 UK Biobank participants}, volume={599}, ISSN={1476-4687}, url={http://dx.doi.org/10.1038/s41586-021-04103-z}, DOI={10.1038/s41586-021-04103-z}, number={7886}, journal={Nature}, publisher={Springer Science and Business Media LLC}, author={Backman, Joshua D. and Li, Alexander H. and Marcketta, Anthony and Sun, Dylan and Mbatchou, Joelle and Kessler, Michael D. and Benner, Christian and Liu, Daren and Locke, Adam E. and Balasubramanian, Suganthi and Yadav, Ashish and Banerjee, Nilanjana and Gillies, Christopher E. and Damask, Amy and Liu, Simon and Bai, Xiaodong and Hawes, Alicia and Maxwell, Evan and Gurski, Lauren and Watanabe, Kyoko and Kosmicki, Jack A. and Rajagopal, Veera and Mighty, Jason and Jones, Marcus and Mitnaul, Lyndon and Stahl, Eli and Coppola, Giovanni and Jorgenson, Eric and Habegger, Lukas and Salerno, William J. and Shuldiner, Alan R. and Lotta, Luca A. and Overton, John D. and Cantor, Michael N. and Reid, Jeffrey G. and Yancopoulos, George and Kang, Hyun M. and Marchini, Jonathan and Baras, Aris and Abecasis, Gonçalo R. and Ferreira, Manuel A. R.}, year={2021}, month=oct, pages={628–634} }
</div></div><br><br><br><br><br><br></div>