<link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/bootstrap/4.0.0/css/bootstrap.min.css" integrity="sha384-Gn5384xqQ1aoWXA+058RXPxPg6fy4IWvTNh0E263XmFcJlSAwiGgFAW/dAiS6JXm" crossorigin="anonymous"><div class='container mt-4 bg-light p-4 text-center'><h1 class='text-primary'>Phenotype: hypertension</h1><p class='text-muted'><strong>URL:</strong> http://ftp.ebi.ac.uk/pub/databases/gwas/summary_statistics/GCST90038001-GCST90039000/GCST90038604</p><table border="1" class="dataframe table table-bordered">
  <thead>
    <tr style="text-align: right;">
      <th>firstAuthor</th>
      <th>accessionId</th>
      <th>publicationDate</th>
      <th>journal</th>
      <th>title</th>
      <th>reportedTrait</th>
      <th>efoTraits</th>
      <th>bgTraits</th>
      <th>discoverySampleAncestry</th>
      <th>replicationSampleAncestry</th>
      <th>associationCount</th>
      <th>summaryStatistics</th>
      <th>ssApiFlag</th>
      <th>agreedToCc0</th>
      <th>genotypingTechnologies</th>
      <th>pubmedId</th>
      <th>initialSampleDescription</th>
      <th>replicateSampleDescription</th>
      <th>UKB_FIELD</th>
      <th>FuzzySimilarity_hypertension</th>
      <th>SearchPhenotype</th>
    </tr>
  </thead>
  <tbody>
    <tr>
      <td>D??nerta?� HM</td>
      <td>GCST90038604</td>
      <td>4/8/2021</td>
      <td>Nat Aging</td>
      <td>Common genetic associations between age-related diseases.</td>
      <td>hypertension</td>
      <td>hypertension</td>
      <td>-</td>
      <td>484598 nr</td>
      <td>NaN</td>
      <td>0</td>
      <td>http://ftp.ebi.ac.uk/pub/databases/gwas/summary_statistics/GCST90038001-GCST90039000/GCST90038604</td>
      <td>no</td>
      <td>yes</td>
      <td>Genome-wide genotyping array</td>
      <td>33959723</td>
      <td>129,909 cases,354,689 controls</td>
      <td>NaN</td>
      <td>-</td>
      <td>100</td>
      <td>hypertension</td>
    </tr>
  </tbody>
</table><h2 class='text-muted'>Information file</h2><table border="1" class="dataframe table table-bordered">
  <thead>
    <tr style="text-align: right;">
      <th>Unnamed: 0</th>
      <th>Name</th>
      <th>Last modified</th>
      <th>Size</th>
      <th>Description</th>
      <th>URL</th>
    </tr>
  </thead>
  <tbody>
    <tr>
      <td>1</td>
      <td>GCST90038604_buildGRCh37.tsv-meta.yaml</td>
      <td>2024-01-04 00:23</td>
      <td>663</td>
      <td>NaN</td>
      <td>http://ftp.ebi.ac.uk/pub/databases/gwas/summary_statistics/GCST90038001-GCST90039000/GCST90038604/GCST90038604_buildGRCh37.tsv-meta.yaml</td>
    </tr>
    <tr>
      <td>2</td>
      <td>harmonised/</td>
      <td>2024-02-02 15:16</td>
      <td>-</td>
      <td>NaN</td>
      <td>http://ftp.ebi.ac.uk/pub/databases/gwas/summary_statistics/GCST90038001-GCST90039000/GCST90038604/harmonised/</td>
    </tr>
    <tr>
      <td>3</td>
      <td>md5sum.txt</td>
      <td>2021-11-14 00:30</td>
      <td>62</td>
      <td>NaN</td>
      <td>http://ftp.ebi.ac.uk/pub/databases/gwas/summary_statistics/GCST90038001-GCST90039000/GCST90038604/md5sum.txt</td>
    </tr>
    <tr>
      <td>0</td>
      <td>GCST90038604_buildGRCh37.tsv</td>
      <td>2021-11-14 00:30</td>
      <td>1.2G</td>
      <td>NaN</td>
      <td>http://ftp.ebi.ac.uk/pub/databases/gwas/summary_statistics/GCST90038001-GCST90039000/GCST90038604/GCST90038604_buildGRCh37.tsv</td>
    </tr>
  </tbody>
</table><h2 class='text-muted'>Processed file</h2><p>Processing file: GCST90038604_buildGRCh37.tsv!</p><h2 class='text-muted'>GWAS file head</h2><table border="1" class="dataframe table table-bordered">
  <thead>
    <tr style="text-align: right;">
      <th>variant_id</th>
      <th>chromosome</th>
      <th>base_pair_location</th>
      <th>genpos</th>
      <th>effect_allele</th>
      <th>other_allele</th>
      <th>effect_allele_frequency</th>
      <th>info</th>
      <th>chisq_linreg</th>
      <th>p_linreg</th>
      <th>beta</th>
      <th>standard_error</th>
      <th>chisq_bolt_lmm_inf</th>
      <th>p_bolt_lmm_inf</th>
      <th>chisq_bolt_lmm</th>
      <th>p_value</th>
    </tr>
  </thead>
  <tbody>
    <tr>
      <td>rs533090414</td>
      <td>1</td>
      <td>18849</td>
      <td>0.0</td>
      <td>C</td>
      <td>G</td>
      <td>0.026465</td>
      <td>0.536412</td>
      <td>4.25927</td>
      <td>0.039</td>
      <td>-0.007614</td>
      <td>0.003562</td>
      <td>4.56988</td>
      <td>0.033</td>
      <td>4.82569</td>
      <td>0.028</td>
    </tr>
    <tr>
      <td>rs568927205</td>
      <td>1</td>
      <td>54712</td>
      <td>0.0</td>
      <td>T</td>
      <td>TTTTC</td>
      <td>0.415825</td>
      <td>0.538572</td>
      <td>3.85829</td>
      <td>0.050</td>
      <td>0.001961</td>
      <td>0.001147</td>
      <td>2.92236</td>
      <td>0.087</td>
      <td>2.90307</td>
      <td>0.088</td>
    </tr>
  </tbody>
</table></div><div class='container mt-4 bg-light p-4 text-center'><h2 class='text-muted'>Original GWAS file headers</h2><table border="1" class="dataframe table table-bordered">
  <thead>
    <tr style="text-align: right;">
      <th>variant_id</th>
      <th>chromosome</th>
      <th>base_pair_location</th>
      <th>genpos</th>
      <th>effect_allele</th>
      <th>other_allele</th>
      <th>effect_allele_frequency</th>
      <th>info</th>
      <th>chisq_linreg</th>
      <th>p_linreg</th>
      <th>beta</th>
      <th>standard_error</th>
      <th>chisq_bolt_lmm_inf</th>
      <th>p_bolt_lmm_inf</th>
      <th>chisq_bolt_lmm</th>
      <th>p_value</th>
    </tr>
  </thead>
  <tbody>
    <tr>
      <td>rs533090414</td>
      <td>1</td>
      <td>18849</td>
      <td>0.0</td>
      <td>C</td>
      <td>G</td>
      <td>0.026465</td>
      <td>0.536412</td>
      <td>4.25927</td>
      <td>0.039</td>
      <td>-0.007614</td>
      <td>0.003562</td>
      <td>4.56988</td>
      <td>0.033</td>
      <td>4.82569</td>
      <td>0.028</td>
    </tr>
    <tr>
      <td>rs568927205</td>
      <td>1</td>
      <td>54712</td>
      <td>0.0</td>
      <td>T</td>
      <td>TTTTC</td>
      <td>0.415825</td>
      <td>0.538572</td>
      <td>3.85829</td>
      <td>0.050</td>
      <td>0.001961</td>
      <td>0.001147</td>
      <td>2.92236</td>
      <td>0.087</td>
      <td>2.90307</td>
      <td>0.088</td>
    </tr>
  </tbody>
</table><p style='font-weight: bold; color: blue;'>CHR -> ['chromosome']</p><p style='font-weight: bold; color: blue;'>SNP -> ['variant_id']</p><p style='font-weight: bold; color: blue;'>BP -> ['base_pair_location', 'genpos']</p><p style='font-weight: bold; color: blue;'>A1 -> ['effect_allele']</p><p style='font-weight: bold; color: blue;'>A2 -> ['other_allele']</p><p style='font-weight: bold; color: blue;'>N -> []</p><p style='font-weight: bold; color: blue;'>P -> ['p_value']</p><p style='font-weight: bold; color: blue;'>INFO -> ['info']</p><p style='font-weight: bold; color: blue;'>MAF -> ['effect_allele_frequency']</p><p style='font-weight: bold; color: blue;'>BETA -> ['beta', 'p_bolt_lmm_inf']</p><p style='font-weight: bold; color: blue;'>SE -> ['standard_error']</p><p style='font-weight: bold; color: blue;'>OR -> []</p><p style='font-weight: bold; color: blue;'>Z -> []</p><p style='font-weight: bold; color: blue;'>D -> []</p><h2 class='text-muted'>GWAS columns</h2>variant_id, chromosome, base_pair_location, genpos, effect_allele, other_allele, effect_allele_frequency, info, chisq_linreg, p_linreg, beta, standard_error, chisq_bolt_lmm_inf, p_bolt_lmm_inf, chisq_bolt_lmm, p_value<h2 class='text-muted'>Identified columns</h2>chromosome, variant_id, base_pair_location, genpos, effect_allele, other_allele, p_value, info, effect_allele_frequency, beta, p_bolt_lmm_inf, standard_error<h2 class='text-muted'>Unidentified Columns headers</h2>p_linreg, chisq_bolt_lmm, chisq_bolt_lmm_inf, chisq_linreg<h2 class='text-muted'>PMID</h2>33959723<h2 class='text-muted'>DOI</h2>10.1038/s43587-021-00051-5<div class='bibtex-container'><h2 class='text-muted'>BibTeX</h2><div style='font-family: monospace; white-space: pre-wrap;'> @article{D_nerta__2021, title={Common genetic associations between age-related diseases}, volume={1}, ISSN={2662-8465}, url={http://dx.doi.org/10.1038/s43587-021-00051-5}, DOI={10.1038/s43587-021-00051-5}, number={4}, journal={Nature Aging}, publisher={Springer Science and Business Media LLC}, author={Dönertaş, Handan Melike and Fabian, Daniel K. and Fuentealba, Matías and Partridge, Linda and Thornton, Janet M.}, year={2021}, month=apr, pages={400–412} }
</div></div><br><br><br><br><br><br></div><div class='container mt-4 bg-light p-4 text-center'><h1 class='text-primary'>Phenotype: hypertension</h1><p class='text-muted'><strong>URL:</strong> http://ftp.ebi.ac.uk/pub/databases/gwas/summary_statistics/GCST007001-GCST008000/GCST007610</p><table border="1" class="dataframe table table-bordered">
  <thead>
    <tr style="text-align: right;">
      <th>firstAuthor</th>
      <th>accessionId</th>
      <th>publicationDate</th>
      <th>journal</th>
      <th>title</th>
      <th>reportedTrait</th>
      <th>efoTraits</th>
      <th>bgTraits</th>
      <th>discoverySampleAncestry</th>
      <th>replicationSampleAncestry</th>
      <th>associationCount</th>
      <th>summaryStatistics</th>
      <th>ssApiFlag</th>
      <th>agreedToCc0</th>
      <th>genotypingTechnologies</th>
      <th>pubmedId</th>
      <th>initialSampleDescription</th>
      <th>replicateSampleDescription</th>
      <th>UKB_FIELD</th>
      <th>FuzzySimilarity_hypertension</th>
      <th>SearchPhenotype</th>
    </tr>
  </thead>
  <tbody>
    <tr>
      <td>Zhu Z</td>
      <td>GCST007610</td>
      <td>4/2/2019</td>
      <td>Respir Res</td>
      <td>Genetic overlap of chronic obstructive pulmonary disease and cardiovascular disease-related traits: a large-scale genome-wide cross-trait analysis.</td>
      <td>hypertension</td>
      <td>hypertension</td>
      <td>-</td>
      <td>458554 european</td>
      <td>NaN</td>
      <td>0</td>
      <td>http://ftp.ebi.ac.uk/pub/databases/gwas/summary_statistics/GCST007001-GCST008000/GCST007610</td>
      <td>no</td>
      <td>-</td>
      <td>Genome-wide genotyping array</td>
      <td>30940143</td>
      <td>144,793 European ancestry cases,313,761 European ancestry controls</td>
      <td>NaN</td>
      <td>-</td>
      <td>100</td>
      <td>hypertension</td>
    </tr>
  </tbody>
</table><h2 class='text-muted'>Information file</h2><table border="1" class="dataframe table table-bordered">
  <thead>
    <tr style="text-align: right;">
      <th>Unnamed: 0</th>
      <th>Name</th>
      <th>Last modified</th>
      <th>Size</th>
      <th>Description</th>
      <th>URL</th>
    </tr>
  </thead>
  <tbody>
    <tr>
      <td>1</td>
      <td>readme.txt</td>
      <td>2019-05-03 12:49</td>
      <td>494</td>
      <td>NaN</td>
      <td>http://ftp.ebi.ac.uk/pub/databases/gwas/summary_statistics/GCST007001-GCST008000/GCST007610/readme.txt</td>
    </tr>
    <tr>
      <td>0</td>
      <td>ZhuZ_30940143_ukbb.bolt_460K_selfRepWhite.doctor_highbloodpressure.assoc.gz</td>
      <td>2019-05-03 12:49</td>
      <td>143M</td>
      <td>NaN</td>
      <td>http://ftp.ebi.ac.uk/pub/databases/gwas/summary_statistics/GCST007001-GCST008000/GCST007610/ZhuZ_30940143_ukbb.bolt_460K_selfRepWhite.doctor_highbloodpressure.assoc.gz</td>
    </tr>
  </tbody>
</table><h2 class='text-muted'>Processed file</h2><p>Processing file: ZhuZ_30940143_ukbb.bolt_460K_selfRepWhite.doctor_highbloodpressure.assoc.gz!</p><h2 class='text-muted'>GWAS file head</h2><table border="1" class="dataframe table table-bordered">
  <thead>
    <tr style="text-align: right;">
      <th>snp</th>
      <th>chr</th>
      <th>bp</th>
      <th>a1</th>
      <th>a0</th>
      <th>maf</th>
      <th>hwep</th>
      <th>info</th>
      <th>beta</th>
      <th>se</th>
      <th>p</th>
    </tr>
  </thead>
  <tbody>
    <tr>
      <td>10_48698435_A_G</td>
      <td>10</td>
      <td>48698435</td>
      <td>A</td>
      <td>G</td>
      <td>0.01353</td>
      <td>0.02719</td>
      <td>0.960443</td>
      <td>-0.01552</td>
      <td>0.015790</td>
      <td>0.23</td>
    </tr>
    <tr>
      <td>11_1373806_C_A</td>
      <td>11</td>
      <td>1373806</td>
      <td>C</td>
      <td>A</td>
      <td>0.02781</td>
      <td>0.04388</td>
      <td>0.894853</td>
      <td>0.00809</td>
      <td>0.011217</td>
      <td>0.48</td>
    </tr>
  </tbody>
</table></div><div class='container mt-4 bg-light p-4 text-center'><h2 class='text-muted'>Original GWAS file headers</h2><table border="1" class="dataframe table table-bordered">
  <thead>
    <tr style="text-align: right;">
      <th>snp</th>
      <th>chr</th>
      <th>bp</th>
      <th>a1</th>
      <th>a0</th>
      <th>maf</th>
      <th>hwep</th>
      <th>info</th>
      <th>beta</th>
      <th>se</th>
      <th>p</th>
    </tr>
  </thead>
  <tbody>
    <tr>
      <td>10_48698435_A_G</td>
      <td>10</td>
      <td>48698435</td>
      <td>A</td>
      <td>G</td>
      <td>0.01353</td>
      <td>0.02719</td>
      <td>0.960443</td>
      <td>-0.01552</td>
      <td>0.015790</td>
      <td>0.23</td>
    </tr>
    <tr>
      <td>11_1373806_C_A</td>
      <td>11</td>
      <td>1373806</td>
      <td>C</td>
      <td>A</td>
      <td>0.02781</td>
      <td>0.04388</td>
      <td>0.894853</td>
      <td>0.00809</td>
      <td>0.011217</td>
      <td>0.48</td>
    </tr>
  </tbody>
</table><p style='font-weight: bold; color: blue;'>CHR -> ['chr']</p><p style='font-weight: bold; color: blue;'>SNP -> ['snp']</p><p style='font-weight: bold; color: blue;'>BP -> ['bp']</p><p style='font-weight: bold; color: blue;'>A1 -> ['a1', 'a0']</p><p style='font-weight: bold; color: blue;'>A2 -> ['a1']</p><p style='font-weight: bold; color: blue;'>N -> []</p><p style='font-weight: bold; color: blue;'>P -> ['p']</p><p style='font-weight: bold; color: blue;'>INFO -> ['info']</p><p style='font-weight: bold; color: blue;'>MAF -> ['maf']</p><p style='font-weight: bold; color: blue;'>BETA -> ['beta']</p><p style='font-weight: bold; color: blue;'>SE -> ['se']</p><p style='font-weight: bold; color: blue;'>OR -> []</p><p style='font-weight: bold; color: blue;'>Z -> []</p><p style='font-weight: bold; color: blue;'>D -> []</p><h2 class='text-muted'>GWAS columns</h2>snp, chr, bp, a1, a0, maf, hwep, info, beta, se, p<h2 class='text-muted'>Identified columns</h2>chr, snp, bp, a1, a0, a1, p, info, maf, beta, se<h2 class='text-muted'>Unidentified Columns headers</h2>hwep<h2 class='text-muted'>PMID</h2>30940143<h2 class='text-muted'>DOI</h2>10.1186/s12931-019-1036-8<div class='bibtex-container'><h2 class='text-muted'>BibTeX</h2><div style='font-family: monospace; white-space: pre-wrap;'> @article{Zhu_2019, title={Genetic overlap of chronic obstructive pulmonary disease and cardiovascular disease-related traits: a large-scale genome-wide cross-trait analysis}, volume={20}, ISSN={1465-993X}, url={http://dx.doi.org/10.1186/s12931-019-1036-8}, DOI={10.1186/s12931-019-1036-8}, number={1}, journal={Respiratory Research}, publisher={Springer Science and Business Media LLC}, author={Zhu, Zhaozhong and Wang, Xiaofang and Li, Xihao and Lin, Yifei and Shen, Sipeng and Liu, Cong-Lin and Hobbs, Brain D. and Hasegawa, Kohei and Liang, Liming and Boezen, H. Marike and Camargo, Carlos A. and Cho, Michael H. and Christiani, David C.}, year={2019}, month=apr }
</div></div><br><br><br><br><br><br></div><div class='container mt-4 bg-light p-4 text-center'><h1 class='text-primary'>Phenotype: hypertension</h1><p class='text-muted'><strong>URL:</strong> http://ftp.ebi.ac.uk/pub/databases/gwas/summary_statistics/GCST90077001-GCST90078000/GCST90077645</p><table border="1" class="dataframe table table-bordered">
  <thead>
    <tr style="text-align: right;">
      <th>firstAuthor</th>
      <th>accessionId</th>
      <th>publicationDate</th>
      <th>journal</th>
      <th>title</th>
      <th>reportedTrait</th>
      <th>efoTraits</th>
      <th>bgTraits</th>
      <th>discoverySampleAncestry</th>
      <th>replicationSampleAncestry</th>
      <th>associationCount</th>
      <th>summaryStatistics</th>
      <th>ssApiFlag</th>
      <th>agreedToCc0</th>
      <th>genotypingTechnologies</th>
      <th>pubmedId</th>
      <th>initialSampleDescription</th>
      <th>replicateSampleDescription</th>
      <th>UKB_FIELD</th>
      <th>FuzzySimilarity_hypertension</th>
      <th>SearchPhenotype</th>
    </tr>
  </thead>
  <tbody>
    <tr>
      <td>Backman JD</td>
      <td>GCST90077645</td>
      <td>10/18/2021</td>
      <td>Nature</td>
      <td>Exome sequencing and analysis of 454,787 UK Biobank participants.</td>
      <td>hypertension</td>
      <td>hypertension</td>
      <td>-</td>
      <td>331754 european</td>
      <td>118739 European</td>
      <td>0</td>
      <td>http://ftp.ebi.ac.uk/pub/databases/gwas/summary_statistics/GCST90077001-GCST90078000/GCST90077645</td>
      <td>no</td>
      <td>yes</td>
      <td>Exome-wide sequencing,Genome-wide genotyping array</td>
      <td>34662886</td>
      <td>115,454 European ancestry cases,216,300 European ancestry controls</td>
      <td>65,295 European ancestry cases,53,444 European ancestry controls</td>
      <td>-</td>
      <td>100</td>
      <td>hypertension</td>
    </tr>
  </tbody>
</table><h2 class='text-muted'>Information file</h2><table border="1" class="dataframe table table-bordered">
  <thead>
    <tr style="text-align: right;">
      <th>Unnamed: 0</th>
      <th>Name</th>
      <th>Last modified</th>
      <th>Size</th>
      <th>Description</th>
      <th>URL</th>
    </tr>
  </thead>
  <tbody>
    <tr>
      <td>1</td>
      <td>GCST90077645_buildGRCh38.tsv.gz-meta.yaml</td>
      <td>2024-01-04 00:31</td>
      <td>884</td>
      <td>NaN</td>
      <td>http://ftp.ebi.ac.uk/pub/databases/gwas/summary_statistics/GCST90077001-GCST90078000/GCST90077645/GCST90077645_buildGRCh38.tsv.gz-meta.yaml</td>
    </tr>
    <tr>
      <td>2</td>
      <td>README.txt</td>
      <td>2021-11-18 00:30</td>
      <td>698</td>
      <td>NaN</td>
      <td>http://ftp.ebi.ac.uk/pub/databases/gwas/summary_statistics/GCST90077001-GCST90078000/GCST90077645/README.txt</td>
    </tr>
    <tr>
      <td>3</td>
      <td>harmonised/</td>
      <td>2024-02-02 15:16</td>
      <td>-</td>
      <td>NaN</td>
      <td>http://ftp.ebi.ac.uk/pub/databases/gwas/summary_statistics/GCST90077001-GCST90078000/GCST90077645/harmonised/</td>
    </tr>
    <tr>
      <td>4</td>
      <td>md5sum.txt</td>
      <td>2021-11-18 00:30</td>
      <td>109</td>
      <td>NaN</td>
      <td>http://ftp.ebi.ac.uk/pub/databases/gwas/summary_statistics/GCST90077001-GCST90078000/GCST90077645/md5sum.txt</td>
    </tr>
    <tr>
      <td>0</td>
      <td>GCST90077645_buildGRCh38.tsv.gz</td>
      <td>2021-11-18 00:30</td>
      <td>15M</td>
      <td>NaN</td>
      <td>http://ftp.ebi.ac.uk/pub/databases/gwas/summary_statistics/GCST90077001-GCST90078000/GCST90077645/GCST90077645_buildGRCh38.tsv.gz</td>
    </tr>
  </tbody>
</table><h2 class='text-muted'>Processed file</h2><p>Processing file: GCST90077645_buildGRCh38.tsv.gz!</p><h2 class='text-muted'>GWAS file head</h2><table border="1" class="dataframe table table-bordered">
  <thead>
    <tr style="text-align: right;">
      <th>name</th>
      <th>chromosome</th>
      <th>base_pair_location</th>
      <th>other_allele</th>
      <th>effect_allele</th>
      <th>trait</th>
      <th>cohort</th>
      <th>model</th>
      <th>odds_ratio</th>
      <th>ci_lower</th>
      <th>ci_upper</th>
      <th>p_value</th>
      <th>effect_allele_frequency</th>
      <th>standard_error</th>
    </tr>
  </thead>
  <tbody>
    <tr>
      <td>1_930165_G_A</td>
      <td>1</td>
      <td>930165</td>
      <td>G</td>
      <td>A</td>
      <td>20002_Illness_code_SR_1065_hypertension_AnyInst__BIN</td>
      <td>UKB_Freeze_450_EUR</td>
      <td>ADD-WGR-FIRTH</td>
      <td>1.32885</td>
      <td>0.577734</td>
      <td>3.05649</td>
      <td>0.503499</td>
      <td>0.000042</td>
      <td>0.424984</td>
    </tr>
    <tr>
      <td>1_930204_G_A</td>
      <td>1</td>
      <td>930204</td>
      <td>G</td>
      <td>A</td>
      <td>20002_Illness_code_SR_1065_hypertension_AnyInst__BIN</td>
      <td>UKB_Freeze_450_EUR</td>
      <td>ADD-WGR-FIRTH</td>
      <td>2.12246</td>
      <td>0.896652</td>
      <td>5.02408</td>
      <td>0.086928</td>
      <td>0.000041</td>
      <td>0.439633</td>
    </tr>
  </tbody>
</table></div><div class='container mt-4 bg-light p-4 text-center'><h2 class='text-muted'>Original GWAS file headers</h2><table border="1" class="dataframe table table-bordered">
  <thead>
    <tr style="text-align: right;">
      <th>name</th>
      <th>chromosome</th>
      <th>base_pair_location</th>
      <th>other_allele</th>
      <th>effect_allele</th>
      <th>trait</th>
      <th>cohort</th>
      <th>model</th>
      <th>odds_ratio</th>
      <th>ci_lower</th>
      <th>ci_upper</th>
      <th>p_value</th>
      <th>effect_allele_frequency</th>
      <th>standard_error</th>
    </tr>
  </thead>
  <tbody>
    <tr>
      <td>1_930165_G_A</td>
      <td>1</td>
      <td>930165</td>
      <td>G</td>
      <td>A</td>
      <td>20002_Illness_code_SR_1065_hypertension_AnyInst__BIN</td>
      <td>UKB_Freeze_450_EUR</td>
      <td>ADD-WGR-FIRTH</td>
      <td>1.32885</td>
      <td>0.577734</td>
      <td>3.05649</td>
      <td>0.503499</td>
      <td>0.000042</td>
      <td>0.424984</td>
    </tr>
    <tr>
      <td>1_930204_G_A</td>
      <td>1</td>
      <td>930204</td>
      <td>G</td>
      <td>A</td>
      <td>20002_Illness_code_SR_1065_hypertension_AnyInst__BIN</td>
      <td>UKB_Freeze_450_EUR</td>
      <td>ADD-WGR-FIRTH</td>
      <td>2.12246</td>
      <td>0.896652</td>
      <td>5.02408</td>
      <td>0.086928</td>
      <td>0.000041</td>
      <td>0.439633</td>
    </tr>
  </tbody>
</table><p style='font-weight: bold; color: blue;'>CHR -> ['chromosome']</p><p style='font-weight: bold; color: blue;'>SNP -> ['name']</p><p style='font-weight: bold; color: blue;'>BP -> ['name', 'base_pair_location']</p><p style='font-weight: bold; color: blue;'>A1 -> ['effect_allele']</p><p style='font-weight: bold; color: blue;'>A2 -> ['other_allele']</p><p style='font-weight: bold; color: blue;'>N -> []</p><p style='font-weight: bold; color: blue;'>P -> ['p_value']</p><p style='font-weight: bold; color: blue;'>INFO -> []</p><p style='font-weight: bold; color: blue;'>MAF -> ['effect_allele_frequency']</p><p style='font-weight: bold; color: blue;'>BETA -> []</p><p style='font-weight: bold; color: blue;'>SE -> ['standard_error']</p><p style='font-weight: bold; color: blue;'>OR -> ['odds_ratio']</p><p style='font-weight: bold; color: blue;'>Z -> []</p><p style='font-weight: bold; color: blue;'>D -> []</p><h2 class='text-muted'>GWAS columns</h2>name, chromosome, base_pair_location, other_allele, effect_allele, trait, cohort, model, odds_ratio, ci_lower, ci_upper, p_value, effect_allele_frequency, standard_error<h2 class='text-muted'>Identified columns</h2>chromosome, name, name, base_pair_location, effect_allele, other_allele, p_value, effect_allele_frequency, standard_error, odds_ratio<h2 class='text-muted'>Unidentified Columns headers</h2>model, cohort, trait, ci_upper, ci_lower<h2 class='text-muted'>PMID</h2>34662886<h2 class='text-muted'>DOI</h2>10.1038/s41586-021-04103-z<div class='bibtex-container'><h2 class='text-muted'>BibTeX</h2><div style='font-family: monospace; white-space: pre-wrap;'> @article{Backman_2021, title={Exome sequencing and analysis of 454,787 UK Biobank participants}, volume={599}, ISSN={1476-4687}, url={http://dx.doi.org/10.1038/s41586-021-04103-z}, DOI={10.1038/s41586-021-04103-z}, number={7886}, journal={Nature}, publisher={Springer Science and Business Media LLC}, author={Backman, Joshua D. and Li, Alexander H. and Marcketta, Anthony and Sun, Dylan and Mbatchou, Joelle and Kessler, Michael D. and Benner, Christian and Liu, Daren and Locke, Adam E. and Balasubramanian, Suganthi and Yadav, Ashish and Banerjee, Nilanjana and Gillies, Christopher E. and Damask, Amy and Liu, Simon and Bai, Xiaodong and Hawes, Alicia and Maxwell, Evan and Gurski, Lauren and Watanabe, Kyoko and Kosmicki, Jack A. and Rajagopal, Veera and Mighty, Jason and Jones, Marcus and Mitnaul, Lyndon and Stahl, Eli and Coppola, Giovanni and Jorgenson, Eric and Habegger, Lukas and Salerno, William J. and Shuldiner, Alan R. and Lotta, Luca A. and Overton, John D. and Cantor, Michael N. and Reid, Jeffrey G. and Yancopoulos, George and Kang, Hyun M. and Marchini, Jonathan and Baras, Aris and Abecasis, Gonçalo R. and Ferreira, Manuel A. R.}, year={2021}, month=oct, pages={628–634} }
</div></div><br><br><br><br><br><br></div><div class='container mt-4 bg-light p-4 text-center'><h1 class='text-primary'>Phenotype: hypertension</h1><p class='text-muted'><strong>URL:</strong> http://ftp.ebi.ac.uk/pub/databases/gwas/summary_statistics/GCST90000001-GCST90001000/GCST90000064</p><table border="1" class="dataframe table table-bordered">
  <thead>
    <tr style="text-align: right;">
      <th>firstAuthor</th>
      <th>accessionId</th>
      <th>publicationDate</th>
      <th>journal</th>
      <th>title</th>
      <th>reportedTrait</th>
      <th>efoTraits</th>
      <th>bgTraits</th>
      <th>discoverySampleAncestry</th>
      <th>replicationSampleAncestry</th>
      <th>associationCount</th>
      <th>summaryStatistics</th>
      <th>ssApiFlag</th>
      <th>agreedToCc0</th>
      <th>genotypingTechnologies</th>
      <th>pubmedId</th>
      <th>initialSampleDescription</th>
      <th>replicateSampleDescription</th>
      <th>UKB_FIELD</th>
      <th>FuzzySimilarity_hypertension</th>
      <th>SearchPhenotype</th>
    </tr>
  </thead>
  <tbody>
    <tr>
      <td>Surendran P</td>
      <td>GCST90000064</td>
      <td>11/23/2020</td>
      <td>Nat Genet</td>
      <td>Discovery of rare variants associated with blood pressure regulation through meta-analysis of 1.3 million individuals.</td>
      <td>hypertension</td>
      <td>hypertension</td>
      <td>-</td>
      <td>810865 european,21077 african american or afro-caribbean,33689 south asian,4586 hispanic or latin american</td>
      <td>NaN</td>
      <td>4</td>
      <td>http://ftp.ebi.ac.uk/pub/databases/gwas/summary_statistics/GCST90000001-GCST90001000/GCST90000064</td>
      <td>no</td>
      <td>yes</td>
      <td>Exome genotyping array</td>
      <td>33230300</td>
      <td>810,865 European ancestry individuals,21,077 African American individuals,33,689 South Asian ancestry individuals,4,586 Hispanic individuals</td>
      <td>NaN</td>
      <td>-</td>
      <td>100</td>
      <td>hypertension</td>
    </tr>
  </tbody>
</table><h2 class='text-muted'>Information file</h2><table border="1" class="dataframe table table-bordered">
  <thead>
    <tr style="text-align: right;">
      <th>Unnamed: 0</th>
      <th>Name</th>
      <th>Last modified</th>
      <th>Size</th>
      <th>Description</th>
      <th>URL</th>
    </tr>
  </thead>
  <tbody>
    <tr>
      <td>3</td>
      <td>harmonised/</td>
      <td>2024-02-02 15:18</td>
      <td>-</td>
      <td>NaN</td>
      <td>http://ftp.ebi.ac.uk/pub/databases/gwas/summary_statistics/GCST90000001-GCST90001000/GCST90000064/harmonised/</td>
    </tr>
    <tr>
      <td>4</td>
      <td>md5sum.txt</td>
      <td>2021-10-08 10:21</td>
      <td>124</td>
      <td>NaN</td>
      <td>http://ftp.ebi.ac.uk/pub/databases/gwas/summary_statistics/GCST90000001-GCST90001000/GCST90000064/md5sum.txt</td>
    </tr>
    <tr>
      <td>2</td>
      <td>GCST90000064_buildGRCh37.tsv-meta.yaml</td>
      <td>2024-01-04 00:14</td>
      <td>1.1K</td>
      <td>NaN</td>
      <td>http://ftp.ebi.ac.uk/pub/databases/gwas/summary_statistics/GCST90000001-GCST90001000/GCST90000064/GCST90000064_buildGRCh37.tsv-meta.yaml</td>
    </tr>
    <tr>
      <td>0</td>
      <td>BP-ICE_PA_HTN_15-04-2020.txt</td>
      <td>2021-10-08 10:21</td>
      <td>24M</td>
      <td>NaN</td>
      <td>http://ftp.ebi.ac.uk/pub/databases/gwas/summary_statistics/GCST90000001-GCST90001000/GCST90000064/BP-ICE_PA_HTN_15-04-2020.txt</td>
    </tr>
    <tr>
      <td>1</td>
      <td>GCST90000064_buildGRCh37.tsv</td>
      <td>2021-10-08 10:21</td>
      <td>24M</td>
      <td>NaN</td>
      <td>http://ftp.ebi.ac.uk/pub/databases/gwas/summary_statistics/GCST90000001-GCST90001000/GCST90000064/GCST90000064_buildGRCh37.tsv</td>
    </tr>
  </tbody>
</table><h2 class='text-muted'>Processed file</h2><p>Processing file: BP-ICE_PA_HTN_15-04-2020.txt.1!</p><h2 class='text-muted'>GWAS file head</h2><table border="1" class="dataframe table table-bordered">
  <thead>
    <tr style="text-align: right;">
      <th>rsid</th>
      <th>markername</th>
      <th>allele1</th>
      <th>allele2</th>
      <th>freq1</th>
      <th>freqse</th>
      <th>minfreq</th>
      <th>maxfreq</th>
      <th>weight</th>
      <th>zscore</th>
      <th>p-value</th>
      <th>direction</th>
      <th>hetisq</th>
      <th>hetchisq</th>
      <th>hetdf</th>
      <th>hetpval</th>
      <th>n</th>
    </tr>
  </thead>
  <tbody>
    <tr>
      <td>rs75333668</td>
      <td>1_762320_C_T</td>
      <td>t</td>
      <td>c</td>
      <td>0.0031</td>
      <td>0.0026</td>
      <td>0.0009</td>
      <td>0.0060</td>
      <td>644545</td>
      <td>-0.490145</td>
      <td>0.624</td>
      <td>+-</td>
      <td>66.0</td>
      <td>2.941</td>
      <td>1</td>
      <td>0.08636</td>
      <td>644545</td>
    </tr>
    <tr>
      <td>rs200686669</td>
      <td>1_861349_C_T</td>
      <td>t</td>
      <td>c</td>
      <td>0.0006</td>
      <td>0.0001</td>
      <td>0.0005</td>
      <td>0.0007</td>
      <td>517947</td>
      <td>0.917323</td>
      <td>0.359</td>
      <td>+-</td>
      <td>39.6</td>
      <td>1.656</td>
      <td>1</td>
      <td>0.19810</td>
      <td>517947</td>
    </tr>
  </tbody>
</table></div><div class='container mt-4 bg-light p-4 text-center'><h2 class='text-muted'>Original GWAS file headers</h2><table border="1" class="dataframe table table-bordered">
  <thead>
    <tr style="text-align: right;">
      <th>rsid</th>
      <th>markername</th>
      <th>allele1</th>
      <th>allele2</th>
      <th>freq1</th>
      <th>freqse</th>
      <th>minfreq</th>
      <th>maxfreq</th>
      <th>weight</th>
      <th>zscore</th>
      <th>p-value</th>
      <th>direction</th>
      <th>hetisq</th>
      <th>hetchisq</th>
      <th>hetdf</th>
      <th>hetpval</th>
      <th>n</th>
    </tr>
  </thead>
  <tbody>
    <tr>
      <td>rs75333668</td>
      <td>1_762320_C_T</td>
      <td>t</td>
      <td>c</td>
      <td>0.0031</td>
      <td>0.0026</td>
      <td>0.0009</td>
      <td>0.0060</td>
      <td>644545</td>
      <td>-0.490145</td>
      <td>0.624</td>
      <td>+-</td>
      <td>66.0</td>
      <td>2.941</td>
      <td>1</td>
      <td>0.08636</td>
      <td>644545</td>
    </tr>
    <tr>
      <td>rs200686669</td>
      <td>1_861349_C_T</td>
      <td>t</td>
      <td>c</td>
      <td>0.0006</td>
      <td>0.0001</td>
      <td>0.0005</td>
      <td>0.0007</td>
      <td>517947</td>
      <td>0.917323</td>
      <td>0.359</td>
      <td>+-</td>
      <td>39.6</td>
      <td>1.656</td>
      <td>1</td>
      <td>0.19810</td>
      <td>517947</td>
    </tr>
  </tbody>
</table><p style='font-weight: bold; color: blue;'>CHR -> []</p><p style='font-weight: bold; color: blue;'>SNP -> ['rsid', 'markername']</p><p style='font-weight: bold; color: blue;'>BP -> ['markername']</p><p style='font-weight: bold; color: blue;'>A1 -> ['allele1']</p><p style='font-weight: bold; color: blue;'>A2 -> ['allele1', 'allele2']</p><p style='font-weight: bold; color: blue;'>N -> ['weight', 'n']</p><p style='font-weight: bold; color: blue;'>P -> ['p-value', 'hetpval']</p><p style='font-weight: bold; color: blue;'>INFO -> []</p><p style='font-weight: bold; color: blue;'>MAF -> ['freq1']</p><p style='font-weight: bold; color: blue;'>BETA -> []</p><p style='font-weight: bold; color: blue;'>SE -> ['freqse']</p><p style='font-weight: bold; color: blue;'>OR -> []</p><p style='font-weight: bold; color: blue;'>Z -> ['zscore']</p><p style='font-weight: bold; color: blue;'>D -> ['direction']</p><h2 class='text-muted'>GWAS columns</h2>rsid, markername, allele1, allele2, freq1, freqse, minfreq, maxfreq, weight, zscore, p-value, direction, hetisq, hetchisq, hetdf, hetpval, n<h2 class='text-muted'>Identified columns</h2>rsid, markername, markername, allele1, allele1, allele2, weight, n, p-value, hetpval, freq1, freqse, zscore, direction<h2 class='text-muted'>Unidentified Columns headers</h2>hetisq, hetchisq, minfreq, hetdf, maxfreq<h2 class='text-muted'>PMID</h2>33230300<h2 class='text-muted'>DOI</h2>10.1038/s41588-020-00713-x<div class='bibtex-container'><h2 class='text-muted'>BibTeX</h2><div style='font-family: monospace; white-space: pre-wrap;'> @article{Surendran_2020, title={Discovery of rare variants associated with blood pressure regulation through meta-analysis of 1.3 million individuals}, volume={52}, ISSN={1546-1718}, url={http://dx.doi.org/10.1038/s41588-020-00713-x}, DOI={10.1038/s41588-020-00713-x}, number={12}, journal={Nature Genetics}, publisher={Springer Science and Business Media LLC}, author={Surendran, Praveen and Feofanova, Elena V. and Lahrouchi, Najim and Ntalla, Ioanna and Karthikeyan, Savita and Cook, James and Chen, Lingyan and Mifsud, Borbala and Yao, Chen and Kraja, Aldi T. and Cartwright, James H. and Hellwege, Jacklyn N. and Giri, Ayush and Tragante, Vinicius and Thorleifsson, Gudmar and Liu, Dajiang J. and Prins, Bram P. and Stewart, Isobel D. and Cabrera, Claudia P. and Eales, James M. and Akbarov, Artur and Auer, Paul L. and Bielak, Lawrence F. and Bis, Joshua C. and Braithwaite, Vickie S. and Brody, Jennifer A. and Daw, E. Warwick and Warren, Helen R. and Drenos, Fotios and Nielsen, Sune Fallgaard and Faul, Jessica D. and Fauman, Eric B. and Fava, Cristiano and Ferreira, Teresa and Foley, Christopher N. and Franceschini, Nora and Gao, He and Giannakopoulou, Olga and Giulianini, Franco and Gudbjartsson, Daniel F. and Guo, Xiuqing and Harris, Sarah E. and Havulinna, Aki S. and Helgadottir, Anna and Huffman, Jennifer E. and Hwang, Shih-Jen and Kanoni, Stavroula and Kontto, Jukka and Larson, Martin G. and Li-Gao, Ruifang and Lindström, Jaana and Lotta, Luca A. and Lu, Yingchang and Luan, Jian’an and Mahajan, Anubha and Malerba, Giovanni and Masca, Nicholas G. D. and Mei, Hao and Menni, Cristina and Mook-Kanamori, Dennis O. and Mosen-Ansorena, David and Müller-Nurasyid, Martina and Paré, Guillaume and Paul, Dirk S. and Perola, Markus and Poveda, Alaitz and Rauramaa, Rainer and Richard, Melissa and Richardson, Tom G. and Sepúlveda, Nuno and Sim, Xueling and Smith, Albert V. and Smith, Jennifer A. and Staley, James R. and Stanáková, Alena and Sulem, Patrick and Thériault, Sébastien and Thorsteinsdottir, Unnur and Trompet, Stella and Varga, Tibor V. and Velez Edwards, Digna R. and Veronesi, Giovanni and Weiss, Stefan and Willems, Sara M. and Yao, Jie and Young, Robin and Yu, Bing and Zhang, Weihua and Zhao, Jing-Hua and Zhao, Wei and Zhao, Wei and Evangelou, Evangelos and Aeschbacher, Stefanie and Asllanaj, Eralda and Blankenberg, Stefan and Bonnycastle, Lori L. and Bork-Jensen, Jette and Brandslund, Ivan and Braund, Peter S. and Burgess, Stephen and Cho, Kelly and Christensen, Cramer and Connell, John and Mutsert, Renée de and Dominiczak, Anna F. and Dörr, Marcus and Eiriksdottir, Gudny and Farmaki, Aliki-Eleni and Gaziano, J. Michael and Grarup, Niels and Grove, Megan L. and Hallmans, Göran and Hansen, Torben and Have, Christian T. and Heiss, Gerardo and Jørgensen, Marit E. and Jousilahti, Pekka and Kajantie, Eero and Kamat, Mihir and Käräjämäki, AnneMari and Karpe, Fredrik and Koistinen, Heikki A. and Kovesdy, Csaba P. and Kuulasmaa, Kari and Laatikainen, Tiina and Lannfelt, Lars and Lee, I-Te and Lee, Wen-Jane and de Boer, Rudolf A. and van der Harst, Pim and van der Meer, Peter and Verweij, Niek and Linneberg, Allan and Martin, Lisa W. and Moitry, Marie and Nadkarni, Girish and Neville, Matt J. and Palmer, Colin N. A. and Papanicolaou, George J. and Pedersen, Oluf and Peters, James and Poulter, Neil and Rasheed, Asif and Rasmussen, Katrine L. and Rayner, N. William and Mägi, Reedik and Renström, Frida and Rettig, Rainer and Rossouw, Jacques and Schreiner, Pamela J. and Sever, Peter S. and Sigurdsson, Emil L. and Skaaby, Tea and Sun, Yan V. and Sundstrom, Johan and Thorgeirsson, Gudmundur and Esko, Tõnu and Trabetti, Elisabetta and Tsao, Philip S. and Tuomi, Tiinamaija and Turner, Stephen T. and Tzoulaki, Ioanna and Vaartjes, Ilonca and Vergnaud, Anne-Claire and Willer, Cristen J. and Wilson, Peter W. F. and Witte, Daniel R. and Yonova-Doing, Ekaterina and Zhang, He and Aliya, Naheed and Almgren, Peter and Amouyel, Philippe and Asselbergs, Folkert W. and Barnes, Michael R. and Blakemore, Alexandra I. and Boehnke, Michael and Bots, Michiel L. and Bottinger, Erwin P. and Buring, Julie E. and Chambers, John C. and Chen, Yii-Der Ida and Chowdhury, Rajiv and Conen, David and Correa, Adolfo and Davey Smith, George and Boer, Rudolf A. de and Deary, Ian J. and Dedoussis, George and Deloukas, Panos and Di Angelantonio, Emanuele and Elliott, Paul and Butterworth, Adam S. and Danesh, John and Langenberg, Claudia and Deloukas, Panos and McCarthy, Mark I. and Franks, Paul W. and Rolandsson, Olov and Wareham, Nicholas J. and Felix, Stephan B. and Ferrières, Jean and Ford, Ian and Fornage, Myriam and Franks, Paul W. and Franks, Stephen and Frossard, Philippe and Gambaro, Giovanni and Gaunt, Tom R. and Groop, Leif and Gudnason, Vilmundur and Harris, Tamara B. and Hayward, Caroline and Hennig, Branwen J. and Herzig, Karl-Heinz and Ingelsson, Erik and Tuomilehto, Jaakko and Järvelin, Marjo-Riitta and Jukema, J. Wouter and Kardia, Sharon L. R. and Kee, Frank and Kooner, Jaspal S. and Kooperberg, Charles and Launer, Lenore J. and Lind, Lars and Loos, Ruth J. F. and Majumder, Abdulla al Shafi. and Laakso, Markku and McCarthy, Mark I. and Melander, Olle and Mohlke, Karen L. and Murray, Alison D. and Nordestgaard, Børge Grønne and Orho-Melander, Marju and Packard, Chris J. and Padmanabhan, Sandosh and Palmas, Walter and Polasek, Ozren and Porteous, David J. and Prentice, Andrew M. and Province, Michael A. and Relton, Caroline L. and Rice, Kenneth and Ridker, Paul M. and Rolandsson, Olov and Rosendaal, Frits R. and Rotter, Jerome I. and Rudan, Igor and Salomaa, Veikko and Samani, Nilesh J. and Sattar, Naveed and Sheu, Wayne H.-H. and Smith, Blair H. and Soranzo, Nicole and Spector, Timothy D. and Starr, John M. and Sebert, Sylvain and Taylor, Kent D. and Lakka, Timo A. and Timpson, Nicholas J. and Tobin, Martin D. and Prins, Bram P. and Zeggini, Eleftheria and van der Harst, Pim and van der Meer, Peter and Ramachandran, Vasan S. and Verweij, Niek and Virtamo, Jarmo and Völker, Uwe and Weir, David R. and Zeggini, Eleftheria and Charchar, Fadi J. and Hellwege, Jacklyn N. and Giri, Ayush and Edwards, Digna R. Velez and Cho, Kelly and Gaziano, J. Michael and Kovesdy, Csaba P. and Sun, Yan V. and Tsao, Philip S. and Wilson, Peter W. F. and Edwards, Todd L. and Hung, Adriana M. and O’Donnell, Christopher J. and Wareham, Nicholas J. and Langenberg, Claudia and Tomaszewski, Maciej and Butterworth, Adam S. and Caulfield, Mark J. and Danesh, John and Edwards, Todd L. and Holm, Hilma and Hung, Adriana M. and Lindgren, Cecilia M. and Liu, Chunyu and Manning, Alisa K. and Morris, Andrew P. and Morrison, Alanna C. and O’Donnell, Christopher J. and Psaty, Bruce M. and Saleheen, Danish and Stefansson, Kari and Boerwinkle, Eric and Chasman, Daniel I. and Levy, Daniel and Newton-Cheh, Christopher and Munroe, Patricia B. and Howson, Joanna M. M.}, year={2020}, month=nov, pages={1314–1332} }
</div></div><br><br><br><br><br><br></div><div class='container mt-4 bg-light p-4 text-center'><h1 class='text-primary'>Phenotype: hypertension</h1><p class='text-muted'><strong>URL:</strong> http://ftp.ebi.ac.uk/pub/databases/gwas/summary_statistics/GCST90000001-GCST90001000/GCST90000060</p><table border="1" class="dataframe table table-bordered">
  <thead>
    <tr style="text-align: right;">
      <th>firstAuthor</th>
      <th>accessionId</th>
      <th>publicationDate</th>
      <th>journal</th>
      <th>title</th>
      <th>reportedTrait</th>
      <th>efoTraits</th>
      <th>bgTraits</th>
      <th>discoverySampleAncestry</th>
      <th>replicationSampleAncestry</th>
      <th>associationCount</th>
      <th>summaryStatistics</th>
      <th>ssApiFlag</th>
      <th>agreedToCc0</th>
      <th>genotypingTechnologies</th>
      <th>pubmedId</th>
      <th>initialSampleDescription</th>
      <th>replicateSampleDescription</th>
      <th>UKB_FIELD</th>
      <th>FuzzySimilarity_hypertension</th>
      <th>SearchPhenotype</th>
    </tr>
  </thead>
  <tbody>
    <tr>
      <td>Surendran P</td>
      <td>GCST90000060</td>
      <td>11/23/2020</td>
      <td>Nat Genet</td>
      <td>Discovery of rare variants associated with blood pressure regulation through meta-analysis of 1.3 million individuals.</td>
      <td>hypertension</td>
      <td>hypertension</td>
      <td>-</td>
      <td>810865 european</td>
      <td>NaN</td>
      <td>32</td>
      <td>http://ftp.ebi.ac.uk/pub/databases/gwas/summary_statistics/GCST90000001-GCST90001000/GCST90000060</td>
      <td>no</td>
      <td>yes</td>
      <td>Exome genotyping array</td>
      <td>33230300</td>
      <td>810,865 European ancestry individuals</td>
      <td>NaN</td>
      <td>-</td>
      <td>100</td>
      <td>hypertension</td>
    </tr>
  </tbody>
</table><h2 class='text-muted'>Information file</h2><table border="1" class="dataframe table table-bordered">
  <thead>
    <tr style="text-align: right;">
      <th>Unnamed: 0</th>
      <th>Name</th>
      <th>Last modified</th>
      <th>Size</th>
      <th>Description</th>
      <th>URL</th>
    </tr>
  </thead>
  <tbody>
    <tr>
      <td>2</td>
      <td>GCST90000060_buildGRCh37.tsv-meta.yaml</td>
      <td>2024-01-04 00:21</td>
      <td>701</td>
      <td>NaN</td>
      <td>http://ftp.ebi.ac.uk/pub/databases/gwas/summary_statistics/GCST90000001-GCST90001000/GCST90000060/GCST90000060_buildGRCh37.tsv-meta.yaml</td>
    </tr>
    <tr>
      <td>3</td>
      <td>harmonised/</td>
      <td>2024-02-02 15:18</td>
      <td>-</td>
      <td>NaN</td>
      <td>http://ftp.ebi.ac.uk/pub/databases/gwas/summary_statistics/GCST90000001-GCST90001000/GCST90000060/harmonised/</td>
    </tr>
    <tr>
      <td>4</td>
      <td>md5sum.txt</td>
      <td>2021-10-08 10:20</td>
      <td>125</td>
      <td>NaN</td>
      <td>http://ftp.ebi.ac.uk/pub/databases/gwas/summary_statistics/GCST90000001-GCST90001000/GCST90000060/md5sum.txt</td>
    </tr>
    <tr>
      <td>0</td>
      <td>BP-ICE_EUR_HTN_15-04-2020.txt</td>
      <td>2021-10-08 10:20</td>
      <td>23M</td>
      <td>NaN</td>
      <td>http://ftp.ebi.ac.uk/pub/databases/gwas/summary_statistics/GCST90000001-GCST90001000/GCST90000060/BP-ICE_EUR_HTN_15-04-2020.txt</td>
    </tr>
    <tr>
      <td>1</td>
      <td>GCST90000060_buildGRCh37.tsv</td>
      <td>2021-10-08 10:20</td>
      <td>23M</td>
      <td>NaN</td>
      <td>http://ftp.ebi.ac.uk/pub/databases/gwas/summary_statistics/GCST90000001-GCST90001000/GCST90000060/GCST90000060_buildGRCh37.tsv</td>
    </tr>
  </tbody>
</table><h2 class='text-muted'>Processed file</h2><p>Processing file: BP-ICE_EUR_HTN_15-04-2020.txt.1!</p><h2 class='text-muted'>GWAS file head</h2><table border="1" class="dataframe table table-bordered">
  <thead>
    <tr style="text-align: right;">
      <th>rsid</th>
      <th>markername</th>
      <th>allele1</th>
      <th>allele2</th>
      <th>freq1</th>
      <th>freqse</th>
      <th>minfreq</th>
      <th>maxfreq</th>
      <th>weight</th>
      <th>zscore</th>
      <th>p-value</th>
      <th>direction</th>
      <th>hetisq</th>
      <th>hetchisq</th>
      <th>hetdf</th>
      <th>hetpval</th>
      <th>n</th>
    </tr>
  </thead>
  <tbody>
    <tr>
      <td>rs75333668</td>
      <td>1_762320_C_T</td>
      <td>t</td>
      <td>c</td>
      <td>0.0009</td>
      <td>0.0000</td>
      <td>0.0009</td>
      <td>0.0009</td>
      <td>600720</td>
      <td>-0.550115</td>
      <td>0.5822</td>
      <td>+-</td>
      <td>72.2</td>
      <td>3.603</td>
      <td>1</td>
      <td>0.05767</td>
      <td>600720</td>
    </tr>
    <tr>
      <td>rs200686669</td>
      <td>1_861349_C_T</td>
      <td>t</td>
      <td>c</td>
      <td>0.0006</td>
      <td>0.0001</td>
      <td>0.0005</td>
      <td>0.0007</td>
      <td>515929</td>
      <td>0.874267</td>
      <td>0.3820</td>
      <td>+-</td>
      <td>45.7</td>
      <td>1.841</td>
      <td>1</td>
      <td>0.17480</td>
      <td>515929</td>
    </tr>
  </tbody>
</table></div><div class='container mt-4 bg-light p-4 text-center'><h2 class='text-muted'>Original GWAS file headers</h2><table border="1" class="dataframe table table-bordered">
  <thead>
    <tr style="text-align: right;">
      <th>rsid</th>
      <th>markername</th>
      <th>allele1</th>
      <th>allele2</th>
      <th>freq1</th>
      <th>freqse</th>
      <th>minfreq</th>
      <th>maxfreq</th>
      <th>weight</th>
      <th>zscore</th>
      <th>p-value</th>
      <th>direction</th>
      <th>hetisq</th>
      <th>hetchisq</th>
      <th>hetdf</th>
      <th>hetpval</th>
      <th>n</th>
    </tr>
  </thead>
  <tbody>
    <tr>
      <td>rs75333668</td>
      <td>1_762320_C_T</td>
      <td>t</td>
      <td>c</td>
      <td>0.0009</td>
      <td>0.0000</td>
      <td>0.0009</td>
      <td>0.0009</td>
      <td>600720</td>
      <td>-0.550115</td>
      <td>0.5822</td>
      <td>+-</td>
      <td>72.2</td>
      <td>3.603</td>
      <td>1</td>
      <td>0.05767</td>
      <td>600720</td>
    </tr>
    <tr>
      <td>rs200686669</td>
      <td>1_861349_C_T</td>
      <td>t</td>
      <td>c</td>
      <td>0.0006</td>
      <td>0.0001</td>
      <td>0.0005</td>
      <td>0.0007</td>
      <td>515929</td>
      <td>0.874267</td>
      <td>0.3820</td>
      <td>+-</td>
      <td>45.7</td>
      <td>1.841</td>
      <td>1</td>
      <td>0.17480</td>
      <td>515929</td>
    </tr>
  </tbody>
</table><p style='font-weight: bold; color: blue;'>CHR -> []</p><p style='font-weight: bold; color: blue;'>SNP -> ['rsid', 'markername']</p><p style='font-weight: bold; color: blue;'>BP -> ['markername']</p><p style='font-weight: bold; color: blue;'>A1 -> ['allele1']</p><p style='font-weight: bold; color: blue;'>A2 -> ['allele1', 'allele2']</p><p style='font-weight: bold; color: blue;'>N -> ['weight', 'n']</p><p style='font-weight: bold; color: blue;'>P -> ['p-value', 'hetpval']</p><p style='font-weight: bold; color: blue;'>INFO -> []</p><p style='font-weight: bold; color: blue;'>MAF -> ['freq1']</p><p style='font-weight: bold; color: blue;'>BETA -> []</p><p style='font-weight: bold; color: blue;'>SE -> ['freqse']</p><p style='font-weight: bold; color: blue;'>OR -> []</p><p style='font-weight: bold; color: blue;'>Z -> ['zscore']</p><p style='font-weight: bold; color: blue;'>D -> ['direction']</p><h2 class='text-muted'>GWAS columns</h2>rsid, markername, allele1, allele2, freq1, freqse, minfreq, maxfreq, weight, zscore, p-value, direction, hetisq, hetchisq, hetdf, hetpval, n<h2 class='text-muted'>Identified columns</h2>rsid, markername, markername, allele1, allele1, allele2, weight, n, p-value, hetpval, freq1, freqse, zscore, direction<h2 class='text-muted'>Unidentified Columns headers</h2>hetisq, hetchisq, minfreq, hetdf, maxfreq<h2 class='text-muted'>PMID</h2>33230300<h2 class='text-muted'>DOI</h2>10.1038/s41588-020-00713-x<div class='bibtex-container'><h2 class='text-muted'>BibTeX</h2><div style='font-family: monospace; white-space: pre-wrap;'> @article{Surendran_2020, title={Discovery of rare variants associated with blood pressure regulation through meta-analysis of 1.3 million individuals}, volume={52}, ISSN={1546-1718}, url={http://dx.doi.org/10.1038/s41588-020-00713-x}, DOI={10.1038/s41588-020-00713-x}, number={12}, journal={Nature Genetics}, publisher={Springer Science and Business Media LLC}, author={Surendran, Praveen and Feofanova, Elena V. and Lahrouchi, Najim and Ntalla, Ioanna and Karthikeyan, Savita and Cook, James and Chen, Lingyan and Mifsud, Borbala and Yao, Chen and Kraja, Aldi T. and Cartwright, James H. and Hellwege, Jacklyn N. and Giri, Ayush and Tragante, Vinicius and Thorleifsson, Gudmar and Liu, Dajiang J. and Prins, Bram P. and Stewart, Isobel D. and Cabrera, Claudia P. and Eales, James M. and Akbarov, Artur and Auer, Paul L. and Bielak, Lawrence F. and Bis, Joshua C. and Braithwaite, Vickie S. and Brody, Jennifer A. and Daw, E. Warwick and Warren, Helen R. and Drenos, Fotios and Nielsen, Sune Fallgaard and Faul, Jessica D. and Fauman, Eric B. and Fava, Cristiano and Ferreira, Teresa and Foley, Christopher N. and Franceschini, Nora and Gao, He and Giannakopoulou, Olga and Giulianini, Franco and Gudbjartsson, Daniel F. and Guo, Xiuqing and Harris, Sarah E. and Havulinna, Aki S. and Helgadottir, Anna and Huffman, Jennifer E. and Hwang, Shih-Jen and Kanoni, Stavroula and Kontto, Jukka and Larson, Martin G. and Li-Gao, Ruifang and Lindström, Jaana and Lotta, Luca A. and Lu, Yingchang and Luan, Jian’an and Mahajan, Anubha and Malerba, Giovanni and Masca, Nicholas G. D. and Mei, Hao and Menni, Cristina and Mook-Kanamori, Dennis O. and Mosen-Ansorena, David and Müller-Nurasyid, Martina and Paré, Guillaume and Paul, Dirk S. and Perola, Markus and Poveda, Alaitz and Rauramaa, Rainer and Richard, Melissa and Richardson, Tom G. and Sepúlveda, Nuno and Sim, Xueling and Smith, Albert V. and Smith, Jennifer A. and Staley, James R. and Stanáková, Alena and Sulem, Patrick and Thériault, Sébastien and Thorsteinsdottir, Unnur and Trompet, Stella and Varga, Tibor V. and Velez Edwards, Digna R. and Veronesi, Giovanni and Weiss, Stefan and Willems, Sara M. and Yao, Jie and Young, Robin and Yu, Bing and Zhang, Weihua and Zhao, Jing-Hua and Zhao, Wei and Zhao, Wei and Evangelou, Evangelos and Aeschbacher, Stefanie and Asllanaj, Eralda and Blankenberg, Stefan and Bonnycastle, Lori L. and Bork-Jensen, Jette and Brandslund, Ivan and Braund, Peter S. and Burgess, Stephen and Cho, Kelly and Christensen, Cramer and Connell, John and Mutsert, Renée de and Dominiczak, Anna F. and Dörr, Marcus and Eiriksdottir, Gudny and Farmaki, Aliki-Eleni and Gaziano, J. Michael and Grarup, Niels and Grove, Megan L. and Hallmans, Göran and Hansen, Torben and Have, Christian T. and Heiss, Gerardo and Jørgensen, Marit E. and Jousilahti, Pekka and Kajantie, Eero and Kamat, Mihir and Käräjämäki, AnneMari and Karpe, Fredrik and Koistinen, Heikki A. and Kovesdy, Csaba P. and Kuulasmaa, Kari and Laatikainen, Tiina and Lannfelt, Lars and Lee, I-Te and Lee, Wen-Jane and de Boer, Rudolf A. and van der Harst, Pim and van der Meer, Peter and Verweij, Niek and Linneberg, Allan and Martin, Lisa W. and Moitry, Marie and Nadkarni, Girish and Neville, Matt J. and Palmer, Colin N. A. and Papanicolaou, George J. and Pedersen, Oluf and Peters, James and Poulter, Neil and Rasheed, Asif and Rasmussen, Katrine L. and Rayner, N. William and Mägi, Reedik and Renström, Frida and Rettig, Rainer and Rossouw, Jacques and Schreiner, Pamela J. and Sever, Peter S. and Sigurdsson, Emil L. and Skaaby, Tea and Sun, Yan V. and Sundstrom, Johan and Thorgeirsson, Gudmundur and Esko, Tõnu and Trabetti, Elisabetta and Tsao, Philip S. and Tuomi, Tiinamaija and Turner, Stephen T. and Tzoulaki, Ioanna and Vaartjes, Ilonca and Vergnaud, Anne-Claire and Willer, Cristen J. and Wilson, Peter W. F. and Witte, Daniel R. and Yonova-Doing, Ekaterina and Zhang, He and Aliya, Naheed and Almgren, Peter and Amouyel, Philippe and Asselbergs, Folkert W. and Barnes, Michael R. and Blakemore, Alexandra I. and Boehnke, Michael and Bots, Michiel L. and Bottinger, Erwin P. and Buring, Julie E. and Chambers, John C. and Chen, Yii-Der Ida and Chowdhury, Rajiv and Conen, David and Correa, Adolfo and Davey Smith, George and Boer, Rudolf A. de and Deary, Ian J. and Dedoussis, George and Deloukas, Panos and Di Angelantonio, Emanuele and Elliott, Paul and Butterworth, Adam S. and Danesh, John and Langenberg, Claudia and Deloukas, Panos and McCarthy, Mark I. and Franks, Paul W. and Rolandsson, Olov and Wareham, Nicholas J. and Felix, Stephan B. and Ferrières, Jean and Ford, Ian and Fornage, Myriam and Franks, Paul W. and Franks, Stephen and Frossard, Philippe and Gambaro, Giovanni and Gaunt, Tom R. and Groop, Leif and Gudnason, Vilmundur and Harris, Tamara B. and Hayward, Caroline and Hennig, Branwen J. and Herzig, Karl-Heinz and Ingelsson, Erik and Tuomilehto, Jaakko and Järvelin, Marjo-Riitta and Jukema, J. Wouter and Kardia, Sharon L. R. and Kee, Frank and Kooner, Jaspal S. and Kooperberg, Charles and Launer, Lenore J. and Lind, Lars and Loos, Ruth J. F. and Majumder, Abdulla al Shafi. and Laakso, Markku and McCarthy, Mark I. and Melander, Olle and Mohlke, Karen L. and Murray, Alison D. and Nordestgaard, Børge Grønne and Orho-Melander, Marju and Packard, Chris J. and Padmanabhan, Sandosh and Palmas, Walter and Polasek, Ozren and Porteous, David J. and Prentice, Andrew M. and Province, Michael A. and Relton, Caroline L. and Rice, Kenneth and Ridker, Paul M. and Rolandsson, Olov and Rosendaal, Frits R. and Rotter, Jerome I. and Rudan, Igor and Salomaa, Veikko and Samani, Nilesh J. and Sattar, Naveed and Sheu, Wayne H.-H. and Smith, Blair H. and Soranzo, Nicole and Spector, Timothy D. and Starr, John M. and Sebert, Sylvain and Taylor, Kent D. and Lakka, Timo A. and Timpson, Nicholas J. and Tobin, Martin D. and Prins, Bram P. and Zeggini, Eleftheria and van der Harst, Pim and van der Meer, Peter and Ramachandran, Vasan S. and Verweij, Niek and Virtamo, Jarmo and Völker, Uwe and Weir, David R. and Zeggini, Eleftheria and Charchar, Fadi J. and Hellwege, Jacklyn N. and Giri, Ayush and Edwards, Digna R. Velez and Cho, Kelly and Gaziano, J. Michael and Kovesdy, Csaba P. and Sun, Yan V. and Tsao, Philip S. and Wilson, Peter W. F. and Edwards, Todd L. and Hung, Adriana M. and O’Donnell, Christopher J. and Wareham, Nicholas J. and Langenberg, Claudia and Tomaszewski, Maciej and Butterworth, Adam S. and Caulfield, Mark J. and Danesh, John and Edwards, Todd L. and Holm, Hilma and Hung, Adriana M. and Lindgren, Cecilia M. and Liu, Chunyu and Manning, Alisa K. and Morris, Andrew P. and Morrison, Alanna C. and O’Donnell, Christopher J. and Psaty, Bruce M. and Saleheen, Danish and Stefansson, Kari and Boerwinkle, Eric and Chasman, Daniel I. and Levy, Daniel and Newton-Cheh, Christopher and Munroe, Patricia B. and Howson, Joanna M. M.}, year={2020}, month=nov, pages={1314–1332} }
</div></div><br><br><br><br><br><br></div><div class='container mt-4 bg-light p-4 text-center'><h1 class='text-primary'>Phenotype: hypertension</h1><p class='text-muted'><strong>URL:</strong> http://ftp.ebi.ac.uk/pub/databases/gwas/summary_statistics/GCST009001-GCST010000/GCST009654</p><table border="1" class="dataframe table table-bordered">
  <thead>
    <tr style="text-align: right;">
      <th>firstAuthor</th>
      <th>accessionId</th>
      <th>publicationDate</th>
      <th>journal</th>
      <th>title</th>
      <th>reportedTrait</th>
      <th>efoTraits</th>
      <th>bgTraits</th>
      <th>discoverySampleAncestry</th>
      <th>replicationSampleAncestry</th>
      <th>associationCount</th>
      <th>summaryStatistics</th>
      <th>ssApiFlag</th>
      <th>agreedToCc0</th>
      <th>genotypingTechnologies</th>
      <th>pubmedId</th>
      <th>initialSampleDescription</th>
      <th>replicateSampleDescription</th>
      <th>UKB_FIELD</th>
      <th>FuzzySimilarity_hypertension</th>
      <th>SearchPhenotype</th>
    </tr>
  </thead>
  <tbody>
    <tr>
      <td>Oh SW</td>
      <td>GCST009654</td>
      <td>1/7/2020</td>
      <td>PLoS One</td>
      <td>Genome-wide association study of metabolic syndrome in Korean populations.</td>
      <td>hypertension</td>
      <td>hypertension</td>
      <td>-</td>
      <td>2253 east asian</td>
      <td>NaN</td>
      <td>0</td>
      <td>http://ftp.ebi.ac.uk/pub/databases/gwas/summary_statistics/GCST009001-GCST010000/GCST009654</td>
      <td>no</td>
      <td>-</td>
      <td>Genome-wide genotyping array</td>
      <td>31910446</td>
      <td>2,253 Korean ancestry cases and controls</td>
      <td>NaN</td>
      <td>-</td>
      <td>100</td>
      <td>hypertension</td>
    </tr>
  </tbody>
</table><h2 class='text-muted'>Information file</h2><table border="1" class="dataframe table table-bordered">
  <thead>
    <tr style="text-align: right;">
      <th>Unnamed: 0</th>
      <th>Name</th>
      <th>Last modified</th>
      <th>Size</th>
      <th>Description</th>
      <th>URL</th>
    </tr>
  </thead>
  <tbody>
    <tr>
      <td>1</td>
      <td>md5sum.txt</td>
      <td>2020-02-28 17:44</td>
      <td>65</td>
      <td>NaN</td>
      <td>http://ftp.ebi.ac.uk/pub/databases/gwas/summary_statistics/GCST009001-GCST010000/GCST009654/md5sum.txt</td>
    </tr>
    <tr>
      <td>0</td>
      <td>REPORT_high_BP_replication.xlsx</td>
      <td>2020-02-28 17:44</td>
      <td>89M</td>
      <td>NaN</td>
      <td>http://ftp.ebi.ac.uk/pub/databases/gwas/summary_statistics/GCST009001-GCST010000/GCST009654/REPORT_high_BP_replication.xlsx</td>
    </tr>
  </tbody>
</table><h2 class='text-muted'>GWAS file</h2><p>We were unable to process the GWAS file!</p><div class='container mt-4 bg-light p-4 text-center'><h1 class='text-primary'>Phenotype: hypertension</h1><p class='text-muted'><strong>URL:</strong> http://ftp.ebi.ac.uk/pub/databases/gwas/summary_statistics/GCST90086001-GCST90087000/GCST90086089</p><table border="1" class="dataframe table table-bordered">
  <thead>
    <tr style="text-align: right;">
      <th>firstAuthor</th>
      <th>accessionId</th>
      <th>publicationDate</th>
      <th>journal</th>
      <th>title</th>
      <th>reportedTrait</th>
      <th>efoTraits</th>
      <th>bgTraits</th>
      <th>discoverySampleAncestry</th>
      <th>replicationSampleAncestry</th>
      <th>associationCount</th>
      <th>summaryStatistics</th>
      <th>ssApiFlag</th>
      <th>agreedToCc0</th>
      <th>genotypingTechnologies</th>
      <th>pubmedId</th>
      <th>initialSampleDescription</th>
      <th>replicateSampleDescription</th>
      <th>UKB_FIELD</th>
      <th>FuzzySimilarity_hypertension</th>
      <th>SearchPhenotype</th>
    </tr>
  </thead>
  <tbody>
    <tr>
      <td>Guindo-Mart?�nez M</td>
      <td>GCST90086089</td>
      <td>4/23/2021</td>
      <td>Nat Commun</td>
      <td>The impact of non-additive genetic associations on age-related complex diseases.</td>
      <td>hypertension</td>
      <td>hypertension</td>
      <td>-</td>
      <td>56637 european</td>
      <td>NaN</td>
      <td>0</td>
      <td>http://ftp.ebi.ac.uk/pub/databases/gwas/summary_statistics/GCST90086001-GCST90087000/GCST90086089</td>
      <td>no</td>
      <td>yes</td>
      <td>Genome-wide genotyping array</td>
      <td>33893285</td>
      <td>28,391 European ancestry cases,28,246 European ancestry controls</td>
      <td>NaN</td>
      <td>-</td>
      <td>100</td>
      <td>hypertension</td>
    </tr>
  </tbody>
</table><h2 class='text-muted'>Information file</h2><table border="1" class="dataframe table table-bordered">
  <thead>
    <tr style="text-align: right;">
      <th>Unnamed: 0</th>
      <th>Name</th>
      <th>Last modified</th>
      <th>Size</th>
      <th>Description</th>
      <th>URL</th>
    </tr>
  </thead>
  <tbody>
    <tr>
      <td>1</td>
      <td>GCST90086089_buildGRCh37.tsv-meta.yaml</td>
      <td>2024-01-04 00:12</td>
      <td>697</td>
      <td>NaN</td>
      <td>http://ftp.ebi.ac.uk/pub/databases/gwas/summary_statistics/GCST90086001-GCST90087000/GCST90086089/GCST90086089_buildGRCh37.tsv-meta.yaml</td>
    </tr>
    <tr>
      <td>2</td>
      <td>md5sum.txt</td>
      <td>2023-09-09 00:12</td>
      <td>63</td>
      <td>NaN</td>
      <td>http://ftp.ebi.ac.uk/pub/databases/gwas/summary_statistics/GCST90086001-GCST90087000/GCST90086089/md5sum.txt</td>
    </tr>
    <tr>
      <td>0</td>
      <td>GCST90086089_buildGRCh37.tsv</td>
      <td>2023-09-09 00:12</td>
      <td>1.3G</td>
      <td>NaN</td>
      <td>http://ftp.ebi.ac.uk/pub/databases/gwas/summary_statistics/GCST90086001-GCST90087000/GCST90086089/GCST90086089_buildGRCh37.tsv</td>
    </tr>
  </tbody>
</table><h2 class='text-muted'>Processed file</h2><p>Processing file: GCST90086089_buildGRCh37.tsv.1!</p><h2 class='text-muted'>GWAS file head</h2><table border="1" class="dataframe table table-bordered">
  <thead>
    <tr style="text-align: right;">
      <th>chromosome</th>
      <th>base_pair_location</th>
      <th>rs_id_all</th>
      <th>info_all</th>
      <th>other_allele</th>
      <th>effect_allele</th>
      <th>all_maf</th>
      <th>all_total</th>
      <th>freq_effect_allele</th>
      <th>standard_error</th>
      <th>frequentist_gen_se_2</th>
      <th>beta</th>
      <th>frequentist_gen_beta_2</th>
      <th>p_value</th>
    </tr>
  </thead>
  <tbody>
    <tr>
      <td>1</td>
      <td>248000051</td>
      <td>1_248000051_A_G</td>
      <td>0.994</td>
      <td>A</td>
      <td>G</td>
      <td>0.062105</td>
      <td>56637</td>
      <td>0.062105</td>
      <td>0.069732</td>
      <td>0.074959</td>
      <td>0.009762</td>
      <td>-0.020279</td>
      <td>0.927446</td>
    </tr>
    <tr>
      <td>1</td>
      <td>248000108</td>
      <td>1_248000108_G_A</td>
      <td>0.995</td>
      <td>G</td>
      <td>A</td>
      <td>0.184855</td>
      <td>56637</td>
      <td>0.184856</td>
      <td>0.025663</td>
      <td>0.030853</td>
      <td>0.007105</td>
      <td>0.003667</td>
      <td>0.851831</td>
    </tr>
  </tbody>
</table></div><div class='container mt-4 bg-light p-4 text-center'><h2 class='text-muted'>Original GWAS file headers</h2><table border="1" class="dataframe table table-bordered">
  <thead>
    <tr style="text-align: right;">
      <th>chromosome</th>
      <th>base_pair_location</th>
      <th>rs_id_all</th>
      <th>info_all</th>
      <th>other_allele</th>
      <th>effect_allele</th>
      <th>all_maf</th>
      <th>all_total</th>
      <th>freq_effect_allele</th>
      <th>standard_error</th>
      <th>frequentist_gen_se_2</th>
      <th>beta</th>
      <th>frequentist_gen_beta_2</th>
      <th>p_value</th>
    </tr>
  </thead>
  <tbody>
    <tr>
      <td>1</td>
      <td>248000051</td>
      <td>1_248000051_A_G</td>
      <td>0.994</td>
      <td>A</td>
      <td>G</td>
      <td>0.062105</td>
      <td>56637</td>
      <td>0.062105</td>
      <td>0.069732</td>
      <td>0.074959</td>
      <td>0.009762</td>
      <td>-0.020279</td>
      <td>0.927446</td>
    </tr>
    <tr>
      <td>1</td>
      <td>248000108</td>
      <td>1_248000108_G_A</td>
      <td>0.995</td>
      <td>G</td>
      <td>A</td>
      <td>0.184855</td>
      <td>56637</td>
      <td>0.184856</td>
      <td>0.025663</td>
      <td>0.030853</td>
      <td>0.007105</td>
      <td>0.003667</td>
      <td>0.851831</td>
    </tr>
  </tbody>
</table><p style='font-weight: bold; color: blue;'>CHR -> ['chromosome']</p><p style='font-weight: bold; color: blue;'>SNP -> ['rs_id_all']</p><p style='font-weight: bold; color: blue;'>BP -> ['base_pair_location']</p><p style='font-weight: bold; color: blue;'>A1 -> ['effect_allele']</p><p style='font-weight: bold; color: blue;'>A2 -> ['other_allele']</p><p style='font-weight: bold; color: blue;'>N -> ['all_total']</p><p style='font-weight: bold; color: blue;'>P -> ['p_value']</p><p style='font-weight: bold; color: blue;'>INFO -> ['info_all']</p><p style='font-weight: bold; color: blue;'>MAF -> ['all_maf', 'freq_effect_allele']</p><p style='font-weight: bold; color: blue;'>BETA -> ['beta']</p><p style='font-weight: bold; color: blue;'>SE -> ['standard_error']</p><p style='font-weight: bold; color: blue;'>OR -> []</p><p style='font-weight: bold; color: blue;'>Z -> []</p><p style='font-weight: bold; color: blue;'>D -> []</p><h2 class='text-muted'>GWAS columns</h2>chromosome, base_pair_location, rs_id_all, info_all, other_allele, effect_allele, all_maf, all_total, freq_effect_allele, standard_error, frequentist_gen_se_2, beta, frequentist_gen_beta_2, p_value<h2 class='text-muted'>Identified columns</h2>chromosome, rs_id_all, base_pair_location, effect_allele, other_allele, all_total, p_value, info_all, all_maf, freq_effect_allele, beta, standard_error<h2 class='text-muted'>Unidentified Columns headers</h2>frequentist_gen_beta_2, frequentist_gen_se_2<h2 class='text-muted'>PMID</h2>33893285<h2 class='text-muted'>DOI</h2>10.1038/s41467-021-21952-4<div class='bibtex-container'><h2 class='text-muted'>BibTeX</h2><div style='font-family: monospace; white-space: pre-wrap;'> @article{Guindo_Mart_nez_2021, title={The impact of non-additive genetic associations on age-related complex diseases}, volume={12}, ISSN={2041-1723}, url={http://dx.doi.org/10.1038/s41467-021-21952-4}, DOI={10.1038/s41467-021-21952-4}, number={1}, journal={Nature Communications}, publisher={Springer Science and Business Media LLC}, author={Guindo-Martínez, Marta and Amela, Ramon and Bonàs-Guarch, Silvia and Puiggròs, Montserrat and Salvoro, Cecilia and Miguel-Escalada, Irene and Carey, Caitlin E. and Cole, Joanne B. and Rüeger, Sina and Atkinson, Elizabeth and Leong, Aaron and Sanchez, Friman and Ramon-Cortes, Cristian and Ejarque, Jorge and Palmer, Duncan S. and Kurki, Mitja and Aragam, Krishna and Florez, Jose C. and Badia, Rosa M. and Mercader, Josep M. and Torrents, David}, year={2021}, month=apr }
</div></div><br><br><br><br><br><br></div><div class='container mt-4 bg-light p-4 text-center'><h1 class='text-primary'>Phenotype: hypertension</h1><p class='text-muted'><strong>URL:</strong> http://ftp.ebi.ac.uk/pub/databases/gwas/summary_statistics/GCST009001-GCST010000/GCST009659</p><table border="1" class="dataframe table table-bordered">
  <thead>
    <tr style="text-align: right;">
      <th>firstAuthor</th>
      <th>accessionId</th>
      <th>publicationDate</th>
      <th>journal</th>
      <th>title</th>
      <th>reportedTrait</th>
      <th>efoTraits</th>
      <th>bgTraits</th>
      <th>discoverySampleAncestry</th>
      <th>replicationSampleAncestry</th>
      <th>associationCount</th>
      <th>summaryStatistics</th>
      <th>ssApiFlag</th>
      <th>agreedToCc0</th>
      <th>genotypingTechnologies</th>
      <th>pubmedId</th>
      <th>initialSampleDescription</th>
      <th>replicateSampleDescription</th>
      <th>UKB_FIELD</th>
      <th>FuzzySimilarity_hypertension</th>
      <th>SearchPhenotype</th>
    </tr>
  </thead>
  <tbody>
    <tr>
      <td>Oh SW</td>
      <td>GCST009659</td>
      <td>1/7/2020</td>
      <td>PLoS One</td>
      <td>Genome-wide association study of metabolic syndrome in Korean populations.</td>
      <td>hypertension</td>
      <td>hypertension</td>
      <td>-</td>
      <td>7423 east asian</td>
      <td>NaN</td>
      <td>0</td>
      <td>http://ftp.ebi.ac.uk/pub/databases/gwas/summary_statistics/GCST009001-GCST010000/GCST009659</td>
      <td>no</td>
      <td>-</td>
      <td>Genome-wide genotyping array</td>
      <td>31910446</td>
      <td>7,423 Korean ancestry cases and controls</td>
      <td>NaN</td>
      <td>-</td>
      <td>100</td>
      <td>hypertension</td>
    </tr>
  </tbody>
</table><h2 class='text-muted'>Information file</h2><table border="1" class="dataframe table table-bordered">
  <thead>
    <tr style="text-align: right;">
      <th>Unnamed: 0</th>
      <th>Name</th>
      <th>Last modified</th>
      <th>Size</th>
      <th>Description</th>
      <th>URL</th>
    </tr>
  </thead>
  <tbody>
    <tr>
      <td>1</td>
      <td>md5sum.txt</td>
      <td>2020-02-28 17:43</td>
      <td>63</td>
      <td>NaN</td>
      <td>http://ftp.ebi.ac.uk/pub/databases/gwas/summary_statistics/GCST009001-GCST010000/GCST009659/md5sum.txt</td>
    </tr>
    <tr>
      <td>0</td>
      <td>REPORT_high_BP_discovery.xlsx</td>
      <td>2020-02-28 17:43</td>
      <td>91M</td>
      <td>NaN</td>
      <td>http://ftp.ebi.ac.uk/pub/databases/gwas/summary_statistics/GCST009001-GCST010000/GCST009659/REPORT_high_BP_discovery.xlsx</td>
    </tr>
  </tbody>
</table><h2 class='text-muted'>Processed file</h2><p>Processing file: REPORT_high_BP_discovery.xlsx.1!</p><h2 class='text-muted'>GWAS file head</h2><table border="1" class="dataframe table table-bordered">
  <thead>
    <tr style="text-align: right;">
      <th>&lt;?xml</th>
      <th>version=1.0</th>
      <th>encoding=utf-8</th>
      <th>standalone=yes?&gt;</th>
    </tr>
  </thead>
  <tbody>
    <tr>
      <td>PartName=/docProps/core.xml</td>
      <td>ContentType=application/vnd.openxmlformats-package.core-properties+xml/&gt;&lt;Override</td>
      <td>PartName=/docProps/app.xml</td>
      <td>ContentType=application/vnd.openxmlformats-officedocument.extended-properties+xml/&gt;&lt;/Types&gt;</td>
    </tr>
  </tbody>
</table></div><div class='container mt-4 bg-light p-4 text-center'><h2 class='text-muted'>Original GWAS file headers</h2><table border="1" class="dataframe table table-bordered">
  <thead>
    <tr style="text-align: right;">
      <th>&lt;?xml</th>
      <th>version=1.0</th>
      <th>encoding=utf-8</th>
      <th>standalone=yes?&gt;</th>
    </tr>
  </thead>
  <tbody>
    <tr>
      <td>PartName=/docProps/core.xml</td>
      <td>ContentType=application/vnd.openxmlformats-package.core-properties+xml/&gt;&lt;Override</td>
      <td>PartName=/docProps/app.xml</td>
      <td>ContentType=application/vnd.openxmlformats-officedocument.extended-properties+xml/&gt;&lt;/Types&gt;</td>
    </tr>
  </tbody>
</table><p style='font-weight: bold; color: blue;'>CHR -> []</p><p style='font-weight: bold; color: blue;'>SNP -> []</p><p style='font-weight: bold; color: blue;'>BP -> []</p><p style='font-weight: bold; color: blue;'>A1 -> []</p><p style='font-weight: bold; color: blue;'>A2 -> []</p><p style='font-weight: bold; color: blue;'>N -> []</p><p style='font-weight: bold; color: blue;'>P -> []</p><p style='font-weight: bold; color: blue;'>INFO -> []</p><p style='font-weight: bold; color: blue;'>MAF -> []</p><p style='font-weight: bold; color: blue;'>BETA -> []</p><p style='font-weight: bold; color: blue;'>SE -> []</p><p style='font-weight: bold; color: blue;'>OR -> []</p><p style='font-weight: bold; color: blue;'>Z -> []</p><p style='font-weight: bold; color: blue;'>D -> []</p><h2 class='text-muted'>GWAS columns</h2><?xml, version=1.0, encoding=utf-8, standalone=yes?><h2 class='text-muted'>Identified columns</h2><h2 class='text-muted'>Unidentified Columns headers</h2>version=1.0, standalone=yes?>, encoding=utf-8, <?xml<h2 class='text-muted'>PMID</h2>31910446<h2 class='text-muted'>DOI</h2>10.1371/journal.pone.0227357<div class='bibtex-container'><h2 class='text-muted'>BibTeX</h2><div style='font-family: monospace; white-space: pre-wrap;'> @article{Oh_2020, title={Genome-wide association study of metabolic syndrome in Korean populations}, volume={15}, ISSN={1932-6203}, url={http://dx.doi.org/10.1371/journal.pone.0227357}, DOI={10.1371/journal.pone.0227357}, number={1}, journal={PLOS ONE}, publisher={Public Library of Science (PLoS)}, author={Oh, Seung-Won and Lee, Jong-Eun and Shin, Eunsoon and Kwon, Hyuktae and Choe, Eun Kyung and Choi, Su-Yeon and Rhee, Hwanseok and Choi, Seung Ho}, editor={Maeda, Shiro}, year={2020}, month=jan, pages={e0227357} }
</div></div><br><br><br><br><br><br></div><div class='container mt-4 bg-light p-4 text-center'><h1 class='text-primary'>Phenotype: hypertension</h1><p class='text-muted'><strong>URL:</strong> http://ftp.ebi.ac.uk/pub/databases/gwas/summary_statistics/GCST90301001-GCST90302000/GCST90301363</p><table border="1" class="dataframe table table-bordered">
  <thead>
    <tr style="text-align: right;">
      <th>firstAuthor</th>
      <th>accessionId</th>
      <th>publicationDate</th>
      <th>journal</th>
      <th>title</th>
      <th>reportedTrait</th>
      <th>efoTraits</th>
      <th>bgTraits</th>
      <th>discoverySampleAncestry</th>
      <th>replicationSampleAncestry</th>
      <th>associationCount</th>
      <th>summaryStatistics</th>
      <th>ssApiFlag</th>
      <th>agreedToCc0</th>
      <th>genotypingTechnologies</th>
      <th>pubmedId</th>
      <th>initialSampleDescription</th>
      <th>replicateSampleDescription</th>
      <th>UKB_FIELD</th>
      <th>FuzzySimilarity_hypertension</th>
      <th>SearchPhenotype</th>
    </tr>
  </thead>
  <tbody>
    <tr>
      <td>Singh S</td>
      <td>GCST90301363</td>
      <td>12/16/2023</td>
      <td>Nat Commun</td>
      <td>Genome-wide association study meta-analysis of blood pressure traits and hypertension in sub-Saharan African populations: an AWI-Gen study.</td>
      <td>hypertension</td>
      <td>hypertension</td>
      <td>-</td>
      <td>10772 sub-saharan african</td>
      <td>NaN</td>
      <td>38</td>
      <td>http://ftp.ebi.ac.uk/pub/databases/gwas/summary_statistics/GCST90301001-GCST90302000/GCST90301363</td>
      <td>no</td>
      <td>yes</td>
      <td>Genome-wide genotyping array</td>
      <td>38104120</td>
      <td>10,772 Sub-Saharan African ancestry individuals</td>
      <td>NaN</td>
      <td>-</td>
      <td>100</td>
      <td>hypertension</td>
    </tr>
  </tbody>
</table><h2 class='text-muted'>Information file</h2><table border="1" class="dataframe table table-bordered">
  <thead>
    <tr style="text-align: right;">
      <th>Unnamed: 0</th>
      <th>Name</th>
      <th>Last modified</th>
      <th>Size</th>
      <th>Description</th>
      <th>URL</th>
    </tr>
  </thead>
  <tbody>
    <tr>
      <td>2</td>
      <td>harmonised/</td>
      <td>2024-02-07 22:30</td>
      <td>-</td>
      <td>NaN</td>
      <td>http://ftp.ebi.ac.uk/pub/databases/gwas/summary_statistics/GCST90301001-GCST90302000/GCST90301363/harmonised/</td>
    </tr>
    <tr>
      <td>3</td>
      <td>md5sum.txt</td>
      <td>2024-01-04 00:07</td>
      <td>116</td>
      <td>NaN</td>
      <td>http://ftp.ebi.ac.uk/pub/databases/gwas/summary_statistics/GCST90301001-GCST90302000/GCST90301363/md5sum.txt</td>
    </tr>
    <tr>
      <td>1</td>
      <td>GCST90301363.tsv.gz-meta.yaml</td>
      <td>2024-01-17 00:07</td>
      <td>1.2K</td>
      <td>NaN</td>
      <td>http://ftp.ebi.ac.uk/pub/databases/gwas/summary_statistics/GCST90301001-GCST90302000/GCST90301363/GCST90301363.tsv.gz-meta.yaml</td>
    </tr>
    <tr>
      <td>0</td>
      <td>GCST90301363.tsv.gz</td>
      <td>2024-01-04 00:07</td>
      <td>379M</td>
      <td>NaN</td>
      <td>http://ftp.ebi.ac.uk/pub/databases/gwas/summary_statistics/GCST90301001-GCST90302000/GCST90301363/GCST90301363.tsv.gz</td>
    </tr>
  </tbody>
</table><h2 class='text-muted'>Processed file</h2><p>Processing file: GCST90301363.tsv.gz!</p><h2 class='text-muted'>GWAS file head</h2><table border="1" class="dataframe table table-bordered">
  <thead>
    <tr style="text-align: right;">
      <th>chromosome</th>
      <th>base_pair_location</th>
      <th>effect_allele</th>
      <th>other_allele</th>
      <th>beta</th>
      <th>standard_error</th>
      <th>effect_allele_frequency</th>
      <th>p_value</th>
      <th>variant_id</th>
      <th>n</th>
    </tr>
  </thead>
  <tbody>
    <tr>
      <td>1</td>
      <td>723918</td>
      <td>G</td>
      <td>A</td>
      <td>-0.002731</td>
      <td>0.010825</td>
      <td>0.890295</td>
      <td>0.888008</td>
      <td>rs144434834</td>
      <td>11000</td>
    </tr>
    <tr>
      <td>1</td>
      <td>724103</td>
      <td>A</td>
      <td>G</td>
      <td>0.000479</td>
      <td>0.007758</td>
      <td>0.808843</td>
      <td>0.971799</td>
      <td>rs111533735</td>
      <td>11000</td>
    </tr>
  </tbody>
</table></div><div class='container mt-4 bg-light p-4 text-center'><h2 class='text-muted'>Original GWAS file headers</h2><table border="1" class="dataframe table table-bordered">
  <thead>
    <tr style="text-align: right;">
      <th>chromosome</th>
      <th>base_pair_location</th>
      <th>effect_allele</th>
      <th>other_allele</th>
      <th>beta</th>
      <th>standard_error</th>
      <th>effect_allele_frequency</th>
      <th>p_value</th>
      <th>variant_id</th>
      <th>n</th>
    </tr>
  </thead>
  <tbody>
    <tr>
      <td>1</td>
      <td>723918</td>
      <td>G</td>
      <td>A</td>
      <td>-0.002731</td>
      <td>0.010825</td>
      <td>0.890295</td>
      <td>0.888008</td>
      <td>rs144434834</td>
      <td>11000</td>
    </tr>
    <tr>
      <td>1</td>
      <td>724103</td>
      <td>A</td>
      <td>G</td>
      <td>0.000479</td>
      <td>0.007758</td>
      <td>0.808843</td>
      <td>0.971799</td>
      <td>rs111533735</td>
      <td>11000</td>
    </tr>
  </tbody>
</table><p style='font-weight: bold; color: blue;'>CHR -> ['chromosome']</p><p style='font-weight: bold; color: blue;'>SNP -> ['variant_id']</p><p style='font-weight: bold; color: blue;'>BP -> ['base_pair_location']</p><p style='font-weight: bold; color: blue;'>A1 -> ['effect_allele']</p><p style='font-weight: bold; color: blue;'>A2 -> ['other_allele']</p><p style='font-weight: bold; color: blue;'>N -> ['n']</p><p style='font-weight: bold; color: blue;'>P -> ['p_value']</p><p style='font-weight: bold; color: blue;'>INFO -> []</p><p style='font-weight: bold; color: blue;'>MAF -> ['effect_allele_frequency']</p><p style='font-weight: bold; color: blue;'>BETA -> ['beta']</p><p style='font-weight: bold; color: blue;'>SE -> ['standard_error']</p><p style='font-weight: bold; color: blue;'>OR -> []</p><p style='font-weight: bold; color: blue;'>Z -> []</p><p style='font-weight: bold; color: blue;'>D -> []</p><h2 class='text-muted'>GWAS columns</h2>chromosome, base_pair_location, effect_allele, other_allele, beta, standard_error, effect_allele_frequency, p_value, variant_id, n<h2 class='text-muted'>Identified columns</h2>chromosome, variant_id, base_pair_location, effect_allele, other_allele, n, p_value, effect_allele_frequency, beta, standard_error<h2 class='text-muted'>Unidentified Columns headers</h2><h2 class='text-muted'>PMID</h2>38104120<h2 class='text-muted'>DOI</h2>10.1038/s41467-023-44079-0<div class='bibtex-container'><h2 class='text-muted'>BibTeX</h2><div style='font-family: monospace; white-space: pre-wrap;'> @article{Singh_2023, title={Genome-wide association study meta-analysis of blood pressure traits and hypertension in sub-Saharan African populations: an AWI-Gen study}, volume={14}, ISSN={2041-1723}, url={http://dx.doi.org/10.1038/s41467-023-44079-0}, DOI={10.1038/s41467-023-44079-0}, number={1}, journal={Nature Communications}, publisher={Springer Science and Business Media LLC}, author={Singh, Surina and Choudhury, Ananyo and Hazelhurst, Scott and Crowther, Nigel J. and Boua, Palwendé R. and Sorgho, Hermann and Agongo, Godfred and Nonterah, Engelbert A. and Micklesfield, Lisa K. and Norris, Shane A. and Kisiangani, Isaac and Mohamed, Shukri and Gómez-Olivé, Francesc X. and Tollman, Stephen M. and Choma, Solomon and Brandenburg, J-T. and Ramsay, Michèle}, year={2023}, month=dec }
</div></div><br><br><br><br><br><br></div><div class='container mt-4 bg-light p-4 text-center'><h1 class='text-primary'>Phenotype: hypertension</h1><p class='text-muted'><strong>URL:</strong> http://ftp.ebi.ac.uk/pub/databases/gwas/summary_statistics/GCST006001-GCST007000/GCST006229</p><table border="1" class="dataframe table table-bordered">
  <thead>
    <tr style="text-align: right;">
      <th>firstAuthor</th>
      <th>accessionId</th>
      <th>publicationDate</th>
      <th>journal</th>
      <th>title</th>
      <th>reportedTrait</th>
      <th>efoTraits</th>
      <th>bgTraits</th>
      <th>discoverySampleAncestry</th>
      <th>replicationSampleAncestry</th>
      <th>associationCount</th>
      <th>summaryStatistics</th>
      <th>ssApiFlag</th>
      <th>agreedToCc0</th>
      <th>genotypingTechnologies</th>
      <th>pubmedId</th>
      <th>initialSampleDescription</th>
      <th>replicateSampleDescription</th>
      <th>UKB_FIELD</th>
      <th>FuzzySimilarity_hypertension</th>
      <th>SearchPhenotype</th>
    </tr>
  </thead>
  <tbody>
    <tr>
      <td>Liu C</td>
      <td>GCST006229</td>
      <td>10/1/2016</td>
      <td>Nat Genet</td>
      <td>Meta-analysis identifies common and rare variants influencing blood pressure and overlapping with metabolic trait loci.</td>
      <td>hypertension</td>
      <td>hypertension</td>
      <td>-</td>
      <td>21503 african american or afro-caribbean,4586 hispanic or latin american,120473 european</td>
      <td>26183 South Asian,154543 European</td>
      <td>5</td>
      <td>http://ftp.ebi.ac.uk/pub/databases/gwas/summary_statistics/GCST006001-GCST007000/GCST006229</td>
      <td>yes</td>
      <td>-</td>
      <td>Exome genotyping array</td>
      <td>27618448</td>
      <td>120,473 European ancestry individuals,21,503 African American individuals,4,586 Hispanic individuals</td>
      <td>154,543 European ancestry individuals,26,183 South Asian ancestry individuals</td>
      <td>-</td>
      <td>100</td>
      <td>hypertension</td>
    </tr>
  </tbody>
</table><h2 class='text-muted'>Information file</h2><table border="1" class="dataframe table table-bordered">
  <thead>
    <tr style="text-align: right;">
      <th>Unnamed: 0</th>
      <th>Name</th>
      <th>Last modified</th>
      <th>Size</th>
      <th>Description</th>
      <th>URL</th>
    </tr>
  </thead>
  <tbody>
    <tr>
      <td>4</td>
      <td>harmonised/</td>
      <td>2018-11-12 16:52</td>
      <td>-</td>
      <td>NaN</td>
      <td>http://ftp.ebi.ac.uk/pub/databases/gwas/summary_statistics/GCST006001-GCST007000/GCST006229/harmonised/</td>
    </tr>
    <tr>
      <td>2</td>
      <td>HTN_HIS_CHARGE_lookup_dbGap2.csv</td>
      <td>2016-11-04 14:19</td>
      <td>4.6M</td>
      <td>NaN</td>
      <td>http://ftp.ebi.ac.uk/pub/databases/gwas/summary_statistics/GCST006001-GCST007000/GCST006229/HTN_HIS_CHARGE_lookup_dbGap2.csv</td>
    </tr>
    <tr>
      <td>0</td>
      <td>HTN_AA_CHARGE_lookup_dbGap2.csv</td>
      <td>2016-11-04 14:19</td>
      <td>5.6M</td>
      <td>NaN</td>
      <td>http://ftp.ebi.ac.uk/pub/databases/gwas/summary_statistics/GCST006001-GCST007000/GCST006229/HTN_AA_CHARGE_lookup_dbGap2.csv</td>
    </tr>
    <tr>
      <td>1</td>
      <td>HTN_EA_CHARGE_lookup_dbGap2.csv</td>
      <td>2016-11-04 14:19</td>
      <td>6.1M</td>
      <td>NaN</td>
      <td>http://ftp.ebi.ac.uk/pub/databases/gwas/summary_statistics/GCST006001-GCST007000/GCST006229/HTN_EA_CHARGE_lookup_dbGap2.csv</td>
    </tr>
    <tr>
      <td>3</td>
      <td>HTN_all_ethnic_CHARGE_lookup_dbGap2.csv</td>
      <td>2016-11-04 14:19</td>
      <td>6.5M</td>
      <td>NaN</td>
      <td>http://ftp.ebi.ac.uk/pub/databases/gwas/summary_statistics/GCST006001-GCST007000/GCST006229/HTN_all_ethnic_CHARGE_lookup_dbGap2.csv</td>
    </tr>
  </tbody>
</table><h2 class='text-muted'>Processed file</h2><p>Processing file: HTN_all_ethnic_CHARGE_lookup_dbGap2.csv.1!</p><h2 class='text-muted'>GWAS file head</h2><table border="1" class="dataframe table table-bordered">
  <thead>
    <tr style="text-align: right;">
      <th>markername</th>
      <th>p.value</th>
    </tr>
  </thead>
  <tbody>
    <tr>
      <td>rs75333668</td>
      <td>0.479123</td>
    </tr>
    <tr>
      <td>rs200686669</td>
      <td>0.456393</td>
    </tr>
  </tbody>
</table></div><div class='container mt-4 bg-light p-4 text-center'><h2 class='text-muted'>Original GWAS file headers</h2><table border="1" class="dataframe table table-bordered">
  <thead>
    <tr style="text-align: right;">
      <th>markername</th>
      <th>p.value</th>
    </tr>
  </thead>
  <tbody>
    <tr>
      <td>rs75333668</td>
      <td>0.479123</td>
    </tr>
    <tr>
      <td>rs200686669</td>
      <td>0.456393</td>
    </tr>
  </tbody>
</table><p style='font-weight: bold; color: blue;'>CHR -> []</p><p style='font-weight: bold; color: blue;'>SNP -> ['markername']</p><p style='font-weight: bold; color: blue;'>BP -> ['markername']</p><p style='font-weight: bold; color: blue;'>A1 -> []</p><p style='font-weight: bold; color: blue;'>A2 -> []</p><p style='font-weight: bold; color: blue;'>N -> []</p><p style='font-weight: bold; color: blue;'>P -> ['p.value']</p><p style='font-weight: bold; color: blue;'>INFO -> []</p><p style='font-weight: bold; color: blue;'>MAF -> []</p><p style='font-weight: bold; color: blue;'>BETA -> []</p><p style='font-weight: bold; color: blue;'>SE -> []</p><p style='font-weight: bold; color: blue;'>OR -> []</p><p style='font-weight: bold; color: blue;'>Z -> []</p><p style='font-weight: bold; color: blue;'>D -> []</p><h2 class='text-muted'>GWAS columns</h2>markername, p.value<h2 class='text-muted'>Identified columns</h2>markername, markername, p.value<h2 class='text-muted'>Unidentified Columns headers</h2><h2 class='text-muted'>PMID</h2>27618448<h2 class='text-muted'>DOI</h2>10.1038/ng.3660<div class='bibtex-container'><h2 class='text-muted'>BibTeX</h2><div style='font-family: monospace; white-space: pre-wrap;'> @article{Liu_2016, title={Meta-analysis identifies common and rare variants influencing blood pressure and overlapping with metabolic trait loci}, volume={48}, ISSN={1546-1718}, url={http://dx.doi.org/10.1038/ng.3660}, DOI={10.1038/ng.3660}, number={10}, journal={Nature Genetics}, publisher={Springer Science and Business Media LLC}, author={Liu, Chunyu and Kraja, Aldi T and Smith, Jennifer A and Brody, Jennifer A and Franceschini, Nora and Bis, Joshua C and Rice, Kenneth and Morrison, Alanna C and Lu, Yingchang and Weiss, Stefan and Guo, Xiuqing and Palmas, Walter and Martin, Lisa W and Chen, Yii-Der Ida and Surendran, Praveen and Drenos, Fotios and Cook, James P and Auer, Paul L and Chu, Audrey Y and Giri, Ayush and Zhao, Wei and Jakobsdottir, Johanna and Lin, Li-An and Stafford, Jeanette M and Amin, Najaf and Mei, Hao and Yao, Jie and Voorman, Arend and Larson, Martin G and Grove, Megan L and Smith, Albert V and Hwang, Shih-Jen and Chen, Han and Huan, Tianxiao and Kosova, Gulum and Stitziel, Nathan O and Kathiresan, Sekar and Samani, Nilesh and Schunkert, Heribert and Deloukas, Panos and Li, Man and Fuchsberger, Christian and Pattaro, Cristian and Gorski, Mathias and Kooperberg, Charles and Papanicolaou, George J and Rossouw, Jacques E and Faul, Jessica D and Kardia, Sharon L R and Bouchard, Claude and Raffel, Leslie J and Uitterlinden, André G and Franco, Oscar H and Vasan, Ramachandran S and O’Donnell, Christopher J and Taylor, Kent D and Liu, Kiang and Bottinger, Erwin P and Gottesman, Omri and Daw, E Warwick and Giulianini, Franco and Ganesh, Santhi and Salfati, Elias and Harris, Tamara B and Launer, Lenore J and Dörr, Marcus and Felix, Stephan B and Rettig, Rainer and Völzke, Henry and Kim, Eric and Lee, Wen-Jane and Lee, I-Te and Sheu, Wayne H-H and Tsosie, Krystal S and Edwards, Digna R Velez and Liu, Yongmei and Correa, Adolfo and Weir, David R and Völker, Uwe and Ridker, Paul M and Boerwinkle, Eric and Gudnason, Vilmundur and Reiner, Alexander P and van Duijn, Cornelia M and Borecki, Ingrid B and Edwards, Todd L and Chakravarti, Aravinda and Rotter, Jerome I and Psaty, Bruce M and Loos, Ruth J F and Fornage, Myriam and Ehret, Georg B and Newton-Cheh, Christopher and Levy, Daniel and Chasman, Daniel I}, year={2016}, month=sep, pages={1162–1170} }
</div></div><br><br><br><br><br><br></div><div class='container mt-4 bg-light p-4 text-center'><h1 class='text-primary'>Phenotype: hypertension</h1><p class='text-muted'><strong>URL:</strong> http://ftp.ebi.ac.uk/pub/databases/gwas/summary_statistics/GCST008001-GCST009000/GCST008036</p><table border="1" class="dataframe table table-bordered">
  <thead>
    <tr style="text-align: right;">
      <th>firstAuthor</th>
      <th>accessionId</th>
      <th>publicationDate</th>
      <th>journal</th>
      <th>title</th>
      <th>reportedTrait</th>
      <th>efoTraits</th>
      <th>bgTraits</th>
      <th>discoverySampleAncestry</th>
      <th>replicationSampleAncestry</th>
      <th>associationCount</th>
      <th>summaryStatistics</th>
      <th>ssApiFlag</th>
      <th>agreedToCc0</th>
      <th>genotypingTechnologies</th>
      <th>pubmedId</th>
      <th>initialSampleDescription</th>
      <th>replicateSampleDescription</th>
      <th>UKB_FIELD</th>
      <th>FuzzySimilarity_hypertension</th>
      <th>SearchPhenotype</th>
    </tr>
  </thead>
  <tbody>
    <tr>
      <td>Wojcik GL</td>
      <td>GCST008036</td>
      <td>6/19/2019</td>
      <td>Nature</td>
      <td>Genetic analyses of diverse populations improves discovery for complex traits.</td>
      <td>hypertension</td>
      <td>hypertension</td>
      <td>-</td>
      <td>4663 east asian, asian unspecified,3705 oceanian,21936 hispanic or latin american,635 native american,1050 nr,17152 african american or afro-caribbean</td>
      <td>NaN</td>
      <td>2</td>
      <td>http://ftp.ebi.ac.uk/pub/databases/gwas/summary_statistics/GCST008001-GCST009000/GCST008036</td>
      <td>yes</td>
      <td>-</td>
      <td>Genome-wide genotyping array</td>
      <td>31217584</td>
      <td>11,863 African American cases,8,663 Hispanic/Latino cases,3,102 Asian ancestry cases,2,645 Native Hawaiian ancestry cases, 342 Native American ancestry cases, 508 cases,5,289 African American controls,13,273 Hispanic/Latino controls,1,561 Asian ancestry controls,1,060 Native Hawaiian ancestry controls, 293 Native American ancestry controls, 542 controls</td>
      <td>NaN</td>
      <td>-</td>
      <td>100</td>
      <td>hypertension</td>
    </tr>
  </tbody>
</table><h2 class='text-muted'>Information file</h2><table border="1" class="dataframe table table-bordered">
  <thead>
    <tr style="text-align: right;">
      <th>Unnamed: 0</th>
      <th>Name</th>
      <th>Last modified</th>
      <th>Size</th>
      <th>Description</th>
      <th>URL</th>
    </tr>
  </thead>
  <tbody>
    <tr>
      <td>2</td>
      <td>harmonised/</td>
      <td>2019-12-06 05:42</td>
      <td>-</td>
      <td>NaN</td>
      <td>http://ftp.ebi.ac.uk/pub/databases/gwas/summary_statistics/GCST008001-GCST009000/GCST008036/harmonised/</td>
    </tr>
    <tr>
      <td>0</td>
      <td>README</td>
      <td>2019-07-12 12:10</td>
      <td>3.0K</td>
      <td>NaN</td>
      <td>http://ftp.ebi.ac.uk/pub/databases/gwas/summary_statistics/GCST008001-GCST009000/GCST008036/README</td>
    </tr>
    <tr>
      <td>1</td>
      <td>WojcikG_PMID_htn.gz</td>
      <td>2019-07-12 12:10</td>
      <td>1.1G</td>
      <td>NaN</td>
      <td>http://ftp.ebi.ac.uk/pub/databases/gwas/summary_statistics/GCST008001-GCST009000/GCST008036/WojcikG_PMID_htn.gz</td>
    </tr>
  </tbody>
</table><h2 class='text-muted'>Processed file</h2><p>Processing file: WojcikG_PMID_htn.gz!</p><h2 class='text-muted'>GWAS file head</h2><table border="1" class="dataframe table table-bordered">
  <thead>
    <tr style="text-align: right;">
      <th>chr</th>
      <th>position_hg19</th>
      <th>snp</th>
      <th>other-allele</th>
      <th>effect-allele</th>
      <th>effect-allele-frequency</th>
      <th>sample-size</th>
      <th>effect-allele-frequency-cases</th>
      <th>sample-size-cases</th>
      <th>beta</th>
      <th>se</th>
      <th>p-val</th>
      <th>info-score</th>
      <th>rsid</th>
    </tr>
  </thead>
  <tbody>
    <tr>
      <td>1</td>
      <td>10539</td>
      <td>1_10539_C_A</td>
      <td>C</td>
      <td>A</td>
      <td>0.004378</td>
      <td>49141</td>
      <td>0.003604</td>
      <td>27123</td>
      <td>-0.104166</td>
      <td>0.168609</td>
      <td>0.536709</td>
      <td>0.460</td>
      <td>rs537182016</td>
    </tr>
    <tr>
      <td>1</td>
      <td>10616</td>
      <td>rs376342519_10616_CCGCCGTTGCAAAGGCGCGCCG_C</td>
      <td>CCGCCGTTGCAAAGGCGCGCCG</td>
      <td>C</td>
      <td>0.991634</td>
      <td>49141</td>
      <td>0.990179</td>
      <td>27123</td>
      <td>-0.173881</td>
      <td>0.109543</td>
      <td>0.112437</td>
      <td>0.604</td>
      <td>rs376342519</td>
    </tr>
  </tbody>
</table></div><div class='container mt-4 bg-light p-4 text-center'><h2 class='text-muted'>Original GWAS file headers</h2><table border="1" class="dataframe table table-bordered">
  <thead>
    <tr style="text-align: right;">
      <th>chr</th>
      <th>position_hg19</th>
      <th>snp</th>
      <th>other-allele</th>
      <th>effect-allele</th>
      <th>effect-allele-frequency</th>
      <th>sample-size</th>
      <th>effect-allele-frequency-cases</th>
      <th>sample-size-cases</th>
      <th>beta</th>
      <th>se</th>
      <th>p-val</th>
      <th>info-score</th>
      <th>rsid</th>
    </tr>
  </thead>
  <tbody>
    <tr>
      <td>1</td>
      <td>10539</td>
      <td>1_10539_C_A</td>
      <td>C</td>
      <td>A</td>
      <td>0.004378</td>
      <td>49141</td>
      <td>0.003604</td>
      <td>27123</td>
      <td>-0.104166</td>
      <td>0.168609</td>
      <td>0.536709</td>
      <td>0.460</td>
      <td>rs537182016</td>
    </tr>
    <tr>
      <td>1</td>
      <td>10616</td>
      <td>rs376342519_10616_CCGCCGTTGCAAAGGCGCGCCG_C</td>
      <td>CCGCCGTTGCAAAGGCGCGCCG</td>
      <td>C</td>
      <td>0.991634</td>
      <td>49141</td>
      <td>0.990179</td>
      <td>27123</td>
      <td>-0.173881</td>
      <td>0.109543</td>
      <td>0.112437</td>
      <td>0.604</td>
      <td>rs376342519</td>
    </tr>
  </tbody>
</table><p style='font-weight: bold; color: blue;'>CHR -> ['chr']</p><p style='font-weight: bold; color: blue;'>SNP -> ['snp', 'rsid']</p><p style='font-weight: bold; color: blue;'>BP -> ['position_hg19']</p><p style='font-weight: bold; color: blue;'>A1 -> ['effect-allele']</p><p style='font-weight: bold; color: blue;'>A2 -> ['other-allele']</p><p style='font-weight: bold; color: blue;'>N -> ['sample-size', 'sample-size-cases']</p><p style='font-weight: bold; color: blue;'>P -> ['p-val']</p><p style='font-weight: bold; color: blue;'>INFO -> ['info-score']</p><p style='font-weight: bold; color: blue;'>MAF -> ['effect-allele-frequency']</p><p style='font-weight: bold; color: blue;'>BETA -> ['beta']</p><p style='font-weight: bold; color: blue;'>SE -> ['se']</p><p style='font-weight: bold; color: blue;'>OR -> []</p><p style='font-weight: bold; color: blue;'>Z -> []</p><p style='font-weight: bold; color: blue;'>D -> []</p><h2 class='text-muted'>GWAS columns</h2>chr, position_hg19, snp, other-allele, effect-allele, effect-allele-frequency, sample-size, effect-allele-frequency-cases, sample-size-cases, beta, se, p-val, info-score, rsid<h2 class='text-muted'>Identified columns</h2>chr, snp, rsid, position_hg19, effect-allele, other-allele, sample-size, sample-size-cases, p-val, info-score, effect-allele-frequency, beta, se<h2 class='text-muted'>Unidentified Columns headers</h2>effect-allele-frequency-cases<h2 class='text-muted'>PMID</h2>31217584<h2 class='text-muted'>DOI</h2>10.1038/s41586-019-1310-4<div class='bibtex-container'><h2 class='text-muted'>BibTeX</h2><div style='font-family: monospace; white-space: pre-wrap;'> @article{Wojcik_2019, title={Genetic analyses of diverse populations improves discovery for complex traits}, volume={570}, ISSN={1476-4687}, url={http://dx.doi.org/10.1038/s41586-019-1310-4}, DOI={10.1038/s41586-019-1310-4}, number={7762}, journal={Nature}, publisher={Springer Science and Business Media LLC}, author={Wojcik, Genevieve L. and Graff, Mariaelisa and Nishimura, Katherine K. and Tao, Ran and Haessler, Jeffrey and Gignoux, Christopher R. and Highland, Heather M. and Patel, Yesha M. and Sorokin, Elena P. and Avery, Christy L. and Belbin, Gillian M. and Bien, Stephanie A. and Cheng, Iona and Cullina, Sinead and Hodonsky, Chani J. and Hu, Yao and Huckins, Laura M. and Jeff, Janina and Justice, Anne E. and Kocarnik, Jonathan M. and Lim, Unhee and Lin, Bridget M. and Lu, Yingchang and Nelson, Sarah C. and Park, Sung-Shim L. and Poisner, Hannah and Preuss, Michael H. and Richard, Melissa A. and Schurmann, Claudia and Setiawan, Veronica W. and Sockell, Alexandra and Vahi, Karan and Verbanck, Marie and Vishnu, Abhishek and Walker, Ryan W. and Young, Kristin L. and Zubair, Niha and Acuña-Alonso, Victor and Ambite, Jose Luis and Barnes, Kathleen C. and Boerwinkle, Eric and Bottinger, Erwin P. and Bustamante, Carlos D. and Caberto, Christian and Canizales-Quinteros, Samuel and Conomos, Matthew P. and Deelman, Ewa and Do, Ron and Doheny, Kimberly and Fernández-Rhodes, Lindsay and Fornage, Myriam and Hailu, Benyam and Heiss, Gerardo and Henn, Brenna M. and Hindorff, Lucia A. and Jackson, Rebecca D. and Laurie, Cecelia A. and Laurie, Cathy C. and Li, Yuqing and Lin, Dan-Yu and Moreno-Estrada, Andres and Nadkarni, Girish and Norman, Paul J. and Pooler, Loreall C. and Reiner, Alexander P. and Romm, Jane and Sabatti, Chiara and Sandoval, Karla and Sheng, Xin and Stahl, Eli A. and Stram, Daniel O. and Thornton, Timothy A. and Wassel, Christina L. and Wilkens, Lynne R. and Winkler, Cheryl A. and Yoneyama, Sachi and Buyske, Steven and Haiman, Christopher A. and Kooperberg, Charles and Le Marchand, Loic and Loos, Ruth J. F. and Matise, Tara C. and North, Kari E. and Peters, Ulrike and Kenny, Eimear E. and Carlson, Christopher S.}, year={2019}, month=jun, pages={514–518} }
</div></div><br><br><br><br><br><br></div>