<link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/bootstrap/4.0.0/css/bootstrap.min.css" integrity="sha384-Gn5384xqQ1aoWXA+058RXPxPg6fy4IWvTNh0E263XmFcJlSAwiGgFAW/dAiS6JXm" crossorigin="anonymous"><div class='container mt-4 bg-light p-4 text-center'><h1 class='text-primary'>Phenotype: irritable bowel syndrome</h1><p class='text-muted'><strong>URL:</strong> http://ftp.ebi.ac.uk/pub/databases/gwas/summary_statistics/GCST90077001-GCST90078000/GCST90077699</p><table border="1" class="dataframe table table-bordered">
  <thead>
    <tr style="text-align: right;">
      <th>firstAuthor</th>
      <th>accessionId</th>
      <th>publicationDate</th>
      <th>journal</th>
      <th>title</th>
      <th>reportedTrait</th>
      <th>efoTraits</th>
      <th>bgTraits</th>
      <th>discoverySampleAncestry</th>
      <th>replicationSampleAncestry</th>
      <th>associationCount</th>
      <th>summaryStatistics</th>
      <th>ssApiFlag</th>
      <th>agreedToCc0</th>
      <th>genotypingTechnologies</th>
      <th>pubmedId</th>
      <th>initialSampleDescription</th>
      <th>replicateSampleDescription</th>
      <th>UKB_FIELD</th>
      <th>FuzzySimilarity_irritable bowel syndrome</th>
      <th>SearchPhenotype</th>
    </tr>
  </thead>
  <tbody>
    <tr>
      <td>Backman JD</td>
      <td>GCST90077699</td>
      <td>10/18/2021</td>
      <td>Nature</td>
      <td>Exome sequencing and analysis of 454,787 UK Biobank participants.</td>
      <td>irritable bowel syndrome</td>
      <td>irritable bowel syndrome</td>
      <td>-</td>
      <td>331754 european</td>
      <td>NaN</td>
      <td>0</td>
      <td>http://ftp.ebi.ac.uk/pub/databases/gwas/summary_statistics/GCST90077001-GCST90078000/GCST90077699</td>
      <td>no</td>
      <td>yes</td>
      <td>Exome-wide sequencing,Genome-wide genotyping array</td>
      <td>34662886</td>
      <td>11,343 European ancestry cases,320,411 European ancestry controls</td>
      <td>NaN</td>
      <td>-</td>
      <td>100</td>
      <td>irritable bowel syndrome</td>
    </tr>
  </tbody>
</table><h2 class='text-muted'>Information file</h2><table border="1" class="dataframe table table-bordered">
  <thead>
    <tr style="text-align: right;">
      <th>Unnamed: 0</th>
      <th>Name</th>
      <th>Last modified</th>
      <th>Size</th>
      <th>Description</th>
      <th>URL</th>
    </tr>
  </thead>
  <tbody>
    <tr>
      <td>1</td>
      <td>GCST90077699_buildGRCh38.tsv.gz-meta.yaml</td>
      <td>2024-01-04 00:17</td>
      <td>896</td>
      <td>NaN</td>
      <td>http://ftp.ebi.ac.uk/pub/databases/gwas/summary_statistics/GCST90077001-GCST90078000/GCST90077699/GCST90077699_buildGRCh38.tsv.gz-meta.yaml</td>
    </tr>
    <tr>
      <td>2</td>
      <td>README.txt</td>
      <td>2021-11-18 01:02</td>
      <td>698</td>
      <td>NaN</td>
      <td>http://ftp.ebi.ac.uk/pub/databases/gwas/summary_statistics/GCST90077001-GCST90078000/GCST90077699/README.txt</td>
    </tr>
    <tr>
      <td>3</td>
      <td>harmonised/</td>
      <td>2024-02-02 15:16</td>
      <td>-</td>
      <td>NaN</td>
      <td>http://ftp.ebi.ac.uk/pub/databases/gwas/summary_statistics/GCST90077001-GCST90078000/GCST90077699/harmonised/</td>
    </tr>
    <tr>
      <td>4</td>
      <td>md5sum.txt</td>
      <td>2021-11-18 01:02</td>
      <td>109</td>
      <td>NaN</td>
      <td>http://ftp.ebi.ac.uk/pub/databases/gwas/summary_statistics/GCST90077001-GCST90078000/GCST90077699/md5sum.txt</td>
    </tr>
    <tr>
      <td>0</td>
      <td>GCST90077699_buildGRCh38.tsv.gz</td>
      <td>2021-11-18 01:02</td>
      <td>15M</td>
      <td>NaN</td>
      <td>http://ftp.ebi.ac.uk/pub/databases/gwas/summary_statistics/GCST90077001-GCST90078000/GCST90077699/GCST90077699_buildGRCh38.tsv.gz</td>
    </tr>
  </tbody>
</table><h2 class='text-muted'>Processed file</h2><p>Processing file: GCST90077699_buildGRCh38.tsv.gz!</p><h2 class='text-muted'>GWAS file head</h2><table border="1" class="dataframe table table-bordered">
  <thead>
    <tr style="text-align: right;">
      <th>name</th>
      <th>chromosome</th>
      <th>base_pair_location</th>
      <th>other_allele</th>
      <th>effect_allele</th>
      <th>trait</th>
      <th>cohort</th>
      <th>model</th>
      <th>odds_ratio</th>
      <th>ci_lower</th>
      <th>ci_upper</th>
      <th>p_value</th>
      <th>effect_allele_frequency</th>
      <th>standard_error</th>
    </tr>
  </thead>
  <tbody>
    <tr>
      <td>1_930165_G_A</td>
      <td>1</td>
      <td>930165</td>
      <td>G</td>
      <td>A</td>
      <td>20002_Illness_code_SR_1154_irritable_bowel_syndrome_AnyInst__BIN</td>
      <td>UKB_Freeze_450_EUR</td>
      <td>ADD-WGR-FIRTH</td>
      <td>0.910421</td>
      <td>0.132693</td>
      <td>6.24649</td>
      <td>0.92391</td>
      <td>0.000042</td>
      <td>0.982604</td>
    </tr>
    <tr>
      <td>1_930204_G_A</td>
      <td>1</td>
      <td>930204</td>
      <td>G</td>
      <td>A</td>
      <td>20002_Illness_code_SR_1154_irritable_bowel_syndrome_AnyInst__BIN</td>
      <td>UKB_Freeze_450_EUR</td>
      <td>ADD-WGR-FIRTH</td>
      <td>1.342730</td>
      <td>0.136936</td>
      <td>13.16620</td>
      <td>0.80026</td>
      <td>0.000041</td>
      <td>1.164790</td>
    </tr>
  </tbody>
</table></div><div class='container mt-4 bg-light p-4 text-center'><h2 class='text-muted'>Original GWAS file headers</h2><table border="1" class="dataframe table table-bordered">
  <thead>
    <tr style="text-align: right;">
      <th>name</th>
      <th>chromosome</th>
      <th>base_pair_location</th>
      <th>other_allele</th>
      <th>effect_allele</th>
      <th>trait</th>
      <th>cohort</th>
      <th>model</th>
      <th>odds_ratio</th>
      <th>ci_lower</th>
      <th>ci_upper</th>
      <th>p_value</th>
      <th>effect_allele_frequency</th>
      <th>standard_error</th>
    </tr>
  </thead>
  <tbody>
    <tr>
      <td>1_930165_G_A</td>
      <td>1</td>
      <td>930165</td>
      <td>G</td>
      <td>A</td>
      <td>20002_Illness_code_SR_1154_irritable_bowel_syndrome_AnyInst__BIN</td>
      <td>UKB_Freeze_450_EUR</td>
      <td>ADD-WGR-FIRTH</td>
      <td>0.910421</td>
      <td>0.132693</td>
      <td>6.24649</td>
      <td>0.92391</td>
      <td>0.000042</td>
      <td>0.982604</td>
    </tr>
    <tr>
      <td>1_930204_G_A</td>
      <td>1</td>
      <td>930204</td>
      <td>G</td>
      <td>A</td>
      <td>20002_Illness_code_SR_1154_irritable_bowel_syndrome_AnyInst__BIN</td>
      <td>UKB_Freeze_450_EUR</td>
      <td>ADD-WGR-FIRTH</td>
      <td>1.342730</td>
      <td>0.136936</td>
      <td>13.16620</td>
      <td>0.80026</td>
      <td>0.000041</td>
      <td>1.164790</td>
    </tr>
  </tbody>
</table><p style='font-weight: bold; color: blue;'>CHR -> ['chromosome']</p><p style='font-weight: bold; color: blue;'>SNP -> ['name']</p><p style='font-weight: bold; color: blue;'>BP -> ['name', 'base_pair_location']</p><p style='font-weight: bold; color: blue;'>A1 -> ['effect_allele']</p><p style='font-weight: bold; color: blue;'>A2 -> ['other_allele']</p><p style='font-weight: bold; color: blue;'>N -> []</p><p style='font-weight: bold; color: blue;'>P -> ['p_value']</p><p style='font-weight: bold; color: blue;'>INFO -> []</p><p style='font-weight: bold; color: blue;'>MAF -> ['effect_allele_frequency']</p><p style='font-weight: bold; color: blue;'>BETA -> []</p><p style='font-weight: bold; color: blue;'>SE -> ['standard_error']</p><p style='font-weight: bold; color: blue;'>OR -> ['odds_ratio']</p><p style='font-weight: bold; color: blue;'>Z -> []</p><p style='font-weight: bold; color: blue;'>D -> []</p><h2 class='text-muted'>GWAS columns</h2>name, chromosome, base_pair_location, other_allele, effect_allele, trait, cohort, model, odds_ratio, ci_lower, ci_upper, p_value, effect_allele_frequency, standard_error<h2 class='text-muted'>Identified columns</h2>chromosome, name, name, base_pair_location, effect_allele, other_allele, p_value, effect_allele_frequency, standard_error, odds_ratio<h2 class='text-muted'>Unidentified Columns headers</h2>ci_upper, cohort, trait, model, ci_lower<h2 class='text-muted'>PMID</h2>34662886<h2 class='text-muted'>DOI</h2>10.1038/s41586-021-04103-z<div class='bibtex-container'><h2 class='text-muted'>BibTeX</h2><div style='font-family: monospace; white-space: pre-wrap;'> @article{Backman_2021, title={Exome sequencing and analysis of 454,787 UK Biobank participants}, volume={599}, ISSN={1476-4687}, url={http://dx.doi.org/10.1038/s41586-021-04103-z}, DOI={10.1038/s41586-021-04103-z}, number={7886}, journal={Nature}, publisher={Springer Science and Business Media LLC}, author={Backman, Joshua D. and Li, Alexander H. and Marcketta, Anthony and Sun, Dylan and Mbatchou, Joelle and Kessler, Michael D. and Benner, Christian and Liu, Daren and Locke, Adam E. and Balasubramanian, Suganthi and Yadav, Ashish and Banerjee, Nilanjana and Gillies, Christopher E. and Damask, Amy and Liu, Simon and Bai, Xiaodong and Hawes, Alicia and Maxwell, Evan and Gurski, Lauren and Watanabe, Kyoko and Kosmicki, Jack A. and Rajagopal, Veera and Mighty, Jason and Jones, Marcus and Mitnaul, Lyndon and Stahl, Eli and Coppola, Giovanni and Jorgenson, Eric and Habegger, Lukas and Salerno, William J. and Shuldiner, Alan R. and Lotta, Luca A. and Overton, John D. and Cantor, Michael N. and Reid, Jeffrey G. and Yancopoulos, George and Kang, Hyun M. and Marchini, Jonathan and Baras, Aris and Abecasis, Gonçalo R. and Ferreira, Manuel A. R.}, year={2021}, month=oct, pages={628–634} }
</div></div><br><br><br><br><br><br></div><div class='container mt-4 bg-light p-4 text-center'><h1 class='text-primary'>Phenotype: irritable bowel syndrome</h1><p class='text-muted'><strong>URL:</strong> http://ftp.ebi.ac.uk/pub/databases/gwas/summary_statistics/GCST90016001-GCST90017000/GCST90016564</p><table border="1" class="dataframe table table-bordered">
  <thead>
    <tr style="text-align: right;">
      <th>firstAuthor</th>
      <th>accessionId</th>
      <th>publicationDate</th>
      <th>journal</th>
      <th>title</th>
      <th>reportedTrait</th>
      <th>efoTraits</th>
      <th>bgTraits</th>
      <th>discoverySampleAncestry</th>
      <th>replicationSampleAncestry</th>
      <th>associationCount</th>
      <th>summaryStatistics</th>
      <th>ssApiFlag</th>
      <th>agreedToCc0</th>
      <th>genotypingTechnologies</th>
      <th>pubmedId</th>
      <th>initialSampleDescription</th>
      <th>replicateSampleDescription</th>
      <th>UKB_FIELD</th>
      <th>FuzzySimilarity_irritable bowel syndrome</th>
      <th>SearchPhenotype</th>
    </tr>
  </thead>
  <tbody>
    <tr>
      <td>Eijsbouts C</td>
      <td>GCST90016564</td>
      <td>11/5/2021</td>
      <td>Nat Genet</td>
      <td>Genome-wide analysis of 53,400 people with irritable bowel syndrome highlights shared genetic pathways with mood and anxiety disorders.</td>
      <td>irritable bowel syndrome</td>
      <td>irritable bowel syndrome</td>
      <td>-</td>
      <td>486601 european</td>
      <td>1589307 European</td>
      <td>6</td>
      <td>http://ftp.ebi.ac.uk/pub/databases/gwas/summary_statistics/GCST90016001-GCST90017000/GCST90016564</td>
      <td>no</td>
      <td>yes</td>
      <td>Genome-wide genotyping array</td>
      <td>34741163</td>
      <td>53,400 European ancestry cases,433,201 European ancestry controls</td>
      <td>205,252 European ancestry cases,1,384,055 European ancestry controls</td>
      <td>-</td>
      <td>100</td>
      <td>irritable bowel syndrome</td>
    </tr>
  </tbody>
</table><h2 class='text-muted'>Information file</h2><table border="1" class="dataframe table table-bordered">
  <thead>
    <tr style="text-align: right;">
      <th>Unnamed: 0</th>
      <th>Name</th>
      <th>Last modified</th>
      <th>Size</th>
      <th>Description</th>
      <th>URL</th>
    </tr>
  </thead>
  <tbody>
    <tr>
      <td>2</td>
      <td>README.txt</td>
      <td>2021-07-03 00:01</td>
      <td>926</td>
      <td>NaN</td>
      <td>http://ftp.ebi.ac.uk/pub/databases/gwas/summary_statistics/GCST90016001-GCST90017000/GCST90016564/README.txt</td>
    </tr>
    <tr>
      <td>3</td>
      <td>harmonised/</td>
      <td>2024-02-02 15:18</td>
      <td>-</td>
      <td>NaN</td>
      <td>http://ftp.ebi.ac.uk/pub/databases/gwas/summary_statistics/GCST90016001-GCST90017000/GCST90016564/harmonised/</td>
    </tr>
    <tr>
      <td>4</td>
      <td>md5sum.txt</td>
      <td>2021-07-03 00:01</td>
      <td>106</td>
      <td>NaN</td>
      <td>http://ftp.ebi.ac.uk/pub/databases/gwas/summary_statistics/GCST90016001-GCST90017000/GCST90016564/md5sum.txt</td>
    </tr>
    <tr>
      <td>1</td>
      <td>GCST90016564_buildGRCh37.tsv-meta.yaml</td>
      <td>2024-01-04 00:06</td>
      <td>1.6K</td>
      <td>NaN</td>
      <td>http://ftp.ebi.ac.uk/pub/databases/gwas/summary_statistics/GCST90016001-GCST90017000/GCST90016564/GCST90016564_buildGRCh37.tsv-meta.yaml</td>
    </tr>
    <tr>
      <td>0</td>
      <td>GCST90016564_buildGRCh37.tsv</td>
      <td>2021-07-03 00:01</td>
      <td>638M</td>
      <td>NaN</td>
      <td>http://ftp.ebi.ac.uk/pub/databases/gwas/summary_statistics/GCST90016001-GCST90017000/GCST90016564/GCST90016564_buildGRCh37.tsv</td>
    </tr>
  </tbody>
</table><h2 class='text-muted'>Processed file</h2><p>Processing file: GCST90016564_buildGRCh37.tsv!</p><h2 class='text-muted'>GWAS file head</h2><table border="1" class="dataframe table table-bordered">
  <thead>
    <tr style="text-align: right;">
      <th>variant_id</th>
      <th>p_value</th>
      <th>chromosome</th>
      <th>base_pair_location</th>
      <th>effect_allele</th>
      <th>other_allele</th>
      <th>effect_allele_frequency</th>
      <th>beta</th>
      <th>standard_error</th>
      <th>n_case</th>
      <th>n_control</th>
    </tr>
  </thead>
  <tbody>
    <tr>
      <td>rs367896724</td>
      <td>0.1537</td>
      <td>1</td>
      <td>10177</td>
      <td>A</td>
      <td>AC</td>
      <td>0.6024</td>
      <td>0.0171</td>
      <td>0.0120</td>
      <td>40548</td>
      <td>293220</td>
    </tr>
    <tr>
      <td>rs201106462</td>
      <td>0.5196</td>
      <td>1</td>
      <td>10352</td>
      <td>T</td>
      <td>TA</td>
      <td>0.6076</td>
      <td>0.0079</td>
      <td>0.0123</td>
      <td>40548</td>
      <td>293220</td>
    </tr>
  </tbody>
</table></div><div class='container mt-4 bg-light p-4 text-center'><h2 class='text-muted'>Original GWAS file headers</h2><table border="1" class="dataframe table table-bordered">
  <thead>
    <tr style="text-align: right;">
      <th>variant_id</th>
      <th>p_value</th>
      <th>chromosome</th>
      <th>base_pair_location</th>
      <th>effect_allele</th>
      <th>other_allele</th>
      <th>effect_allele_frequency</th>
      <th>beta</th>
      <th>standard_error</th>
      <th>n_case</th>
      <th>n_control</th>
    </tr>
  </thead>
  <tbody>
    <tr>
      <td>rs367896724</td>
      <td>0.1537</td>
      <td>1</td>
      <td>10177</td>
      <td>A</td>
      <td>AC</td>
      <td>0.6024</td>
      <td>0.0171</td>
      <td>0.0120</td>
      <td>40548</td>
      <td>293220</td>
    </tr>
    <tr>
      <td>rs201106462</td>
      <td>0.5196</td>
      <td>1</td>
      <td>10352</td>
      <td>T</td>
      <td>TA</td>
      <td>0.6076</td>
      <td>0.0079</td>
      <td>0.0123</td>
      <td>40548</td>
      <td>293220</td>
    </tr>
  </tbody>
</table><p style='font-weight: bold; color: blue;'>CHR -> ['chromosome']</p><p style='font-weight: bold; color: blue;'>SNP -> ['variant_id']</p><p style='font-weight: bold; color: blue;'>BP -> ['base_pair_location']</p><p style='font-weight: bold; color: blue;'>A1 -> ['effect_allele']</p><p style='font-weight: bold; color: blue;'>A2 -> ['other_allele']</p><p style='font-weight: bold; color: blue;'>N -> ['n_case', 'n_control']</p><p style='font-weight: bold; color: blue;'>P -> ['p_value']</p><p style='font-weight: bold; color: blue;'>INFO -> []</p><p style='font-weight: bold; color: blue;'>MAF -> ['effect_allele_frequency']</p><p style='font-weight: bold; color: blue;'>BETA -> ['beta']</p><p style='font-weight: bold; color: blue;'>SE -> ['standard_error']</p><p style='font-weight: bold; color: blue;'>OR -> []</p><p style='font-weight: bold; color: blue;'>Z -> []</p><p style='font-weight: bold; color: blue;'>D -> []</p><h2 class='text-muted'>GWAS columns</h2>variant_id, p_value, chromosome, base_pair_location, effect_allele, other_allele, effect_allele_frequency, beta, standard_error, n_case, n_control<h2 class='text-muted'>Identified columns</h2>chromosome, variant_id, base_pair_location, effect_allele, other_allele, n_case, n_control, p_value, effect_allele_frequency, beta, standard_error<h2 class='text-muted'>Unidentified Columns headers</h2><h2 class='text-muted'>PMID</h2>34741163<h2 class='text-muted'>DOI</h2>10.1038/s41588-021-00950-8<div class='bibtex-container'><h2 class='text-muted'>BibTeX</h2><div style='font-family: monospace; white-space: pre-wrap;'> @article{Eijsbouts_2021, title={Genome-wide analysis of 53,400 people with irritable bowel syndrome highlights shared genetic pathways with mood and anxiety disorders}, volume={53}, ISSN={1546-1718}, url={http://dx.doi.org/10.1038/s41588-021-00950-8}, DOI={10.1038/s41588-021-00950-8}, number={11}, journal={Nature Genetics}, publisher={Springer Science and Business Media LLC}, author={Eijsbouts, Chris and Zheng, Tenghao and Kennedy, Nicholas A. and Bonfiglio, Ferdinando and Anderson, Carl A. and Moutsianas, Loukas and Holliday, Joanne and Shi, Jingchunzi and Shringarpure, Suyash and Agee, Michelle and Aslibekyan, Stella and Auton, Adam and Bell, Robert K. and Bryc, Katarzyna and Clark, Sarah K. and Elson, Sarah L. and Fletez-Brant, Kipper and Fontanillas, Pierre and Furlotte, Nicholas A. and Gandhi, Pooja M. and Heilbron, Karl and Hicks, Barry and Hinds, David A. and Huber, Karen E. and Jewett, Ethan M. and Jiang, Yunxuan and Kleinman, Aaron and Lin, Keng-Han and Litterman, Nadia K. and Luff, Marie K. and McCreight, Jey C. and McIntyre, Matthew H. and McManus, Kimberly F. and Mountain, Joanna L. and Mozaffari, Sahar V. and Nandakumar, Priyanka and Noblin, Elizabeth S. and Northover, Carrie A. M. and O’Connell, Jared and Petrakovitz, Aaron A. and Pitts, Steven J. and Poznik, G. David and Sathirapongsasuti, J. Fah and Shastri, Anjali J. and Shelton, Janie F. and Tian, Chao and Tung, Joyce Y. and Tunney, Robert J. and Vacic, Vladimir and Wang, Xin and Zare, Amir S. and Voda, Alexandru-Ioan and Kashyap, Purna and Chang, Lin and Mayer, Emeran and Heitkemper, Margaret and Sayuk, Gregory S. and Ringel-Kulka, Tamar and Ringel, Yehuda and Chey, William D. and Eswaran, Shanti and Merchant, Juanita L. and Shulman, Robert J. and Bujanda, Luis and Garcia-Etxebarria, Koldo and Dlugosz, Aldona and Lindberg, Greger and Schmidt, Peter T. and Karling, Pontus and Ohlsson, Bodil and Walter, Susanna and Faresjö, Åshild O. and Simren, Magnus and Halfvarson, Jonas and Portincasa, Piero and Barbara, Giovanni and Usai-Satta, Paolo and Neri, Matteo and Nardone, Gerardo and Cuomo, Rosario and Galeazzi, Francesca and Bellini, Massimo and Latiano, Anna and Houghton, Lesley and Jonkers, Daisy and Kurilshikov, Alexander and Weersma, Rinse K. and Netea, Mihai and Tesarz, Jonas and Gauss, Annika and Goebel-Stengel, Miriam and Andresen, Viola and Frieling, Thomas and Pehl, Christian and Schaefert, Rainer and Niesler, Beate and Lieb, Wolfgang and Hanevik, Kurt and Langeland, Nina and Wensaas, Knut-Arne and Litleskare, Sverre and Gabrielsen, Maiken E. and Thomas, Laurent and Thijs, Vincent and Lemmens, Robin and Van Oudenhove, Lukas and Wouters, Mira and Farrugia, Gianrico and Franke, Andre and Hübenthal, Matthias and Abecasis, Gonçalo and Zawistowski, Matthew and Skogholt, Anne Heidi and Ness-Jensen, Eivind and Hveem, Kristian and Esko, Tõnu and Teder-Laving, Maris and Zhernakova, Alexandra and Camilleri, Michael and Boeckxstaens, Guy and Whorwell, Peter J. and Spiller, Robin and McVean, Gil and D’Amato, Mauro and Jostins, Luke and Parkes, Miles}, year={2021}, month=nov, pages={1543–1552} }
</div></div><br><br><br><br><br><br></div><div class='container mt-4 bg-light p-4 text-center'><h1 class='text-primary'>Phenotype: irritable bowel syndrome</h1><p class='text-muted'><strong>URL:</strong> http://ftp.ebi.ac.uk/pub/databases/gwas/summary_statistics/GCST006001-GCST007000/GCST006304</p><table border="1" class="dataframe table table-bordered">
  <thead>
    <tr style="text-align: right;">
      <th>firstAuthor</th>
      <th>accessionId</th>
      <th>publicationDate</th>
      <th>journal</th>
      <th>title</th>
      <th>reportedTrait</th>
      <th>efoTraits</th>
      <th>bgTraits</th>
      <th>discoverySampleAncestry</th>
      <th>replicationSampleAncestry</th>
      <th>associationCount</th>
      <th>summaryStatistics</th>
      <th>ssApiFlag</th>
      <th>agreedToCc0</th>
      <th>genotypingTechnologies</th>
      <th>pubmedId</th>
      <th>initialSampleDescription</th>
      <th>replicateSampleDescription</th>
      <th>UKB_FIELD</th>
      <th>FuzzySimilarity_irritable bowel syndrome</th>
      <th>SearchPhenotype</th>
    </tr>
  </thead>
  <tbody>
    <tr>
      <td>Bonfiglio F</td>
      <td>GCST006304</td>
      <td>4/4/2018</td>
      <td>Gastroenterology</td>
      <td>Female-specific Association Between Variants on Chromosome 9 and Self-reported Diagnosis of Irritable Bowel Syndrome.</td>
      <td>irritable bowel syndrome</td>
      <td>irritable bowel syndrome</td>
      <td>-</td>
      <td>346075 european</td>
      <td>NaN</td>
      <td>15</td>
      <td>http://ftp.ebi.ac.uk/pub/databases/gwas/summary_statistics/GCST006001-GCST007000/GCST006304</td>
      <td>no</td>
      <td>-</td>
      <td>Genome-wide genotyping array</td>
      <td>29626450</td>
      <td>7,130 British ancestry women cases,2,446 British ancestry men cases,178,076 British ancestry women controls,158,423 British ancestry men controls</td>
      <td>NaN</td>
      <td>-</td>
      <td>100</td>
      <td>irritable bowel syndrome</td>
    </tr>
  </tbody>
</table><h2 class='text-muted'>Information file</h2><table border="1" class="dataframe table table-bordered">
  <thead>
    <tr style="text-align: right;">
      <th>Unnamed: 0</th>
      <th>Name</th>
      <th>Last modified</th>
      <th>Size</th>
      <th>Description</th>
      <th>URL</th>
    </tr>
  </thead>
  <tbody>
    <tr>
      <td>0</td>
      <td>BonfiglioF_29626450.FieldDescription.txt</td>
      <td>2020-01-17 00:02</td>
      <td>264</td>
      <td>NaN</td>
      <td>http://ftp.ebi.ac.uk/pub/databases/gwas/summary_statistics/GCST006001-GCST007000/GCST006304/BonfiglioF_29626450.FieldDescription.txt</td>
    </tr>
    <tr>
      <td>2</td>
      <td>md5sum.txt</td>
      <td>2020-01-17 00:02</td>
      <td>134</td>
      <td>NaN</td>
      <td>http://ftp.ebi.ac.uk/pub/databases/gwas/summary_statistics/GCST006001-GCST007000/GCST006304/md5sum.txt</td>
    </tr>
    <tr>
      <td>1</td>
      <td>BonfiglioF_29626450.txt.gz</td>
      <td>2020-01-17 00:02</td>
      <td>206M</td>
      <td>NaN</td>
      <td>http://ftp.ebi.ac.uk/pub/databases/gwas/summary_statistics/GCST006001-GCST007000/GCST006304/BonfiglioF_29626450.txt.gz</td>
    </tr>
  </tbody>
</table><h2 class='text-muted'>Processed file</h2><p>Processing file: BonfiglioF_29626450.txt.gz!</p><h2 class='text-muted'>GWAS file head</h2><table border="1" class="dataframe table table-bordered">
  <thead>
    <tr style="text-align: right;">
      <th>snp</th>
      <th>chr</th>
      <th>pos</th>
      <th>a1</th>
      <th>a2</th>
      <th>eaf</th>
      <th>or</th>
      <th>se</th>
      <th>p</th>
      <th>n</th>
      <th>info</th>
    </tr>
  </thead>
  <tbody>
    <tr>
      <td>rs10</td>
      <td>7</td>
      <td>92383888</td>
      <td>A</td>
      <td>C</td>
      <td>0.052794</td>
      <td>1.012710</td>
      <td>0.033135</td>
      <td>0.703092</td>
      <td>336785</td>
      <td>0.939077</td>
    </tr>
    <tr>
      <td>rs1000000</td>
      <td>12</td>
      <td>126890980</td>
      <td>G</td>
      <td>A</td>
      <td>0.772911</td>
      <td>0.969541</td>
      <td>0.017405</td>
      <td>0.075533</td>
      <td>346020</td>
      <td>0.999152</td>
    </tr>
  </tbody>
</table></div><div class='container mt-4 bg-light p-4 text-center'><h2 class='text-muted'>Original GWAS file headers</h2><table border="1" class="dataframe table table-bordered">
  <thead>
    <tr style="text-align: right;">
      <th>snp</th>
      <th>chr</th>
      <th>pos</th>
      <th>a1</th>
      <th>a2</th>
      <th>eaf</th>
      <th>or</th>
      <th>se</th>
      <th>p</th>
      <th>n</th>
      <th>info</th>
    </tr>
  </thead>
  <tbody>
    <tr>
      <td>rs10</td>
      <td>7</td>
      <td>92383888</td>
      <td>A</td>
      <td>C</td>
      <td>0.052794</td>
      <td>1.012710</td>
      <td>0.033135</td>
      <td>0.703092</td>
      <td>336785</td>
      <td>0.939077</td>
    </tr>
    <tr>
      <td>rs1000000</td>
      <td>12</td>
      <td>126890980</td>
      <td>G</td>
      <td>A</td>
      <td>0.772911</td>
      <td>0.969541</td>
      <td>0.017405</td>
      <td>0.075533</td>
      <td>346020</td>
      <td>0.999152</td>
    </tr>
  </tbody>
</table><p style='font-weight: bold; color: blue;'>CHR -> ['chr']</p><p style='font-weight: bold; color: blue;'>SNP -> ['snp']</p><p style='font-weight: bold; color: blue;'>BP -> ['pos']</p><p style='font-weight: bold; color: blue;'>A1 -> ['a1']</p><p style='font-weight: bold; color: blue;'>A2 -> ['a1', 'a2']</p><p style='font-weight: bold; color: blue;'>N -> ['n']</p><p style='font-weight: bold; color: blue;'>P -> ['p']</p><p style='font-weight: bold; color: blue;'>INFO -> ['info']</p><p style='font-weight: bold; color: blue;'>MAF -> ['eaf']</p><p style='font-weight: bold; color: blue;'>BETA -> []</p><p style='font-weight: bold; color: blue;'>SE -> ['se']</p><p style='font-weight: bold; color: blue;'>OR -> ['or']</p><p style='font-weight: bold; color: blue;'>Z -> []</p><p style='font-weight: bold; color: blue;'>D -> []</p><h2 class='text-muted'>GWAS columns</h2>snp, chr, pos, a1, a2, eaf, or, se, p, n, info<h2 class='text-muted'>Identified columns</h2>chr, snp, pos, a1, a1, a2, n, p, info, eaf, se, or<h2 class='text-muted'>Unidentified Columns headers</h2><h2 class='text-muted'>PMID</h2>29626450<h2 class='text-muted'>DOI</h2>10.1053/j.gastro.2018.03.064<div class='bibtex-container'><h2 class='text-muted'>BibTeX</h2><div style='font-family: monospace; white-space: pre-wrap;'> @article{Bonfiglio_2018, title={Female-Specific Association Between Variants on Chromosome 9 and Self-Reported Diagnosis of Irritable Bowel Syndrome}, volume={155}, ISSN={0016-5085}, url={http://dx.doi.org/10.1053/j.gastro.2018.03.064}, DOI={10.1053/j.gastro.2018.03.064}, number={1}, journal={Gastroenterology}, publisher={Elsevier BV}, author={Bonfiglio, Ferdinando and Zheng, Tenghao and Garcia-Etxebarria, Koldo and Hadizadeh, Fatemeh and Bujanda, Luis and Bresso, Francesca and Agreus, Lars and Andreasson, Anna and Dlugosz, Aldona and Lindberg, Greger and Schmidt, Peter T. and Karling, Pontus and Ohlsson, Bodil and Simren, Magnus and Walter, Susanna and Nardone, Gerardo and Cuomo, Rosario and Usai-Satta, Paolo and Galeazzi, Francesca and Neri, Matteo and Portincasa, Piero and Bellini, Massimo and Barbara, Giovanni and Latiano, Anna and Hübenthal, Matthias and Thijs, Vincent and Netea, Mihai G. and Jonkers, Daisy and Chang, Lin and Mayer, Emeran A. and Wouters, Mira M. and Boeckxstaens, Guy and Camilleri, Michael and Franke, Andre and Zhernakova, Alexandra and D’Amato, Mauro}, year={2018}, month=jul, pages={168–179} }
</div></div><br><br><br><br><br><br></div><div class='container mt-4 bg-light p-4 text-center'><h1 class='text-primary'>Phenotype: irritable bowel syndrome</h1><p class='text-muted'><strong>URL:</strong> http://ftp.ebi.ac.uk/pub/databases/gwas/summary_statistics/GCST90038001-GCST90039000/GCST90038626</p><table border="1" class="dataframe table table-bordered">
  <thead>
    <tr style="text-align: right;">
      <th>firstAuthor</th>
      <th>accessionId</th>
      <th>publicationDate</th>
      <th>journal</th>
      <th>title</th>
      <th>reportedTrait</th>
      <th>efoTraits</th>
      <th>bgTraits</th>
      <th>discoverySampleAncestry</th>
      <th>replicationSampleAncestry</th>
      <th>associationCount</th>
      <th>summaryStatistics</th>
      <th>ssApiFlag</th>
      <th>agreedToCc0</th>
      <th>genotypingTechnologies</th>
      <th>pubmedId</th>
      <th>initialSampleDescription</th>
      <th>replicateSampleDescription</th>
      <th>UKB_FIELD</th>
      <th>FuzzySimilarity_irritable bowel syndrome</th>
      <th>SearchPhenotype</th>
    </tr>
  </thead>
  <tbody>
    <tr>
      <td>D??nerta?� HM</td>
      <td>GCST90038626</td>
      <td>4/8/2021</td>
      <td>Nat Aging</td>
      <td>Common genetic associations between age-related diseases.</td>
      <td>irritable bowel syndrome</td>
      <td>irritable bowel syndrome</td>
      <td>-</td>
      <td>484598 nr</td>
      <td>NaN</td>
      <td>0</td>
      <td>http://ftp.ebi.ac.uk/pub/databases/gwas/summary_statistics/GCST90038001-GCST90039000/GCST90038626</td>
      <td>no</td>
      <td>yes</td>
      <td>Genome-wide genotyping array</td>
      <td>33959723</td>
      <td>11,168 cases,473,430 controls</td>
      <td>NaN</td>
      <td>-</td>
      <td>100</td>
      <td>irritable bowel syndrome</td>
    </tr>
  </tbody>
</table><h2 class='text-muted'>Information file</h2><table border="1" class="dataframe table table-bordered">
  <thead>
    <tr style="text-align: right;">
      <th>Unnamed: 0</th>
      <th>Name</th>
      <th>Last modified</th>
      <th>Size</th>
      <th>Description</th>
      <th>URL</th>
    </tr>
  </thead>
  <tbody>
    <tr>
      <td>1</td>
      <td>GCST90038626_buildGRCh37.tsv-meta.yaml</td>
      <td>2024-01-04 00:24</td>
      <td>675</td>
      <td>NaN</td>
      <td>http://ftp.ebi.ac.uk/pub/databases/gwas/summary_statistics/GCST90038001-GCST90039000/GCST90038626/GCST90038626_buildGRCh37.tsv-meta.yaml</td>
    </tr>
    <tr>
      <td>2</td>
      <td>harmonised/</td>
      <td>2024-02-02 15:16</td>
      <td>-</td>
      <td>NaN</td>
      <td>http://ftp.ebi.ac.uk/pub/databases/gwas/summary_statistics/GCST90038001-GCST90039000/GCST90038626/harmonised/</td>
    </tr>
    <tr>
      <td>3</td>
      <td>md5sum.txt</td>
      <td>2021-11-14 00:23</td>
      <td>62</td>
      <td>NaN</td>
      <td>http://ftp.ebi.ac.uk/pub/databases/gwas/summary_statistics/GCST90038001-GCST90039000/GCST90038626/md5sum.txt</td>
    </tr>
    <tr>
      <td>0</td>
      <td>GCST90038626_buildGRCh37.tsv</td>
      <td>2021-11-14 00:23</td>
      <td>1.2G</td>
      <td>NaN</td>
      <td>http://ftp.ebi.ac.uk/pub/databases/gwas/summary_statistics/GCST90038001-GCST90039000/GCST90038626/GCST90038626_buildGRCh37.tsv</td>
    </tr>
  </tbody>
</table><h2 class='text-muted'>Processed file</h2><p>Processing file: GCST90038626_buildGRCh37.tsv!</p><h2 class='text-muted'>GWAS file head</h2><table border="1" class="dataframe table table-bordered">
  <thead>
    <tr style="text-align: right;">
      <th>variant_id</th>
      <th>chromosome</th>
      <th>base_pair_location</th>
      <th>genpos</th>
      <th>effect_allele</th>
      <th>other_allele</th>
      <th>effect_allele_frequency</th>
      <th>info</th>
      <th>chisq_linreg</th>
      <th>p_linreg</th>
      <th>beta</th>
      <th>standard_error</th>
      <th>chisq_bolt_lmm_inf</th>
      <th>p_bolt_lmm_inf</th>
      <th>chisq_bolt_lmm</th>
      <th>p_value</th>
    </tr>
  </thead>
  <tbody>
    <tr>
      <td>rs533090414</td>
      <td>1</td>
      <td>18849</td>
      <td>0.0</td>
      <td>C</td>
      <td>G</td>
      <td>0.026465</td>
      <td>0.536412</td>
      <td>6.76961</td>
      <td>0.0093</td>
      <td>-0.003456</td>
      <td>0.001314</td>
      <td>6.91981</td>
      <td>0.0085</td>
      <td>6.92088</td>
      <td>0.0085</td>
    </tr>
    <tr>
      <td>rs568927205</td>
      <td>1</td>
      <td>54712</td>
      <td>0.0</td>
      <td>T</td>
      <td>TTTTC</td>
      <td>0.415825</td>
      <td>0.538572</td>
      <td>3.42641</td>
      <td>0.0640</td>
      <td>-0.000796</td>
      <td>0.000423</td>
      <td>3.53499</td>
      <td>0.0600</td>
      <td>3.53488</td>
      <td>0.0600</td>
    </tr>
  </tbody>
</table></div><div class='container mt-4 bg-light p-4 text-center'><h2 class='text-muted'>Original GWAS file headers</h2><table border="1" class="dataframe table table-bordered">
  <thead>
    <tr style="text-align: right;">
      <th>variant_id</th>
      <th>chromosome</th>
      <th>base_pair_location</th>
      <th>genpos</th>
      <th>effect_allele</th>
      <th>other_allele</th>
      <th>effect_allele_frequency</th>
      <th>info</th>
      <th>chisq_linreg</th>
      <th>p_linreg</th>
      <th>beta</th>
      <th>standard_error</th>
      <th>chisq_bolt_lmm_inf</th>
      <th>p_bolt_lmm_inf</th>
      <th>chisq_bolt_lmm</th>
      <th>p_value</th>
    </tr>
  </thead>
  <tbody>
    <tr>
      <td>rs533090414</td>
      <td>1</td>
      <td>18849</td>
      <td>0.0</td>
      <td>C</td>
      <td>G</td>
      <td>0.026465</td>
      <td>0.536412</td>
      <td>6.76961</td>
      <td>0.0093</td>
      <td>-0.003456</td>
      <td>0.001314</td>
      <td>6.91981</td>
      <td>0.0085</td>
      <td>6.92088</td>
      <td>0.0085</td>
    </tr>
    <tr>
      <td>rs568927205</td>
      <td>1</td>
      <td>54712</td>
      <td>0.0</td>
      <td>T</td>
      <td>TTTTC</td>
      <td>0.415825</td>
      <td>0.538572</td>
      <td>3.42641</td>
      <td>0.0640</td>
      <td>-0.000796</td>
      <td>0.000423</td>
      <td>3.53499</td>
      <td>0.0600</td>
      <td>3.53488</td>
      <td>0.0600</td>
    </tr>
  </tbody>
</table><p style='font-weight: bold; color: blue;'>CHR -> ['chromosome']</p><p style='font-weight: bold; color: blue;'>SNP -> ['variant_id']</p><p style='font-weight: bold; color: blue;'>BP -> ['base_pair_location', 'genpos']</p><p style='font-weight: bold; color: blue;'>A1 -> ['effect_allele']</p><p style='font-weight: bold; color: blue;'>A2 -> ['other_allele']</p><p style='font-weight: bold; color: blue;'>N -> []</p><p style='font-weight: bold; color: blue;'>P -> ['p_value']</p><p style='font-weight: bold; color: blue;'>INFO -> ['info']</p><p style='font-weight: bold; color: blue;'>MAF -> ['effect_allele_frequency']</p><p style='font-weight: bold; color: blue;'>BETA -> ['beta', 'p_bolt_lmm_inf']</p><p style='font-weight: bold; color: blue;'>SE -> ['standard_error']</p><p style='font-weight: bold; color: blue;'>OR -> []</p><p style='font-weight: bold; color: blue;'>Z -> []</p><p style='font-weight: bold; color: blue;'>D -> []</p><h2 class='text-muted'>GWAS columns</h2>variant_id, chromosome, base_pair_location, genpos, effect_allele, other_allele, effect_allele_frequency, info, chisq_linreg, p_linreg, beta, standard_error, chisq_bolt_lmm_inf, p_bolt_lmm_inf, chisq_bolt_lmm, p_value<h2 class='text-muted'>Identified columns</h2>chromosome, variant_id, base_pair_location, genpos, effect_allele, other_allele, p_value, info, effect_allele_frequency, beta, p_bolt_lmm_inf, standard_error<h2 class='text-muted'>Unidentified Columns headers</h2>chisq_bolt_lmm, chisq_linreg, chisq_bolt_lmm_inf, p_linreg<h2 class='text-muted'>PMID</h2>33959723<h2 class='text-muted'>DOI</h2>10.1038/s43587-021-00051-5<div class='bibtex-container'><h2 class='text-muted'>BibTeX</h2><div style='font-family: monospace; white-space: pre-wrap;'> @article{D_nerta__2021, title={Common genetic associations between age-related diseases}, volume={1}, ISSN={2662-8465}, url={http://dx.doi.org/10.1038/s43587-021-00051-5}, DOI={10.1038/s43587-021-00051-5}, number={4}, journal={Nature Aging}, publisher={Springer Science and Business Media LLC}, author={Dönertaş, Handan Melike and Fabian, Daniel K. and Fuentealba, Matías and Partridge, Linda and Thornton, Janet M.}, year={2021}, month=apr, pages={400–412} }
</div></div><br><br><br><br><br><br></div>