<link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/bootstrap/4.0.0/css/bootstrap.min.css" integrity="sha384-Gn5384xqQ1aoWXA+058RXPxPg6fy4IWvTNh0E263XmFcJlSAwiGgFAW/dAiS6JXm" crossorigin="anonymous"><div class='container mt-4 bg-light p-4 text-center'><h1 class='text-primary'>Phenotype: osteoarthritis</h1><p class='text-muted'><strong>URL:</strong> http://ftp.ebi.ac.uk/pub/databases/gwas/summary_statistics/GCST007001-GCST008000/GCST007093</p><table border="1" class="dataframe table table-bordered">
  <thead>
    <tr style="text-align: right;">
      <th>firstAuthor</th>
      <th>accessionId</th>
      <th>publicationDate</th>
      <th>journal</th>
      <th>title</th>
      <th>reportedTrait</th>
      <th>efoTraits</th>
      <th>bgTraits</th>
      <th>discoverySampleAncestry</th>
      <th>replicationSampleAncestry</th>
      <th>associationCount</th>
      <th>summaryStatistics</th>
      <th>ssApiFlag</th>
      <th>agreedToCc0</th>
      <th>genotypingTechnologies</th>
      <th>pubmedId</th>
      <th>initialSampleDescription</th>
      <th>replicateSampleDescription</th>
      <th>UKB_FIELD</th>
      <th>FuzzySimilarity_osteoarthritis</th>
      <th>SearchPhenotype</th>
    </tr>
  </thead>
  <tbody>
    <tr>
      <td>Tachmazidou I</td>
      <td>GCST007093</td>
      <td>1/21/2019</td>
      <td>Nat Genet</td>
      <td>Identification of new therapeutic targets for osteoarthritis through genome-wide analyses of UK Biobank data.</td>
      <td>osteoarthritis</td>
      <td>osteoarthritis</td>
      <td>-</td>
      <td>455221 european</td>
      <td>NaN</td>
      <td>35</td>
      <td>http://ftp.ebi.ac.uk/pub/databases/gwas/summary_statistics/GCST007001-GCST008000/GCST007093</td>
      <td>no</td>
      <td>-</td>
      <td>Genome-wide genotyping array</td>
      <td>30664745</td>
      <td>77,052 European ancestry cases,378,169 European ancestry controls</td>
      <td>NaN</td>
      <td>-</td>
      <td>100</td>
      <td>osteoarthritis</td>
    </tr>
  </tbody>
</table><h2 class='text-muted'>Information file</h2><table border="1" class="dataframe table table-bordered">
  <thead>
    <tr style="text-align: right;">
      <th>Unnamed: 0</th>
      <th>Name</th>
      <th>Last modified</th>
      <th>Size</th>
      <th>Description</th>
      <th>URL</th>
    </tr>
  </thead>
  <tbody>
    <tr>
      <td>1</td>
      <td>Tachmazidou_30664745_md5sum.txt</td>
      <td>2019-03-13 12:51</td>
      <td>273</td>
      <td>NaN</td>
      <td>http://ftp.ebi.ac.uk/pub/databases/gwas/summary_statistics/GCST007001-GCST008000/GCST007093/Tachmazidou_30664745_md5sum.txt</td>
    </tr>
    <tr>
      <td>2</td>
      <td>Tachmazidou_30664745_readme.txt</td>
      <td>2019-03-13 12:51</td>
      <td>2.6K</td>
      <td>NaN</td>
      <td>http://ftp.ebi.ac.uk/pub/databases/gwas/summary_statistics/GCST007001-GCST008000/GCST007093/Tachmazidou_30664745_readme.txt</td>
    </tr>
    <tr>
      <td>0</td>
      <td>Tachmazidou_30664745_ALLOA.txt.gz</td>
      <td>2019-03-13 12:51</td>
      <td>778M</td>
      <td>NaN</td>
      <td>http://ftp.ebi.ac.uk/pub/databases/gwas/summary_statistics/GCST007001-GCST008000/GCST007093/Tachmazidou_30664745_ALLOA.txt.gz</td>
    </tr>
  </tbody>
</table><h2 class='text-muted'>Processed file</h2><p>Processing file: Tachmazidou_30664745_ALLOA.txt.gz!</p><h2 class='text-muted'>GWAS file head</h2><table border="1" class="dataframe table table-bordered">
  <thead>
    <tr style="text-align: right;">
      <th>markername</th>
      <th>allele1</th>
      <th>allele2</th>
      <th>freq1</th>
      <th>freqse</th>
      <th>minfreq</th>
      <th>maxfreq</th>
      <th>effect</th>
      <th>stderr</th>
      <th>p-value</th>
      <th>direction</th>
      <th>hetisq</th>
      <th>hetchisq</th>
      <th>hetdf</th>
      <th>hetpval</th>
    </tr>
  </thead>
  <tbody>
    <tr>
      <td>3_132917622_A_G</td>
      <td>a</td>
      <td>g</td>
      <td>0.0002</td>
      <td>0.0</td>
      <td>0.0002</td>
      <td>0.0002</td>
      <td>0.4870</td>
      <td>0.2279</td>
      <td>0.03262</td>
      <td>?+</td>
      <td>0.0</td>
      <td>0.0</td>
      <td>0</td>
      <td>1.0</td>
    </tr>
    <tr>
      <td>9_105246255_G_T</td>
      <td>t</td>
      <td>g</td>
      <td>0.9992</td>
      <td>0.0</td>
      <td>0.9992</td>
      <td>0.9992</td>
      <td>-0.2118</td>
      <td>0.1114</td>
      <td>0.05719</td>
      <td>?-</td>
      <td>0.0</td>
      <td>0.0</td>
      <td>0</td>
      <td>1.0</td>
    </tr>
  </tbody>
</table></div><div class='container mt-4 bg-light p-4 text-center'><h2 class='text-muted'>Original GWAS file headers</h2><table border="1" class="dataframe table table-bordered">
  <thead>
    <tr style="text-align: right;">
      <th>markername</th>
      <th>allele1</th>
      <th>allele2</th>
      <th>freq1</th>
      <th>freqse</th>
      <th>minfreq</th>
      <th>maxfreq</th>
      <th>effect</th>
      <th>stderr</th>
      <th>p-value</th>
      <th>direction</th>
      <th>hetisq</th>
      <th>hetchisq</th>
      <th>hetdf</th>
      <th>hetpval</th>
    </tr>
  </thead>
  <tbody>
    <tr>
      <td>3_132917622_A_G</td>
      <td>a</td>
      <td>g</td>
      <td>0.0002</td>
      <td>0.0</td>
      <td>0.0002</td>
      <td>0.0002</td>
      <td>0.4870</td>
      <td>0.2279</td>
      <td>0.03262</td>
      <td>?+</td>
      <td>0.0</td>
      <td>0.0</td>
      <td>0</td>
      <td>1.0</td>
    </tr>
    <tr>
      <td>9_105246255_G_T</td>
      <td>t</td>
      <td>g</td>
      <td>0.9992</td>
      <td>0.0</td>
      <td>0.9992</td>
      <td>0.9992</td>
      <td>-0.2118</td>
      <td>0.1114</td>
      <td>0.05719</td>
      <td>?-</td>
      <td>0.0</td>
      <td>0.0</td>
      <td>0</td>
      <td>1.0</td>
    </tr>
  </tbody>
</table><p style='font-weight: bold; color: blue;'>CHR -> []</p><p style='font-weight: bold; color: blue;'>SNP -> ['markername']</p><p style='font-weight: bold; color: blue;'>BP -> ['markername']</p><p style='font-weight: bold; color: blue;'>A1 -> ['allele1']</p><p style='font-weight: bold; color: blue;'>A2 -> ['allele1', 'allele2']</p><p style='font-weight: bold; color: blue;'>N -> []</p><p style='font-weight: bold; color: blue;'>P -> ['p-value', 'hetpval']</p><p style='font-weight: bold; color: blue;'>INFO -> []</p><p style='font-weight: bold; color: blue;'>MAF -> ['freq1']</p><p style='font-weight: bold; color: blue;'>BETA -> ['effect']</p><p style='font-weight: bold; color: blue;'>SE -> ['freqse', 'stderr']</p><p style='font-weight: bold; color: blue;'>OR -> []</p><p style='font-weight: bold; color: blue;'>Z -> []</p><p style='font-weight: bold; color: blue;'>D -> ['direction']</p><h2 class='text-muted'>GWAS columns</h2>markername, allele1, allele2, freq1, freqse, minfreq, maxfreq, effect, stderr, p-value, direction, hetisq, hetchisq, hetdf, hetpval<h2 class='text-muted'>Identified columns</h2>markername, markername, allele1, allele1, allele2, p-value, hetpval, freq1, effect, freqse, stderr, direction<h2 class='text-muted'>Unidentified Columns headers</h2>minfreq, hetchisq, hetdf, maxfreq, hetisq<h2 class='text-muted'>PMID</h2>30664745<h2 class='text-muted'>DOI</h2>10.1038/s41588-018-0327-1<div class='bibtex-container'><h2 class='text-muted'>BibTeX</h2><div style='font-family: monospace; white-space: pre-wrap;'> @article{Tachmazidou_2019, title={Identification of new therapeutic targets for osteoarthritis through genome-wide analyses of UK Biobank data}, volume={51}, ISSN={1546-1718}, url={http://dx.doi.org/10.1038/s41588-018-0327-1}, DOI={10.1038/s41588-018-0327-1}, number={2}, journal={Nature Genetics}, publisher={Springer Science and Business Media LLC}, author={Tachmazidou, Ioanna and Hatzikotoulas, Konstantinos and Southam, Lorraine and Esparza-Gordillo, Jorge and Haberland, Valeriia and Zheng, Jie and Johnson, Toby and Koprulu, Mine and Zengini, Eleni and Steinberg, Julia and Wilkinson, Jeremy M. and Bhatnagar, Sahir and Hoffman, Joshua D. and Buchan, Natalie and Süveges, Dániel and Yerges-Armstrong, Laura and Smith, George Davey and Gaunt, Tom R. and Scott, Robert A. and McCarthy, Linda C. and Zeggini, Eleftheria}, year={2019}, month=jan, pages={230–236} }
</div></div><br><br><br><br><br><br></div><div class='container mt-4 bg-light p-4 text-center'><h1 class='text-primary'>Phenotype: osteoarthritis</h1><p class='text-muted'><strong>URL:</strong> http://ftp.ebi.ac.uk/pub/databases/gwas/summary_statistics/GCST90134001-GCST90135000/GCST90134279</p><table border="1" class="dataframe table table-bordered">
  <thead>
    <tr style="text-align: right;">
      <th>firstAuthor</th>
      <th>accessionId</th>
      <th>publicationDate</th>
      <th>journal</th>
      <th>title</th>
      <th>reportedTrait</th>
      <th>efoTraits</th>
      <th>bgTraits</th>
      <th>discoverySampleAncestry</th>
      <th>replicationSampleAncestry</th>
      <th>associationCount</th>
      <th>summaryStatistics</th>
      <th>ssApiFlag</th>
      <th>agreedToCc0</th>
      <th>genotypingTechnologies</th>
      <th>pubmedId</th>
      <th>initialSampleDescription</th>
      <th>replicateSampleDescription</th>
      <th>UKB_FIELD</th>
      <th>FuzzySimilarity_osteoarthritis</th>
      <th>SearchPhenotype</th>
    </tr>
  </thead>
  <tbody>
    <tr>
      <td>McDonald MN</td>
      <td>GCST90134279</td>
      <td>11/21/2022</td>
      <td>Nat Genet</td>
      <td>Novel genetic loci associated with osteoarthritis in multi-ancestry analyses in the Million Veteran Program and UK Biobank.</td>
      <td>osteoarthritis</td>
      <td>osteoarthritis</td>
      <td>-</td>
      <td>293819 european</td>
      <td>NaN</td>
      <td>61</td>
      <td>http://ftp.ebi.ac.uk/pub/databases/gwas/summary_statistics/GCST90134001-GCST90135000/GCST90134279</td>
      <td>no</td>
      <td>yes</td>
      <td>Genome-wide genotyping array</td>
      <td>36411363</td>
      <td>54,039 European ancestry cases,239,780 European ancestry controls</td>
      <td>NaN</td>
      <td>-</td>
      <td>100</td>
      <td>osteoarthritis</td>
    </tr>
  </tbody>
</table><h2 class='text-muted'>Information file</h2><table border="1" class="dataframe table table-bordered">
  <thead>
    <tr style="text-align: right;">
      <th>Unnamed: 0</th>
      <th>Name</th>
      <th>Last modified</th>
      <th>Size</th>
      <th>Description</th>
      <th>URL</th>
    </tr>
  </thead>
  <tbody>
    <tr>
      <td>1</td>
      <td>GCST90134279_buildGRCh37.csv-meta.yaml</td>
      <td>2024-01-04 00:30</td>
      <td>673</td>
      <td>NaN</td>
      <td>http://ftp.ebi.ac.uk/pub/databases/gwas/summary_statistics/GCST90134001-GCST90135000/GCST90134279/GCST90134279_buildGRCh37.csv-meta.yaml</td>
    </tr>
    <tr>
      <td>2</td>
      <td>md5sum.txt</td>
      <td>2022-09-06 16:18</td>
      <td>62</td>
      <td>NaN</td>
      <td>http://ftp.ebi.ac.uk/pub/databases/gwas/summary_statistics/GCST90134001-GCST90135000/GCST90134279/md5sum.txt</td>
    </tr>
    <tr>
      <td>0</td>
      <td>GCST90134279_buildGRCh37.csv</td>
      <td>2022-09-06 16:18</td>
      <td>2.3G</td>
      <td>NaN</td>
      <td>http://ftp.ebi.ac.uk/pub/databases/gwas/summary_statistics/GCST90134001-GCST90135000/GCST90134279/GCST90134279_buildGRCh37.csv</td>
    </tr>
  </tbody>
</table><h2 class='text-muted'>Processed file</h2><p>Processing file: GCST90134279_buildGRCh37.csv.1!</p><h2 class='text-muted'>GWAS file head</h2><table border="1" class="dataframe table table-bordered">
  <thead>
    <tr style="text-align: right;">
      <th>variant_id</th>
      <th>p_value</th>
      <th>chromosome</th>
      <th>base_pair_location</th>
      <th>effect_allele</th>
      <th>other_allele</th>
      <th>effect_allele_frequency</th>
      <th>odds_ratio</th>
      <th>ci_lower</th>
      <th>ci_upper</th>
      <th>beta</th>
      <th>standard_error</th>
    </tr>
  </thead>
  <tbody>
    <tr>
      <td>NaN</td>
      <td>0.628044</td>
      <td>10</td>
      <td>10001340</td>
      <td>G</td>
      <td>C</td>
      <td>0.000453</td>
      <td>1.10005</td>
      <td>0.747967</td>
      <td>1.61786</td>
      <td>0.095356</td>
      <td>0.196814</td>
    </tr>
    <tr>
      <td>NaN</td>
      <td>0.685891</td>
      <td>10</td>
      <td>10001753</td>
      <td>T</td>
      <td>TAAAG</td>
      <td>0.049836</td>
      <td>1.00678</td>
      <td>0.974344</td>
      <td>1.04030</td>
      <td>0.006757</td>
      <td>0.016709</td>
    </tr>
  </tbody>
</table></div><div class='container mt-4 bg-light p-4 text-center'><h2 class='text-muted'>Original GWAS file headers</h2><table border="1" class="dataframe table table-bordered">
  <thead>
    <tr style="text-align: right;">
      <th>variant_id</th>
      <th>p_value</th>
      <th>chromosome</th>
      <th>base_pair_location</th>
      <th>effect_allele</th>
      <th>other_allele</th>
      <th>effect_allele_frequency</th>
      <th>odds_ratio</th>
      <th>ci_lower</th>
      <th>ci_upper</th>
      <th>beta</th>
      <th>standard_error</th>
    </tr>
  </thead>
  <tbody>
    <tr>
      <td>NaN</td>
      <td>0.628044</td>
      <td>10</td>
      <td>10001340</td>
      <td>G</td>
      <td>C</td>
      <td>0.000453</td>
      <td>1.10005</td>
      <td>0.747967</td>
      <td>1.61786</td>
      <td>0.095356</td>
      <td>0.196814</td>
    </tr>
    <tr>
      <td>NaN</td>
      <td>0.685891</td>
      <td>10</td>
      <td>10001753</td>
      <td>T</td>
      <td>TAAAG</td>
      <td>0.049836</td>
      <td>1.00678</td>
      <td>0.974344</td>
      <td>1.04030</td>
      <td>0.006757</td>
      <td>0.016709</td>
    </tr>
  </tbody>
</table><p style='font-weight: bold; color: blue;'>CHR -> ['chromosome']</p><p style='font-weight: bold; color: blue;'>SNP -> ['variant_id']</p><p style='font-weight: bold; color: blue;'>BP -> ['base_pair_location']</p><p style='font-weight: bold; color: blue;'>A1 -> ['effect_allele']</p><p style='font-weight: bold; color: blue;'>A2 -> ['other_allele']</p><p style='font-weight: bold; color: blue;'>N -> []</p><p style='font-weight: bold; color: blue;'>P -> ['p_value']</p><p style='font-weight: bold; color: blue;'>INFO -> []</p><p style='font-weight: bold; color: blue;'>MAF -> ['effect_allele_frequency']</p><p style='font-weight: bold; color: blue;'>BETA -> ['beta']</p><p style='font-weight: bold; color: blue;'>SE -> ['standard_error']</p><p style='font-weight: bold; color: blue;'>OR -> ['odds_ratio']</p><p style='font-weight: bold; color: blue;'>Z -> []</p><p style='font-weight: bold; color: blue;'>D -> []</p><h2 class='text-muted'>GWAS columns</h2>variant_id, p_value, chromosome, base_pair_location, effect_allele, other_allele, effect_allele_frequency, odds_ratio, ci_lower, ci_upper, beta, standard_error<h2 class='text-muted'>Identified columns</h2>chromosome, variant_id, base_pair_location, effect_allele, other_allele, p_value, effect_allele_frequency, beta, standard_error, odds_ratio<h2 class='text-muted'>Unidentified Columns headers</h2>ci_lower, ci_upper<h2 class='text-muted'>PMID</h2>36411363<h2 class='text-muted'>DOI</h2>Could not find DOI!<div class='bibtex-container'><h2 class='text-muted'>BibTeX</h2><div style='font-family: monospace; white-space: pre-wrap;'></div></div><br><br><br><br><br><br></div><div class='container mt-4 bg-light p-4 text-center'><h1 class='text-primary'>Phenotype: osteoarthritis</h1><p class='text-muted'><strong>URL:</strong> http://ftp.ebi.ac.uk/pub/databases/gwas/summary_statistics/GCST90038001-GCST90039000/GCST90038686</p><table border="1" class="dataframe table table-bordered">
  <thead>
    <tr style="text-align: right;">
      <th>firstAuthor</th>
      <th>accessionId</th>
      <th>publicationDate</th>
      <th>journal</th>
      <th>title</th>
      <th>reportedTrait</th>
      <th>efoTraits</th>
      <th>bgTraits</th>
      <th>discoverySampleAncestry</th>
      <th>replicationSampleAncestry</th>
      <th>associationCount</th>
      <th>summaryStatistics</th>
      <th>ssApiFlag</th>
      <th>agreedToCc0</th>
      <th>genotypingTechnologies</th>
      <th>pubmedId</th>
      <th>initialSampleDescription</th>
      <th>replicateSampleDescription</th>
      <th>UKB_FIELD</th>
      <th>FuzzySimilarity_osteoarthritis</th>
      <th>SearchPhenotype</th>
    </tr>
  </thead>
  <tbody>
    <tr>
      <td>D??nerta?� HM</td>
      <td>GCST90038686</td>
      <td>4/8/2021</td>
      <td>Nat Aging</td>
      <td>Common genetic associations between age-related diseases.</td>
      <td>osteoarthritis</td>
      <td>osteoarthritis</td>
      <td>-</td>
      <td>484598 nr</td>
      <td>NaN</td>
      <td>0</td>
      <td>http://ftp.ebi.ac.uk/pub/databases/gwas/summary_statistics/GCST90038001-GCST90039000/GCST90038686</td>
      <td>no</td>
      <td>yes</td>
      <td>Genome-wide genotyping array</td>
      <td>33959723</td>
      <td>39,515 cases,445,083 controls</td>
      <td>NaN</td>
      <td>-</td>
      <td>100</td>
      <td>osteoarthritis</td>
    </tr>
  </tbody>
</table><h2 class='text-muted'>Information file</h2><table border="1" class="dataframe table table-bordered">
  <thead>
    <tr style="text-align: right;">
      <th>Unnamed: 0</th>
      <th>Name</th>
      <th>Last modified</th>
      <th>Size</th>
      <th>Description</th>
      <th>URL</th>
    </tr>
  </thead>
  <tbody>
    <tr>
      <td>1</td>
      <td>GCST90038686_buildGRCh37.tsv-meta.yaml</td>
      <td>2024-01-04 00:19</td>
      <td>667</td>
      <td>NaN</td>
      <td>http://ftp.ebi.ac.uk/pub/databases/gwas/summary_statistics/GCST90038001-GCST90039000/GCST90038686/GCST90038686_buildGRCh37.tsv-meta.yaml</td>
    </tr>
    <tr>
      <td>2</td>
      <td>harmonised/</td>
      <td>2024-02-02 15:16</td>
      <td>-</td>
      <td>NaN</td>
      <td>http://ftp.ebi.ac.uk/pub/databases/gwas/summary_statistics/GCST90038001-GCST90039000/GCST90038686/harmonised/</td>
    </tr>
    <tr>
      <td>3</td>
      <td>md5sum.txt</td>
      <td>2021-11-14 00:24</td>
      <td>62</td>
      <td>NaN</td>
      <td>http://ftp.ebi.ac.uk/pub/databases/gwas/summary_statistics/GCST90038001-GCST90039000/GCST90038686/md5sum.txt</td>
    </tr>
    <tr>
      <td>0</td>
      <td>GCST90038686_buildGRCh37.tsv</td>
      <td>2021-11-14 00:24</td>
      <td>1.2G</td>
      <td>NaN</td>
      <td>http://ftp.ebi.ac.uk/pub/databases/gwas/summary_statistics/GCST90038001-GCST90039000/GCST90038686/GCST90038686_buildGRCh37.tsv</td>
    </tr>
  </tbody>
</table><h2 class='text-muted'>Processed file</h2><p>Processing file: GCST90038686_buildGRCh37.tsv!</p><h2 class='text-muted'>GWAS file head</h2><table border="1" class="dataframe table table-bordered">
  <thead>
    <tr style="text-align: right;">
      <th>variant_id</th>
      <th>chromosome</th>
      <th>base_pair_location</th>
      <th>genpos</th>
      <th>effect_allele</th>
      <th>other_allele</th>
      <th>effect_allele_frequency</th>
      <th>info</th>
      <th>chisq_linreg</th>
      <th>p_linreg</th>
      <th>beta</th>
      <th>standard_error</th>
      <th>chisq_bolt_lmm_inf</th>
      <th>p_bolt_lmm_inf</th>
      <th>chisq_bolt_lmm</th>
      <th>p_value</th>
    </tr>
  </thead>
  <tbody>
    <tr>
      <td>rs533090414</td>
      <td>1</td>
      <td>18849</td>
      <td>0.0</td>
      <td>C</td>
      <td>G</td>
      <td>0.026465</td>
      <td>0.536412</td>
      <td>0.005782</td>
      <td>0.94</td>
      <td>0.000325</td>
      <td>0.002341</td>
      <td>0.019297</td>
      <td>0.89</td>
      <td>0.018784</td>
      <td>0.89</td>
    </tr>
    <tr>
      <td>rs568927205</td>
      <td>1</td>
      <td>54712</td>
      <td>0.0</td>
      <td>T</td>
      <td>TTTTC</td>
      <td>0.415825</td>
      <td>0.538572</td>
      <td>2.009400</td>
      <td>0.16</td>
      <td>-0.001080</td>
      <td>0.000754</td>
      <td>2.050360</td>
      <td>0.15</td>
      <td>2.045640</td>
      <td>0.15</td>
    </tr>
  </tbody>
</table></div><div class='container mt-4 bg-light p-4 text-center'><h2 class='text-muted'>Original GWAS file headers</h2><table border="1" class="dataframe table table-bordered">
  <thead>
    <tr style="text-align: right;">
      <th>variant_id</th>
      <th>chromosome</th>
      <th>base_pair_location</th>
      <th>genpos</th>
      <th>effect_allele</th>
      <th>other_allele</th>
      <th>effect_allele_frequency</th>
      <th>info</th>
      <th>chisq_linreg</th>
      <th>p_linreg</th>
      <th>beta</th>
      <th>standard_error</th>
      <th>chisq_bolt_lmm_inf</th>
      <th>p_bolt_lmm_inf</th>
      <th>chisq_bolt_lmm</th>
      <th>p_value</th>
    </tr>
  </thead>
  <tbody>
    <tr>
      <td>rs533090414</td>
      <td>1</td>
      <td>18849</td>
      <td>0.0</td>
      <td>C</td>
      <td>G</td>
      <td>0.026465</td>
      <td>0.536412</td>
      <td>0.005782</td>
      <td>0.94</td>
      <td>0.000325</td>
      <td>0.002341</td>
      <td>0.019297</td>
      <td>0.89</td>
      <td>0.018784</td>
      <td>0.89</td>
    </tr>
    <tr>
      <td>rs568927205</td>
      <td>1</td>
      <td>54712</td>
      <td>0.0</td>
      <td>T</td>
      <td>TTTTC</td>
      <td>0.415825</td>
      <td>0.538572</td>
      <td>2.009400</td>
      <td>0.16</td>
      <td>-0.001080</td>
      <td>0.000754</td>
      <td>2.050360</td>
      <td>0.15</td>
      <td>2.045640</td>
      <td>0.15</td>
    </tr>
  </tbody>
</table><p style='font-weight: bold; color: blue;'>CHR -> ['chromosome']</p><p style='font-weight: bold; color: blue;'>SNP -> ['variant_id']</p><p style='font-weight: bold; color: blue;'>BP -> ['base_pair_location', 'genpos']</p><p style='font-weight: bold; color: blue;'>A1 -> ['effect_allele']</p><p style='font-weight: bold; color: blue;'>A2 -> ['other_allele']</p><p style='font-weight: bold; color: blue;'>N -> []</p><p style='font-weight: bold; color: blue;'>P -> ['p_value']</p><p style='font-weight: bold; color: blue;'>INFO -> ['info']</p><p style='font-weight: bold; color: blue;'>MAF -> ['effect_allele_frequency']</p><p style='font-weight: bold; color: blue;'>BETA -> ['beta', 'p_bolt_lmm_inf']</p><p style='font-weight: bold; color: blue;'>SE -> ['standard_error']</p><p style='font-weight: bold; color: blue;'>OR -> []</p><p style='font-weight: bold; color: blue;'>Z -> []</p><p style='font-weight: bold; color: blue;'>D -> []</p><h2 class='text-muted'>GWAS columns</h2>variant_id, chromosome, base_pair_location, genpos, effect_allele, other_allele, effect_allele_frequency, info, chisq_linreg, p_linreg, beta, standard_error, chisq_bolt_lmm_inf, p_bolt_lmm_inf, chisq_bolt_lmm, p_value<h2 class='text-muted'>Identified columns</h2>chromosome, variant_id, base_pair_location, genpos, effect_allele, other_allele, p_value, info, effect_allele_frequency, beta, p_bolt_lmm_inf, standard_error<h2 class='text-muted'>Unidentified Columns headers</h2>chisq_linreg, chisq_bolt_lmm_inf, chisq_bolt_lmm, p_linreg<h2 class='text-muted'>PMID</h2>33959723<h2 class='text-muted'>DOI</h2>10.1038/s43587-021-00051-5<div class='bibtex-container'><h2 class='text-muted'>BibTeX</h2><div style='font-family: monospace; white-space: pre-wrap;'> @article{D_nerta__2021, title={Common genetic associations between age-related diseases}, volume={1}, ISSN={2662-8465}, url={http://dx.doi.org/10.1038/s43587-021-00051-5}, DOI={10.1038/s43587-021-00051-5}, number={4}, journal={Nature Aging}, publisher={Springer Science and Business Media LLC}, author={Dönertaş, Handan Melike and Fabian, Daniel K. and Fuentealba, Matías and Partridge, Linda and Thornton, Janet M.}, year={2021}, month=apr, pages={400–412} }
</div></div><br><br><br><br><br><br></div><div class='container mt-4 bg-light p-4 text-center'><h1 class='text-primary'>Phenotype: osteoarthritis</h1><p class='text-muted'><strong>URL:</strong> http://ftp.ebi.ac.uk/pub/databases/gwas/summary_statistics/GCST90134001-GCST90135000/GCST90134283</p><table border="1" class="dataframe table table-bordered">
  <thead>
    <tr style="text-align: right;">
      <th>firstAuthor</th>
      <th>accessionId</th>
      <th>publicationDate</th>
      <th>journal</th>
      <th>title</th>
      <th>reportedTrait</th>
      <th>efoTraits</th>
      <th>bgTraits</th>
      <th>discoverySampleAncestry</th>
      <th>replicationSampleAncestry</th>
      <th>associationCount</th>
      <th>summaryStatistics</th>
      <th>ssApiFlag</th>
      <th>agreedToCc0</th>
      <th>genotypingTechnologies</th>
      <th>pubmedId</th>
      <th>initialSampleDescription</th>
      <th>replicateSampleDescription</th>
      <th>UKB_FIELD</th>
      <th>FuzzySimilarity_osteoarthritis</th>
      <th>SearchPhenotype</th>
    </tr>
  </thead>
  <tbody>
    <tr>
      <td>McDonald MN</td>
      <td>GCST90134283</td>
      <td>11/21/2022</td>
      <td>Nat Genet</td>
      <td>Novel genetic loci associated with osteoarthritis in multi-ancestry analyses in the Million Veteran Program and UK Biobank.</td>
      <td>osteoarthritis</td>
      <td>osteoarthritis</td>
      <td>-</td>
      <td>321359 european, south asian, east asian, african unspecified</td>
      <td>NaN</td>
      <td>0</td>
      <td>http://ftp.ebi.ac.uk/pub/databases/gwas/summary_statistics/GCST90134001-GCST90135000/GCST90134283</td>
      <td>no</td>
      <td>yes</td>
      <td>Genome-wide genotyping array</td>
      <td>36411363</td>
      <td>58,574 European ancestry cases,262,785 European ancestry controls,58,574 South Asian ancestry cases,262,785 South Asian ancestry controls,58,574 East Asian ancestry cases,262,785 East Asian ancestry controls,58,574 African ancestry cases,262,785 African ancestry controls</td>
      <td>NaN</td>
      <td>-</td>
      <td>100</td>
      <td>osteoarthritis</td>
    </tr>
  </tbody>
</table><h2 class='text-muted'>Information file</h2><table border="1" class="dataframe table table-bordered">
  <thead>
    <tr style="text-align: right;">
      <th>Unnamed: 0</th>
      <th>Name</th>
      <th>Last modified</th>
      <th>Size</th>
      <th>Description</th>
      <th>URL</th>
    </tr>
  </thead>
  <tbody>
    <tr>
      <td>1</td>
      <td>GCST90134283_buildGRCh37.csv-meta.yaml</td>
      <td>2024-01-04 00:15</td>
      <td>740</td>
      <td>NaN</td>
      <td>http://ftp.ebi.ac.uk/pub/databases/gwas/summary_statistics/GCST90134001-GCST90135000/GCST90134283/GCST90134283_buildGRCh37.csv-meta.yaml</td>
    </tr>
    <tr>
      <td>2</td>
      <td>md5sum.txt</td>
      <td>2022-09-06 16:32</td>
      <td>62</td>
      <td>NaN</td>
      <td>http://ftp.ebi.ac.uk/pub/databases/gwas/summary_statistics/GCST90134001-GCST90135000/GCST90134283/md5sum.txt</td>
    </tr>
    <tr>
      <td>0</td>
      <td>GCST90134283_buildGRCh37.csv</td>
      <td>2022-09-06 16:32</td>
      <td>3.3G</td>
      <td>NaN</td>
      <td>http://ftp.ebi.ac.uk/pub/databases/gwas/summary_statistics/GCST90134001-GCST90135000/GCST90134283/GCST90134283_buildGRCh37.csv</td>
    </tr>
  </tbody>
</table><h2 class='text-muted'>Processed file</h2><p>Processing file: GCST90134283_buildGRCh37.csv!</p><h2 class='text-muted'>GWAS file head</h2><table border="1" class="dataframe table table-bordered">
  <thead>
    <tr style="text-align: right;">
      <th>variant_id</th>
      <th>p_value</th>
      <th>chromosome</th>
      <th>base_pair_location</th>
      <th>effect_allele</th>
      <th>other_allele</th>
      <th>effect_allele_frequency</th>
      <th>odds_ratio</th>
      <th>ci_lower</th>
      <th>ci_upper</th>
      <th>beta</th>
      <th>standard_error</th>
    </tr>
  </thead>
  <tbody>
    <tr>
      <td>rs534229142</td>
      <td>0.031</td>
      <td>1</td>
      <td>10511</td>
      <td>G</td>
      <td>A</td>
      <td>0.999586</td>
      <td>0.952151</td>
      <td>0.907509</td>
      <td>0.996793</td>
      <td>-0.049032</td>
      <td>0.022776</td>
    </tr>
    <tr>
      <td>NaN</td>
      <td>0.650</td>
      <td>1</td>
      <td>10616</td>
      <td>CCGCCGTTGCAAAGGCGCGCCG</td>
      <td>C</td>
      <td>0.001612</td>
      <td>0.994727</td>
      <td>0.971980</td>
      <td>1.017475</td>
      <td>-0.005287</td>
      <td>0.011606</td>
    </tr>
  </tbody>
</table></div><div class='container mt-4 bg-light p-4 text-center'><h2 class='text-muted'>Original GWAS file headers</h2><table border="1" class="dataframe table table-bordered">
  <thead>
    <tr style="text-align: right;">
      <th>variant_id</th>
      <th>p_value</th>
      <th>chromosome</th>
      <th>base_pair_location</th>
      <th>effect_allele</th>
      <th>other_allele</th>
      <th>effect_allele_frequency</th>
      <th>odds_ratio</th>
      <th>ci_lower</th>
      <th>ci_upper</th>
      <th>beta</th>
      <th>standard_error</th>
    </tr>
  </thead>
  <tbody>
    <tr>
      <td>rs534229142</td>
      <td>0.031</td>
      <td>1</td>
      <td>10511</td>
      <td>G</td>
      <td>A</td>
      <td>0.999586</td>
      <td>0.952151</td>
      <td>0.907509</td>
      <td>0.996793</td>
      <td>-0.049032</td>
      <td>0.022776</td>
    </tr>
    <tr>
      <td>NaN</td>
      <td>0.650</td>
      <td>1</td>
      <td>10616</td>
      <td>CCGCCGTTGCAAAGGCGCGCCG</td>
      <td>C</td>
      <td>0.001612</td>
      <td>0.994727</td>
      <td>0.971980</td>
      <td>1.017475</td>
      <td>-0.005287</td>
      <td>0.011606</td>
    </tr>
  </tbody>
</table><p style='font-weight: bold; color: blue;'>CHR -> ['chromosome']</p><p style='font-weight: bold; color: blue;'>SNP -> ['variant_id']</p><p style='font-weight: bold; color: blue;'>BP -> ['base_pair_location']</p><p style='font-weight: bold; color: blue;'>A1 -> ['effect_allele']</p><p style='font-weight: bold; color: blue;'>A2 -> ['other_allele']</p><p style='font-weight: bold; color: blue;'>N -> []</p><p style='font-weight: bold; color: blue;'>P -> ['p_value']</p><p style='font-weight: bold; color: blue;'>INFO -> []</p><p style='font-weight: bold; color: blue;'>MAF -> ['effect_allele_frequency']</p><p style='font-weight: bold; color: blue;'>BETA -> ['beta']</p><p style='font-weight: bold; color: blue;'>SE -> ['standard_error']</p><p style='font-weight: bold; color: blue;'>OR -> ['odds_ratio']</p><p style='font-weight: bold; color: blue;'>Z -> []</p><p style='font-weight: bold; color: blue;'>D -> []</p><h2 class='text-muted'>GWAS columns</h2>variant_id, p_value, chromosome, base_pair_location, effect_allele, other_allele, effect_allele_frequency, odds_ratio, ci_lower, ci_upper, beta, standard_error<h2 class='text-muted'>Identified columns</h2>chromosome, variant_id, base_pair_location, effect_allele, other_allele, p_value, effect_allele_frequency, beta, standard_error, odds_ratio<h2 class='text-muted'>Unidentified Columns headers</h2>ci_lower, ci_upper<h2 class='text-muted'>PMID</h2>36411363<h2 class='text-muted'>DOI</h2>Could not find DOI!<div class='bibtex-container'><h2 class='text-muted'>BibTeX</h2><div style='font-family: monospace; white-space: pre-wrap;'></div></div><br><br><br><br><br><br></div><div class='container mt-4 bg-light p-4 text-center'><h1 class='text-primary'>Phenotype: osteoarthritis</h1><p class='text-muted'><strong>URL:</strong> http://ftp.ebi.ac.uk/pub/databases/gwas/summary_statistics/GCST90134001-GCST90135000/GCST90134290</p><table border="1" class="dataframe table table-bordered">
  <thead>
    <tr style="text-align: right;">
      <th>firstAuthor</th>
      <th>accessionId</th>
      <th>publicationDate</th>
      <th>journal</th>
      <th>title</th>
      <th>reportedTrait</th>
      <th>efoTraits</th>
      <th>bgTraits</th>
      <th>discoverySampleAncestry</th>
      <th>replicationSampleAncestry</th>
      <th>associationCount</th>
      <th>summaryStatistics</th>
      <th>ssApiFlag</th>
      <th>agreedToCc0</th>
      <th>genotypingTechnologies</th>
      <th>pubmedId</th>
      <th>initialSampleDescription</th>
      <th>replicateSampleDescription</th>
      <th>UKB_FIELD</th>
      <th>FuzzySimilarity_osteoarthritis</th>
      <th>SearchPhenotype</th>
    </tr>
  </thead>
  <tbody>
    <tr>
      <td>McDonald MN</td>
      <td>GCST90134290</td>
      <td>11/21/2022</td>
      <td>Nat Genet</td>
      <td>Novel genetic loci associated with osteoarthritis in multi-ancestry analyses in the Million Veteran Program and UK Biobank.</td>
      <td>osteoarthritis</td>
      <td>osteoarthritis</td>
      <td>-</td>
      <td>484374 european, south asian, east asian, african unspecified, hispanic or latin american</td>
      <td>NaN</td>
      <td>0</td>
      <td>http://ftp.ebi.ac.uk/pub/databases/gwas/summary_statistics/GCST90134001-GCST90135000/GCST90134290</td>
      <td>no</td>
      <td>yes</td>
      <td>Genome-wide genotyping array</td>
      <td>36411363</td>
      <td>140,025 European ancestry cases,344,349 European ancestry controls,140,025 South Asian ancestry cases,344,349 South Asian ancestry controls,140,025 East Asian ancestry cases,344,349 East Asian ancestry controls,140,025 African ancestry cases,344,349 African ancestry controls,140,025 Hispanic or Latin American cases,344,349 Hispanic or Latin American controls</td>
      <td>NaN</td>
      <td>-</td>
      <td>100</td>
      <td>osteoarthritis</td>
    </tr>
  </tbody>
</table><h2 class='text-muted'>Information file</h2><table border="1" class="dataframe table table-bordered">
  <thead>
    <tr style="text-align: right;">
      <th>Unnamed: 0</th>
      <th>Name</th>
      <th>Last modified</th>
      <th>Size</th>
      <th>Description</th>
      <th>URL</th>
    </tr>
  </thead>
  <tbody>
    <tr>
      <td>1</td>
      <td>GCST90134290_buildGRCh37.csv-meta.yaml</td>
      <td>2024-01-04 00:30</td>
      <td>775</td>
      <td>NaN</td>
      <td>http://ftp.ebi.ac.uk/pub/databases/gwas/summary_statistics/GCST90134001-GCST90135000/GCST90134290/GCST90134290_buildGRCh37.csv-meta.yaml</td>
    </tr>
    <tr>
      <td>2</td>
      <td>md5sum.txt</td>
      <td>2022-09-06 16:25</td>
      <td>62</td>
      <td>NaN</td>
      <td>http://ftp.ebi.ac.uk/pub/databases/gwas/summary_statistics/GCST90134001-GCST90135000/GCST90134290/md5sum.txt</td>
    </tr>
    <tr>
      <td>0</td>
      <td>GCST90134290_buildGRCh37.csv</td>
      <td>2022-09-06 16:25</td>
      <td>1.1G</td>
      <td>NaN</td>
      <td>http://ftp.ebi.ac.uk/pub/databases/gwas/summary_statistics/GCST90134001-GCST90135000/GCST90134290/GCST90134290_buildGRCh37.csv</td>
    </tr>
  </tbody>
</table><h2 class='text-muted'>Processed file</h2><p>Processing file: GCST90134290_buildGRCh37.csv.1!</p><h2 class='text-muted'>GWAS file head</h2><table border="1" class="dataframe table table-bordered">
  <thead>
    <tr style="text-align: right;">
      <th>variant_id</th>
      <th>p_value</th>
      <th>chromosome</th>
      <th>base_pair_location</th>
      <th>effect_allele</th>
      <th>other_allele</th>
      <th>odds_ratio</th>
      <th>ci_lower</th>
      <th>ci_upper</th>
      <th>beta</th>
      <th>standard_error</th>
    </tr>
  </thead>
  <tbody>
    <tr>
      <td>NaN</td>
      <td>0.1363</td>
      <td>1</td>
      <td>600318</td>
      <td>a</td>
      <td>g</td>
      <td>0.838702</td>
      <td>0.607226</td>
      <td>1.070178</td>
      <td>-0.1759</td>
      <td>0.1181</td>
    </tr>
    <tr>
      <td>NaN</td>
      <td>0.1018</td>
      <td>1</td>
      <td>603755</td>
      <td>a</td>
      <td>g</td>
      <td>0.755104</td>
      <td>0.418768</td>
      <td>1.091440</td>
      <td>-0.2809</td>
      <td>0.1716</td>
    </tr>
  </tbody>
</table></div><div class='container mt-4 bg-light p-4 text-center'><h2 class='text-muted'>Original GWAS file headers</h2><table border="1" class="dataframe table table-bordered">
  <thead>
    <tr style="text-align: right;">
      <th>variant_id</th>
      <th>p_value</th>
      <th>chromosome</th>
      <th>base_pair_location</th>
      <th>effect_allele</th>
      <th>other_allele</th>
      <th>odds_ratio</th>
      <th>ci_lower</th>
      <th>ci_upper</th>
      <th>beta</th>
      <th>standard_error</th>
    </tr>
  </thead>
  <tbody>
    <tr>
      <td>NaN</td>
      <td>0.1363</td>
      <td>1</td>
      <td>600318</td>
      <td>a</td>
      <td>g</td>
      <td>0.838702</td>
      <td>0.607226</td>
      <td>1.070178</td>
      <td>-0.1759</td>
      <td>0.1181</td>
    </tr>
    <tr>
      <td>NaN</td>
      <td>0.1018</td>
      <td>1</td>
      <td>603755</td>
      <td>a</td>
      <td>g</td>
      <td>0.755104</td>
      <td>0.418768</td>
      <td>1.091440</td>
      <td>-0.2809</td>
      <td>0.1716</td>
    </tr>
  </tbody>
</table><p style='font-weight: bold; color: blue;'>CHR -> ['chromosome']</p><p style='font-weight: bold; color: blue;'>SNP -> ['variant_id']</p><p style='font-weight: bold; color: blue;'>BP -> ['base_pair_location']</p><p style='font-weight: bold; color: blue;'>A1 -> ['effect_allele']</p><p style='font-weight: bold; color: blue;'>A2 -> ['other_allele']</p><p style='font-weight: bold; color: blue;'>N -> []</p><p style='font-weight: bold; color: blue;'>P -> ['p_value']</p><p style='font-weight: bold; color: blue;'>INFO -> []</p><p style='font-weight: bold; color: blue;'>MAF -> []</p><p style='font-weight: bold; color: blue;'>BETA -> ['beta']</p><p style='font-weight: bold; color: blue;'>SE -> ['standard_error']</p><p style='font-weight: bold; color: blue;'>OR -> ['odds_ratio']</p><p style='font-weight: bold; color: blue;'>Z -> []</p><p style='font-weight: bold; color: blue;'>D -> []</p><h2 class='text-muted'>GWAS columns</h2>variant_id, p_value, chromosome, base_pair_location, effect_allele, other_allele, odds_ratio, ci_lower, ci_upper, beta, standard_error<h2 class='text-muted'>Identified columns</h2>chromosome, variant_id, base_pair_location, effect_allele, other_allele, p_value, beta, standard_error, odds_ratio<h2 class='text-muted'>Unidentified Columns headers</h2>ci_lower, ci_upper<h2 class='text-muted'>PMID</h2>36411363<h2 class='text-muted'>DOI</h2>Could not find DOI!<div class='bibtex-container'><h2 class='text-muted'>BibTeX</h2><div style='font-family: monospace; white-space: pre-wrap;'></div></div><br><br><br><br><br><br></div><div class='container mt-4 bg-light p-4 text-center'><h1 class='text-primary'>Phenotype: osteoarthritis</h1><p class='text-muted'><strong>URL:</strong> http://ftp.ebi.ac.uk/pub/databases/gwas/summary_statistics/GCST90134001-GCST90135000/GCST90134284</p><table border="1" class="dataframe table table-bordered">
  <thead>
    <tr style="text-align: right;">
      <th>firstAuthor</th>
      <th>accessionId</th>
      <th>publicationDate</th>
      <th>journal</th>
      <th>title</th>
      <th>reportedTrait</th>
      <th>efoTraits</th>
      <th>bgTraits</th>
      <th>discoverySampleAncestry</th>
      <th>replicationSampleAncestry</th>
      <th>associationCount</th>
      <th>summaryStatistics</th>
      <th>ssApiFlag</th>
      <th>agreedToCc0</th>
      <th>genotypingTechnologies</th>
      <th>pubmedId</th>
      <th>initialSampleDescription</th>
      <th>replicateSampleDescription</th>
      <th>UKB_FIELD</th>
      <th>FuzzySimilarity_osteoarthritis</th>
      <th>SearchPhenotype</th>
    </tr>
  </thead>
  <tbody>
    <tr>
      <td>McDonald MN</td>
      <td>GCST90134284</td>
      <td>11/21/2022</td>
      <td>Nat Genet</td>
      <td>Novel genetic loci associated with osteoarthritis in multi-ancestry analyses in the Million Veteran Program and UK Biobank.</td>
      <td>osteoarthritis</td>
      <td>osteoarthritis</td>
      <td>-</td>
      <td>484374 european, south asian, east asian, african unspecified, hispanic or latin american</td>
      <td>NaN</td>
      <td>27</td>
      <td>http://ftp.ebi.ac.uk/pub/databases/gwas/summary_statistics/GCST90134001-GCST90135000/GCST90134284</td>
      <td>no</td>
      <td>yes</td>
      <td>Genome-wide genotyping array</td>
      <td>36411363</td>
      <td>140,025 European ancestry cases,344,349 European ancestry controls,140,025 South Asian ancestry cases,344,349 South Asian ancestry controls,140,025 East Asian ancestry cases,344,349 East Asian ancestry controls,140,025 African ancestry cases,344,349 African ancestry controls,140,025 Hispanic or Latin American cases,344,349 Hispanic or Latin American controls</td>
      <td>NaN</td>
      <td>-</td>
      <td>100</td>
      <td>osteoarthritis</td>
    </tr>
  </tbody>
</table><h2 class='text-muted'>Information file</h2><table border="1" class="dataframe table table-bordered">
  <thead>
    <tr style="text-align: right;">
      <th>Unnamed: 0</th>
      <th>Name</th>
      <th>Last modified</th>
      <th>Size</th>
      <th>Description</th>
      <th>URL</th>
    </tr>
  </thead>
  <tbody>
    <tr>
      <td>1</td>
      <td>GCST90134284_buildGRCh37.csv-meta.yaml</td>
      <td>2024-01-04 00:35</td>
      <td>775</td>
      <td>NaN</td>
      <td>http://ftp.ebi.ac.uk/pub/databases/gwas/summary_statistics/GCST90134001-GCST90135000/GCST90134284/GCST90134284_buildGRCh37.csv-meta.yaml</td>
    </tr>
    <tr>
      <td>2</td>
      <td>md5sum.txt</td>
      <td>2022-09-06 16:24</td>
      <td>62</td>
      <td>NaN</td>
      <td>http://ftp.ebi.ac.uk/pub/databases/gwas/summary_statistics/GCST90134001-GCST90135000/GCST90134284/md5sum.txt</td>
    </tr>
    <tr>
      <td>0</td>
      <td>GCST90134284_buildGRCh37.csv</td>
      <td>2022-09-06 16:24</td>
      <td>1.8G</td>
      <td>NaN</td>
      <td>http://ftp.ebi.ac.uk/pub/databases/gwas/summary_statistics/GCST90134001-GCST90135000/GCST90134284/GCST90134284_buildGRCh37.csv</td>
    </tr>
  </tbody>
</table><h2 class='text-muted'>Processed file</h2><p>Processing file: GCST90134284_buildGRCh37.csv.1!</p><h2 class='text-muted'>GWAS file head</h2><table border="1" class="dataframe table table-bordered">
  <thead>
    <tr style="text-align: right;">
      <th>variant_id</th>
      <th>p_value</th>
      <th>chromosome</th>
      <th>base_pair_location</th>
      <th>effect_allele</th>
      <th>other_allele</th>
      <th>odds_ratio</th>
      <th>ci_lower</th>
      <th>ci_upper</th>
      <th>beta</th>
      <th>standard_error</th>
    </tr>
  </thead>
  <tbody>
    <tr>
      <td>NaN</td>
      <td>0.2224</td>
      <td>1</td>
      <td>570002</td>
      <td>A</td>
      <td>G</td>
      <td>1.025725</td>
      <td>0.984957</td>
      <td>1.066493</td>
      <td>0.0254</td>
      <td>0.0208</td>
    </tr>
    <tr>
      <td>NaN</td>
      <td>0.5564</td>
      <td>1</td>
      <td>592088</td>
      <td>T</td>
      <td>C</td>
      <td>1.010252</td>
      <td>0.976148</td>
      <td>1.044356</td>
      <td>0.0102</td>
      <td>0.0174</td>
    </tr>
  </tbody>
</table></div><div class='container mt-4 bg-light p-4 text-center'><h2 class='text-muted'>Original GWAS file headers</h2><table border="1" class="dataframe table table-bordered">
  <thead>
    <tr style="text-align: right;">
      <th>variant_id</th>
      <th>p_value</th>
      <th>chromosome</th>
      <th>base_pair_location</th>
      <th>effect_allele</th>
      <th>other_allele</th>
      <th>odds_ratio</th>
      <th>ci_lower</th>
      <th>ci_upper</th>
      <th>beta</th>
      <th>standard_error</th>
    </tr>
  </thead>
  <tbody>
    <tr>
      <td>NaN</td>
      <td>0.2224</td>
      <td>1</td>
      <td>570002</td>
      <td>A</td>
      <td>G</td>
      <td>1.025725</td>
      <td>0.984957</td>
      <td>1.066493</td>
      <td>0.0254</td>
      <td>0.0208</td>
    </tr>
    <tr>
      <td>NaN</td>
      <td>0.5564</td>
      <td>1</td>
      <td>592088</td>
      <td>T</td>
      <td>C</td>
      <td>1.010252</td>
      <td>0.976148</td>
      <td>1.044356</td>
      <td>0.0102</td>
      <td>0.0174</td>
    </tr>
  </tbody>
</table><p style='font-weight: bold; color: blue;'>CHR -> ['chromosome']</p><p style='font-weight: bold; color: blue;'>SNP -> ['variant_id']</p><p style='font-weight: bold; color: blue;'>BP -> ['base_pair_location']</p><p style='font-weight: bold; color: blue;'>A1 -> ['effect_allele']</p><p style='font-weight: bold; color: blue;'>A2 -> ['other_allele']</p><p style='font-weight: bold; color: blue;'>N -> []</p><p style='font-weight: bold; color: blue;'>P -> ['p_value']</p><p style='font-weight: bold; color: blue;'>INFO -> []</p><p style='font-weight: bold; color: blue;'>MAF -> []</p><p style='font-weight: bold; color: blue;'>BETA -> ['beta']</p><p style='font-weight: bold; color: blue;'>SE -> ['standard_error']</p><p style='font-weight: bold; color: blue;'>OR -> ['odds_ratio']</p><p style='font-weight: bold; color: blue;'>Z -> []</p><p style='font-weight: bold; color: blue;'>D -> []</p><h2 class='text-muted'>GWAS columns</h2>variant_id, p_value, chromosome, base_pair_location, effect_allele, other_allele, odds_ratio, ci_lower, ci_upper, beta, standard_error<h2 class='text-muted'>Identified columns</h2>chromosome, variant_id, base_pair_location, effect_allele, other_allele, p_value, beta, standard_error, odds_ratio<h2 class='text-muted'>Unidentified Columns headers</h2>ci_lower, ci_upper<h2 class='text-muted'>PMID</h2>36411363<h2 class='text-muted'>DOI</h2>Could not find DOI!<div class='bibtex-container'><h2 class='text-muted'>BibTeX</h2><div style='font-family: monospace; white-space: pre-wrap;'></div></div><br><br><br><br><br><br></div><div class='container mt-4 bg-light p-4 text-center'><h1 class='text-primary'>Phenotype: osteoarthritis</h1><p class='text-muted'><strong>URL:</strong> http://ftp.ebi.ac.uk/pub/databases/gwas/summary_statistics/GCST90134001-GCST90135000/GCST90134286</p><table border="1" class="dataframe table table-bordered">
  <thead>
    <tr style="text-align: right;">
      <th>firstAuthor</th>
      <th>accessionId</th>
      <th>publicationDate</th>
      <th>journal</th>
      <th>title</th>
      <th>reportedTrait</th>
      <th>efoTraits</th>
      <th>bgTraits</th>
      <th>discoverySampleAncestry</th>
      <th>replicationSampleAncestry</th>
      <th>associationCount</th>
      <th>summaryStatistics</th>
      <th>ssApiFlag</th>
      <th>agreedToCc0</th>
      <th>genotypingTechnologies</th>
      <th>pubmedId</th>
      <th>initialSampleDescription</th>
      <th>replicateSampleDescription</th>
      <th>UKB_FIELD</th>
      <th>FuzzySimilarity_osteoarthritis</th>
      <th>SearchPhenotype</th>
    </tr>
  </thead>
  <tbody>
    <tr>
      <td>McDonald MN</td>
      <td>GCST90134286</td>
      <td>11/21/2022</td>
      <td>Nat Genet</td>
      <td>Novel genetic loci associated with osteoarthritis in multi-ancestry analyses in the Million Veteran Program and UK Biobank.</td>
      <td>osteoarthritis</td>
      <td>osteoarthritis</td>
      <td>-</td>
      <td>469444 european, south asian, east asian, african unspecified, hispanic or latin american</td>
      <td>NaN</td>
      <td>60</td>
      <td>http://ftp.ebi.ac.uk/pub/databases/gwas/summary_statistics/GCST90134001-GCST90135000/GCST90134286</td>
      <td>no</td>
      <td>yes</td>
      <td>Genome-wide genotyping array</td>
      <td>36411363</td>
      <td>133,506 European ancestry cases,335,938 European ancestry controls,133,506 South Asian ancestry cases,335,938 South Asian ancestry controls,133,506 East Asian ancestry cases,335,938 East Asian ancestry controls,133,506 African ancestry cases,335,938 African ancestry controls,133,506 Hispanic or Latin American cases,335,938 Hispanic or Latin American controls</td>
      <td>NaN</td>
      <td>-</td>
      <td>100</td>
      <td>osteoarthritis</td>
    </tr>
  </tbody>
</table><h2 class='text-muted'>Information file</h2><table border="1" class="dataframe table table-bordered">
  <thead>
    <tr style="text-align: right;">
      <th>Unnamed: 0</th>
      <th>Name</th>
      <th>Last modified</th>
      <th>Size</th>
      <th>Description</th>
      <th>URL</th>
    </tr>
  </thead>
  <tbody>
    <tr>
      <td>1</td>
      <td>GCST90134286_buildGRCh37.csv-meta.yaml</td>
      <td>2024-01-04 00:08</td>
      <td>775</td>
      <td>NaN</td>
      <td>http://ftp.ebi.ac.uk/pub/databases/gwas/summary_statistics/GCST90134001-GCST90135000/GCST90134286/GCST90134286_buildGRCh37.csv-meta.yaml</td>
    </tr>
    <tr>
      <td>2</td>
      <td>md5sum.txt</td>
      <td>2022-09-06 16:27</td>
      <td>62</td>
      <td>NaN</td>
      <td>http://ftp.ebi.ac.uk/pub/databases/gwas/summary_statistics/GCST90134001-GCST90135000/GCST90134286/md5sum.txt</td>
    </tr>
    <tr>
      <td>0</td>
      <td>GCST90134286_buildGRCh37.csv</td>
      <td>2022-09-06 16:27</td>
      <td>1.1G</td>
      <td>NaN</td>
      <td>http://ftp.ebi.ac.uk/pub/databases/gwas/summary_statistics/GCST90134001-GCST90135000/GCST90134286/GCST90134286_buildGRCh37.csv</td>
    </tr>
  </tbody>
</table><h2 class='text-muted'>Processed file</h2><p>Processing file: GCST90134286_buildGRCh37.csv.1!</p><h2 class='text-muted'>GWAS file head</h2><table border="1" class="dataframe table table-bordered">
  <thead>
    <tr style="text-align: right;">
      <th>variant_id</th>
      <th>snp</th>
      <th>p_value</th>
      <th>chromosome</th>
      <th>base_pair_location</th>
      <th>effect_allele</th>
      <th>other_allele</th>
      <th>odds_ratio</th>
      <th>ci_lower</th>
      <th>ci_upper</th>
      <th>beta</th>
      <th>standard_error</th>
      <th>snp.1</th>
    </tr>
  </thead>
  <tbody>
    <tr>
      <td>rs147241137</td>
      <td>1_600495_C_G</td>
      <td>0.3451</td>
      <td>1</td>
      <td>600495</td>
      <td>C</td>
      <td>G</td>
      <td>1.044982</td>
      <td>0.953646</td>
      <td>1.136318</td>
      <td>0.0440</td>
      <td>0.0466</td>
      <td>1_600495_C_G</td>
    </tr>
    <tr>
      <td>NaN</td>
      <td>1_604780_A_C</td>
      <td>0.4391</td>
      <td>1</td>
      <td>604780</td>
      <td>A</td>
      <td>C</td>
      <td>1.116948</td>
      <td>0.836668</td>
      <td>1.397228</td>
      <td>0.1106</td>
      <td>0.1430</td>
      <td>1_604780_A_C</td>
    </tr>
  </tbody>
</table></div><div class='container mt-4 bg-light p-4 text-center'><h2 class='text-muted'>Original GWAS file headers</h2><table border="1" class="dataframe table table-bordered">
  <thead>
    <tr style="text-align: right;">
      <th>variant_id</th>
      <th>snp</th>
      <th>p_value</th>
      <th>chromosome</th>
      <th>base_pair_location</th>
      <th>effect_allele</th>
      <th>other_allele</th>
      <th>odds_ratio</th>
      <th>ci_lower</th>
      <th>ci_upper</th>
      <th>beta</th>
      <th>standard_error</th>
      <th>snp.1</th>
    </tr>
  </thead>
  <tbody>
    <tr>
      <td>rs147241137</td>
      <td>1_600495_C_G</td>
      <td>0.3451</td>
      <td>1</td>
      <td>600495</td>
      <td>C</td>
      <td>G</td>
      <td>1.044982</td>
      <td>0.953646</td>
      <td>1.136318</td>
      <td>0.0440</td>
      <td>0.0466</td>
      <td>1_600495_C_G</td>
    </tr>
    <tr>
      <td>NaN</td>
      <td>1_604780_A_C</td>
      <td>0.4391</td>
      <td>1</td>
      <td>604780</td>
      <td>A</td>
      <td>C</td>
      <td>1.116948</td>
      <td>0.836668</td>
      <td>1.397228</td>
      <td>0.1106</td>
      <td>0.1430</td>
      <td>1_604780_A_C</td>
    </tr>
  </tbody>
</table><p style='font-weight: bold; color: blue;'>CHR -> ['chromosome']</p><p style='font-weight: bold; color: blue;'>SNP -> ['variant_id', 'snp']</p><p style='font-weight: bold; color: blue;'>BP -> ['base_pair_location']</p><p style='font-weight: bold; color: blue;'>A1 -> ['effect_allele']</p><p style='font-weight: bold; color: blue;'>A2 -> ['other_allele']</p><p style='font-weight: bold; color: blue;'>N -> []</p><p style='font-weight: bold; color: blue;'>P -> ['p_value']</p><p style='font-weight: bold; color: blue;'>INFO -> []</p><p style='font-weight: bold; color: blue;'>MAF -> []</p><p style='font-weight: bold; color: blue;'>BETA -> ['beta']</p><p style='font-weight: bold; color: blue;'>SE -> ['standard_error']</p><p style='font-weight: bold; color: blue;'>OR -> ['odds_ratio']</p><p style='font-weight: bold; color: blue;'>Z -> []</p><p style='font-weight: bold; color: blue;'>D -> []</p><h2 class='text-muted'>GWAS columns</h2>variant_id, snp, p_value, chromosome, base_pair_location, effect_allele, other_allele, odds_ratio, ci_lower, ci_upper, beta, standard_error, snp.1<h2 class='text-muted'>Identified columns</h2>chromosome, variant_id, snp, base_pair_location, effect_allele, other_allele, p_value, beta, standard_error, odds_ratio<h2 class='text-muted'>Unidentified Columns headers</h2>snp.1, ci_lower, ci_upper<h2 class='text-muted'>PMID</h2>36411363<h2 class='text-muted'>DOI</h2>Could not find DOI!<div class='bibtex-container'><h2 class='text-muted'>BibTeX</h2><div style='font-family: monospace; white-space: pre-wrap;'></div></div><br><br><br><br><br><br></div><div class='container mt-4 bg-light p-4 text-center'><h1 class='text-primary'>Phenotype: osteoarthritis</h1><p class='text-muted'><strong>URL:</strong> http://ftp.ebi.ac.uk/pub/databases/gwas/summary_statistics/GCST90134001-GCST90135000/GCST90134287</p><table border="1" class="dataframe table table-bordered">
  <thead>
    <tr style="text-align: right;">
      <th>firstAuthor</th>
      <th>accessionId</th>
      <th>publicationDate</th>
      <th>journal</th>
      <th>title</th>
      <th>reportedTrait</th>
      <th>efoTraits</th>
      <th>bgTraits</th>
      <th>discoverySampleAncestry</th>
      <th>replicationSampleAncestry</th>
      <th>associationCount</th>
      <th>summaryStatistics</th>
      <th>ssApiFlag</th>
      <th>agreedToCc0</th>
      <th>genotypingTechnologies</th>
      <th>pubmedId</th>
      <th>initialSampleDescription</th>
      <th>replicateSampleDescription</th>
      <th>UKB_FIELD</th>
      <th>FuzzySimilarity_osteoarthritis</th>
      <th>SearchPhenotype</th>
    </tr>
  </thead>
  <tbody>
    <tr>
      <td>McDonald MN</td>
      <td>GCST90134287</td>
      <td>11/21/2022</td>
      <td>Nat Genet</td>
      <td>Novel genetic loci associated with osteoarthritis in multi-ancestry analyses in the Million Veteran Program and UK Biobank.</td>
      <td>osteoarthritis</td>
      <td>osteoarthritis</td>
      <td>-</td>
      <td>35753 african unspecified</td>
      <td>NaN</td>
      <td>13</td>
      <td>http://ftp.ebi.ac.uk/pub/databases/gwas/summary_statistics/GCST90134001-GCST90135000/GCST90134287</td>
      <td>no</td>
      <td>yes</td>
      <td>Genome-wide genotyping array</td>
      <td>36411363</td>
      <td>17,275 African ancestry cases,18,478 African ancestry controls</td>
      <td>NaN</td>
      <td>-</td>
      <td>100</td>
      <td>osteoarthritis</td>
    </tr>
  </tbody>
</table><h2 class='text-muted'>Information file</h2><table border="1" class="dataframe table table-bordered">
  <thead>
    <tr style="text-align: right;">
      <th>Unnamed: 0</th>
      <th>Name</th>
      <th>Last modified</th>
      <th>Size</th>
      <th>Description</th>
      <th>URL</th>
    </tr>
  </thead>
  <tbody>
    <tr>
      <td>1</td>
      <td>GCST90134287_buildGRCh37.csv-meta.yaml</td>
      <td>2024-01-04 00:33</td>
      <td>683</td>
      <td>NaN</td>
      <td>http://ftp.ebi.ac.uk/pub/databases/gwas/summary_statistics/GCST90134001-GCST90135000/GCST90134287/GCST90134287_buildGRCh37.csv-meta.yaml</td>
    </tr>
    <tr>
      <td>2</td>
      <td>md5sum.txt</td>
      <td>2022-09-06 16:31</td>
      <td>62</td>
      <td>NaN</td>
      <td>http://ftp.ebi.ac.uk/pub/databases/gwas/summary_statistics/GCST90134001-GCST90135000/GCST90134287/md5sum.txt</td>
    </tr>
    <tr>
      <td>0</td>
      <td>GCST90134287_buildGRCh37.csv</td>
      <td>2022-09-06 16:31</td>
      <td>1.8G</td>
      <td>NaN</td>
      <td>http://ftp.ebi.ac.uk/pub/databases/gwas/summary_statistics/GCST90134001-GCST90135000/GCST90134287/GCST90134287_buildGRCh37.csv</td>
    </tr>
  </tbody>
</table><h2 class='text-muted'>Processed file</h2><p>Processing file: GCST90134287_buildGRCh37.csv.1!</p><h2 class='text-muted'>GWAS file head</h2><table border="1" class="dataframe table table-bordered">
  <thead>
    <tr style="text-align: right;">
      <th>variant_id</th>
      <th>p_value</th>
      <th>chromosome</th>
      <th>base_pair_location</th>
      <th>effect_allele</th>
      <th>other_allele</th>
      <th>odds_ratio</th>
      <th>ci_lower</th>
      <th>ci_upper</th>
      <th>beta</th>
      <th>standard_error</th>
    </tr>
  </thead>
  <tbody>
    <tr>
      <td>NaN</td>
      <td>0.4063</td>
      <td>1</td>
      <td>570002</td>
      <td>a</td>
      <td>g</td>
      <td>1.097023</td>
      <td>0.878483</td>
      <td>1.315563</td>
      <td>0.0926</td>
      <td>0.1115</td>
    </tr>
    <tr>
      <td>NaN</td>
      <td>0.6343</td>
      <td>1</td>
      <td>594846</td>
      <td>t</td>
      <td>g</td>
      <td>0.911102</td>
      <td>0.527726</td>
      <td>1.294478</td>
      <td>-0.0931</td>
      <td>0.1956</td>
    </tr>
  </tbody>
</table></div><div class='container mt-4 bg-light p-4 text-center'><h2 class='text-muted'>Original GWAS file headers</h2><table border="1" class="dataframe table table-bordered">
  <thead>
    <tr style="text-align: right;">
      <th>variant_id</th>
      <th>p_value</th>
      <th>chromosome</th>
      <th>base_pair_location</th>
      <th>effect_allele</th>
      <th>other_allele</th>
      <th>odds_ratio</th>
      <th>ci_lower</th>
      <th>ci_upper</th>
      <th>beta</th>
      <th>standard_error</th>
    </tr>
  </thead>
  <tbody>
    <tr>
      <td>NaN</td>
      <td>0.4063</td>
      <td>1</td>
      <td>570002</td>
      <td>a</td>
      <td>g</td>
      <td>1.097023</td>
      <td>0.878483</td>
      <td>1.315563</td>
      <td>0.0926</td>
      <td>0.1115</td>
    </tr>
    <tr>
      <td>NaN</td>
      <td>0.6343</td>
      <td>1</td>
      <td>594846</td>
      <td>t</td>
      <td>g</td>
      <td>0.911102</td>
      <td>0.527726</td>
      <td>1.294478</td>
      <td>-0.0931</td>
      <td>0.1956</td>
    </tr>
  </tbody>
</table><p style='font-weight: bold; color: blue;'>CHR -> ['chromosome']</p><p style='font-weight: bold; color: blue;'>SNP -> ['variant_id']</p><p style='font-weight: bold; color: blue;'>BP -> ['base_pair_location']</p><p style='font-weight: bold; color: blue;'>A1 -> ['effect_allele']</p><p style='font-weight: bold; color: blue;'>A2 -> ['other_allele']</p><p style='font-weight: bold; color: blue;'>N -> []</p><p style='font-weight: bold; color: blue;'>P -> ['p_value']</p><p style='font-weight: bold; color: blue;'>INFO -> []</p><p style='font-weight: bold; color: blue;'>MAF -> []</p><p style='font-weight: bold; color: blue;'>BETA -> ['beta']</p><p style='font-weight: bold; color: blue;'>SE -> ['standard_error']</p><p style='font-weight: bold; color: blue;'>OR -> ['odds_ratio']</p><p style='font-weight: bold; color: blue;'>Z -> []</p><p style='font-weight: bold; color: blue;'>D -> []</p><h2 class='text-muted'>GWAS columns</h2>variant_id, p_value, chromosome, base_pair_location, effect_allele, other_allele, odds_ratio, ci_lower, ci_upper, beta, standard_error<h2 class='text-muted'>Identified columns</h2>chromosome, variant_id, base_pair_location, effect_allele, other_allele, p_value, beta, standard_error, odds_ratio<h2 class='text-muted'>Unidentified Columns headers</h2>ci_lower, ci_upper<h2 class='text-muted'>PMID</h2>36411363<h2 class='text-muted'>DOI</h2>Could not find DOI!<div class='bibtex-container'><h2 class='text-muted'>BibTeX</h2><div style='font-family: monospace; white-space: pre-wrap;'></div></div><br><br><br><br><br><br></div><div class='container mt-4 bg-light p-4 text-center'><h1 class='text-primary'>Phenotype: osteoarthritis</h1><p class='text-muted'><strong>URL:</strong> http://ftp.ebi.ac.uk/pub/databases/gwas/summary_statistics/GCST90134001-GCST90135000/GCST90134288</p><table border="1" class="dataframe table table-bordered">
  <thead>
    <tr style="text-align: right;">
      <th>firstAuthor</th>
      <th>accessionId</th>
      <th>publicationDate</th>
      <th>journal</th>
      <th>title</th>
      <th>reportedTrait</th>
      <th>efoTraits</th>
      <th>bgTraits</th>
      <th>discoverySampleAncestry</th>
      <th>replicationSampleAncestry</th>
      <th>associationCount</th>
      <th>summaryStatistics</th>
      <th>ssApiFlag</th>
      <th>agreedToCc0</th>
      <th>genotypingTechnologies</th>
      <th>pubmedId</th>
      <th>initialSampleDescription</th>
      <th>replicateSampleDescription</th>
      <th>UKB_FIELD</th>
      <th>FuzzySimilarity_osteoarthritis</th>
      <th>SearchPhenotype</th>
    </tr>
  </thead>
  <tbody>
    <tr>
      <td>McDonald MN</td>
      <td>GCST90134288</td>
      <td>11/21/2022</td>
      <td>Nat Genet</td>
      <td>Novel genetic loci associated with osteoarthritis in multi-ancestry analyses in the Million Veteran Program and UK Biobank.</td>
      <td>osteoarthritis</td>
      <td>osteoarthritis</td>
      <td>-</td>
      <td>413170 european</td>
      <td>NaN</td>
      <td>74</td>
      <td>http://ftp.ebi.ac.uk/pub/databases/gwas/summary_statistics/GCST90134001-GCST90135000/GCST90134288</td>
      <td>no</td>
      <td>yes</td>
      <td>Genome-wide genotyping array</td>
      <td>36411363</td>
      <td>109,358 European ancestry cases,303,812 European ancestry controls</td>
      <td>NaN</td>
      <td>-</td>
      <td>100</td>
      <td>osteoarthritis</td>
    </tr>
  </tbody>
</table><h2 class='text-muted'>Information file</h2><table border="1" class="dataframe table table-bordered">
  <thead>
    <tr style="text-align: right;">
      <th>Unnamed: 0</th>
      <th>Name</th>
      <th>Last modified</th>
      <th>Size</th>
      <th>Description</th>
      <th>URL</th>
    </tr>
  </thead>
  <tbody>
    <tr>
      <td>1</td>
      <td>GCST90134288_buildGRCh37.csv-meta.yaml</td>
      <td>2024-01-04 00:16</td>
      <td>673</td>
      <td>NaN</td>
      <td>http://ftp.ebi.ac.uk/pub/databases/gwas/summary_statistics/GCST90134001-GCST90135000/GCST90134288/GCST90134288_buildGRCh37.csv-meta.yaml</td>
    </tr>
    <tr>
      <td>2</td>
      <td>md5sum.txt</td>
      <td>2022-09-06 16:27</td>
      <td>62</td>
      <td>NaN</td>
      <td>http://ftp.ebi.ac.uk/pub/databases/gwas/summary_statistics/GCST90134001-GCST90135000/GCST90134288/md5sum.txt</td>
    </tr>
    <tr>
      <td>0</td>
      <td>GCST90134288_buildGRCh37.csv</td>
      <td>2022-09-06 16:27</td>
      <td>1.3G</td>
      <td>NaN</td>
      <td>http://ftp.ebi.ac.uk/pub/databases/gwas/summary_statistics/GCST90134001-GCST90135000/GCST90134288/GCST90134288_buildGRCh37.csv</td>
    </tr>
  </tbody>
</table><h2 class='text-muted'>Processed file</h2><p>Processing file: GCST90134288_buildGRCh37.csv!</p><h2 class='text-muted'>GWAS file head</h2><table border="1" class="dataframe table table-bordered">
  <thead>
    <tr style="text-align: right;">
      <th>variant_id</th>
      <th>p_value</th>
      <th>chromosome</th>
      <th>base_pair_location</th>
      <th>effect_allele</th>
      <th>other_allele</th>
      <th>odds_ratio</th>
      <th>ci_lower</th>
      <th>ci_upper</th>
      <th>beta</th>
      <th>standard_error</th>
    </tr>
  </thead>
  <tbody>
    <tr>
      <td>rs147241137</td>
      <td>0.5005</td>
      <td>1</td>
      <td>600495</td>
      <td>c</td>
      <td>g</td>
      <td>0.900505</td>
      <td>0.595725</td>
      <td>1.205285</td>
      <td>-0.1048</td>
      <td>0.1555</td>
    </tr>
    <tr>
      <td>NaN</td>
      <td>0.1948</td>
      <td>1</td>
      <td>603755</td>
      <td>a</td>
      <td>g</td>
      <td>0.822094</td>
      <td>0.525938</td>
      <td>1.118250</td>
      <td>-0.1959</td>
      <td>0.1511</td>
    </tr>
  </tbody>
</table></div><div class='container mt-4 bg-light p-4 text-center'><h2 class='text-muted'>Original GWAS file headers</h2><table border="1" class="dataframe table table-bordered">
  <thead>
    <tr style="text-align: right;">
      <th>variant_id</th>
      <th>p_value</th>
      <th>chromosome</th>
      <th>base_pair_location</th>
      <th>effect_allele</th>
      <th>other_allele</th>
      <th>odds_ratio</th>
      <th>ci_lower</th>
      <th>ci_upper</th>
      <th>beta</th>
      <th>standard_error</th>
    </tr>
  </thead>
  <tbody>
    <tr>
      <td>rs147241137</td>
      <td>0.5005</td>
      <td>1</td>
      <td>600495</td>
      <td>c</td>
      <td>g</td>
      <td>0.900505</td>
      <td>0.595725</td>
      <td>1.205285</td>
      <td>-0.1048</td>
      <td>0.1555</td>
    </tr>
    <tr>
      <td>NaN</td>
      <td>0.1948</td>
      <td>1</td>
      <td>603755</td>
      <td>a</td>
      <td>g</td>
      <td>0.822094</td>
      <td>0.525938</td>
      <td>1.118250</td>
      <td>-0.1959</td>
      <td>0.1511</td>
    </tr>
  </tbody>
</table><p style='font-weight: bold; color: blue;'>CHR -> ['chromosome']</p><p style='font-weight: bold; color: blue;'>SNP -> ['variant_id']</p><p style='font-weight: bold; color: blue;'>BP -> ['base_pair_location']</p><p style='font-weight: bold; color: blue;'>A1 -> ['effect_allele']</p><p style='font-weight: bold; color: blue;'>A2 -> ['other_allele']</p><p style='font-weight: bold; color: blue;'>N -> []</p><p style='font-weight: bold; color: blue;'>P -> ['p_value']</p><p style='font-weight: bold; color: blue;'>INFO -> []</p><p style='font-weight: bold; color: blue;'>MAF -> []</p><p style='font-weight: bold; color: blue;'>BETA -> ['beta']</p><p style='font-weight: bold; color: blue;'>SE -> ['standard_error']</p><p style='font-weight: bold; color: blue;'>OR -> ['odds_ratio']</p><p style='font-weight: bold; color: blue;'>Z -> []</p><p style='font-weight: bold; color: blue;'>D -> []</p><h2 class='text-muted'>GWAS columns</h2>variant_id, p_value, chromosome, base_pair_location, effect_allele, other_allele, odds_ratio, ci_lower, ci_upper, beta, standard_error<h2 class='text-muted'>Identified columns</h2>chromosome, variant_id, base_pair_location, effect_allele, other_allele, p_value, beta, standard_error, odds_ratio<h2 class='text-muted'>Unidentified Columns headers</h2>ci_lower, ci_upper<h2 class='text-muted'>PMID</h2>36411363<h2 class='text-muted'>DOI</h2>Could not find DOI!<div class='bibtex-container'><h2 class='text-muted'>BibTeX</h2><div style='font-family: monospace; white-space: pre-wrap;'></div></div><br><br><br><br><br><br></div>